{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer ="<?xml version=\"1.0\" encoding=\"utf-8\" ?><data>   <entry name=\"Rooted tree\" value=\"A rooted tree is one in which the root of the phylogenetic tree is determined by using the mid-point rooting or out-group sequences.\" />   <entry name=\"RSCU\" value=\"Many amino acids are coded by more than one codon; thus multiple codons for a given amino acid are synonymous.  However, many genes display a non-random usage of synonymous codons for specific amino acids.  A measure of the extent of this non-randomness is given by the Relative Synonymous Codon Usage (RSCU) (Sharp et al. 1986).   The RSCU for a particular codon (i) is given by                               	RSCUi  =  Xi / åå XI /n  where Xi is the number of times the ith codon has been used for a given amino acid, and n is the number of synonymous codons for that amino acid.\" />   <entry name=\"RTDT (RelTime with Dated Tips)\" value=\"RelTime method applied using tip dates for calibrating the timetree. To use this method, tip dates must be available for all molecular ingroup taxa. RTDT is suitable for the analysis of DNA or protein sequence from fast evolving pathogens or those generated from ancient data.\" />   <entry name=\"Singleton Sites\" value=\"A singleton site contains at least two types of nucleotides (or amino acids) with, at most, one occurring multiple times.  MEGA identifies a site as a singleton site if at least three sequences contain unambiguous nucleotides or amino acids.\" />   <entry name=\"Site Label\" value=\"The individual sites in nucleotide or amino acid data can be labeled to construct non-contiguous sets of sites.  The Setup Genes and Domains dialog can be used to assign or edit site labels, in addition to specifying them in the input data files. This is shown in the following example of three-sequences in which the sites in the Third Gene are labeled with a ‘+’ mark.  An underscore marks an absence of any labels.  !Gene=FirstGene  Domain=Exon1  Property=Coding;  #Human_{Mammal}  ATGGTTTCTAGTCAGGTCACCATGATAGGTCTCAAT  #Mouse_{Mammal}  ATGGTTTCTAGTCAGGTCACCATGATAGGTCCCAAT  #Chicken_{Aves}  ATGGTTTCTAGTCAGCTCACCATGATAGGTCTCAAT    !Gene=SecondGene  Domain=AnIntron Property=Noncoding;  #Human     ATTCCCAGGGAATTCCCGGGGGGTTTAAGGCCCCTTTAAAGAAAGAT  #Mouse     GTAGCGCGCGTCGTCAGAGCTCCCAAGGGTAGCAGTCACAGAAAGAT  #Chicken   GTAAAAAAAAAAGTCAGAGCTCCCCCCAATATATATCACAGAAAGAT    !Gene=ThirdGene  Domain=Exon2  Property=Coding;  #Human      ATCTGCTCTCGAGTACTGATACAAATGACTTCTGCGTACAACTGA  #Mouse      ATCTGATCTCGTGTGCTGGTACGAATGATTTCTGCGTTCAACTGA  #Chicken    ATCTGCTCTCGAGTACTGCTACCAATGACTTCTGCGTACAACTGA  !Label      +++__-+++-a-+++-L-+++-k-+++123+++-_-+++---+++;    Each site can be associated with only one label.  A label can be a letter or a number.  For analyses that require codons, MEGA includes only those codons in which all three positions are given the same label.  This site labeling system facilitates the analysis of specific sites, as often is required for comparing sequences of regulatory elements, intron-splice sites, and antigen recognition sites in the genes of applications such as the Major Histocompatibility Complex.\" />   <entry name=\"Staden\" value=\"The Staden file format is used to store data from DNA sequencing instruments. Each file contains the data for a single reading and includes the called sequence as well as additional data obtained from the reading. This file format was first described in   	Dear, S and Staden, R. “A Standard file format for data from DNA sequencing instruments”, DNA Sequence 3, 107-110, (1992)  MEGA is able to display the contents of a Staden-formatted trace file using MEGA’s Trace File Editor, which is part of the Alignment Explorer.\" />   <entry name=\"Statements in input files\" value=\"All statements in MEGA files start with an exclamation mark (!) and end with a semicolon (;).  They are useful in specifying various attributes of the data and the data file.  There are three common statements for all types of data: Title, Format, and Description.  There also are other statements that can be used in MEGA files, depending on the type of data being analyzed.\" />   <entry name=\"Subtree-Pruning-Regrafting (SPR)\" value=\"For any tree searching method, exhaustive search, where all possible topologies are considered is unfeasible for even a small number of taxa. Subtree Pruning And Regrafting is a tree topology search heuristic which reduces the number of topologies searched by performing the following operations on the tree.        First, a subtree of the current best tree is selected and detached (pruned). Second, the detached subtree is regrafted onto another branch of the remaining tree, in such a way that a new topology is created and then likelihood of the new topology is calculated. This procedure is repeated for all regrafting positions that produce new topologies using the pruned subtree. The procedure is also repeated for each subtree (within the designated search level) and if the topology with best likelihood among those scored gives sufficient improvement over the current best tree, that topology becomes the current best tree. This is repeated until no significant further likelihood improvements are obtained.        A single pass of the SPR algorithm examines O(N2) new trees, where N is the number of leaves in the original tree. This is because, for each subtree there are O(N) possible regraftings, and there are O(N) possible subtrees to consider. In contrast, NNI examines O(N) topologies at each pass of the algorithm.\" />   <entry name=\"Swap\" value=\"This command changes the cursor to the 'Flip' icon.  Then, you click on an interior branch, MEGA swaps the two subtrees defined by this branch.  If each of the subtrees is an individual taxon, then Swap is the same as Flip.  The cursor will revert to the arrow if you click on the arrow icon on the left-hand side of the Tree Explorer.\" />   <entry name=\"Synonymous change\" value=\"A nucleotide change is synonymous if it does not cause the codon to code for a different amino acid.  A nucleotide site in which one or more changes is synonymous is referred to as a synonymous site.  If only one of three possible nucleotide changes at that site is synonymous, then the site is 1/3 synonymous.  If two of three nucleotide changes are synonymous, then the site is 2/3 synonymous and 1/3 nonsynonymous.  And, if all three possible nucleotide changes are synonymous, then the site is completely synonymous.\" />   <entry name=\"Taxa\" value=\"A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on the study, “taxa” may refer to species, populations, individuals, or sequences within an individual.\" />   <entry name=\"Topological distance\" value=\"The topological distance quantifies the extent of topological differences between two given trees.  For unrooted, bifurcating trees, this distance is twice the number of interior branches at which the taxa are partitioned differently.\" />   <entry name=\"Topology\" value=\"The branching pattern of a tree is its topology.\" />   <entry name=\"Transition\" value=\"A transition occurs when a purine is substituted by a purine, or a pyrimidine by a pyrimidine.\" />   <entry name=\"Transition Matrix\" value=\"A transition matrix specifies the probability of every possible substitution among the nucleotides or amino acids.\" />   <entry name=\"Transition/Transversion Ratio (R)\" value=\"This is the ratio of the number of transitions to the number of transversions for a pair of sequences.  R becomes 0.5 when there is no bias towards either transitional or transversional substitution because, when the two kinds of substitution are equally probable, there are twice as many possible transversions as transitions.   MEGA allows you to conduct an analysis of your data with a specified value of R.   Note that R should not be confused with the ratio of the transition and transversion rates (k = a/b).\" />   <entry name=\"Translation\" value=\"Translation is the process whereby each codon in the mRNA is translated into a particular amino acid, according to the genetic code specific to the species and its DNA, and added to the growing polypeptide chain.\" />   <entry name=\"Transversion\" value=\"A change from a purine to a pyrimidine, or vice versa, is a transversion.\" />   <entry name=\"Tree Bisection Reconnection (TBR)\" value=\"Tree Bisection and Reconnection is a search heuristic which reduces the number of topologies searched, so that we don’t have to perform an exhaustive search of all possible tree topologies.    TBR selects a subtree, and deletes the internal branch which connects it to the main tree.  This subtree is now not connected to the main tree, which leaves us with 2 trees.  The subtree is then reconnected to the main tree in all possible connections between the branches of the two trees.  If any of the reconnected trees score better than the currently best tree, the algorithm selects that tree as the new best tree and performs another round of bisection, reconnection, and scoring.\" />   <entry name=\"Tree Length\" value=\"Tree length is the criterion used by the Maximum Parsimony method to search for the best tree.  It is defined as the sum of the minimum numbers of substitutions over all sites for the given topology.   To compute the tree length for the unweighted parsimony method, we use the procedure described in Fitch (1971), which is based on the two rules described below.  For a given site these rules are applied to each node and the sum of substitutions over all nodes and over all sites are taken.  Note that the estimation of the minimum number of substitutions is not affected by the position of the root.  Rule 1.  When the two descendent nodes of an ancestral node have some states (nucleotides or amino acids) in common, the ancestral node is assigned to the set of common states.  In this case, the most parsimonious explanation does not require any substitutions.  Rule 2.  When the two descendant nodes have no states in common, then all states in the descendent nodes are combined to form the set of possible states at the ancestral node.  In this case, one substitution is required.\" />   <entry name=\"Unrooted Tree\" value=\"An unrooted tree is one in which no assumption is made regarding the ancestor of all the taxa in the tree.\" />  </data>";
