{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var glossary =  {"type":"data","entrys":[{"type":"entry","name":"Groups of Taxa","value":"A group of taxa is a set of one or more taxa. Members of a group can be specified in the input data file, and created and edited in the Setup Taxa and Groups dialog.  Groups of taxa often are constructed based on their evolutionary relatedness.  For example, sequences may be grouped based on the geographic origin of the source individual, or sequences from a multi-gene family may be arranged into groups consisting of orthologous sequences."},{"type":"entry","name":"Indels","value":"Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During the alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps, or indels."},{"type":"entry","name":"Independent Sites","value":"In a sequence alignment, all sites that have not been assigned to any gene or domain are classified as independent."},{"type":"entry","name":"Inferred Tree","value":"A tree reconstructed from the observed sequence or other appropriate data using any tree-making method (such as UPGMA, NJ, ME, or MP) is known as an inferred or reconstructed tree."},{"type":"entry","name":"Intron","value":"Introns are the non-coding segments of DNA in a gene that are interspersed among the exons."},{"type":"entry","name":"Labeled Sites","value":"Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily."},{"type":"entry","name":"Max-mini branch-and-bound search","value":"This is an algorithm for searching for the MP tree using the branch-and bound search method.  See Nei & Kumar (2000) for details."},{"type":"entry","name":"Maximum Composite Likelihood","value":"In general, a composite likelihood is defined as a sum of log-likelihoods for related estimates. In MEGA4, the maximum composite likelihood is used for describing the sum of log-likelihoods for all pairwise distances in a distance matrix (Tamura et al. 2004) estimated by using the Tamura-Nei (1993) model (see related Tamura-Nei distance). Further information is in the Maximum Composite Likelihood Method."},{"type":"entry","name":"Maximum Parsimony Principle","value":"For any given topology, the sum of the minimum possible substitutions over all sites is known as the tree length for that topology.  The topology with the minimum tree length is known as the Maximum Parsimony tree."},{"type":"entry","name":"MEGA-CC","value":"MEGA-CC is the compute core that is used by MEGA and is distributed as a command-line application that is useful for iterative and batch analysis. It can be obtained from www.megasoftware.net ."},{"type":"entry","name":"Mid-point rooting","value":"In the mid-point rooting method, the root of an unrooted tree is placed at the mid-point of the longest distance between two taxa in a tree."},{"type":"entry","name":"Min-mini algorithm","value":"This is a heuristic search algorithm for finding the MP tree, and is somewhat similar to the branch-and bound search method.  However, in this algorithm, many trees that are unlikely to have a small local tree length are eliminated from the computation of their L values. Thus while the algorithm speeds up the search for the MP tree, as compared to the branch-and-bound search, the final tree or trees may not be the true MP tree(s).  The user can specify a search factor to control the extensiveness of the search and MEGA adds the user specified search factor to the current local upper bound.  Of course, the larger the search factor, the slower the search, since many more trees will be examined.     (See also Nei & Kumar (2000), pages 122, 125)"},{"type":"entry","name":"Monophyletic","value":"A cluster of taxa that shared a common ancestor comparatively recently in the evolutionary history of a phylogenetic tree is monophyletic.  The term reflects the close relationship of the taxa with each other."},{"type":"entry","name":"mRNA","value":"Protein-coding genes are first transcribed into messenger RNAs (mRNA), which are, in turn, translated into amino acid sequences to make proteins."},{"type":"entry","name":"NCBI","value":"An acronym that stands for “National Center for Biotechnology Information”. NCBI is a federally funded resource for molecular biology information. NCBI creates databases, conducts research in computational biology, develops software and tools for analyzing genome data, and disseminates biomedical information. You can find out more about NCBI by visiting the NCBI website (http://www.ncbi.nlm.nih.gov)."},{"type":"entry","name":"Newick Format","value":"NEWICK is a simple format used to write out trees in a text file.  While this is a hard-to-read format for humans, it is very useful for exchanging trees between different types of software. An example of the contents of a NEWICK format tree file is given below (note that semi-colon is needed to end the tree).  Further information on this format can be found at Joe Felsenstein’s website.   ((raccoon, bear),((sea_lion,seal),((monkey,cat), weasel)),dog);  The above tree with branch lengths will look as follows:  ((raccoon:19.19959,bear:6.80041):0.84600,((sea_lion:11.99700, seal:12.00300):7.52973,((monkey:100.85930,cat:47.14069):20.59201, weasel:18.87953):2.09460):3.87382,dog:25.46154);  If you wish to specify bootstrap values then they could appear before the branch lengths (e.g., in .DND files produced by CLUSTAL) or after the branch lengths (e.g., in .PHB files produced by CLUSTAL).  In these cases, the format might look like:  ((raccoon:19.19959,bear:6.80041)50:0.84600,((sea_lion:11.99700, seal:12.00300)100:7.52973,((monkey:100.85930,cat:47.14069)80:20.59201, weasel:18.87953)75:2.09460)50:3.87382,dog:25.46154);  or  ((raccoon:19.19959,bear:6.80041):0.84600[50],((sea_lion:11.99700, seal:12.00300):7.52973[100],((monkey:100.85930,cat:47.14069):20.59201[80], weasel:18.87953):2.09460[75]):3.87382[50],dog:25.46154);"},{"type":"entry","name":"Node","value":"A node in a phylogenetic tree represents a taxon, the external or terminal nodes represent the extant taxa and the internal nodes represent the ancestral taxa."},{"type":"entry","name":"Nonsynonymous change","value":"A nucleotide change is nonsynonymous if it changes the amino acid encoded by the original codon.  A nucleotide site in which one or more changes are nonsynonymous is referred to as a nonsynonymous site.  If only one of three possible nucleotide changes at that site is nonsynonymous, then the site is 1/3 nonsynonymous.  If two of three nucleotide changes are nonsynonymous, then the site is 2/3 nonsynonymous.  And, if all three possible nucleotide changes are nonsynonymous, then the site is completely nonsynonymous."},{"type":"entry","name":"OLS branch length estimates","value":"The ordinary least squares estimate of a branch length (b) is given by  {bmct OLSBranchLenEst_d1.bmp}  where dij is the pairwise distance between sequences i and j.  The coefficients wij’s depend on whether the branch under consideration is internal or external.    Coefficients wij’s for an internal branch  {bmct OLSBranchLenEst_d2.bmp}  where, mA, mB, mC, and mC are the numbers of sequences in clusters A, B, C, and D, respectively.    Coefficients wij’s for an external branch  {bmct OLSBranchLenEst_d3.bmp}   where, mA and mB are the numbers of sequences in clusters A and B."},{"type":"entry","name":"Orthologous Genes","value":"Two genes are said to be orthologous if they are the result of a speciation event."},{"type":"entry","name":"Out-group","value":"An out-group is a sequence (or set of sequences) that is known to be a sister taxa to all other sequences in the dataset."},{"type":"entry","name":"Pairwise-deletion option","value":"In the pairwise-deletion option, sites containing missing data or alignment gaps are removed from the analysis as the need arises (e.g., pairwise distance computation). This is in contrast to the complete-deletion option in which all such sites are removed prior to the analysis."},{"type":"entry","name":"Parsimony-informative site","value":"A site is parsimony-informative if it contains at least two types of nucleotides (or amino acids), and at least two of them occur with a minimum frequency of two."},{"type":"entry","name":"Partial-Deletion Option","value":"In the partial-deletion option, sites are removed before the analysis begins.  A site is removed if it has a higher percentage of ambiguous sites than is specified in the Site Coverage Cutoff parameter.  This is similar to complete deletion, in that it will have the same result if the Site Coverage is set to 100% (because 100% means there are no ambiguous sites).  This along with complete deletion is in contrast with the Pairwise-deletion in which sites are removed during the analysis as the need arises (eg., pairwise distance computation)."},{"type":"entry","name":"Polypeptide","value":"A polypeptide is a chain of many amino acids."},{"type":"entry","name":"Positive selection","value":"At the DNA sequence level, positive selection refers to selection in favor of nonsynonymous substitutions.  In this case, the evolutionary distance based on nonsynonymous substitutions is expected to be greater than synonymous substitutions."},{"type":"entry","name":"Protein parsimony","value":"A Maximum Parsimony analysis on protein sequences is known as protein parsimony."},{"type":"entry","name":"Purifying selection","value":"Purifying selection refers to selection against nonsynonymous substitutions at the DNA level.  In this case, the evolutionary distance based on synonymous substitutions is expected to be greater than the distance based on nonsynonymous substitutions."},{"type":"entry","name":"Purines","value":"The nucleotides adenine (A) and guanine (G) are known as purines."},{"type":"entry","name":"Pyramidines","value":"The nucleotides cytosine (C) and thymine (T) are known as pyrimidines."},{"type":"entry","name":"Random addition trees","value":"This refers to the generation of random initial trees for a heuristic search to find MP trees.  In this case, a tree is generated by randomly selecting a sequence and adding it to the growing tree on a randomly-selected branch."}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), glossary, { sync:true });
})();
