{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer ="<?xml version=\"1.0\" encoding=\"utf-8\" ?><data>   <entry name=\"ABI File Format\" value=\"The ABI File Format is a binary file that is produced by ABI sequencer software. This data file, referred to as a “trace file” is viewable in MEGA’s Trace File Editor, which is part of the Alignment Explorer.\" />   <entry name=\"Alignment Gaps\" value=\"Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.\" />   <entry name=\"Alignment Session\" value=\"When working in MEGA’s Alignment Explorer you can choose to save the current state of all data and settings in the alignment explorer to a file so you can archive your work, or save it to resume editing in the future. An alignment session is a binary file format that is saved with the .MAS file extension.\" />   <entry name=\"Bifurcating Tree\" value=\"A bifurcating tree is one in which each ancestral lineage gives rise to exactly two descendent lineages.  A tree with only bifurcating nodes is called a bifurcating tree.\" />   <entry name=\"Branch\" value=\"A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.\" />   <entry name=\"Branch Swap Filter\" value=\"For analyses that use a Maximum Likelihood approach, the ‘branch swap filter’ analysis setting is used to set the stringency of optimization with respect to branch lengths and improvements in log likelihood values. A weaker filter will result in more exhaustive optimization so that the search may take longer but potentially, a larger search space will be explored. A stronger filter will result in less exhaustive optimization so that the search may require less time but potentially, a smaller search space will be explored.\" />   <entry name=\"Branch-and-Bound algorithm\" value=\"The branch-and-bound algorithm is used to find all the MP trees.  It guarantees to find all the MP trees without conducting an exhaustive search.  MEGA also employs the Max-mini branch-and-bound search, which is described in detail in Kumar et al. (1993) and Nei and Kumar (2000, page 123).\" />   <entry name=\"Close-Neighbor-Interchange (CNI)\" value=\"In any method, examining all possible topologies is very time consuming.  This algorithm reduces the time spent searching by first producing a temporary tree, (e.g., an NJ tree when an ME tree is being sought), and then examining all of the topologies that are different from this temporary tree by a topological distance of dT = 2 and 4.  If this is repeated many times, and all the topologies previously examined are avoided, one can usually obtain the tree being sought.    For the MP method, the CNI search can start with a tree generated by the random addition of sequences.  This process can be repeated multiple times to find the MP tree.  See Nei &amp; Kumar (2000) for details.\" />   <entry name=\"ClustalW\" value=\"ClustalW is a general purpose multiple sequence alignment program for DNA or proteins. You can learn more about ClustalW by visiting its website (http://www.ebi.ac.uk/clustalw/).\" />   <entry name=\"Codon\" value=\"A codon is triplet of nucleotides that codes for a specific amino acid.\" />   <entry name=\"Complete Deletion Option\" value=\"In the complete-deletion option, sites containing missing data or alignment gaps are removed before the analysis begins.  This is in contrast to the pairwise-deletion option in which sites are removed during the analysis as the need arises (e.g., pairwise distance computation).\" />   <entry name=\"Composition Distance\" value=\"Composition distance is a measure of the difference in nucleotide (or amino acid) composition for a given pair of sequences.  It is one half the sum of squared difference in counts of bases (or residues).  MEGA computes and presents the Composition Distance per site, which is given by the total composition distance between two sequences divided by the number of positions compared, excluding gaps and missing data.\" />   <entry name=\"Compress/Uncompress\" value=\"This command changes the cursor to the 'Compress/Uncompress' icon.  If you click on an interior branch, MEGA will prompt you to give a name to the group that will be formed.  It then will compress all the lineages defined by this branch into a solid elongated triangle whose thickness is proportional to the number of taxa condensed. Clicking on the branch again will uncompress it.  The cursor may be reverted to the arrow by clicking on the arrow icon on the left hand side of the Tree Explorer.\" />   <entry name=\"Condensed Tree\" value=\"When interior branches in a phylogenetic tree do not have statistically significant lengths, choosing this command condenses the tree into a topology in which each branch with less than the desired statistical significance is collapsed.\" />   <entry name=\"Consensus Tree\" value=\"The MP method produces many equally parsimonious trees.  Choosing this command produces a composite tree that is a consensus among all such trees, for example, either as a strict consensus, in which all conflicting branching patterns among the trees are resolved by making those nodes multifurcating or as a Majority-Rule consensus, in which conflicting branching patterns are resolved by selecting the pattern seen in more than 50% of the trees.   (Details are given in Nei and Kumar [2000], page 130).\" />   <entry name=\"Constant Site\" value=\"A site containing the same nucleotide or amino acid in all sequences is referred to as a constant site.  MEGA identifies a site as a constant site only if at least two sequences contain unambiguous nucleotides or amino acids.\" />   <entry name=\"CorrTest\" value=\"A method for detecting correlation of evolutionary rates in a phylogenetic tree (Tao et al. 2018)\" />   <entry name=\"Data Coverage\" value=\"Data coverage at a given internal node in a tree is defined as the proportion of sites where there is at least one taxon (sequence) in each descendent lineage that has available sequence data. This value is only available for timetrees and is useful for identifying problematic nodes in the tree where divergence time estimates may be unreliable due to lack of available data (see Filipski et al.).\" />   <entry name=\"Degeneracy\" value=\"0-fold degenerate sites are those at which all changes are nonsynonymous.  2-fold degenerate sites are those at which one out of three changes is synonymous.  (All sites at which two out of three changes are synonymous also are included in this category.)  4-fold degenerate sites are those at which all changes are synonymous.\" />   <entry name=\"Disparity Index\" value=\"Disparity Index measures the observed difference in substitution patterns for a pair of sequences.  It works by comparing the nucleotide (or amino acid) frequencies in given pair of sequences and using the number of observed differences between sequences.  MEGA computes and presents the Disparity Index per site, which is given by the total disparity index between two sequences divided by the number of positions compared, excluding gaps and missing data.  It is more powerful than a chi-square test of the equality of base frequencies between sequences.\" />   <entry name=\"Disparity Index Test\" value=\"Tests the null hypothesis that sequences have evolved with the same pattern of substitution, as judged from the extent of differences in base composition biases between sequences (Kumar and Gadagkar 2001).\" />   <entry name=\"Domains\" value=\"A domain is a continuous block of sites in a sequence alignment.  A domain can be free-standing or assigned to genes and protein-coding (e.g., exons) or non-coding (e.g., introns). Domains can be defined in the input data, and can be defined and edited in the Setup Genes Domains dialog.\" />   <entry name=\"Exon\" value=\"A protein-coding gene typically consists of multiple coding regions, known as exons, interspersed with non-coding DNA (introns)\" />   <entry name=\"Extant Taxa\" value=\"The taxa whose sequences, other genetic information or morphological characters, etc. are being used for a phylogenetic analysis are known as extant taxa, irrespective of whether the individuals or species to which the sequences and other information belong are extant or extinct.\" />   <entry name=\"Flip\" value=\"This command changes the cursor to the 'Flip' icon. Then, if you click on an interior branch, MEGA reverses the order of the lineages defined by this branch.  The cursor will revert to the arrow if you click on the arrow icon on the left hand side of the Tree Explorer.\" />   <entry name=\"Format Command\" value=\"A format command in a data file begins with! Format and contains at least the data type included in the file.\" />   <entry name=\"Gamma Parameter\" value=\"According to the gamma distribution, the substitution rate often varies from site to site within a sequence.  The shape of this distribution is determined by a value known as the gamma parameter, which is also known as the shape parameter.\" />   <entry name=\"Gene\" value=\"A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.\" />   <entry name=\"Genetic Codes\" value=\"A genetic code table specifies the amino acid residues encoded by the various codons.  Vertebrate mitochondria, Drosophila mitochondria, and yeast mitochondria all have their own genetic code tables, which are slightly different from the most common table, the Standard Genetic Code Table.\" />  </data>";
