{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Data Menu","topics":[{"type":"topic","name":"Data Menu","url":"Data_Menu.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Data Menu (Sequence Data Explorer)","url":"Data_Menu_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Guide to Notations Used","url":"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm"},{"type":"topic","name":"Select Genetic Code (in Sequence Data Explorer)","url":"Select_Genetic_Code_(in_Sequence_Data_Explorer).htm"},{"type":"topic","name":"Select Genetic Code Tables","url":"Select_Genetic_Code_Tables.htm"},{"type":"topic","name":"Sequence Data Subset Selection","url":"Sequence_Data_Subset_Selection.htm"},{"type":"topic","name":"Setup/Select Genes & Domains (Sequence Data Explorer)","url":"Setup_Select_Genes___Domains_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Setup/Select Taxa & Groups (in Sequence Data Explorer)","url":"Setup_Select_Taxa_Groups_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Data menu to select the","topics":[{"type":"topic","name":"Select Genetic Code Table","url":"Select_Genetic_Code_Table.htm"}]},{"type":"key","name":"Data Miner","topics":[{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"}]},{"type":"key","name":"Data Subset","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"}],"keys":[{"type":"key","name":"Use","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"}]}]},{"type":"key","name":"Data Type","topics":[{"type":"topic","name":"Data Explorer","url":"Data_Explorer.htm"},{"type":"topic","name":"Input Data Format Dialog","url":"Input_Data_Format_Dialog.htm"}]},{"type":"key","name":"DataFormat","topics":[{"type":"topic","name":"Keywords for Format Statement","url":"Keywords_for_Format_Statement.htm"}]},{"type":"key","name":"dataset","topics":[{"type":"topic","name":"Close Data","url":"Close_Data.htm"},{"type":"topic","name":"Codon Usage","url":"Codon_Usage.htm"},{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Inapplicable Computation Requested","url":"Inapplicable_Computation_Requested.htm"},{"type":"topic","name":"Keywords for Format Statement","url":"Keywords_for_Format_Statement.htm"},{"type":"topic","name":"Not Enough Groups Selected","url":"Not_Enough_Groups_Selected.htm"},{"type":"topic","name":"Not Enough Taxa Selected","url":"Not_Enough_Taxa_Selected.htm"},{"type":"topic","name":"Nucleotide and Amino Acid Compositions","url":"Nucleotide_and_Amino_Acid_Compositions.htm"},{"type":"topic","name":"Open Data","url":"Open_Data.htm"},{"type":"topic","name":"Out-group","url":"Out-group.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Translate/Untranslate (in Sequence Data Explorer)","url":"Translate_Untranslate_(in_Sequence_Data_Explorer).htm"}]},{"type":"key","name":"datasets","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"}]},{"type":"key","name":"DataType","topics":[{"type":"topic","name":"Keywords for Format Statement","url":"Keywords_for_Format_Statement.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Rules for Format Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Format_Statement.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"DataType setting","topics":[{"type":"topic","name":"Rules for Format Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Format_Statement.htm"}]},{"type":"key","name":"Dayhoff","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Dayhoff 1979","url":"Dayhoff_1979.htm"},{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"},{"type":"topic","name":"Dayhoff/JTT Distance Could Not Be Computed","url":"Dayhoff_JTT_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Gamma distance (Amino acids)","url":"Gamma_distance_Amino_acids.htm"}],"keys":[{"type":"key","name":"estimating","topics":[{"type":"topic","name":"Gamma distance (Amino acids)","url":"Gamma_distance_Amino_acids.htm"}]}]},{"type":"key","name":"Dayhoff 1966","topics":[{"type":"topic","name":"Eck and Dayhoff 1966","url":"Eck_and_Dayhoff_1966.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"}]},{"type":"key","name":"Dayhoff 1978","topics":[{"type":"topic","name":"Dayhoff 1978","url":"Dayhoff_1978.htm"}]},{"type":"key","name":"Dayhoff 1979","topics":[{"type":"topic","name":"Dayhoff 1979","url":"Dayhoff_1979.htm"},{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"}]},{"type":"key","name":"Dayhoff MO","topics":[{"type":"topic","name":"Dayhoff 1978","url":"Dayhoff_1978.htm"},{"type":"topic","name":"Eck and Dayhoff 1966","url":"Eck_and_Dayhoff_1966.htm"}]},{"type":"key","name":"Dayhoff/JTT Distance Could Not Be Computed","topics":[{"type":"topic","name":"Dayhoff/JTT Distance Could Not Be Computed","url":"Dayhoff_JTT_Distance_Could_Not_Be_Computed.htm"}]},{"type":"key","name":"dbNSFP","topics":[{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"}]},{"type":"key","name":"dbNSFP_1","topics":[{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"}]},{"type":"key","name":"Debian","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"},{"type":"topic","name":"System Requirements","url":"Part_I_Getting_Started/Installing_MEGA/System_Requirements.htm"}]},{"type":"key","name":"Debian-based","topics":[{"type":"topic","name":"Uninstalling MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Uninstalling_MEGA.htm"}]},{"type":"key","name":"DeBry 1992","topics":[{"type":"topic","name":"DeBry 1992","url":"DeBry_1992.htm"}]},{"type":"key","name":"DeBry RW","topics":[{"type":"topic","name":"DeBry 1992","url":"DeBry_1992.htm"}]},{"type":"key","name":"Decrease Decimal","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"Decrease Precision","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"}]},{"type":"key","name":"Define","topics":[{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"}]},{"type":"key","name":"Define/Edit/Select","topics":[{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"}]},{"type":"key","name":"Defining Genes","topics":[{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"Defining Groups","topics":[{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}],"keys":[{"type":"key","name":"Taxa","topics":[{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}]}]},{"type":"key","name":"Degeneracy","topics":[{"type":"topic","name":"Degeneracy","url":"Degeneracy.htm"}]},{"type":"key","name":"Degenerate Sites","topics":[{"type":"topic","name":"Highlight 0-fold Degenerate Sites","url":"Highlight_0-fold_Degenerate_Sites.htm"},{"type":"topic","name":"Highlight 2-fold Degenerate Sites","url":"Highlight_2-fold_Degenerate_Sites.htm"},{"type":"topic","name":"Highlight 4-fold Degenerate Sites","url":"Highlight_4-fold_Degenerate_Sites.htm"},{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"}]},{"type":"key","name":"Delay Divergent Cutoff","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Delete","topics":[{"type":"topic","name":"Delete (in Text Editor)","url":"Delete_in_Text_Editor.htm"},{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"},{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"},{"type":"topic","name":"Setup/Select Taxa & Groups Dialog","url":"Setup_Select_Taxa_Groups_Dialog.htm"}],"keys":[{"type":"key","name":"Data","topics":[{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"}]}]},{"type":"key","name":"Delete Gap","topics":[{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Edit Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Edit_Menu.htm"},{"type":"topic","name":"Edit Menu (in Alignment Explorer)","url":"Edit_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"deleteriousness","topics":[{"type":"topic","name":"Diagnose Mutations","url":"Diagnose_Mutations.htm"},{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"}],"keys":[{"type":"key","name":"Forecast","topics":[{"type":"topic","name":"Diagnose Mutations","url":"Diagnose_Mutations.htm"}]},{"type":"key","name":"nsSNVs","topics":[{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"}]}]},{"type":"key","name":"Deletion","topics":[{"type":"topic","name":"Alignment Gaps and Sites with Missing Information","url":"Alignment_Gaps_and_Sites_with_Missing_Information.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"Sequence Data Subset Selection","url":"Sequence_Data_Subset_Selection.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
