{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Johnson CS","topics":[{"type":"topic","name":"Estabrook et al. 1975","url":"Estabrook_et_al_1975.htm"}]},{"type":"key","name":"Join","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Bootstrap Test of Phylogeny","url":"Bootstrap_Test_of_Phylogeny.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Interior Branch Test of Phylogeny","url":"Interior_Branch_Test_of_Phylogeny.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"},{"type":"topic","name":"Maximum Likelihood (ML)","url":"Maximum_Likelihood_ML.htm"},{"type":"topic","name":"Minimum Evolution","url":"Minimum_Evolution.htm"},{"type":"topic","name":"Minimum Evolution (Construct Phylogeny)","url":"Minimum_Evolution_Construct_Phylogeny.htm"},{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"},{"type":"topic","name":"Saitou and Nei 1987","url":"Saitou_and_Nei_1987.htm"},{"type":"topic","name":"Studier and Keppler 1988","url":"Studier_and_Keppler_1988.htm"},{"type":"topic","name":"Tamura et al. 2004","url":"Tamura_et_al_2004.htm"},{"type":"topic","name":"Tateno et al. 1994","url":"Tateno_et_al_1994.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]},{"type":"key","name":"Joining Tre","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"Joining Tree","topics":[{"type":"topic","name":"Bootstrap Test of Phylogeny","url":"Bootstrap_Test_of_Phylogeny.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Interior Branch Test of Phylogeny","url":"Interior_Branch_Test_of_Phylogeny.htm"},{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"Joining Tree menu","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]},{"type":"key","name":"Jones","topics":[{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"}]},{"type":"key","name":"Jones DT","topics":[{"type":"topic","name":"Jones et al. 1992","url":"Jones_et_al_1992.htm"}]},{"type":"key","name":"Jones et al","topics":[{"type":"topic","name":"Jones et al. 1992","url":"Jones_et_al_1992.htm"}]},{"type":"key","name":"JTT","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"},{"type":"topic","name":"Dayhoff/JTT Distance Could Not Be Computed","url":"Dayhoff_JTT_Distance_Could_Not_Be_Computed.htm"}]},{"type":"key","name":"JTT distances","topics":[{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"}]},{"type":"key","name":"JTT Models","topics":[{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"}]},{"type":"key","name":"Jukes","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Jukes and Cantor 1969","url":"Jukes_and_Cantor_1969.htm"},{"type":"topic","name":"Jukes-Cantor distance","url":"Jukes-Cantor_distance.htm"},{"type":"topic","name":"Jukes-Cantor Distance Failed","url":"Jukes-Cantor_Distance_Failed.htm"},{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"},{"type":"topic","name":"Tajima Nei Distance (Gamma Rates and Heterogeneous patterns)","url":"Tajima_Nei_Distance_Gamma_Rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima Nei distance (Gamma rates)","url":"Tajima_Nei_distance_Gamma_rates.htm"},{"type":"topic","name":"Tajima Nei Distance (Heterogeneous patterns)","url":"Tajima_Nei_Distance_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima-Nei distance","url":"Tajima-Nei_distance.htm"}],"keys":[{"type":"key","name":"select","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"}]}]},{"type":"key","name":"Jukes TH","topics":[{"type":"topic","name":"Jukes and Cantor 1969","url":"Jukes_and_Cantor_1969.htm"}]},{"type":"key","name":"Jukes_and_Cantor_1969","topics":[{"type":"topic","name":"Jukes-Cantor distance","url":"Jukes-Cantor_distance.htm"},{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"}]},{"type":"key","name":"Jukes-Cantor","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"},{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"},{"type":"topic","name":"Nei-Gojobori Method","url":"Nei-Gojobori_Method.htm"}]},{"type":"key","name":"Jukes-Cantor distance","topics":[{"type":"topic","name":"Jukes-Cantor distance","url":"Jukes-Cantor_distance.htm"}]},{"type":"key","name":"Jukes-Cantor Distance Failed","topics":[{"type":"topic","name":"Jukes-Cantor Distance Failed","url":"Jukes-Cantor_Distance_Failed.htm"}]},{"type":"key","name":"Jukes-Cantor Gamma distance","topics":[{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"}]},{"type":"key","name":"jump","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}],"keys":[{"type":"key","name":"Step","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}]}]},{"type":"key","name":"keep","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}],"keys":[{"type":"key","name":"Cut","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"Mutation Detail View","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"}]}]},{"type":"key","name":"Keep Predefined Gaps","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Keppler 1988","topics":[{"type":"topic","name":"Studier and Keppler 1988","url":"Studier_and_Keppler_1988.htm"}]},{"type":"key","name":"Keppler KL","topics":[{"type":"topic","name":"Studier and Keppler 1988","url":"Studier_and_Keppler_1988.htm"}]},{"type":"key","name":"keyword searching","topics":[{"type":"topic","name":"Index","url":"Index_1.htm"}]},{"type":"key","name":"Keywords","topics":[{"type":"topic","name":"Common Features","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Common_Features.htm"},{"type":"topic","name":"Gene Search Tab","url":"Gene_Search_Tab.htm"},{"type":"topic","name":"Index","url":"Index_1.htm"},{"type":"topic","name":"Keywords","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Keywords.htm"},{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"},{"type":"topic","name":"Keywords for Format Statement","url":"Keywords_for_Format_Statement.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Rules for Format Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Format_Statement.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Comments","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Writing_Comments.htm"}],"keys":[{"type":"key","name":"Format Statement","topics":[{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"}]}]},{"type":"key","name":"Keywords Exon","topics":[{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"}]},{"type":"key","name":"Kimura","topics":[{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"},{"type":"topic","name":"Kimura 1980","url":"Kimura_1980.htm"},{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura Distance Failed","url":"Kimura_Distance_Failed.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"},{"type":"topic","name":"p distance is found to be > 1","url":"p_distance_is_found_to_be_1.htm"}]},{"type":"key","name":"Kimura gamma distance","topics":[{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"}]},{"type":"key","name":"Kimura Two","topics":[{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"}]},{"type":"key","name":"Kimura_1980","topics":[{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"}]},{"type":"key","name":"Kishino","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Felsenstein and Kishino 1993","url":"Felsenstein_and_Kishino_1993.htm"},{"type":"topic","name":"Kishino and Hasegawa 1989","url":"Kishino_and_Hasegawa_1989.htm"}]},{"type":"key","name":"Kishino 1993","topics":[{"type":"topic","name":"Felsenstein and Kishino 1993","url":"Felsenstein_and_Kishino_1993.htm"}]},{"type":"key","name":"Ko","topics":[{"type":"topic","name":"MEGA Software Development Team","url":"MEGA_Software_Development_Team.htm"}]},{"type":"key","name":"Koichiro Tamura","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"}]},{"type":"key","name":"Kondrashov","topics":[{"type":"topic","name":"Adzhubei et al. 2010","url":"Adzhubei_et_al_2010.htm"}]},{"type":"key","name":"Krimbas","topics":[{"type":"topic","name":"Sourdis and Krimbas 1987","url":"Sourdis_and_Krimbas_1987.htm"}]},{"type":"key","name":"Krimbas 1987","topics":[{"type":"topic","name":"Sourdis and Krimbas 1987","url":"Sourdis_and_Krimbas_1987.htm"}]},{"type":"key","name":"Kruskal JB","topics":[{"type":"topic","name":"Sankoff and Cedergren 1983","url":"Sankoff_and_Cedergren_1983.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
