{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"NCBI BLAST","topics":[{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"},{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"},{"type":"topic","name":"Web Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Web_Menu.htm"},{"type":"topic","name":"Web Menu (in Alignment Explorer)","url":"Web_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"NCBI Query and BLAST","topics":[{"type":"topic","name":"Alignment Menu","url":"Alignment_Menu.htm"}]},{"type":"key","name":"NCBIs Blast","topics":[{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"}]},{"type":"key","name":"NCHAR","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"nd","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Fisher's Exact Test Has Failed","url":"Fisher_s_Exact_Test_Has_Failed.htm"},{"type":"topic","name":"Include Sites Option","url":"Include_Sites_Option.htm"}]},{"type":"key","name":"near the","topics":[{"type":"topic","name":"Converting GCG Format","url":"Converting_GCG_Format.htm"}]},{"type":"key","name":"Nearest","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Nearest-Neighbor-Interchange (NNI)","url":"Nearest-Neighbor-Interchange_NNI.htm"}]},{"type":"key","name":"Nearest_Neighbor_Interchange_NNI","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"}]},{"type":"key","name":"Nearest-Neighbor-Interchange","topics":[{"type":"topic","name":"Nearest-Neighbor-Interchange (NNI)","url":"Nearest-Neighbor-Interchange_NNI.htm"}]},{"type":"key","name":"Nei","topics":[{"type":"topic","name":"Acknowledgements","url":"Acknowledgements.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Bootstrap method to compute standard error of distance estimates","url":"Bootstrap_method_to_compute_standard_error_of_distance_estimates.htm"},{"type":"topic","name":"Bootstrap Test of Phylogeny","url":"Bootstrap_Test_of_Phylogeny.htm"},{"type":"topic","name":"Branch-and-Bound algorithm","url":"Branch-and-Bound_algorithm.htm"},{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Codon Based Fisher's Exact Test","url":"Codon_Based_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Codon Based Z-Test (large sample)","url":"Codon_Based_Z-Test_large_sample.htm"},{"type":"topic","name":"Condensed Trees","url":"Condensed_Trees.htm"},{"type":"topic","name":"Consensus Tree","url":"Consensus_Tree.htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Gamma distance (Amino acids)","url":"Gamma_distance_Amino_acids.htm"},{"type":"topic","name":"General Comments on Statistical Tests","url":"General_Comments_on_Statistical_Tests.htm"},{"type":"topic","name":"Genetic Code Statistical Attributes","url":"Genetic_Code_Statistical_Attributes.htm"},{"type":"topic","name":"Interior Branch Test of Phylogeny","url":"Interior_Branch_Test_of_Phylogeny.htm"},{"type":"topic","name":"Jukes-Cantor distance","url":"Jukes-Cantor_distance.htm"},{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"},{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"Kumar et al. 1993","url":"Kumar_et_al_1993.htm"},{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"},{"type":"topic","name":"Large Sample Tests of Selection","url":"Large_Sample_Tests_of_Selection.htm"},{"type":"topic","name":"Li-Wu-Luo Method","url":"Li-Wu-Luo_Method.htm"},{"type":"topic","name":"Max-mini branch-and-bound search","url":"Max-mini_branch-and-bound_search.htm"},{"type":"topic","name":"Maximum Composite Likelihood","url":"Maximum_Composite_Likelihood.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Gamma Rates and Heterogeneous Patterns)","url":"Maximum_Composite_Likelihood_Gamma_Rates_and_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Gamma Rates)","url":"Maximum_Composite_Likelihood_Gamma_Rates.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Heterogeneous Patterns)","url":"Maximum_Composite_Likelihood_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Maximum Composite Likelihood Method","url":"Maximum_Composite_Likelihood_Method.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Min-mini algorithm","url":"Min-mini_algorithm.htm"},{"type":"topic","name":"Minimum Evolution","url":"Minimum_Evolution.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"},{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"},{"type":"topic","name":"Nei 1986","url":"Nei_1986.htm"},{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"},{"type":"topic","name":"Nei and Gojobori 1986","url":"Nei_and_Gojobori_1986.htm"},{"type":"topic","name":"Nei and Jin 1989","url":"Nei_and_Jin_1989.htm"},{"type":"topic","name":"Nei and Kumar 2000","url":"Nei_and_Kumar_2000.htm"},{"type":"topic","name":"Nei et al. 1976","url":"Nei_et_al_1976.htm"},{"type":"topic","name":"Nei et al. 1985","url":"Nei_et_al_1985.htm"},{"type":"topic","name":"Nei et al. 1998","url":"Nei_et_al_1998.htm"},{"type":"topic","name":"Nei-Gojobori Method","url":"Nei-Gojobori_Method.htm"},{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"},{"type":"topic","name":"Neighbor-Joining (NJ) Method","url":"Neighbor-Joining_NJ_Method.htm"},{"type":"topic","name":"No. of differences (Amino acids)","url":"No_of_differences_Amino_acids.htm"},{"type":"topic","name":"No. of differences (Nucleotide)","url":"No_of_differences_Nucleotide.htm"},{"type":"topic","name":"p-distance (Amino acids)","url":"p-distance_Amino_acids.htm"},{"type":"topic","name":"p-distance (Nucleotide)","url":"p-distance_Nucleotide.htm"},{"type":"topic","name":"Pamilo and Nei 1988","url":"Pamilo_and_Nei_1988.htm"},{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"},{"type":"topic","name":"Phylogenetic Inference","url":"Phylogenetic_Inference.htm"},{"type":"topic","name":"Poisson Correction (PC) distance","url":"Poisson_Correction_PC_distance.htm"},{"type":"topic","name":"Rzhetsky and Nei 1992","url":"Rzhetsky_and_Nei_1992.htm"},{"type":"topic","name":"Rzhetsky and Nei 1993","url":"Rzhetsky_and_Nei_1993.htm"},{"type":"topic","name":"Saitou and Nei 1987","url":"Saitou_and_Nei_1987.htm"},{"type":"topic","name":"Select Genetic Code Table Dialog","url":"Select_Genetic_Code_Table_Dialog.htm"},{"type":"topic","name":"Sourdis and Nei 1988","url":"Sourdis_and_Nei_1988.htm"},{"type":"topic","name":"Studier and Keppler 1988","url":"Studier_and_Keppler_1988.htm"},{"type":"topic","name":"Syn-Nonsynonymous","url":"Syn-Nonsynonymous.htm"},{"type":"topic","name":"Tajima and Nei 1982","url":"Tajima_and_Nei_1982.htm"},{"type":"topic","name":"Tajima and Nei 1984","url":"Tajima_and_Nei_1984.htm"},{"type":"topic","name":"Tajima Nei Distance (Gamma Rates and Heterogeneous patterns)","url":"Tajima_Nei_Distance_Gamma_Rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima Nei distance (Gamma rates)","url":"Tajima_Nei_distance_Gamma_rates.htm"},{"type":"topic","name":"Tajima Nei Distance (Heterogeneous patterns)","url":"Tajima_Nei_Distance_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima's Test (Relative Rate)","url":"Tajima_s_Test_Relative_Rate.htm"},{"type":"topic","name":"Tajima's Test of Neutrality","url":"Tajima_s_Test_of_Neutrality.htm"},{"type":"topic","name":"Tajima-Nei distance","url":"Tajima-Nei_distance.htm"},{"type":"topic","name":"Tajima-Nei Distance Could Not Be Computed","url":"Tajima-Nei_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Takahashi and Nei 2000","url":"Takahashi_and_Nei_2000.htm"},{"type":"topic","name":"Takezaki et al. 1995","url":"Takezaki_et_al_1995.htm"},{"type":"topic","name":"Tamura (1992) Distance Could Not Be Computed","url":"Tamura_1992_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"},{"type":"topic","name":"Tamura and Nei 1993","url":"Tamura_and_Nei_1993.htm"},{"type":"topic","name":"Tamura et al. 2004","url":"Tamura_et_al_2004.htm"},{"type":"topic","name":"Tamura et al. 2007","url":"Tamura_et_al_2007.htm"},{"type":"topic","name":"Tamura-Nei distance","url":"Tamura-Nei_distance.htm"},{"type":"topic","name":"Tamura-Nei distance (Gamma rates and Heterogeneous patterns)","url":"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura-Nei distance (Heterogeneous Patterns)","url":"Tamura-Nei_distance_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Tamura-Nei Distance Could Not Be Computed","url":"Tamura-Nei_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Tamura-Nei gamma distance","url":"Tamura-Nei_gamma_distance.htm"},{"type":"topic","name":"Tanaka and Nei 1989","url":"Tanaka_and_Nei_1989.htm"},{"type":"topic","name":"Tateno et al. 1982","url":"Tateno_et_al_1982.htm"},{"type":"topic","name":"Tateno et al. 1994","url":"Tateno_et_al_1994.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"UPGMA","url":"UPGMA.htm"},{"type":"topic","name":"Zhang et al. 1997","url":"Zhang_et_al_1997.htm"},{"type":"topic","name":"Zhang et al. 1998","url":"Zhang_et_al_1998.htm"}]},{"type":"key","name":"Nei 2000","topics":[{"type":"topic","name":"Takahashi and Nei 2000","url":"Takahashi_and_Nei_2000.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
