{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Format","topics":[{"type":"topic","name":"Open Data","url":"Open_Data.htm"},{"type":"topic","name":"Prediction Data Tab","url":"Prediction_Data_Tab.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}],"keys":[{"type":"key","name":"clicking","topics":[{"type":"topic","name":"Prediction Data Tab","url":"Prediction_Data_Tab.htm"}]},{"type":"key","name":"contains","topics":[{"type":"topic","name":"Open Data","url":"Open_Data.htm"}]}]},{"type":"key","name":"Format command","topics":[{"type":"topic","name":"Format command","url":"Format_command.htm"},{"type":"topic","name":"General Considerations (Distance Data Format)","url":"General_Considerations_(Distance_Data_Format).htm"}]},{"type":"key","name":"Format Converter window","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"Format Convertor window","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"}]},{"type":"key","name":"FORMAT DATATYPE","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"Format Selected Sequence","topics":[{"type":"topic","name":"Format Selected Sequence","url":"Format_Selected_Sequence.htm"},{"type":"topic","name":"Format Selected Sequence","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Format_Selected_Sequence.htm"}]},{"type":"key","name":"Format Statement","topics":[{"type":"topic","name":"Common Features","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Common_Features.htm"},{"type":"topic","name":"Keywords for Format Statement","url":"Keywords_for_Format_Statement.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Rules for Format Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Format_Statement.htm"},{"type":"topic","name":"Writing Comments","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Writing_Comments.htm"}],"keys":[{"type":"key","name":"Keywords","topics":[{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"}]}]},{"type":"key","name":"Frequency","topics":[{"type":"topic","name":"Branch tab (in Options dialog box)","url":"Branch_tab_in_Options_dialog_box.htm"}]},{"type":"key","name":"Freydenzon","topics":[{"type":"topic","name":"Filipski et al. 2014","url":"Filipski_et_al_2014.htm"}]},{"type":"key","name":"from the root of","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"}]},{"type":"key","name":"Fuerst PA","topics":[{"type":"topic","name":"Nei et al. 1976","url":"Nei_et_al_1976.htm"}]},{"type":"key","name":"functionalities","topics":[{"type":"topic","name":"Distance Data Explorer","url":"Distance_Data_Explorer.htm"},{"type":"topic","name":"Distance Matrix Explorer","url":"Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Subtree Menu (in Tree Explorer)","url":"Subtree_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"}]},{"type":"key","name":"Functions","topics":[{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"Editing","topics":[{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}]}]},{"type":"key","name":"G., Gibson","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"}]},{"type":"key","name":"Gadagkar 2001","topics":[{"type":"topic","name":"Disparity Index Test","url":"Disparity_Index_Test.htm"},{"type":"topic","name":"Kumar and Gadagkar 2001","url":"Kumar_and_Gadagkar_2001.htm"},{"type":"topic","name":"Pattern tests","url":"Pattern_Tests.htm"}]},{"type":"key","name":"Gadagkar SR","topics":[{"type":"topic","name":"Kumar and Gadagkar 2001","url":"Kumar_and_Gadagkar_2001.htm"}]},{"type":"key","name":"Gamma","topics":[{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Equal Input Model (Gamma rates and Heterogeneous Patterns)","url":"Equal_Input_Model_Gamma_rates_and_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Equal Input Model (Gamma)","url":"Equal_Input_Model_Gamma.htm"},{"type":"topic","name":"Estimate Gamma Parameter for Site Rates (ML)","url":"Estimate_Gamma_Parameter_for_Site_Rates_(ML).htm"},{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"}],"keys":[{"type":"key","name":"using","topics":[{"type":"topic","name":"Equal Input Model (Gamma rates and Heterogeneous Patterns)","url":"Equal_Input_Model_Gamma_rates_and_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Equal Input Model (Gamma)","url":"Equal_Input_Model_Gamma.htm"},{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"}]}]},{"type":"key","name":"Gamma distance","topics":[{"type":"topic","name":"Gamma distance (Amino acids)","url":"Gamma_distance_Amino_acids.htm"},{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"}]},{"type":"key","name":"Gamma Distance Failed Because p","topics":[{"type":"topic","name":"Gamma Distance Failed Because p > 0.99","url":"Gamma_Distance_Failed_Because_p_0_99.htm"}]},{"type":"key","name":"Gamma Parameter","topics":[{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Gamma parameter","url":"Gamma_parameter.htm"}],"keys":[{"type":"key","name":"Computing","topics":[{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"}]}]},{"type":"key","name":"Gamma Rates","topics":[{"type":"topic","name":"Maximum Composite Likelihood (Gamma Rates and Heterogeneous Patterns)","url":"Maximum_Composite_Likelihood_Gamma_Rates_and_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Gamma Rates)","url":"Maximum_Composite_Likelihood_Gamma_Rates.htm"},{"type":"topic","name":"Tajima Nei Distance (Gamma Rates and Heterogeneous patterns)","url":"Tajima_Nei_Distance_Gamma_Rates_and_Heterogeneous_patterns.htm"}]},{"type":"key","name":"Gap Extension Penalty","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Gap Opening Penalty","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Gap Separation Distance","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"}]},{"type":"key","name":"Gaps","topics":[{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}]},{"type":"key","name":"Gaps In Distance Estimation","topics":[{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"}]},{"type":"key","name":"Gaps/Missing Data Treatment","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"}]},{"type":"key","name":"Gascuel","topics":[{"type":"topic","name":"Le and Gascuel 2008","url":"Le_and_Gascuel_2008.htm"}]},{"type":"key","name":"Gascuel 2008","topics":[{"type":"topic","name":"Le and Gascuel 2008","url":"Le_and_Gascuel_2008.htm"}]},{"type":"key","name":"gcg","topics":[{"type":"topic","name":"Built-in Genetic Codes","url":"Built-in_Genetic_Codes.htm"},{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"GCG Format","topics":[{"type":"topic","name":"Converting GCG Format","url":"Converting_GCG_Format.htm"}],"keys":[{"type":"key","name":"Converting","topics":[{"type":"topic","name":"Converting GCG Format","url":"Converting_GCG_Format.htm"}]}]},{"type":"key","name":"GCU","topics":[{"type":"topic","name":"Built-in Genetic Codes","url":"Built-in_Genetic_Codes.htm"}]},{"type":"key","name":"GenBank","topics":[{"type":"topic","name":"About BLAST","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/BLAST/About_BLAST.htm"},{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"},{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"},{"type":"topic","name":"Web Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Web_Menu.htm"},{"type":"topic","name":"Web Menu (in Alignment Explorer)","url":"Web_Menu_in_Alignment_Explorer.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
