{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Close File","topics":[{"type":"topic","name":"Close File (in Text Editor)","url":"Close_File_in_Text_Editor.htm"}]},{"type":"key","name":"Close-Neighbor-Interchange","topics":[{"type":"topic","name":"Close-Neighbor-Interchange","url":"Close-Neighbor-Interchange.htm"},{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"}]},{"type":"key","name":"CLUSTAL","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"},{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Converting Clustal Format","url":"Converting_Clustal_Format.htm"},{"type":"topic","name":"First Time User","url":"First_Time_User.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"},{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"}]},{"type":"key","name":"CLUSTAL file","topics":[{"type":"topic","name":"Converting Clustal Format","url":"Converting_Clustal_Format.htm"}]},{"type":"key","name":"Clustal Format","topics":[{"type":"topic","name":"Converting Clustal Format","url":"Converting_Clustal_Format.htm"}],"keys":[{"type":"key","name":"Converting","topics":[{"type":"topic","name":"Converting Clustal Format","url":"Converting_Clustal_Format.htm"}]}]},{"type":"key","name":"Clustal Options","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"}]},{"type":"key","name":"CLUSTALW","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"},{"type":"topic","name":"About Muscle","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/About_Muscle.htm"},{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW","url":"ClustalW.htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"First Time User","url":"First_Time_User.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"initiates","topics":[{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"sequences data using","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]},{"type":"key","name":"using","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]}]},{"type":"key","name":"ClustalW menu item","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"}]},{"type":"key","name":"ClustalW Options","topics":[{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"ClustalW uses","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"}]},{"type":"key","name":"Cluster Method","topics":[{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Clustering","topics":[{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}],"keys":[{"type":"key","name":"Method","topics":[{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]}]},{"type":"key","name":"CNI","topics":[{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Minimum Evolution (Construct Phylogeny)","url":"Minimum_Evolution_Construct_Phylogeny.htm"}]},{"type":"key","name":"Code Table","topics":[{"type":"topic","name":"Built-in Genetic Codes","url":"Built-in_Genetic_Codes.htm"},{"type":"topic","name":"Genetic Code Statistical Attributes","url":"Genetic_Code_Statistical_Attributes.htm"}]},{"type":"key","name":"Code Table Editor","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Code Table Editor","url":"Code_Table_Editor.htm"}]},{"type":"key","name":"CodeTable","topics":[{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"Coding","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}],"keys":[{"type":"key","name":"positions for","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"}]}]},{"type":"key","name":"Coding Sequences via","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"}],"keys":[{"type":"key","name":"Aligning","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"}]}]},{"type":"key","name":"Codon","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Codon","url":"Codon.htm"},{"type":"topic","name":"Estimate Selection for each Codon (HyPhy)","url":"Estimate_Selection_for_each_Codon_HyPhy.htm"},{"type":"topic","name":"Sequence Data Subset Selection","url":"Sequence_Data_Subset_Selection.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"Codon Based","topics":[{"type":"topic","name":"Codon Based Z-Test (large sample)","url":"Codon_Based_Z-Test_large_sample.htm"},{"type":"topic","name":"Get more information about the codon based Z-test for selection","url":"Get_more_information_about_the_codon_based_Z-test_for_selection.htm"}]},{"type":"key","name":"Codon Based Fisher's Exact Test","topics":[{"type":"topic","name":"Codon Based Fisher's Exact Test","url":"Codon_Based_Fisher_s_Exact_Test.htm"}]},{"type":"key","name":"codon based Z","topics":[{"type":"topic","name":"Codon Based Z-Test (large sample)","url":"Codon_Based_Z-Test_large_sample.htm"},{"type":"topic","name":"Get more information about the codon based Z-test for selection","url":"Get_more_information_about_the_codon_based_Z-test_for_selection.htm"}]},{"type":"key","name":"Codon Positions","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Include Sites Option","url":"Include_Sites_Option.htm"}]},{"type":"key","name":"Codon Start","topics":[{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"Codon Usage","topics":[{"type":"topic","name":"Codon Usage","url":"Codon_Usage.htm"},{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Statistics Menu (in Sequence Data Explorer)","url":"Statistics_Menu_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Codon Usage menu","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"}]},{"type":"key","name":"CodonStart","topics":[{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
