{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"TITLE","topics":[{"type":"topic","name":"Common Features","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Common_Features.htm"},{"type":"topic","name":"General Considerations (Distance Data Format)","url":"General_Considerations_(Distance_Data_Format).htm"},{"type":"topic","name":"Keywords","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Keywords.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Rules for Description Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Description_Statement.htm"},{"type":"topic","name":"Rules for Title Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Title_Statement.htm"},{"type":"topic","name":"Writing Comments","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Writing_Comments.htm"}]},{"type":"key","name":"Title This","topics":[{"type":"topic","name":"Rules for Description Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Description_Statement.htm"},{"type":"topic","name":"Rules for Title Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Title_Statement.htm"}]},{"type":"key","name":"Title, Format","topics":[{"type":"topic","name":"Statements in input files","url":"Statements_in_input_files.htm"}]},{"type":"key","name":"To MEGA Format","topics":[{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"}]},{"type":"key","name":"Toggle Auto Column Width","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"}]},{"type":"key","name":"Toggle Conserved Sites","topics":[{"type":"topic","name":"Display Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm"},{"type":"topic","name":"Display Menu (in Alignment Explorer)","url":"Display_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"toolbar","keys":[{"type":"key","name":"Finished button","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}]}]},{"type":"key","name":"Toolbars","topics":[{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Display Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm"},{"type":"topic","name":"Display Menu (in Alignment Explorer)","url":"Display_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Introduction","url":"Introduction.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"display","topics":[{"type":"topic","name":"Display Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm"}]}]},{"type":"key","name":"Topological distance","topics":[{"type":"topic","name":"Topological distance","url":"Topological_distance.htm"}]},{"type":"key","name":"Topology","topics":[{"type":"topic","name":"Topology","url":"Topology.htm"},{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}],"keys":[{"type":"key","name":"Editing","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]}]},{"type":"key","name":"Topology Editor","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]},{"type":"key","name":"Topology Only","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]},{"type":"key","name":"Total","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"}]},{"type":"key","name":"TpG","topics":[{"type":"topic","name":"Highlight CpG/TpG/CpA","url":"Highlight_CpG_TpG_CpA.htm"}]},{"type":"key","name":"Trace Data File Viewer","topics":[{"type":"topic","name":"Sequencer Menu (in Alignment Explorer)","url":"Sequencer_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"}]},{"type":"key","name":"Traditional","topics":[{"type":"topic","name":"View Menu (in Tree Explorer)","url":"View_Menu_in_Tree_Explorer.htm"}]},{"type":"key","name":"Traditional, Radiation","topics":[{"type":"topic","name":"View Menu (in Tree Explorer)","url":"View_Menu_in_Tree_Explorer.htm"}]},{"type":"key","name":"Transition","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"},{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"},{"type":"topic","name":"Transition","url":"Transition.htm"}],"keys":[{"type":"key","name":"estimates","topics":[{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"},{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"}]}]},{"type":"key","name":"Transition bias","topics":[{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"}]},{"type":"key","name":"Transition Matrix","topics":[{"type":"topic","name":"Transition Matrix","url":"Transition_Matrix.htm"}]},{"type":"key","name":"Transition Weight","topics":[{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Transition/transversion","topics":[{"type":"topic","name":"Tamura 3 parameter (Gamma rates and Heterogeneous patterns)","url":"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm"}]},{"type":"key","name":"Transition/Transversion Ratio","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Transition/Transversion Ratio (R)","url":"Transition_Transversion_Ratio_R.htm"}]},{"type":"key","name":"Transition/transversions","topics":[{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"No. of differences (Nucleotide)","url":"No_of_differences_Nucleotide.htm"},{"type":"topic","name":"p-distance (Nucleotide)","url":"p-distance_Nucleotide.htm"},{"type":"topic","name":"Tamura 3 parameter (Heterogeneous patterns)","url":"Tamura_3_parameter_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3-parameter (Gamma)","url":"Tamura_3-parameter_Gamma.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"},{"type":"topic","name":"Tamura-Nei distance","url":"Tamura-Nei_distance.htm"},{"type":"topic","name":"Tamura-Nei distance (Gamma rates and Heterogeneous patterns)","url":"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura-Nei distance (Heterogeneous Patterns)","url":"Tamura-Nei_distance_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Tamura-Nei gamma distance","url":"Tamura-Nei_gamma_distance.htm"}]},{"type":"key","name":"Transitions & Transversions","topics":[{"type":"topic","name":"Jukes-Cantor distance","url":"Jukes-Cantor_distance.htm"},{"type":"topic","name":"Jukes-Cantor Gamma distance","url":"Jukes-Cantor_Gamma_distance.htm"},{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"No. of differences (Nucleotide)","url":"No_of_differences_Nucleotide.htm"},{"type":"topic","name":"p-distance (Nucleotide)","url":"p-distance_Nucleotide.htm"},{"type":"topic","name":"Tajima Nei Distance (Gamma Rates and Heterogeneous patterns)","url":"Tajima_Nei_Distance_Gamma_Rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima Nei distance (Gamma rates)","url":"Tajima_Nei_distance_Gamma_rates.htm"},{"type":"topic","name":"Tajima Nei Distance (Heterogeneous patterns)","url":"Tajima_Nei_Distance_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima-Nei distance","url":"Tajima-Nei_distance.htm"},{"type":"topic","name":"Tamura 3 parameter (Gamma rates and Heterogeneous patterns)","url":"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3 parameter (Heterogeneous patterns)","url":"Tamura_3_parameter_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3-parameter (Gamma)","url":"Tamura_3-parameter_Gamma.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"},{"type":"topic","name":"Tamura-Nei distance","url":"Tamura-Nei_distance.htm"},{"type":"topic","name":"Tamura-Nei distance (Gamma rates and Heterogeneous patterns)","url":"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura-Nei distance (Heterogeneous Patterns)","url":"Tamura-Nei_distance_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Tamura-Nei gamma distance","url":"Tamura-Nei_gamma_distance.htm"}]},{"type":"key","name":"Translate Sequences","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"Translate/Untranslate","topics":[{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Data Menu (Sequence Data Explorer)","url":"Data_Menu_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Translate/Untranslate (in Sequence Data Explorer)","url":"Translate_Untranslate_(in_Sequence_Data_Explorer).htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
