{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Sequence Data","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analyze User Tree by Least Squares","url":"Analyze_User_Tree_by_Least_Squares.htm"},{"type":"topic","name":"Analyze User Tree by Maximum Likelihood","url":"Analyze_User_Tree_by_Maximum_Likelihood.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Computing statistics on only highlighted sites in Data Explorer","url":"Computing_statistics_on_only_highlighted_sites_in_Data_Explorer.htm"},{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Converting Clustal Format","url":"Converting_Clustal_Format.htm"},{"type":"topic","name":"Converting FASTA Format","url":"Converting_FASTA_Format.htm"},{"type":"topic","name":"Converting IG Format","url":"Converting_IG_Format.htm"},{"type":"topic","name":"Converting MSF Format","url":"Converting_MSF_Format.htm"},{"type":"topic","name":"Data Coverage","url":"Data_Coverage.htm"},{"type":"topic","name":"Data Explorer","url":"Data_Explorer.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Data Menu (Sequence Data Explorer)","url":"Data_Menu_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Display Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm"},{"type":"topic","name":"Display Menu (in Alignment Explorer)","url":"Display_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Display Menu (in Sequence Data Explorer)","url":"Display_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Distance Matrix Explorer","url":"Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Edit Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Edit_Menu.htm"},{"type":"topic","name":"Edit Menu (in Alignment Explorer)","url":"Edit_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Estimate Position-by-Position Rates (ML)","url":"Estimate_Position-by-Position_Rates_ML.htm"},{"type":"topic","name":"Estimate Selection for each Codon (HyPhy)","url":"Estimate_Selection_for_each_Codon_HyPhy.htm"},{"type":"topic","name":"Export Data (Sequence Data Explorer)","url":"Export_Data_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Find Motif","url":"Find_Motif.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"Gene Search Tab","url":"Gene_Search_Tab.htm"},{"type":"topic","name":"General Considerations","url":"General_Considerations.htm"},{"type":"topic","name":"General Description","url":"General_Description.htm"},{"type":"topic","name":"Highlight Menu (in Sequence Data Explorer)","url":"Highlight_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"},{"type":"topic","name":"Inferring Ancestral Sequences (ML)","url":"Inferring_Ancestral_Sequences_ML.htm"},{"type":"topic","name":"Input Data Format Dialog","url":"Input_Data_Format_Dialog.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Kishino and Hasegawa 1989","url":"Kishino_and_Hasegawa_1989.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Mutation Detail View","url":"Mutation_Detail_View.htm"},{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"},{"type":"topic","name":"Open Data","url":"Open_Data.htm"},{"type":"topic","name":"Overview","url":"Overview.htm"},{"type":"topic","name":"Preface","url":"Preface.htm"},{"type":"topic","name":"Query Databanks","url":"Query_Databanks.htm"},{"type":"topic","name":"Quit Data Viewer","url":"Quit_Data_Viewer.htm"},{"type":"topic","name":"Restore Input Order","url":"Restore_Input_Order.htm"},{"type":"topic","name":"Rules for Format Statement","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Format_Statement.htm"},{"type":"topic","name":"Rules for Taxa Names","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Taxa_Names.htm"},{"type":"topic","name":"Save Session (in Sequence Data Explorer)","url":"Save_Session_(in_Sequence_Data_Explorer).htm"},{"type":"topic","name":"Select Genetic Code (in Sequence Data Explorer)","url":"Select_Genetic_Code_(in_Sequence_Data_Explorer).htm"},{"type":"topic","name":"Select Genetic Code Table","url":"Select_Genetic_Code_Table.htm"},{"type":"topic","name":"Select Genetic Code Tables","url":"Select_Genetic_Code_Tables.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Sequence Data Explorer Window","url":"Sequence_Data_Explorer_Window.htm"},{"type":"topic","name":"Sequence Data Subset Selection","url":"Sequence_Data_Subset_Selection.htm"},{"type":"topic","name":"Setup/Select Genes & Domains (Sequence Data Explorer)","url":"Setup_Select_Genes___Domains_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Setup/Select Taxa & Groups (in Sequence Data Explorer)","url":"Setup_Select_Taxa_Groups_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Show Sequence Names","url":"Show_Sequence_Names.htm"},{"type":"topic","name":"Sort Sequences As per Taxa/Group Organizer","url":"Sort_Sequences_As_per_Taxa_Group_Organizer.htm"},{"type":"topic","name":"Sourdis and Krimbas 1987","url":"Sourdis_and_Krimbas_1987.htm"},{"type":"topic","name":"Specify the coordinate information using the Sequence Data Explorer","url":"Specify_the_coordinate_information_using_the_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Statistics Menu (in Sequence Data Explorer)","url":"Statistics_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Translate/Untranslate (in Sequence Data Explorer)","url":"Translate_Untranslate_(in_Sequence_Data_Explorer).htm"},{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"},{"type":"topic","name":"View/Edit Sequencer Files","url":"View_Edit_Sequencer_Files.htm"},{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"},{"type":"topic","name":"Web Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Web_Menu.htm"},{"type":"topic","name":"Web Menu (in Alignment Explorer)","url":"Web_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"},{"type":"topic","name":"Writing only 4-fold degenerate sites to an output file","url":"Writing_only_4-fold_degenerate_sites_to_an_output_file.htm"}]},{"type":"key","name":"sequence data containing","topics":[{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"sequence data editing","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
