{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Coffee","topics":[{"type":"topic","name":"About Muscle","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/About_Muscle.htm"}]},{"type":"key","name":"Color Cells","topics":[{"type":"topic","name":"Color Cells","url":"Color_Cells.htm"},{"type":"topic","name":"Display Menu (in Sequence Data Explorer)","url":"Display_Menu_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Column Sizer","topics":[{"type":"topic","name":"Distance Data Explorer","url":"Distance_Data_Explorer.htm"},{"type":"topic","name":"Distance Matrix Explorer","url":"Distance_Matrix_Explorer.htm"}]},{"type":"key","name":"Columns","topics":[{"type":"topic","name":"Prediction Data Tab","url":"Prediction_Data_Tab.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}]},{"type":"key","name":"comatable","topics":[{"type":"topic","name":"MEGA Saved Sessions","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Saved_Sessions.htm"}]},{"type":"key","name":"Comeron","topics":[{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"}]},{"type":"key","name":"Comeron 1995","topics":[{"type":"topic","name":"Comeron 1995","url":"Comeron_1995.htm"}]},{"type":"key","name":"Comeron JM","topics":[{"type":"topic","name":"Comeron 1995","url":"Comeron_1995.htm"}]},{"type":"key","name":"Comeron_1995","topics":[{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"}]},{"type":"key","name":"Comma","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Statistics Menu (in Sequence Data Explorer)","url":"Statistics_Menu_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Comma Separated Values","topics":[{"type":"topic","name":"Display Results in XL/CSV/Text","url":"Display_Results_in_XL_CSV_Text.htm"}],"keys":[{"type":"key","name":"happen","topics":[{"type":"topic","name":"Display Results in XL/CSV/Text","url":"Display_Results_in_XL_CSV_Text.htm"}]}]},{"type":"key","name":"Comma-separated file","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"Command Line Output","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Estimate Selection for each Codon (HyPhy)","url":"Estimate_Selection_for_each_Codon_HyPhy.htm"}]},{"type":"key","name":"Command Statements","topics":[{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}],"keys":[{"type":"key","name":"Writing","topics":[{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}]}]},{"type":"key","name":"Comments","topics":[{"type":"topic","name":"Writing Comments","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Writing_Comments.htm"}],"keys":[{"type":"key","name":"Writing","topics":[{"type":"topic","name":"Writing Comments","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Writing_Comments.htm"}]}]},{"type":"key","name":"Common Features","topics":[{"type":"topic","name":"Common Features","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Common_Features.htm"}]},{"type":"key","name":"COMMON FILE CONVERSION ATTRIBUTES","topics":[{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"Common Parameters","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Common Sites","topics":[{"type":"topic","name":"No Common Sites","url":"No_Common_Sites.htm"}]},{"type":"key","name":"Compare","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}],"keys":[{"type":"key","name":"CP","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"NJ","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"Compare Distances Using Other Models","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"}]},{"type":"key","name":"compatable","topics":[{"type":"topic","name":"MEGA Saved Sessions","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Saved_Sessions.htm"}]},{"type":"key","name":"comple","topics":[{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"}]},{"type":"key","name":"Complete","topics":[{"type":"topic","name":"Alignment Gaps and Sites with Missing Information","url":"Alignment_Gaps_and_Sites_with_Missing_Information.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"Sequence Data Subset Selection","url":"Sequence_Data_Subset_Selection.htm"}],"keys":[{"type":"key","name":"called","topics":[{"type":"topic","name":"Alignment Gaps and Sites with Missing Information","url":"Alignment_Gaps_and_Sites_with_Missing_Information.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"}]},{"type":"key","name":"using","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"}]}]},{"type":"key","name":"Complete-deletion","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"Include Sites Option","url":"Include_Sites_Option.htm"}]},{"type":"key","name":"Complete-Deletion Option","topics":[{"type":"topic","name":"Complete-Deletion Option","url":"Complete-Deletion_Option.htm"}]},{"type":"key","name":"composite likelihood","topics":[{"type":"topic","name":"Maximum Composite Likelihood Method","url":"Maximum_Composite_Likelihood_Method.htm"}],"keys":[{"type":"key","name":"estimated by maximizing","topics":[{"type":"topic","name":"Maximum Composite Likelihood Method","url":"Maximum_Composite_Likelihood_Method.htm"}]}]},{"type":"key","name":"Composition","topics":[{"type":"topic","name":"Amino Acid Composition","url":"Amino_Acid_Composition.htm"},{"type":"topic","name":"Composition Distance","url":"Composition_Distance.htm"}],"keys":[{"type":"key","name":"presents","topics":[{"type":"topic","name":"Composition Distance","url":"Composition_Distance.htm"}]}]},{"type":"key","name":"Composition Distance","topics":[{"type":"topic","name":"Composition Distance","url":"Composition_Distance.htm"}]},{"type":"key","name":"Compress","topics":[{"type":"topic","name":"Compress/Uncompress","url":"Compress_Uncompress.htm"},{"type":"topic","name":"Subtree Menu (in Tree Explorer)","url":"Subtree_Menu_in_Tree_Explorer.htm"}]},{"type":"key","name":"Compress Subtree","topics":[{"type":"topic","name":"Subtree Drawing Options (in Tree Explorer)","url":"Subtree_Drawing_Options_in_Tree_Explorer.htm"}]},{"type":"key","name":"Compress/Uncompress","topics":[{"type":"topic","name":"Compress/Uncompress","url":"Compress_Uncompress.htm"}]},{"type":"key","name":"computa","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}],"keys":[{"type":"key","name":"begin","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"Computational Core","topics":[{"type":"topic","name":"First Time User","url":"First_Time_User.htm"}]},{"type":"key","name":"computationally","topics":[{"type":"topic","name":"About BLAST","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/BLAST/About_BLAST.htm"}]},{"type":"key","name":"Compute Between Group Means","topics":[{"type":"topic","name":"Compute Between Groups Means","url":"Compute_Between_Groups_Means.htm"},{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"}]},{"type":"key","name":"Compute button","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"Compute Composition Distances","topics":[{"type":"topic","name":"Pattern tests","url":"Pattern_Tests.htm"}]},{"type":"key","name":"Compute MCL Substitution Matrix","topics":[{"type":"topic","name":"Compute MCL Substitution Matrix","url":"Compute_MCL_Substitution_Matrix.htm"}]},{"type":"key","name":"Compute MCL Transversion","topics":[{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"}]},{"type":"key","name":"Compute Menu","topics":[{"type":"topic","name":"Compute Menu (in Tree Explorer)","url":"Compute_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Tree Explorer","url":"TreeExplorer_HC_files/Tree_Explorer.htm"}]},{"type":"key","name":"Compute Net Between Group Means menu","topics":[{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"}]},{"type":"key","name":"Compute Net Between Groups Means","topics":[{"type":"topic","name":"Compute Net Between Groups Means","url":"Compute_Net_Between_Groups_Means.htm"}]},{"type":"key","name":"Compute Overall Mean","topics":[{"type":"topic","name":"Compute Overall Mean","url":"Compute_Overall_Mean.htm"}]},{"type":"key","name":"Compute Pairwise","topics":[{"type":"topic","name":"Compute Pairwise","url":"Compute_Pairwise.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}],"keys":[{"type":"key","name":"Domain Definitions","topics":[{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]}]},{"type":"key","name":"Compute Pattern Disparity Index","topics":[{"type":"topic","name":"Pattern tests","url":"Pattern_Tests.htm"}]},{"type":"key","name":"Compute Sequence Diversity","topics":[{"type":"topic","name":"Compute Sequence Diversity","url":"Compute_Sequence_Diversity.htm"}]},{"type":"key","name":"compute standard error","topics":[{"type":"topic","name":"Bootstrap method to compute standard error of distance estimates","url":"Bootstrap_method_to_compute_standard_error_of_distance_estimates.htm"}],"keys":[{"type":"key","name":"Bootstrap method","topics":[{"type":"topic","name":"Bootstrap method to compute standard error of distance estimates","url":"Bootstrap_method_to_compute_standard_error_of_distance_estimates.htm"}]}]},{"type":"key","name":"Compute Time Tree","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]},{"type":"key","name":"Compute Within Group Mean","topics":[{"type":"topic","name":"Compute Within Groups Mean","url":"Compute_Within_Groups_Mean.htm"},{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"}]},{"type":"key","name":"Computer Applications","topics":[{"type":"topic","name":"Jones et al. 1992","url":"Jones_et_al_1992.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
