{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Translated Protein Sequences","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"}]},{"type":"key","name":"Translating","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}],"keys":[{"type":"key","name":"Sequences","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]}]},{"type":"key","name":"Translation","topics":[{"type":"topic","name":"Translation","url":"Translation.htm"}]},{"type":"key","name":"Transversion","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"},{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"},{"type":"topic","name":"No. of differences (Nucleotide)","url":"No_of_differences_Nucleotide.htm"},{"type":"topic","name":"p-distance (Nucleotide)","url":"p-distance_Nucleotide.htm"},{"type":"topic","name":"Tajima's Test (Relative Rate)","url":"Tajima_s_Test_Relative_Rate.htm"},{"type":"topic","name":"Tamura 3 parameter (Gamma rates and Heterogeneous patterns)","url":"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3 parameter (Heterogeneous patterns)","url":"Tamura_3_parameter_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3-parameter (Gamma)","url":"Tamura_3-parameter_Gamma.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"},{"type":"topic","name":"Tamura-Nei distance","url":"Tamura-Nei_distance.htm"},{"type":"topic","name":"Tamura-Nei distance (Gamma rates and Heterogeneous patterns)","url":"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura-Nei distance (Heterogeneous Patterns)","url":"Tamura-Nei_distance_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Tamura-Nei gamma distance","url":"Tamura-Nei_gamma_distance.htm"},{"type":"topic","name":"Transition/Transversion Ratio (R)","url":"Transition_Transversion_Ratio_R.htm"},{"type":"topic","name":"Transversion","url":"Transversion.htm"}]},{"type":"key","name":"Transversion Bias","topics":[{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"}]},{"type":"key","name":"transversional","topics":[{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"},{"type":"topic","name":"Li-Wu-Luo Method","url":"Li-Wu-Luo_Method.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"},{"type":"topic","name":"No. of differences (Nucleotide)","url":"No_of_differences_Nucleotide.htm"},{"type":"topic","name":"p-distance (Nucleotide)","url":"p-distance_Nucleotide.htm"},{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"},{"type":"topic","name":"Tajima Nei Distance (Gamma Rates and Heterogeneous patterns)","url":"Tajima_Nei_Distance_Gamma_Rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima Nei distance (Gamma rates)","url":"Tajima_Nei_distance_Gamma_rates.htm"},{"type":"topic","name":"Tajima Nei Distance (Heterogeneous patterns)","url":"Tajima_Nei_Distance_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tajima-Nei distance","url":"Tajima-Nei_distance.htm"},{"type":"topic","name":"Tamura 3 parameter (Gamma rates and Heterogeneous patterns)","url":"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3 parameter (Heterogeneous patterns)","url":"Tamura_3_parameter_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3-parameter (Gamma)","url":"Tamura_3-parameter_Gamma.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"},{"type":"topic","name":"Tamura-Nei distance","url":"Tamura-Nei_distance.htm"},{"type":"topic","name":"Tamura-Nei distance (Gamma rates and Heterogeneous patterns)","url":"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura-Nei distance (Heterogeneous Patterns)","url":"Tamura-Nei_distance_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Tamura-Nei gamma distance","url":"Tamura-Nei_gamma_distance.htm"},{"type":"topic","name":"Transition/Transversion Ratio (R)","url":"Transition_Transversion_Ratio_R.htm"}],"keys":[{"type":"key","name":"Number","topics":[{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Kimura gamma distance","url":"Kimura_gamma_distance.htm"},{"type":"topic","name":"Tamura 3 parameter (Gamma rates and Heterogeneous patterns)","url":"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3 parameter (Heterogeneous patterns)","url":"Tamura_3_parameter_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3-parameter (Gamma)","url":"Tamura_3-parameter_Gamma.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"},{"type":"topic","name":"Tamura-Nei distance","url":"Tamura-Nei_distance.htm"},{"type":"topic","name":"Tamura-Nei distance (Gamma rates and Heterogeneous patterns)","url":"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura-Nei distance (Heterogeneous Patterns)","url":"Tamura-Nei_distance_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Tamura-Nei gamma distance","url":"Tamura-Nei_gamma_distance.htm"}]}]},{"type":"key","name":"Tree","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Bifurcating Tree","url":"Bifurcating_Tree.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"First Time User","url":"First_Time_User.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}],"keys":[{"type":"key","name":"Bifurcating","topics":[{"type":"topic","name":"Bifurcating Tree","url":"Bifurcating_Tree.htm"}]},{"type":"key","name":"ML","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"}]},{"type":"key","name":"resize","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]},{"type":"key","name":"Use","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"}]}]},{"type":"key","name":"tree are","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"}],"keys":[{"type":"key","name":"that all tips of","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"}]}]},{"type":"key","name":"Tree Bisection","topics":[{"type":"topic","name":"Tree Bisection Reconnection (TBR)","url":"Tree_Bisection_Reconnection_TBR.htm"}]},{"type":"key","name":"Tree Bisection Reconnection","topics":[{"type":"topic","name":"Tree Bisection Reconnection (TBR)","url":"Tree_Bisection_Reconnection_TBR.htm"}]},{"type":"key","name":"Tree Exp","topics":[{"type":"topic","name":"File Menu (in Tree Explorer)","url":"TreeExplorer_HC_files/File_Menu_in_Tree_Explorer.htm"}],"keys":[{"type":"key","name":"exits","topics":[{"type":"topic","name":"File Menu (in Tree Explorer)","url":"TreeExplorer_HC_files/File_Menu_in_Tree_Explorer.htm"}]}]},{"type":"key","name":"Tree Explore","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}],"keys":[{"type":"key","name":"close","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
