{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer ="<?xml version=\"1.0\" encoding=\"utf-8\" ?><data>  <key name=\"Nonsynonymous\" >    <topic name=\"Analysis Options\" url=\"Analysis_Options.htm\" />    <topic name=\"Analysis Preferences (Distance Computation)\" url=\"Analysis_Preferences_Distance_Computation.htm\" />    <topic name=\"Analysis Preferences (Estimate Selection for each Codon)\" url=\"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm\" />    <topic name=\"Analysis Preferences (Fisher's Exact Test)\" url=\"Analysis_Preferences_Fisher_s_Exact_Test.htm\" />    <topic name=\"Analysis Preferences (Minimum Evolution)\" url=\"Analysis_Preferences_Minimum_Evolution.htm\" />    <topic name=\"Analysis Preferences (NJ/UPGMA)\" url=\"Analysis_Preferences_NJ_UPGMA.htm\" />    <topic name=\"Analysis Preferences (Z-test of Selection)\" url=\"Analysis_Preferences_Z-test_of_Selection.htm\" />    <topic name=\"Codon Based Fisher's Exact Test\" url=\"Codon_Based_Fisher_s_Exact_Test.htm\" />    <topic name=\"Codon Based Z-Test (large sample)\" url=\"Codon_Based_Z-Test_large_sample.htm\" />    <topic name=\"Comeron 1995\" url=\"Comeron_1995.htm\" />    <topic name=\"Degeneracy\" url=\"Degeneracy.htm\" />    <topic name=\"Distance Model Options\" url=\"Distance_Model_Options.htm\" />    <topic name=\"Fisher's Exact Test Has Failed\" url=\"Fisher_s_Exact_Test_Has_Failed.htm\" />    <topic name=\"Genetic Code Statistical Attributes\" url=\"Genetic_Code_Statistical_Attributes.htm\" />    <topic name=\"Get more information about the codon based Z-test for selection\" url=\"Get_more_information_about_the_codon_based_Z-test_for_selection.htm\" />    <topic name=\"Jukes-Cantor Distance Failed\" url=\"Jukes-Cantor_Distance_Failed.htm\" />    <topic name=\"Kimura Distance Failed\" url=\"Kimura_Distance_Failed.htm\" />    <topic name=\"Kumar Method\" url=\"Kumar_Method.htm\" />    <topic name=\"Large Sample Tests of Selection\" url=\"Large_Sample_Tests_of_Selection.htm\" />    <topic name=\"Li 1993\" url=\"Li_1993.htm\" />    <topic name=\"Li et al. 1985\" url=\"Li_et_al_1985.htm\" />    <topic name=\"Li-Wu-Luo Method\" url=\"Li-Wu-Luo_Method.htm\" />    <topic name=\"Menus in MEGA are so short; where are all the options?\" url=\"Menus_in_MEGA_are_so_short_where_are_all_the_options.htm\" />    <topic name=\"Models for estimating distances\" url=\"Models_for_estimating_distances.htm\" />    <topic name=\"Modified Nei-Gojobori Method\" url=\"Modified_Nei-Gojobori_Method.htm\" />    <topic name=\"Nei and Gojobori 1986\" url=\"Nei_and_Gojobori_1986.htm\" />    <topic name=\"Nei-Gojobori Method\" url=\"Nei-Gojobori_Method.htm\" />    <topic name=\"Nonsynonymous change\" url=\"Nonsynonymous_change.htm\" />    <topic name=\"p distance is found to be &gt; 1\" url=\"p_distance_is_found_to_be_1.htm\" />    <topic name=\"Pamilo-Bianchi-Li Method\" url=\"Pamilo-Bianchi-Li_Method.htm\" />    <topic name=\"Positive selection\" url=\"Positive_selection.htm\" />    <topic name=\"Purifying selection\" url=\"Purifying_selection.htm\" />    <topic name=\"Synonymous change\" url=\"Synonymous_change.htm\" />    <key name=\"the number\" >      <topic name=\"Modified Nei-Gojobori Method\" url=\"Modified_Nei-Gojobori_Method.htm\" />    </key>  </key>  <key name=\"Nonsynonymous change\" >    <topic name=\"Nonsynonymous change\" url=\"Nonsynonymous_change.htm\" />  </key>  <key name=\"nonsynonymous rates\" >    <topic name=\"Li et al. 1985\" url=\"Li_et_al_1985.htm\" />  </key>  <key name=\"NOT\" >    <topic name=\"Delete (in Text Editor)\" url=\"Delete_in_Text_Editor.htm\" />  </key>  <key name=\"Notations Used\" >    <topic name=\"Guide to Notations Used\" url=\"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm\" />    <key name=\"Guide\" >      <topic name=\"Guide to Notations Used\" url=\"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm\" />    </key>  </key>  <key name=\"NOTE\" >    <topic name=\"Clustal Options (Protein)\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm\" />    <topic name=\"ClustalW Options (DNA)\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm\" />  </key>  <key name=\"Notepad\" >    <topic name=\"MEGA Format\" url=\"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Format.htm\" />    <topic name=\"Text Editor\" url=\"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm\" />  </key>  <key name=\"NSeqs\" >    <topic name=\"Keywords for Format Statement\" url=\"Keywords_for_Format_Statement.htm\" />    <topic name=\"Keywords for Format Statement (Sequence Data)\" url=\"Keywords_for_Format_Statement_(Sequence_Data).htm\" />    <topic name=\"Writing Command Statement for Defining Genes and Domains\" url=\"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm\" />  </key>  <key name=\"NSites\" >    <topic name=\"Keywords for Format Statement (Sequence Data)\" url=\"Keywords_for_Format_Statement_(Sequence_Data).htm\" />    <topic name=\"Writing Command Statement for Defining Genes and Domains\" url=\"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm\" />  </key>  <key name=\"nSNV\" >    <topic name=\"Gene Search Tab\" url=\"Gene_Search_Tab.htm\" />    <topic name=\"Introduction to myPEG\" url=\"Introduction_to_myPEG.htm\" />    <topic name=\"Main MEGA Window\" url=\"Main_MEGA_Window.htm\" />  </key>  <key name=\"nsSNV\" >    <topic name=\"Diagnose Mutations\" url=\"Diagnose_Mutations.htm\" />    <topic name=\"Introduction to myPEG\" url=\"Introduction_to_myPEG.htm\" />    <topic name=\"MEGA-MDW Server\" url=\"MEGA-MDW_Server.htm\" />    <topic name=\"Mutation Explorer\" url=\"Mutation_Explorer.htm\" />    <topic name=\"Overview\" url=\"Overview.htm\" />    <topic name=\"Sequence Data Explorer Window\" url=\"Sequence_Data_Explorer_Window.htm\" />    <topic name=\"Upload a text file with the coordinate information for all nSNVs of interest\" url=\"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm\" />    <key name=\"deleteriousness\" >      <topic name=\"Introduction to myPEG\" url=\"Introduction_to_myPEG.htm\" />    </key>    <key name=\"numbers\" >      <topic name=\"Upload a text file with the coordinate information for all nSNVs of interest\" url=\"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm\" />    </key>  </key>  <key name=\"nsSNVs from the MEGA-MDW\" >    <topic name=\"Upload a text file with the coordinate information for all nSNVs of interest\" url=\"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm\" />  </key>  <key name=\"NTaxa\" >    <topic name=\"Keywords for Format Statement (Sequence Data)\" url=\"Keywords_for_Format_Statement_(Sequence_Data).htm\" />    <topic name=\"Writing Command Statement for Defining Genes and Domains\" url=\"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm\" />  </key>  <key name=\"Nucleic Acids Res\" >    <topic name=\"About ClustalW\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm\" />  </key>  <key name=\"Nucleic Acids Research 14\" >    <topic name=\"Sharp et al. 1986\" url=\"Sharp_et_al_1986.htm\" />  </key>  <key name=\"Nucleic Acids Research 32\" >    <topic name=\"About Muscle\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/About_Muscle.htm\" />  </key>  <key name=\"Nucleotide\" >    <topic name=\"Analysis Options\" url=\"Analysis_Options.htm\" />    <topic name=\"Analysis Preferences (Distance Computation)\" url=\"Analysis_Preferences_Distance_Computation.htm\" />    <topic name=\"Analysis Preferences (Maximum Likelihood)\" url=\"Analysis_Preferences_Maximum_Likelihood.htm\" />    <topic name=\"Analysis Preferences (Minimum Evolution)\" url=\"Analysis_Preferences_Minimum_Evolution.htm\" />    <topic name=\"Analysis Preferences (NJ/UPGMA)\" url=\"Analysis_Preferences_NJ_UPGMA.htm\" />    <topic name=\"Constructing Likelihood Trees\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm\" />    <topic name=\"Distance Model Options\" url=\"Distance_Model_Options.htm\" />    <topic name=\"Estimating Evolutionary Distances\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm\" />    <topic name=\"Nucleotide and Amino Acid Compositions\" url=\"Nucleotide_and_Amino_Acid_Compositions.htm\" />    <topic name=\"Working With Genes and Domains\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm\" />    <key name=\"select\" >      <topic name=\"Estimating Evolutionary Distances\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm\" />    </key>  </key>  <key name=\"Nucleotide Composition\" >    <topic name=\"Computing Sequence Statistics\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm\" />    <topic name=\"Nucleotide Composition\" url=\"Nucleotide_Composition.htm\" />    <topic name=\"Statistics Menu (in Sequence Data Explorer)\" url=\"Statistics_Menu_in_Sequence_Data_Explorer.htm\" />  </key>  <key name=\"Nucleotide Pair Frequencies\" >    <topic name=\"Computing Sequence Statistics\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm\" />    <topic name=\"Nucleotide Pair Frequencies\" url=\"Nucleotide_Pair_Frequencies.htm\" />    <topic name=\"Statistics Menu (in Sequence Data Explorer)\" url=\"Statistics_Menu_in_Sequence_Data_Explorer.htm\" />  </key>  <key name=\"Nucleotide, Syn\" >    <topic name=\"Models for estimating distances\" url=\"Models_for_estimating_distances.htm\" />  </key>  <key name=\"NUM\" >    <topic name=\"Text Editor\" url=\"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm\" />  </key>  </data>";
