{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Nonsynonymous","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Codon Based Fisher's Exact Test","url":"Codon_Based_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Codon Based Z-Test (large sample)","url":"Codon_Based_Z-Test_large_sample.htm"},{"type":"topic","name":"Comeron 1995","url":"Comeron_1995.htm"},{"type":"topic","name":"Degeneracy","url":"Degeneracy.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Fisher's Exact Test Has Failed","url":"Fisher_s_Exact_Test_Has_Failed.htm"},{"type":"topic","name":"Genetic Code Statistical Attributes","url":"Genetic_Code_Statistical_Attributes.htm"},{"type":"topic","name":"Get more information about the codon based Z-test for selection","url":"Get_more_information_about_the_codon_based_Z-test_for_selection.htm"},{"type":"topic","name":"Jukes-Cantor Distance Failed","url":"Jukes-Cantor_Distance_Failed.htm"},{"type":"topic","name":"Kimura Distance Failed","url":"Kimura_Distance_Failed.htm"},{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"},{"type":"topic","name":"Large Sample Tests of Selection","url":"Large_Sample_Tests_of_Selection.htm"},{"type":"topic","name":"Li 1993","url":"Li_1993.htm"},{"type":"topic","name":"Li et al. 1985","url":"Li_et_al_1985.htm"},{"type":"topic","name":"Li-Wu-Luo Method","url":"Li-Wu-Luo_Method.htm"},{"type":"topic","name":"Menus in MEGA are so short; where are all the options?","url":"Menus_in_MEGA_are_so_short_where_are_all_the_options.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"},{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"},{"type":"topic","name":"Nei and Gojobori 1986","url":"Nei_and_Gojobori_1986.htm"},{"type":"topic","name":"Nei-Gojobori Method","url":"Nei-Gojobori_Method.htm"},{"type":"topic","name":"Nonsynonymous change","url":"Nonsynonymous_change.htm"},{"type":"topic","name":"p distance is found to be > 1","url":"p_distance_is_found_to_be_1.htm"},{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"},{"type":"topic","name":"Positive selection","url":"Positive_selection.htm"},{"type":"topic","name":"Purifying selection","url":"Purifying_selection.htm"},{"type":"topic","name":"Synonymous change","url":"Synonymous_change.htm"}],"keys":[{"type":"key","name":"the number","topics":[{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"}]}]},{"type":"key","name":"Nonsynonymous change","topics":[{"type":"topic","name":"Nonsynonymous change","url":"Nonsynonymous_change.htm"}]},{"type":"key","name":"nonsynonymous rates","topics":[{"type":"topic","name":"Li et al. 1985","url":"Li_et_al_1985.htm"}]},{"type":"key","name":"NOT","topics":[{"type":"topic","name":"Delete (in Text Editor)","url":"Delete_in_Text_Editor.htm"}]},{"type":"key","name":"Notations Used","topics":[{"type":"topic","name":"Guide to Notations Used","url":"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm"}],"keys":[{"type":"key","name":"Guide","topics":[{"type":"topic","name":"Guide to Notations Used","url":"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm"}]}]},{"type":"key","name":"NOTE","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Notepad","topics":[{"type":"topic","name":"MEGA Format","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Format.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}]},{"type":"key","name":"NSeqs","topics":[{"type":"topic","name":"Keywords for Format Statement","url":"Keywords_for_Format_Statement.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"NSites","topics":[{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"nSNV","topics":[{"type":"topic","name":"Gene Search Tab","url":"Gene_Search_Tab.htm"},{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"nsSNV","topics":[{"type":"topic","name":"Diagnose Mutations","url":"Diagnose_Mutations.htm"},{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"},{"type":"topic","name":"MEGA-MDW Server","url":"MEGA-MDW_Server.htm"},{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"},{"type":"topic","name":"Overview","url":"Overview.htm"},{"type":"topic","name":"Sequence Data Explorer Window","url":"Sequence_Data_Explorer_Window.htm"},{"type":"topic","name":"Upload a text file with the coordinate information for all nSNVs of interest","url":"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm"}],"keys":[{"type":"key","name":"deleteriousness","topics":[{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"}]},{"type":"key","name":"numbers","topics":[{"type":"topic","name":"Upload a text file with the coordinate information for all nSNVs of interest","url":"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm"}]}]},{"type":"key","name":"nsSNVs from the MEGA-MDW","topics":[{"type":"topic","name":"Upload a text file with the coordinate information for all nSNVs of interest","url":"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm"}]},{"type":"key","name":"NTaxa","topics":[{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"Nucleic Acids Res","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"}]},{"type":"key","name":"Nucleic Acids Research 14","topics":[{"type":"topic","name":"Sharp et al. 1986","url":"Sharp_et_al_1986.htm"}]},{"type":"key","name":"Nucleic Acids Research 32","topics":[{"type":"topic","name":"About Muscle","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/About_Muscle.htm"}]},{"type":"key","name":"Nucleotide","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Nucleotide and Amino Acid Compositions","url":"Nucleotide_and_Amino_Acid_Compositions.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}],"keys":[{"type":"key","name":"select","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"}]}]},{"type":"key","name":"Nucleotide Composition","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Nucleotide Composition","url":"Nucleotide_Composition.htm"},{"type":"topic","name":"Statistics Menu (in Sequence Data Explorer)","url":"Statistics_Menu_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Nucleotide Pair Frequencies","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Nucleotide Pair Frequencies","url":"Nucleotide_Pair_Frequencies.htm"},{"type":"topic","name":"Statistics Menu (in Sequence Data Explorer)","url":"Statistics_Menu_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Nucleotide, Syn","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"NUM","topics":[{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
