{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Print","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"File Menu (in Tree Explorer)","url":"TreeExplorer_HC_files/File_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Options dialog box","url":"Options_dialog_box.htm"},{"type":"topic","name":"Print (in Text Editor)","url":"Print_in_Text_Editor.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}],"keys":[{"type":"key","name":"NJ Tree","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"Print or Save","topics":[{"type":"topic","name":"Exit","url":"Exit.htm"}]},{"type":"key","name":"Printer Setup","topics":[{"type":"topic","name":"File Menu (in Tree Explorer)","url":"TreeExplorer_HC_files/File_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Printer Setup","url":"Printer_Setup.htm"}]},{"type":"key","name":"Private","topics":[{"type":"topic","name":"How do I prevent the \"MEGA Update Available\" message showing up","url":"How_do_I_prevent_the_MEGA_Update_Available_message_showing_up.htm"}]},{"type":"key","name":"Prob","topics":[{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"problem by","topics":[{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"}]},{"type":"key","name":"produce","topics":[{"type":"topic","name":"Condensed Trees","url":"Condensed_Trees.htm"},{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"},{"type":"topic","name":"Neighbor-Joining (NJ) Method","url":"Neighbor-Joining_NJ_Method.htm"}],"keys":[{"type":"key","name":"multifurcating","topics":[{"type":"topic","name":"Condensed Trees","url":"Condensed_Trees.htm"}]},{"type":"key","name":"unrooted","topics":[{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"},{"type":"topic","name":"Neighbor-Joining (NJ) Method","url":"Neighbor-Joining_NJ_Method.htm"}]}]},{"type":"key","name":"Program Files","topics":[{"type":"topic","name":"How do I prevent the \"MEGA Update Available\" message showing up","url":"How_do_I_prevent_the_MEGA_Update_Available_message_showing_up.htm"}]},{"type":"key","name":"Programmers","topics":[{"type":"topic","name":"MEGA Software Development Team","url":"MEGA_Software_Development_Team.htm"}]},{"type":"key","name":"Progress window wi","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]},{"type":"key","name":"Project Director","topics":[{"type":"topic","name":"MEGA Software Development Team","url":"MEGA_Software_Development_Team.htm"}]},{"type":"key","name":"Property Tab","topics":[{"type":"topic","name":"Subtree Drawing Options (in Tree Explorer)","url":"Subtree_Drawing_Options_in_Tree_Explorer.htm"}]},{"type":"key","name":"Proportion","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"}],"keys":[{"type":"key","name":"Compute","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"}]}]},{"type":"key","name":"Protein","topics":[{"type":"topic","name":"About BLAST","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/BLAST/About_BLAST.htm"},{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"},{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Alignment Menu","url":"Alignment_Menu.htm"},{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Amino Acid Composition","url":"Amino_Acid_Composition.htm"},{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW","url":"ClustalW.htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Codon Usage","url":"Codon_Usage.htm"},{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Converting IG Format","url":"Converting_IG_Format.htm"},{"type":"topic","name":"Converting MSF Format","url":"Converting_MSF_Format.htm"},{"type":"topic","name":"Data Explorer","url":"Data_Explorer.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Data Menu (Sequence Data Explorer)","url":"Data_Menu_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Dayhoff 1978","url":"Dayhoff_1978.htm"},{"type":"topic","name":"Dayhoff 1979","url":"Dayhoff_1979.htm"},{"type":"topic","name":"Display Menu (in Sequence Data Explorer)","url":"Display_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Domains","url":"Domains.htm"},{"type":"topic","name":"Eck and Dayhoff 1966","url":"Eck_and_Dayhoff_1966.htm"},{"type":"topic","name":"Estimate Position-by-Position Rates (ML)","url":"Estimate_Position-by-Position_Rates_ML.htm"},{"type":"topic","name":"Exon","url":"Exon.htm"},{"type":"topic","name":"Find Best DNA/Protein Models (ML)","url":"Find_Best_DNA_Protein_Models_ML.htm"},{"type":"topic","name":"Gene Search Tab","url":"Gene_Search_Tab.htm"},{"type":"topic","name":"Include Sites Option","url":"Include_Sites_Option.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"},{"type":"topic","name":"Invalid special symbol in molecular sequences","url":"Invalid_special_symbol_in_molecular_sequences.htm"},{"type":"topic","name":"IUPAC Single Letter Codes","url":"IUPAC_Single_Letter_Codes.htm"},{"type":"topic","name":"Jones et al. 1992","url":"Jones_et_al_1992.htm"},{"type":"topic","name":"Jukes and Cantor 1969","url":"Jukes_and_Cantor_1969.htm"},{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Maximum Likelihood (ML)","url":"Maximum_Likelihood_ML.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"MEGA Format","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Format.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"},{"type":"topic","name":"mRNA","url":"mRNA.htm"},{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"},{"type":"topic","name":"Overview","url":"Overview.htm"},{"type":"topic","name":"Pauline and Henikoff 2003","url":"Pauline_and_Henikoff_2003.htm"},{"type":"topic","name":"Preface","url":"Preface.htm"},{"type":"topic","name":"Protein parsimony","url":"Protein_parsimony.htm"},{"type":"topic","name":"Select Genetic Code Table","url":"Select_Genetic_Code_Table.htm"},{"type":"topic","name":"Select Genetic Code Tables","url":"Select_Genetic_Code_Tables.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"},{"type":"topic","name":"Syn-Nonsynonymous","url":"Syn-Nonsynonymous.htm"},{"type":"topic","name":"Tajima's Test (Relative Rate)","url":"Tajima_s_Test_Relative_Rate.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Translate/Untranslate (in Sequence Data Explorer)","url":"Translate_Untranslate_(in_Sequence_Data_Explorer).htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"},{"type":"topic","name":"Zhang and Gu 1998","url":"Zhang_and_Gu_1998.htm"},{"type":"topic","name":"Zuckerkandl and Pauling 1965","url":"Zuckerkandl_and_Pauling_1965.htm"}],"keys":[{"type":"key","name":"Atlas","topics":[{"type":"topic","name":"Dayhoff 1979","url":"Dayhoff_1979.htm"}]}]},{"type":"key","name":"protein coding","topics":[{"type":"topic","name":"Amino Acid Composition","url":"Amino_Acid_Composition.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"},{"type":"topic","name":"Tajima's Test (Relative Rate)","url":"Tajima_s_Test_Relative_Rate.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"protein coding DNA","topics":[{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Protein ID","topics":[{"type":"topic","name":"Gene Search Tab","url":"Gene_Search_Tab.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
