{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Newick","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"},{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Calibration File Format","url":"Calibration_File_Format.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Display Newick Tree From File","url":"Display_Newick_Tree_From_File.htm"},{"type":"topic","name":"Display Newick Trees from File","url":"Display_Newick_Trees_from_File.htm"},{"type":"topic","name":"File Menu (in Tree Explorer)","url":"TreeExplorer_HC_files/File_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Gene Duplication Inference","url":"Gene_Duplication_Inference.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"},{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"},{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}],"keys":[{"type":"key","name":"contents","topics":[{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"}]},{"type":"key","name":"load","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]}]},{"type":"key","name":"Newick Format","topics":[{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"}]},{"type":"key","name":"Newick Tree","topics":[{"type":"topic","name":"Display Newick Tree From File","url":"Display_Newick_Tree_From_File.htm"},{"type":"topic","name":"Display Newick Trees from File","url":"Display_Newick_Trees_from_File.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"Newick Tree From File","topics":[{"type":"topic","name":"Display Newick Tree From File","url":"Display_Newick_Tree_From_File.htm"}]},{"type":"key","name":"nex","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"}]},{"type":"key","name":"next semicolon","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"NEXUS","topics":[{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"NEXUS file","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"NEXUS Format","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}],"keys":[{"type":"key","name":"Converting","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]}]},{"type":"key","name":"Nexus, Interleaved","topics":[{"type":"topic","name":"Export Data (Sequence Data Explorer)","url":"Export_Data_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"General Description","url":"General_Description.htm"}]},{"type":"key","name":"Ng","topics":[{"type":"topic","name":"Pauline and Henikoff 2003","url":"Pauline_and_Henikoff_2003.htm"}]},{"type":"key","name":"NJ","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Condensed Trees","url":"Condensed_Trees.htm"},{"type":"topic","name":"Inferred Tree","url":"Inferred_Tree.htm"},{"type":"topic","name":"Minimum Evolution","url":"Minimum_Evolution.htm"},{"type":"topic","name":"Minimum Evolution (Construct Phylogeny)","url":"Minimum_Evolution_Construct_Phylogeny.htm"},{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"},{"type":"topic","name":"Neighbor-Joining (NJ) Method","url":"Neighbor-Joining_NJ_Method.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}],"keys":[{"type":"key","name":"case","topics":[{"type":"topic","name":"Neighbor-Joining (NJ) Method","url":"Neighbor-Joining_NJ_Method.htm"}]},{"type":"key","name":"Compare","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]},{"type":"key","name":"constructing","topics":[{"type":"topic","name":"Neighbor Joining (Construct Phylogeny)","url":"Neighbor_Joining_Construct_Phylogeny.htm"}]},{"type":"key","name":"showing","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]}]},{"type":"key","name":"NJ Tree","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}],"keys":[{"type":"key","name":"Printing","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"NJ/UPGMA","topics":[{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"}]},{"type":"key","name":"nly","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"NNI","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Maximum Likelihood (ML)","url":"Maximum_Likelihood_ML.htm"},{"type":"topic","name":"Nearest-Neighbor-Interchange (NNI)","url":"Nearest-Neighbor-Interchange_NNI.htm"},{"type":"topic","name":"Subtree-Pruning-Regrafting (SPR)","url":"Subtree-Pruning-Regrafting_SPR.htm"}]},{"type":"key","name":"NNI examines","topics":[{"type":"topic","name":"Subtree-Pruning-Regrafting (SPR)","url":"Subtree-Pruning-Regrafting_SPR.htm"}]},{"type":"key","name":"No","keys":[{"type":"key","name":"Discrete Gamma Categories","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"}]}]},{"type":"key","name":"Node","topics":[{"type":"topic","name":"Node","url":"Node.htm"}]},{"type":"key","name":"Node Height Error Bars","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]},{"type":"key","name":"Node Ids","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}],"keys":[{"type":"key","name":"correspond","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]}]},{"type":"key","name":"Node/Subtree Marker","topics":[{"type":"topic","name":"Subtree Drawing Options (in Tree Explorer)","url":"Subtree_Drawing_Options_in_Tree_Explorer.htm"}]},{"type":"key","name":"NodeName","topics":[{"type":"topic","name":"Calibration File Format","url":"Calibration_File_Format.htm"}]},{"type":"key","name":"Non","topics":[{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"Non-interleaved","topics":[{"type":"topic","name":"Converting Phylip (Non-interleaved) Format","url":"Converting_Phylip_(Non-interleaved)_Format.htm"}]},{"type":"key","name":"Noncoding","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Site Label","url":"Site_Label.htm"},{"type":"topic","name":"What is a Site Label?","url":"What_is_a_Site_Label_.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"},{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}]},{"type":"key","name":"None","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"}]},{"type":"key","name":"Nonsyn","topics":[{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
