{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer ="<?xml version=\"1.0\" encoding=\"utf-8\" ?><data>  <key name=\"Use\" >    <topic name=\"About BLAST\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/BLAST/About_BLAST.htm\" />    <topic name=\"About ClustalW\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm\" />    <topic name=\"Aligning Coding Sequences via Protein Sequences\" url=\"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm\" />    <topic name=\"Analysis Options\" url=\"Analysis_Options.htm\" />    <topic name=\"Analysis Preferences (Distance Computation)\" url=\"Analysis_Preferences_Distance_Computation.htm\" />    <topic name=\"Analysis Preferences (Estimate Selection for each Codon)\" url=\"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm\" />    <topic name=\"Analysis Preferences (Fisher's Exact Test)\" url=\"Analysis_Preferences_Fisher_s_Exact_Test.htm\" />    <topic name=\"Analysis Preferences (Maximum Likelihood)\" url=\"Analysis_Preferences_Maximum_Likelihood.htm\" />    <topic name=\"Analysis Preferences (Maximum Parsimony)\" url=\"Analysis_Preferences_Maximum_Parsimony.htm\" />    <topic name=\"Analysis Preferences (Minimum Evolution)\" url=\"Analysis_Preferences_Minimum_Evolution.htm\" />    <topic name=\"Analysis Preferences (NJ/UPGMA)\" url=\"Analysis_Preferences_NJ_UPGMA.htm\" />    <topic name=\"Analysis Preferences (Pattern Homogeneity Analysis)\" url=\"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm\" />    <topic name=\"Analysis Preferences (Z-test of Selection)\" url=\"Analysis_Preferences_Z-test_of_Selection.htm\" />    <topic name=\"Branch Swap Filter\" url=\"Branch_Swap_Filter.htm\" />    <topic name=\"Building Trees With Distance Data\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm\" />    <topic name=\"Citing MEGA in Publications\" url=\"Citing_MEGA_In_Publications.htm\" />    <topic name=\"Clustal Options (Protein)\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm\" />    <topic name=\"ClustalW Options (DNA)\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm\" />    <topic name=\"Codon Based Fisher's Exact Test\" url=\"Codon_Based_Fisher_s_Exact_Test.htm\" />    <topic name=\"Codon Based Z-Test (large sample)\" url=\"Codon_Based_Z-Test_large_sample.htm\" />    <topic name=\"Computing Sequence Statistics\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm\" />    <topic name=\"Computing statistics on only highlighted sites in Data Explorer\" url=\"Computing_statistics_on_only_highlighted_sites_in_Data_Explorer.htm\" />    <topic name=\"Computing the Gamma Parameter (a)\" url=\"Computing_the_Gamma_Parameter_a.htm\" />    <topic name=\"Constructing a Timetree (ML)\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm\" />    <topic name=\"Constructing Likelihood Trees\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm\" />    <topic name=\"Convert To MEGA Format (Main File Menu)\" url=\"Convert_To_MEGA_Format_Main_File_Menu.htm\" />    <topic name=\"Convert to MEGA Format Dialog Box\" url=\"Convert_to_MEGA_Format_Dialog_Box.htm\" />    <topic name=\"Convert to MEGA Format1\" url=\"Convert_to_MEGA_Format1.htm\" />    <topic name=\"Converting XML Format\" url=\"Converting_XML_Format.htm\" />    <topic name=\"Copy (in Text Editor)\" url=\"Copy_in_Text_Editor.htm\" />    <topic name=\"Cut (in Text Editor)\" url=\"Cut_in_Text_Editor.htm\" />    <topic name=\"Data Menu\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm\" />    <topic name=\"Data Menu (in Alignment Explorer)\" url=\"Data_Menu_in_Alignment_Explorer.htm\" />    <topic name=\"Dayhoff and JTT distances (Gamma rates)\" url=\"Dayhoff_and_JTT_distances_Gamma_rates.htm\" />    <topic name=\"Dayhoff and JTT Models\" url=\"Dayhoff_and_JTT_Models.htm\" />    <topic name=\"Dayhoff/JTT Distance Could Not Be Computed\" url=\"Dayhoff_JTT_Distance_Could_Not_Be_Computed.htm\" />    <topic name=\"Disclaimer\" url=\"Disclaimer.htm\" />    <topic name=\"Display Menu\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm\" />    <topic name=\"Display Menu (in Alignment Explorer)\" url=\"Display_Menu_in_Alignment_Explorer.htm\" />    <topic name=\"Display Menu (in Sequence Data Explorer)\" url=\"Display_Menu_in_Sequence_Data_Explorer.htm\" />    <topic name=\"Display Newick Tree From File\" url=\"Display_Newick_Tree_From_File.htm\" />    <topic name=\"Display Newick Trees from File\" url=\"Display_Newick_Trees_from_File.htm\" />    <topic name=\"Display Saved Tree Session\" url=\"Display_Saved_Tree_Session.htm\" />    <topic name=\"Distances Menu\" url=\"Distances_Menu.htm\" />    <topic name=\"Divergence Times in the Outgroup Cluster are not Visible\" url=\"Divergence_Times_in_the_Outgroup_Cluster_are_not_Visible.htm\" />    <topic name=\"Do BLAST Search\" url=\"Do_BLAST_Search.htm\" />    <topic name=\"Equal Input Correction Failed\" url=\"Equal_Input_Correction_Failed.htm\" />    <topic name=\"Estimate Position-by-Position Rates (ML)\" url=\"Estimate_Position-by-Position_Rates_ML.htm\" />    <topic name=\"Estimate Rate Variation among Sites (ML)\" url=\"Estimate_Rate_Variation_among_Sites_ML.htm\" />    <topic name=\"Estimate Selection for each Codon (HyPhy)\" url=\"Estimate_Selection_for_each_Codon_HyPhy.htm\" />    <topic name=\"Estimating Evolutionary Distances\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm\" />    <topic name=\"First Time User\" url=\"First_Time_User.htm\" />    <topic name=\"Fisher's Exact Test Has Failed\" url=\"Fisher_s_Exact_Test_Has_Failed.htm\" />    <topic name=\"Gamma distance (Amino acids)\" url=\"Gamma_distance_Amino_acids.htm\" />    <topic name=\"Gamma Distance Failed Because p &gt; 0.99\" url=\"Gamma_Distance_Failed_Because_p_0_99.htm\" />    <topic name=\"General Considerations\" url=\"General_Considerations.htm\" />    <topic name=\"Get more information about the codon based Z-test for selection\" url=\"Get_more_information_about_the_codon_based_Z-test_for_selection.htm\" />    <topic name=\"Highlight 0-fold Degenerate Sites\" url=\"Highlight_0-fold_Degenerate_Sites.htm\" />    <topic name=\"Highlight 2-fold Degenerate Sites\" url=\"Highlight_2-fold_Degenerate_Sites.htm\" />    <topic name=\"Highlight 4-fold Degenerate Sites\" url=\"Highlight_4-fold_Degenerate_Sites.htm\" />    <topic name=\"Highlight Conserved Sites\" url=\"Highlight_Conserved_Sites.htm\" />    <topic name=\"Highlight CpG/TpG/CpA\" url=\"Highlight_CpG_TpG_CpA.htm\" />    <topic name=\"Highlight Parsimony Informative Sites\" url=\"Highlight_Parsimony_Informative_Sites.htm\" />    <topic name=\"Highlight Singleton Sites\" url=\"Highlight_Singleton_Sites.htm\" />    <topic name=\"Highlight Variable Sites\" url=\"Highlight_Variable_Sites.htm\" />    <topic name=\"Image Menu (in Tree Explorer)\" url=\"Image_Menu_in_Tree_Explorer.htm\" />    <topic name=\"Inferring Ancestral Sequences (ML)\" url=\"Inferring_Ancestral_Sequences_ML.htm\" />    <topic name=\"Inferring Gene Duplications\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm\" />    <topic name=\"Installing MEGA\" url=\"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm\" />    <topic name=\"Introduction\" url=\"Introduction.htm\" />    <topic name=\"Introduction to myPEG\" url=\"Introduction_to_myPEG.htm\" />    <topic name=\"Invalid special symbol in molecular sequences\" url=\"Invalid_special_symbol_in_molecular_sequences.htm\" />    <topic name=\"Jukes-Cantor Distance Failed\" url=\"Jukes-Cantor_Distance_Failed.htm\" />    <topic name=\"Keywords for Format Statement (Sequence Data)\" url=\"Keywords_for_Format_Statement_(Sequence_Data).htm\" />    <topic name=\"Kimura Distance Failed\" url=\"Kimura_Distance_Failed.htm\" />    <topic name=\"Kumar Method\" url=\"Kumar_Method.htm\" />    <topic name=\"Large Sample Tests of Selection\" url=\"Large_Sample_Tests_of_Selection.htm\" />    <topic name=\"LogDet Distance Could Not Be Computed\" url=\"LogDet_Distance_Could_Not_Be_Computed.htm\" />    <topic name=\"Main MEGA Window\" url=\"Main_MEGA_Window.htm\" />    <topic name=\"Maximum Parsimony (Construct Phylogeny)\" url=\"Maximum_Parsimony_Construct_Phylogeny.htm\" />    <topic name=\"MEGA Basics\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm\" />    <topic name=\"Minimum Evolution\" url=\"Minimum_Evolution.htm\" />    <topic name=\"Minimum Evolution (Construct Phylogeny)\" url=\"Minimum_Evolution_Construct_Phylogeny.htm\" />    <topic name=\"Models for estimating distances\" url=\"Models_for_estimating_distances.htm\" />    <topic name=\"Modified Nei-Gojobori Method\" url=\"Modified_Nei-Gojobori_Method.htm\" />    <topic name=\"Muscle Options (DNA)\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm\" />    <topic name=\"MUSCLE Options (Protein)\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm\" />    <topic name=\"Nei-Gojobori Method\" url=\"Nei-Gojobori_Method.htm\" />    <topic name=\"Neighbor Joining (Construct Phylogeny)\" url=\"Neighbor_Joining_Construct_Phylogeny.htm\" />    <topic name=\"Neighbor-Joining (NJ) Method\" url=\"Neighbor-Joining_NJ_Method.htm\" />    <topic name=\"New (in Text Editor)\" url=\"New_in_Text_Editor.htm\" />    <topic name=\"No. of differences (Amino acids)\" url=\"No_of_differences_Amino_acids.htm\" />    <topic name=\"No. of differences (Nucleotide)\" url=\"No_of_differences_Nucleotide.htm\" />    <topic name=\"Open (in Text Editor)\" url=\"Open_in_Text_Editor.htm\" />    <topic name=\"Open Data\" url=\"Open_Data.htm\" />    <topic name=\"Open Saved Alignment Session\" url=\"Open_Saved_Alignment_Session.htm\" />    <topic name=\"Overview\" url=\"Overview.htm\" />    <topic name=\"p distance is found to be &gt; 1\" url=\"p_distance_is_found_to_be_1.htm\" />    <topic name=\"Penny and Hendy 1985\" url=\"Penny_and_Hendy_1985.htm\" />    <topic name=\"Phylogeny Menu\" url=\"Phylogeny_Menu.htm\" />    <topic name=\"Poisson Correction Failed because p &gt; 0.99\" url=\"Poisson_Correction_Failed_because_p_0_99.htm\" />    <topic name=\"Preface\" url=\"Preface.htm\" />    <topic name=\"Pseudorandom number generator\" url=\"Pseudorandom_number_generator.htm\" />    <topic name=\"Query Databanks\" url=\"Query_Databanks.htm\" />    <topic name=\"Reverse Complement\" url=\"Reverse_Complement.htm\" />    <topic name=\"Reverse Complement\" url=\"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Reverse_Complement.htm\" />    <topic name=\"Rules for Taxa Names\" url=\"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Taxa_Names.htm\" />    <topic name=\"Select Genetic Code Table\" url=\"Select_Genetic_Code_Table.htm\" />    <topic name=\"Select Genetic Code Table Dialog\" url=\"Select_Genetic_Code_Table_Dialog.htm\" />    <topic name=\"Select Genetic Code Tables\" url=\"Select_Genetic_Code_Tables.htm\" />    <topic name=\"Setup/Select Genes &amp; Domains Dialog\" url=\"Setup_Select_Genes_Domains_Dialog.htm\" />    <topic name=\"Setup/Select Taxa &amp; Groups Dialog\" url=\"Setup_Select_Taxa_Groups_Dialog.htm\" />    <topic name=\"Show Only Selected Sequences\" url=\"Show_Only_Selected_Sequences.htm\" />    <topic name=\"Show Web Browser\" url=\"Show_Web_Browser.htm\" />    <topic name=\"Statistics Menu (in Sequence Data Explorer)\" url=\"Statistics_Menu_in_Sequence_Data_Explorer.htm\" />    <topic name=\"System Requirements\" url=\"Part_I_Getting_Started/Installing_MEGA/System_Requirements.htm\" />    <topic name=\"Tajima's Test (Relative Rate)\" url=\"Tajima_s_Test_Relative_Rate.htm\" />    <topic name=\"Tajima-Nei Distance Could Not Be Computed\" url=\"Tajima-Nei_Distance_Could_Not_Be_Computed.htm\" />    <topic name=\"Tamura (1992) Distance Could Not Be Computed\" url=\"Tamura_1992_Distance_Could_Not_Be_Computed.htm\" />    <topic name=\"Tamura-Nei Distance Could Not Be Computed\" url=\"Tamura-Nei_Distance_Could_Not_Be_Computed.htm\" />    <topic name=\"Testing Tree Reliability\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm\" />    <topic name=\"Text Editor\" url=\"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm\" />    <topic name=\"Time Tree Tool\" url=\"Time_Tree_Tool.htm\" />    <topic name=\"Trace Data File Viewer/Editor\" url=\"Trace_Data_File_Viewer_Editor.htm\" />    <topic name=\"Trace Data File Viewer/Editor\" url=\"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm\" />    <topic name=\"Tree length\" url=\"Tree_length.htm\" />    <topic name=\"Tree Topology Editor\" url=\"Tree_Topology_Editor.htm\" />    <topic name=\"Undo (in Text Editor)\" url=\"Undo_in_Text_Editor.htm\" />    <topic name=\"Uninstalling MEGA\" url=\"Part_I_Getting_Started/Installing_MEGA/Uninstalling_MEGA.htm\" />    <topic name=\"Use All Selected Sites\" url=\"Use_All_Selected_Sites.htm\" />    <topic name=\"Use Identical Symbol\" url=\"Use_Identical_Symbol.htm\" />    <topic name=\"Use only Highlighted Sites\" url=\"Use_only_Highlighted_Sites.htm\" />    <topic name=\"Using BLAST Within MEGA\" url=\"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/BLAST/Using_BLAST_Within_MEGA.htm\" />    <topic name=\"Using MEGA in the Classroom\" url=\"Part_I_Getting_Started/Features_and_Support/Using_MEGA_in_the_Classroom.htm\" />    <topic name=\"View/Edit Sequencer Files\" url=\"View_Edit_Sequencer_Files.htm\" />    <topic name=\"Web Browser &amp; Data Miner\" url=\"Web_Browser_Data_Miner.htm\" />    <topic name=\"Web Browser and Data Miner\" url=\"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm\" />    <topic name=\"What's New In This Version\" url=\"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm\" />    <topic name=\"Writing Command Statement for Defining Genes and Domains\" url=\"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm\" />    <key name=\"Data Subset\" >      <topic name=\"Analysis Options\" url=\"Analysis_Options.htm\" />      <topic name=\"Analysis Preferences (Distance Computation)\" url=\"Analysis_Preferences_Distance_Computation.htm\" />      <topic name=\"Analysis Preferences (Estimate Selection for each Codon)\" url=\"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm\" />      <topic name=\"Analysis Preferences (Fisher's Exact Test)\" url=\"Analysis_Preferences_Fisher_s_Exact_Test.htm\" />      <topic name=\"Analysis Preferences (Maximum Likelihood)\" url=\"Analysis_Preferences_Maximum_Likelihood.htm\" />      <topic name=\"Analysis Preferences (Maximum Parsimony)\" url=\"Analysis_Preferences_Maximum_Parsimony.htm\" />      <topic name=\"Analysis Preferences (NJ/UPGMA)\" url=\"Analysis_Preferences_NJ_UPGMA.htm\" />      <topic name=\"Analysis Preferences (Pattern Homogeneity Analysis)\" url=\"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm\" />      <topic name=\"Analysis Preferences (Z-test of Selection)\" url=\"Analysis_Preferences_Z-test_of_Selection.htm\" />    </key>    <key name=\"Paste\" >      <topic name=\"Copy (in Text Editor)\" url=\"Copy_in_Text_Editor.htm\" />    </key>    <key name=\"Remember\" >      <topic name=\"MEGA Basics\" url=\"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm\" />    </key>    <key name=\"Tree\" >      <topic name=\"Analysis Preferences (Estimate Selection for each Codon)\" url=\"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm\" />    </key>    <key name=\"wildcards\" >      <topic name=\"Open Data\" url=\"Open_Data.htm\" />    </key>  </key>  </data>";
