{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Pairwise","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"},{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Average Menu (in Distance Data Explorer)","url":"Average_Menu_in_Distance_Data_Explorer.htm"},{"type":"topic","name":"Average Menu (in Distance Matrix Explorer)","url":"Average_Menu_in_Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Codon Based Z-Test (large sample)","url":"Codon_Based_Z-Test_large_sample.htm"},{"type":"topic","name":"Complete-Deletion Option","url":"Complete-Deletion_Option.htm"},{"type":"topic","name":"Compute Between Groups Means","url":"Compute_Between_Groups_Means.htm"},{"type":"topic","name":"Compute Overall Mean","url":"Compute_Overall_Mean.htm"},{"type":"topic","name":"Compute Pairwise","url":"Compute_Pairwise.htm"},{"type":"topic","name":"Compute Within Groups Mean","url":"Compute_Within_Groups_Mean.htm"},{"type":"topic","name":"Dayhoff/JTT Distance Could Not Be Computed","url":"Dayhoff_JTT_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Distance Data Explorer","url":"Distance_Data_Explorer.htm"},{"type":"topic","name":"Distance Matrix Explorer","url":"Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Distances Menu","url":"Distances_Menu.htm"},{"type":"topic","name":"Equal Input Correction Failed","url":"Equal_Input_Correction_Failed.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"File Menu (in Distance Data Explorer)","url":"File_Menu_in_Distance_Data_Explorer.htm"},{"type":"topic","name":"File Menu (in Distance Matrix Explorer)","url":"File_Menu_in_Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Finding the number of sites in pairwise comparisons","url":"Finding_the_number_of_sites_in_pairwise_comparisons.htm"},{"type":"topic","name":"Fisher's Exact Test Has Failed","url":"Fisher_s_Exact_Test_Has_Failed.htm"},{"type":"topic","name":"Gamma Distance Failed Because p > 0.99","url":"Gamma_Distance_Failed_Because_p_0_99.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"General Considerations (Distance Data Format)","url":"General_Considerations_(Distance_Data_Format).htm"},{"type":"topic","name":"Get more information about the codon based Z-test for selection","url":"Get_more_information_about_the_codon_based_Z-test_for_selection.htm"},{"type":"topic","name":"Input Data Format Dialog","url":"Input_Data_Format_Dialog.htm"},{"type":"topic","name":"Jukes-Cantor Distance Failed","url":"Jukes-Cantor_Distance_Failed.htm"},{"type":"topic","name":"Kimura Distance Failed","url":"Kimura_Distance_Failed.htm"},{"type":"topic","name":"Large Sample Tests of Selection","url":"Large_Sample_Tests_of_Selection.htm"},{"type":"topic","name":"LogDet Distance Could Not Be Computed","url":"LogDet_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"},{"type":"topic","name":"Maximum Composite Likelihood Method","url":"Maximum_Composite_Likelihood_Method.htm"},{"type":"topic","name":"No Common Sites","url":"No_Common_Sites.htm"},{"type":"topic","name":"No. of differences (Amino acids)","url":"No_of_differences_Amino_acids.htm"},{"type":"topic","name":"No. of differences (Nucleotide)","url":"No_of_differences_Nucleotide.htm"},{"type":"topic","name":"OLS branch length estimates","url":"OLS_branch_length_estimates.htm"},{"type":"topic","name":"Options dialog box","url":"Options_dialog_box.htm"},{"type":"topic","name":"p distance is found to be > 1","url":"p_distance_is_found_to_be_1.htm"},{"type":"topic","name":"Pairwise-deletion option","url":"Pairwise-deletion_option.htm"},{"type":"topic","name":"Partial-Deletion Option","url":"Partial-Deletion_Option.htm"},{"type":"topic","name":"Pattern tests","url":"Pattern_Tests.htm"},{"type":"topic","name":"Poisson Correction Failed because p > 0.99","url":"Poisson_Correction_Failed_because_p_0_99.htm"},{"type":"topic","name":"Preface","url":"Preface.htm"},{"type":"topic","name":"Tajima-Nei Distance Could Not Be Computed","url":"Tajima-Nei_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Tamura (1992) Distance Could Not Be Computed","url":"Tamura_1992_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Tamura-Nei Distance Could Not Be Computed","url":"Tamura-Nei_Distance_Could_Not_Be_Computed.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"pairwise comparisons","topics":[{"type":"topic","name":"Finding the number of sites in pairwise comparisons","url":"Finding_the_number_of_sites_in_pairwise_comparisons.htm"}]},{"type":"key","name":"Pairwise Deletion","topics":[{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"pairwise distances","topics":[{"type":"topic","name":"Maximum Composite Likelihood","url":"Maximum_Composite_Likelihood.htm"}]},{"type":"key","name":"Pairwise Sequence Alignment","topics":[{"type":"topic","name":"About ClustalW","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/About_ClustalW.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Pairwise-deletion","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"}]},{"type":"key","name":"Pairwise-deletion option","topics":[{"type":"topic","name":"Pairwise-deletion option","url":"Pairwise-deletion_option.htm"}]},{"type":"key","name":"PAM","topics":[{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"}]},{"type":"key","name":"PAM 001","topics":[{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"}]},{"type":"key","name":"Pamilo","topics":[{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"},{"type":"topic","name":"Pamilo and Bianchi 1993","url":"Pamilo_and_Bianchi_1993.htm"},{"type":"topic","name":"Pamilo and Nei 1988","url":"Pamilo_and_Nei_1988.htm"},{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"}]},{"type":"key","name":"Pamilo_and_Bianchi_1993","topics":[{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"}]},{"type":"key","name":"Pamilo-Bianchi-Li Method","topics":[{"type":"topic","name":"Pamilo-Bianchi-Li Method","url":"Pamilo-Bianchi-Li_Method.htm"},{"type":"topic","name":"Syn-Nonsynonymous","url":"Syn-Nonsynonymous.htm"}]},{"type":"key","name":"PAML","topics":[{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Yang 1999","url":"Yang_1999.htm"}]},{"type":"key","name":"PAMP","topics":[{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"}]},{"type":"key","name":"Parameter","topics":[{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"},{"type":"topic","name":"Tamura 3 parameter (Gamma rates and Heterogeneous patterns)","url":"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3 parameter (Heterogeneous patterns)","url":"Tamura_3_parameter_Heterogeneous_patterns.htm"},{"type":"topic","name":"Tamura 3-parameter (Gamma)","url":"Tamura_3-parameter_Gamma.htm"}]},{"type":"key","name":"parameter distance","topics":[{"type":"topic","name":"Kimura 2-parameter distance","url":"Kimura_2-parameter_distance.htm"},{"type":"topic","name":"Tamura 3-parameter distance","url":"Tamura_3-parameter_distance.htm"}]},{"type":"key","name":"Parameter Model","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Parsim","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"}]},{"type":"key","name":"Parsimony","topics":[{"type":"topic","name":"Analyze User Tree by Parsimony","url":"Analyze_User_Tree_by_Parsimony.htm"},{"type":"topic","name":"Phylogenetic Inference","url":"Phylogenetic_Inference.htm"}]},{"type":"key","name":"Parsimony-informative site","topics":[{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Parsimony-informative site","url":"Parsimony-informative_site.htm"}]},{"type":"key","name":"part","topics":[{"type":"topic","name":"First Time User","url":"First_Time_User.htm"},{"type":"topic","name":"Setup/Select Taxa & Groups Dialog","url":"Setup_Select_Taxa_Groups_Dialog.htm"}],"keys":[{"type":"key","name":"MEGA","topics":[{"type":"topic","name":"First Time User","url":"First_Time_User.htm"}]},{"type":"key","name":"Select","topics":[{"type":"topic","name":"Setup/Select Taxa & Groups Dialog","url":"Setup_Select_Taxa_Groups_Dialog.htm"}]}]},{"type":"key","name":"Partial Deletion","topics":[{"type":"topic","name":"Alignment Gaps and Sites with Missing Information","url":"Alignment_Gaps_and_Sites_with_Missing_Information.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Include Sites Option","url":"Include_Sites_Option.htm"}]},{"type":"key","name":"Partial-Deletion Option","topics":[{"type":"topic","name":"Partial-Deletion Option","url":"Partial-Deletion_Option.htm"}]},{"type":"key","name":"Pascal","topics":[{"type":"topic","name":"Press et al. 1993","url":"Press_et_al_1993.htm"}]},{"type":"key","name":"Pass","topics":[{"type":"topic","name":"Subtree-Pruning-Regrafting (SPR)","url":"Subtree-Pruning-Regrafting_SPR.htm"}],"keys":[{"type":"key","name":"SPR","topics":[{"type":"topic","name":"Subtree-Pruning-Regrafting (SPR)","url":"Subtree-Pruning-Regrafting_SPR.htm"}]}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
