{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"PROTEIN INTERLEAVE MISSING","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"Protein Models","topics":[{"type":"topic","name":"Find Best DNA/Protein Models (ML)","url":"Find_Best_DNA_Protein_Models_ML.htm"}]},{"type":"key","name":"Protein parsimony","topics":[{"type":"topic","name":"Protein parsimony","url":"Protein_parsimony.htm"}]},{"type":"key","name":"PROTEIN sequ","topics":[{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Protein Sequences","topics":[{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Dayhoff 1978","url":"Dayhoff_1978.htm"},{"type":"topic","name":"Eck and Dayhoff 1966","url":"Eck_and_Dayhoff_1966.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"Atlas","topics":[{"type":"topic","name":"Dayhoff 1978","url":"Dayhoff_1978.htm"}]}]},{"type":"key","name":"Protein Weight Matrix","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"}]},{"type":"key","name":"Pruning","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"}]},{"type":"key","name":"Pseudorandom number generator","topics":[{"type":"topic","name":"Pseudorandom number generator","url":"Pseudorandom_number_generator.htm"}]},{"type":"key","name":"ption","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"}]},{"type":"key","name":"Publication PDF","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"}]},{"type":"key","name":"Publications","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"}]},{"type":"key","name":"Purdom et al","topics":[{"type":"topic","name":"Purdom et al. 2000","url":"Purdom_et_al_2000.htm"}]},{"type":"key","name":"Purdom PW","topics":[{"type":"topic","name":"Purdom et al. 2000","url":"Purdom_et_al_2000.htm"}]},{"type":"key","name":"Purifying","topics":[{"type":"topic","name":"Purifying selection","url":"Purifying_selection.htm"}],"keys":[{"type":"key","name":"selection","topics":[{"type":"topic","name":"Purifying selection","url":"Purifying_selection.htm"}]}]},{"type":"key","name":"Purines","topics":[{"type":"topic","name":"Purines","url":"Purines.htm"}]},{"type":"key","name":"push","topics":[{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}],"keys":[{"type":"key","name":"Ok button","topics":[{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]}]},{"type":"key","name":"Pyrimidines","topics":[{"type":"topic","name":"Pyrimidines","url":"Pyrimidines.htm"}]},{"type":"key","name":"Pyrimindine","topics":[{"type":"topic","name":"IUPAC Single Letter Codes","url":"IUPAC_Single_Letter_Codes.htm"}]},{"type":"key","name":"Query","topics":[{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"},{"type":"topic","name":"Sequence Data Explorer Window","url":"Sequence_Data_Explorer_Window.htm"}],"keys":[{"type":"key","name":"MEGA","topics":[{"type":"topic","name":"Sequence Data Explorer Window","url":"Sequence_Data_Explorer_Window.htm"}]}]},{"type":"key","name":"Query Databanks","topics":[{"type":"topic","name":"Query Databanks","url":"Query_Databanks.htm"}]},{"type":"key","name":"Query Genbank","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]},{"type":"key","name":"Query Gene Banks","topics":[{"type":"topic","name":"Web Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Web_Menu.htm"},{"type":"topic","name":"Web Menu (in Alignment Explorer)","url":"Web_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"Quick Start","topics":[{"type":"topic","name":"First Time User","url":"First_Time_User.htm"}]},{"type":"key","name":"quit","topics":[{"type":"topic","name":"Options dialog box","url":"Options_dialog_box.htm"}],"keys":[{"type":"key","name":"Options dialog","topics":[{"type":"topic","name":"Options dialog box","url":"Options_dialog_box.htm"}]}]},{"type":"key","name":"Quit Data Viewer","topics":[{"type":"topic","name":"Data Menu (Sequence Data Explorer)","url":"Data_Menu_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"Quit Data Viewer","url":"Quit_Data_Viewer.htm"}]},{"type":"key","name":"Quit Viewer","topics":[{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"File Menu (in Distance Data Explorer)","url":"File_Menu_in_Distance_Data_Explorer.htm"},{"type":"topic","name":"File Menu (in Distance Matrix Explorer)","url":"File_Menu_in_Distance_Matrix_Explorer.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"RAM","topics":[{"type":"topic","name":"System Requirements","url":"Part_I_Getting_Started/Installing_MEGA/System_Requirements.htm"},{"type":"topic","name":"What's New In This Version","url":"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm"}],"keys":[{"type":"key","name":"4GB","topics":[{"type":"topic","name":"What's New In This Version","url":"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm"}]},{"type":"key","name":"MB","topics":[{"type":"topic","name":"System Requirements","url":"Part_I_Getting_Started/Installing_MEGA/System_Requirements.htm"}]}]},{"type":"key","name":"Ramensky","topics":[{"type":"topic","name":"Adzhubei et al. 2010","url":"Adzhubei_et_al_2010.htm"}]},{"type":"key","name":"Random addition trees","topics":[{"type":"topic","name":"Random addition trees","url":"Random_addition_trees.htm"}]},{"type":"key","name":"rate","topics":[{"type":"topic","name":"UPGMA (Construct Phylogeny)","url":"UPGMA_Construct_Phylogeny.htm"}],"keys":[{"type":"key","name":"evolution has","topics":[{"type":"topic","name":"UPGMA (Construct Phylogeny)","url":"UPGMA_Construct_Phylogeny.htm"}]}]},{"type":"key","name":"Rate Uniformily","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Rate Uniformily Among Sites","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"}]},{"type":"key","name":"Rate Uniformity","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Rate Uniformity Among Sites","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Rate Variation","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Rate Variation Among Sites","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Rates | Estimate Gamma Parameter","topics":[{"type":"topic","name":"Estimate Rate Variation among Sites (ML)","url":"Estimate_Rate_Variation_among_Sites_ML.htm"}],"keys":[{"type":"key","name":"Site Rates","topics":[{"type":"topic","name":"Estimate Rate Variation among Sites (ML)","url":"Estimate_Rate_Variation_among_Sites_ML.htm"}]}]},{"type":"key","name":"Rates | Estimate Position","topics":[{"type":"topic","name":"Estimate Position-by-Position Rates (ML)","url":"Estimate_Position-by-Position_Rates_ML.htm"}]},{"type":"key","name":"Rcapsulatu","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"Rcapsulatu DWATVLS","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"Rcapsulatu Len","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"}]},{"type":"key","name":"RCSU","topics":[{"type":"topic","name":"Codon Usage","url":"Codon_Usage.htm"}]},{"type":"key","name":"Read Online","topics":[{"type":"topic","name":"About Muscle","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/About_Muscle.htm"}]},{"type":"key","name":"Reading, Massachusetts","topics":[{"type":"topic","name":"Sankoff and Cedergren 1983","url":"Sankoff_and_Cedergren_1983.htm"}]},{"type":"key","name":"Rearranging","topics":[{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"}],"keys":[{"type":"key","name":"Data","topics":[{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"}]}]},{"type":"key","name":"Recent Advances","topics":[{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"}]},{"type":"key","name":"Recent Trees","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]},{"type":"key","name":"Reconnection","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Tree Bisection Reconnection (TBR)","url":"Tree_Bisection_Reconnection_TBR.htm"}]},{"type":"key","name":"reconstructions","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Rectangular, Straight","topics":[{"type":"topic","name":"View Menu (in Tree Explorer)","url":"View_Menu_in_Tree_Explorer.htm"}]},{"type":"key","name":"Redhat","topics":[{"type":"topic","name":"System Requirements","url":"Part_I_Getting_Started/Installing_MEGA/System_Requirements.htm"}]},{"type":"key","name":"Redhat-like","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"},{"type":"topic","name":"Uninstalling MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Uninstalling_MEGA.htm"}]},{"type":"key","name":"refactoring","topics":[{"type":"topic","name":"Preface","url":"Preface.htm"},{"type":"topic","name":"What's New In This Version","url":"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm"}],"keys":[{"type":"key","name":"64","topics":[{"type":"topic","name":"What's New In This Version","url":"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm"}]},{"type":"key","name":"MEGA","topics":[{"type":"topic","name":"Preface","url":"Preface.htm"}]}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
