{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"HOME","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"},{"type":"topic","name":"Introduction","url":"Introduction.htm"},{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"Hominoidea","topics":[{"type":"topic","name":"Kishino and Hasegawa 1989","url":"Kishino_and_Hasegawa_1989.htm"}]},{"type":"key","name":"hotkey","topics":[{"type":"topic","name":"Hotkeys and Shortcut keys","url":"Hotkeys_and_Shortcut_keys.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}],"keys":[{"type":"key","name":"file menu item","topics":[{"type":"topic","name":"Hotkeys and Shortcut keys","url":"Hotkeys_and_Shortcut_keys.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}]}]},{"type":"key","name":"Hotkeys","topics":[{"type":"topic","name":"Hotkeys and Shortcut keys","url":"Hotkeys_and_Shortcut_keys.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}]},{"type":"key","name":"hsp20","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]},{"type":"key","name":"hsp20_Test.mas","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]},{"type":"key","name":"hsp20.fas","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"}]},{"type":"key","name":"Hughes 1991","topics":[{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"Hydrophobicity Multiplier","topics":[{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"hylogeny","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"HyPhy","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Estimate Selection for each Codon (HyPhy)","url":"Estimate_Selection_for_each_Codon_HyPhy.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"hypothesis rejection","topics":[{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"Iabel","topics":[{"type":"topic","name":"Rules for Taxa Names","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Taxa_Names.htm"}]},{"type":"key","name":"Iabels","topics":[{"type":"topic","name":"Rules for Taxa Names","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/General_Conventions/Rules_for_Taxa_Names.htm"}]},{"type":"key","name":"ID","topics":[{"type":"topic","name":"Sort Sequences by Group and Sequence Names","url":"Sort_Sequences_by_Group_and_Sequence_Names.htm"},{"type":"topic","name":"Sort Sequences by Group Name","url":"Sort_Sequences_by_Group_Name.htm"},{"type":"topic","name":"Sort Sequences By Sequence Name","url":"Sort_Sequences_By_Sequence_Name.htm"}]},{"type":"key","name":"Identical Symbol","topics":[{"type":"topic","name":"Input Data Format Dialog","url":"Input_Data_Format_Dialog.htm"}]},{"type":"key","name":"Ids","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]},{"type":"key","name":"ig","topics":[{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"IG Format","topics":[{"type":"topic","name":"Converting IG Format","url":"Converting_IG_Format.htm"}],"keys":[{"type":"key","name":"Converting","topics":[{"type":"topic","name":"Converting IG Format","url":"Converting_IG_Format.htm"}]}]},{"type":"key","name":"ignore","topics":[{"type":"topic","name":"How can I ignore the current update available message in MEGA's main window","url":"How_can_I_ignore_the_current_update_available_message_in_MEGA_s_main_window.htm"}],"keys":[{"type":"key","name":"current update available message","topics":[{"type":"topic","name":"How can I ignore the current update available message in MEGA's main window","url":"How_can_I_ignore_the_current_update_available_message_in_MEGA_s_main_window.htm"}]}]},{"type":"key","name":"ignore_updates.ini file","topics":[{"type":"topic","name":"How do I prevent the \"MEGA Update Available\" message showing up","url":"How_do_I_prevent_the_MEGA_Update_Available_message_showing_up.htm"}],"keys":[{"type":"key","name":"edit","topics":[{"type":"topic","name":"How do I prevent the \"MEGA Update Available\" message showing up","url":"How_do_I_prevent_the_MEGA_Update_Available_message_showing_up.htm"}]}]},{"type":"key","name":"Image","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}],"keys":[{"type":"key","name":"Select","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"Image Menu","topics":[{"type":"topic","name":"Image Menu (in Tree Explorer)","url":"Image_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Tree Explorer","url":"TreeExplorer_HC_files/Tree_Explorer.htm"}]},{"type":"key","name":"Image Tab","topics":[{"type":"topic","name":"Subtree Drawing Options (in Tree Explorer)","url":"Subtree_Drawing_Options_in_Tree_Explorer.htm"}]},{"type":"key","name":"ImageMagick","topics":[{"type":"topic","name":"What's New In This Version","url":"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm"}]},{"type":"key","name":"immunoglobulins","topics":[{"type":"topic","name":"Tanaka and Nei 1989","url":"Tanaka_and_Nei_1989.htm"}]},{"type":"key","name":"Import Query Data From File","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"},{"type":"topic","name":"Upload a text file with the coordinate information for all nSNVs of interest","url":"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm"}]},{"type":"key","name":"Importing","topics":[{"type":"topic","name":"Calibration Dialog","url":"Calibration_Dialog.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"},{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}],"keys":[{"type":"key","name":"Data","topics":[{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"using","topics":[{"type":"topic","name":"Calibration Dialog","url":"Calibration_Dialog.htm"}]}]},{"type":"key","name":"Importing Data","topics":[{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format Dialog Box","url":"Convert_to_MEGA_Format_Dialog_Box.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"}]},{"type":"key","name":"Improved General Amino Acid Replacement Matrix","topics":[{"type":"topic","name":"Le and Gascuel 2008","url":"Le_and_Gascuel_2008.htm"}]},{"type":"key","name":"in Sequence Data Explorer","topics":[{"type":"topic","name":"Show Group Names","url":"Show_Group_Names.htm"}]},{"type":"key","name":"Inapplicable Computation Requested","topics":[{"type":"topic","name":"Inapplicable Computation Requested","url":"Inapplicable_Computation_Requested.htm"}]},{"type":"key","name":"Include Sites","topics":[{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"}]},{"type":"key","name":"Include Sites Option","topics":[{"type":"topic","name":"Include Sites Option","url":"Include_Sites_Option.htm"}]},{"type":"key","name":"Include/Exclude","topics":[{"type":"topic","name":"Setup/Select Taxa & Groups Dialog","url":"Setup_Select_Taxa_Groups_Dialog.htm"}]},{"type":"key","name":"included in","topics":[{"type":"topic","name":"Upload a text file with the coordinate information for all nSNVs of interest","url":"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm"}]},{"type":"key","name":"Including","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"},{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"Compute Menu (in Tree Explorer)","url":"Compute_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Finding the number of sites in pairwise comparisons","url":"Finding_the_number_of_sites_in_pairwise_comparisons.htm"},{"type":"topic","name":"Setup/Select Taxa & Groups","url":"Setup_Select_Taxa_Groups.htm"},{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"}],"keys":[{"type":"key","name":"Condensed","topics":[{"type":"topic","name":"Compute Menu (in Tree Explorer)","url":"Compute_Menu_in_Tree_Explorer.htm"}]},{"type":"key","name":"Data","topics":[{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"}]},{"type":"key","name":"IUB","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"}]},{"type":"key","name":"MEGA","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"}]},{"type":"key","name":"or excluding","topics":[{"type":"topic","name":"Setup/Select Taxa & Groups","url":"Setup_Select_Taxa_Groups.htm"}]},{"type":"key","name":"Substitutions","topics":[{"type":"topic","name":"Finding the number of sites in pairwise comparisons","url":"Finding_the_number_of_sites_in_pairwise_comparisons.htm"}]}]},{"type":"key","name":"incompatibility","topics":[{"type":"topic","name":"What's New In This Version","url":"Part_I_Getting_Started/Features_and_Support/What_s_New_In_This_Version.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
