{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"File window","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}],"keys":[{"type":"key","name":"Open","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]}]},{"type":"key","name":"filename.pir","topics":[{"type":"topic","name":"Converting PIR Format","url":"Converting_PIR_Format.htm"}]},{"type":"key","name":"filename.xml","topics":[{"type":"topic","name":"Converting XML Format","url":"Converting_XML_Format.htm"}]},{"type":"key","name":"Filipski","topics":[{"type":"topic","name":"Data Coverage","url":"Data_Coverage.htm"},{"type":"topic","name":"Filipski et al. 2014","url":"Filipski_et_al_2014.htm"},{"type":"topic","name":"Tamura et al. 2012","url":"Tamura_et_al_2012.htm"}]},{"type":"key","name":"Filipski et al","topics":[{"type":"topic","name":"Filipski et al. 2014","url":"Filipski_et_al_2014.htm"}]},{"type":"key","name":"Filipski_et_al_2014_1","topics":[{"type":"topic","name":"Data Coverage","url":"Data_Coverage.htm"}]},{"type":"key","name":"fill","topics":[{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"BLAST","topics":[{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}]}]},{"type":"key","name":"Find Again","topics":[{"type":"topic","name":"Find Again (in Text Editor)","url":"Find_Again_in_Text_Editor.htm"}]},{"type":"key","name":"Find Best","topics":[{"type":"topic","name":"Find Best DNA/Protein Models (ML)","url":"Find_Best_DNA_Protein_Models_ML.htm"}]},{"type":"key","name":"Find Gene Duplications","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}]},{"type":"key","name":"Find Marked Site","topics":[{"type":"topic","name":"Search Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Search_Menu.htm"},{"type":"topic","name":"Search Menu (in Alignment Explorer)","url":"Search_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"Find menu","topics":[{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"}]},{"type":"key","name":"Find Motif","topics":[{"type":"topic","name":"Find Motif","url":"Find_Motif.htm"},{"type":"topic","name":"Search Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Search_Menu.htm"},{"type":"topic","name":"Search Menu (in Alignment Explorer)","url":"Search_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"activates","topics":[{"type":"topic","name":"Search Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Search_Menu.htm"},{"type":"topic","name":"Search Menu (in Alignment Explorer)","url":"Search_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}]}]},{"type":"key","name":"Find Next","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"},{"type":"topic","name":"Search Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Search_Menu.htm"},{"type":"topic","name":"Search Menu (in Alignment Explorer)","url":"Search_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"}]},{"type":"key","name":"Find Next Motif","topics":[{"type":"topic","name":"Find Next Motif","url":"Find_Next_Motif.htm"}]},{"type":"key","name":"Find Next Name","topics":[{"type":"topic","name":"Find Next Name","url":"Find_Next_Name.htm"}]},{"type":"key","name":"Find Prev Motif","topics":[{"type":"topic","name":"Find Prev Motif","url":"Find_Prev_Motif.htm"}]},{"type":"key","name":"Find Prev Name","topics":[{"type":"topic","name":"Find Prev Name","url":"Find_Prev_Name.htm"}]},{"type":"key","name":"Find Previous","topics":[{"type":"topic","name":"Search Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Search_Menu.htm"},{"type":"topic","name":"Search Menu (in Alignment Explorer)","url":"Search_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"}]},{"type":"key","name":"Find Sequence Name","topics":[{"type":"topic","name":"Find Sequence Name","url":"Find_Sequence_Name.htm"}]},{"type":"key","name":"Find Text","topics":[{"type":"topic","name":"Find (in Text Editor)","url":"Find_in_Text_Editor.htm"}]},{"type":"key","name":"Finder window","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"}]},{"type":"key","name":"Finding","topics":[{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Find (in Text Editor)","url":"Find_in_Text_Editor.htm"},{"type":"topic","name":"Find Again (in Text Editor)","url":"Find_Again_in_Text_Editor.htm"},{"type":"topic","name":"Finding the number of sites in pairwise comparisons","url":"Finding_the_number_of_sites_in_pairwise_comparisons.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Min-mini algorithm","url":"Min-mini_algorithm.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"}],"keys":[{"type":"key","name":"Average menu","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"ME","topics":[{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"}]},{"type":"key","name":"MP","topics":[{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Min-mini algorithm","url":"Min-mini_algorithm.htm"}]},{"type":"key","name":"number","topics":[{"type":"topic","name":"Finding the number of sites in pairwise comparisons","url":"Finding_the_number_of_sites_in_pairwise_comparisons.htm"}]}]},{"type":"key","name":"Finished button","topics":[{"type":"topic","name":"Calibration Dialog","url":"Calibration_Dialog.htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"},{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}],"keys":[{"type":"key","name":"clicking","topics":[{"type":"topic","name":"Calibration Dialog","url":"Calibration_Dialog.htm"}]},{"type":"key","name":"toolbar","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}]}]},{"type":"key","name":"FireFox","topics":[{"type":"topic","name":"Web Browser & Data Miner","url":"Web_Browser_Data_Miner.htm"},{"type":"topic","name":"Web Browser and Data Miner","url":"Part_II_Assembling_Data_For_Analysis/Web_Browser_and_Data_Miner.htm"}]},{"type":"key","name":"First Time User","topics":[{"type":"topic","name":"First Time User","url":"First_Time_User.htm"}]},{"type":"key","name":"first_exon","topics":[{"type":"topic","name":"Keywords for Command Statements (Genes/Domains)","url":"Keywords_for_Command_Statements_(Genes_Domains).htm"}]},{"type":"key","name":"FirstGene Domain","topics":[{"type":"topic","name":"Site Label","url":"Site_Label.htm"},{"type":"topic","name":"What is a Site Label?","url":"What_is_a_Site_Label_.htm"},{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}]},{"type":"key","name":"Fisher's","topics":[{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"Fisher's Exact test","topics":[{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Codon Based Fisher's Exact Test","url":"Codon_Based_Fisher_s_Exact_Test.htm"}]},{"type":"key","name":"Fisher's Exact Test Has Failed","topics":[{"type":"topic","name":"Fisher's Exact Test Has Failed","url":"Fisher_s_Exact_Test_Has_Failed.htm"}]},{"type":"key","name":"Fitch","topics":[{"type":"topic","name":"Fitch 1971","url":"Fitch_1971.htm"},{"type":"topic","name":"Fitch and Margoliash 1967","url":"Fitch_and_Margoliash_1967.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Tree length","url":"Tree_length.htm"}]},{"type":"key","name":"Fitch WM","topics":[{"type":"topic","name":"Fitch 1971","url":"Fitch_1971.htm"},{"type":"topic","name":"Fitch and Margoliash 1967","url":"Fitch_and_Margoliash_1967.htm"}]},{"type":"key","name":"Fitch_1971","topics":[{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Tree length","url":"Tree_length.htm"}]},{"type":"key","name":"Fixed Column","topics":[{"type":"topic","name":"Distance Data Explorer","url":"Distance_Data_Explorer.htm"},{"type":"topic","name":"Distance Matrix Explorer","url":"Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Fixed Row","topics":[{"type":"topic","name":"Distance Data Explorer","url":"Distance_Data_Explorer.htm"},{"type":"topic","name":"Distance Matrix Explorer","url":"Distance_Matrix_Explorer.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Flannery BP","topics":[{"type":"topic","name":"Press et al. 1993","url":"Press_et_al_1993.htm"}]},{"type":"key","name":"Flip","topics":[{"type":"topic","name":"Flip","url":"Flip.htm"},{"type":"topic","name":"Subtree Menu (in Tree Explorer)","url":"Subtree_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Swap","url":"Swap.htm"}]},{"type":"key","name":"fold Degenerate Sites","topics":[{"type":"topic","name":"Highlight 0-fold Degenerate Sites","url":"Highlight_0-fold_Degenerate_Sites.htm"},{"type":"topic","name":"Highlight 2-fold Degenerate Sites","url":"Highlight_2-fold_Degenerate_Sites.htm"},{"type":"topic","name":"Highlight 4-fold Degenerate Sites","url":"Highlight_4-fold_Degenerate_Sites.htm"},{"type":"topic","name":"Writing only 4-fold degenerate sites to an output file","url":"Writing_only_4-fold_degenerate_sites_to_an_output_file.htm"}],"keys":[{"type":"key","name":"output file","topics":[{"type":"topic","name":"Writing only 4-fold degenerate sites to an output file","url":"Writing_only_4-fold_degenerate_sites_to_an_output_file.htm"}]}]},{"type":"key","name":"Font","topics":[{"type":"topic","name":"Display Menu (in Sequence Data Explorer)","url":"Display_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Font (in Text Editor)","url":"Font_in_Text_Editor.htm"}]},{"type":"key","name":"Font dialog window","topics":[{"type":"topic","name":"Display Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm"},{"type":"topic","name":"Display Menu (in Alignment Explorer)","url":"Display_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"for details","topics":[{"type":"topic","name":"Bootstrap method to compute standard error of distance estimates","url":"Bootstrap_method_to_compute_standard_error_of_distance_estimates.htm"},{"type":"topic","name":"Gamma distance (Amino acids)","url":"Gamma_distance_Amino_acids.htm"}]},{"type":"key","name":"for the NJ","topics":[{"type":"topic","name":"Interior Branch Test of Phylogeny","url":"Interior_Branch_Test_of_Phylogeny.htm"}]},{"type":"key","name":"Forecast","topics":[{"type":"topic","name":"Diagnose Mutations","url":"Diagnose_Mutations.htm"}],"keys":[{"type":"key","name":"deleteriousness","topics":[{"type":"topic","name":"Diagnose Mutations","url":"Diagnose_Mutations.htm"}]}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
