{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Paste","topics":[{"type":"topic","name":"Copy (in Text Editor)","url":"Copy_in_Text_Editor.htm"},{"type":"topic","name":"Cut (in Text Editor)","url":"Cut_in_Text_Editor.htm"},{"type":"topic","name":"Edit Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Edit_Menu.htm"},{"type":"topic","name":"Edit Menu (in Alignment Explorer)","url":"Edit_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Image Menu (in Tree Explorer)","url":"Image_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Paste (in Text Editor)","url":"Paste_in_Text_Editor.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"},{"type":"topic","name":"Toolbar in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Toolbar_in_Alignment_Explorer.htm"},{"type":"topic","name":"Toolbars in Alignment Explorer","url":"Toolbars_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"use","topics":[{"type":"topic","name":"Copy (in Text Editor)","url":"Copy_in_Text_Editor.htm"}]}]},{"type":"key","name":"Pattern Heterogeneity","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Pattern Heterogeneity Between Lineages","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Pattern Heterogeneity Heterogeneity","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Pattern Homogeneity","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]},{"type":"key","name":"Pattern Homogeneity Analysis","topics":[{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"}]},{"type":"key","name":"Pattern tests","topics":[{"type":"topic","name":"Pattern tests","url":"Pattern_Tests.htm"}]},{"type":"key","name":"Patterns","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"}]},{"type":"key","name":"Pauline","topics":[{"type":"topic","name":"Pauline and Henikoff 2003","url":"Pauline_and_Henikoff_2003.htm"}]},{"type":"key","name":"Pauling","topics":[{"type":"topic","name":"Zuckerkandl and Pauling 1965","url":"Zuckerkandl_and_Pauling_1965.htm"}]},{"type":"key","name":"Pauling 1965","topics":[{"type":"topic","name":"Zuckerkandl and Pauling 1965","url":"Zuckerkandl_and_Pauling_1965.htm"}]},{"type":"key","name":"PAUP","topics":[{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"},{"type":"topic","name":"Export Data (Sequence Data Explorer)","url":"Export_Data_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"General Description","url":"General_Description.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Swofford 1993","url":"Swofford_1993.htm"},{"type":"topic","name":"Swofford 1998","url":"Swofford_1998.htm"}]},{"type":"key","name":"PBL","topics":[{"type":"topic","name":"Kumar Method","url":"Kumar_Method.htm"}]},{"type":"key","name":"PC","topics":[{"type":"topic","name":"Poisson Correction (PC) distance","url":"Poisson_Correction_PC_distance.htm"}]},{"type":"key","name":"PDF","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Image Menu (in Tree Explorer)","url":"Image_Menu_in_Tree_Explorer.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}],"keys":[{"type":"key","name":"save","topics":[{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]}]},{"type":"key","name":"PDF File","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"Pennsylvania State University","topics":[{"type":"topic","name":"Acknowledgements","url":"Acknowledgements.htm"},{"type":"topic","name":"Kumar et al. 1993","url":"Kumar_et_al_1993.htm"}]},{"type":"key","name":"Penny","topics":[{"type":"topic","name":"Hendy and Penny 1982","url":"Hendy_and_Penny_1982.htm"},{"type":"topic","name":"Hendy and Penny 1989","url":"Hendy_and_Penny_1989.htm"},{"type":"topic","name":"Penny and Hendy 1985","url":"Penny_and_Hendy_1985.htm"}]},{"type":"key","name":"Penny 1982","topics":[{"type":"topic","name":"Hendy and Penny 1982","url":"Hendy_and_Penny_1982.htm"}]},{"type":"key","name":"Penny 1989","topics":[{"type":"topic","name":"Hendy and Penny 1989","url":"Hendy_and_Penny_1989.htm"}]},{"type":"key","name":"per unit","topics":[{"type":"topic","name":"Branch","url":"Branch.htm"}]},{"type":"key","name":"Perform Tajima's","topics":[{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"performs","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"}],"keys":[{"type":"key","name":"Maximum Likelihood","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"}]}]},{"type":"key","name":"Peroid","topics":[{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"Peshkin","topics":[{"type":"topic","name":"Adzhubei et al. 2010","url":"Adzhubei_et_al_2010.htm"}]},{"type":"key","name":"PHB","topics":[{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"}]},{"type":"key","name":"Phe","topics":[{"type":"topic","name":"IUPAC Single Letter Codes","url":"IUPAC_Single_Letter_Codes.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"PHYLIP","topics":[{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Converting Phylip (Interleaved) Format","url":"Converting_Phylip_(Interleaved)_Format.htm"},{"type":"topic","name":"Converting Phylip (Non-interleaved) Format","url":"Converting_Phylip_(Non-interleaved)_Format.htm"},{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"},{"type":"topic","name":"Dayhoff and JTT Models","url":"Dayhoff_and_JTT_Models.htm"},{"type":"topic","name":"Felsenstein 1993","url":"Felsenstein_1993.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"},{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"}],"keys":[{"type":"key","name":"beginning","topics":[{"type":"topic","name":"Converting Phylip (Interleaved) Format","url":"Converting_Phylip_(Interleaved)_Format.htm"}]},{"type":"key","name":"Converting","topics":[{"type":"topic","name":"Converting Phylip (Interleaved) Format","url":"Converting_Phylip_(Interleaved)_Format.htm"},{"type":"topic","name":"Converting Phylip (Non-interleaved) Format","url":"Converting_Phylip_(Non-interleaved)_Format.htm"}]}]},{"type":"key","name":"PHYLIP 3.0","topics":[{"type":"topic","name":"Export Data (Sequence Data Explorer)","url":"Export_Data_(Sequence_Data_Explorer).htm"},{"type":"topic","name":"General Description","url":"General_Description.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"PHYLIP Interleaved","topics":[{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"PHYLIP Noninterleaved","topics":[{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"phylip2","topics":[{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"Phylogenetic Analysis","topics":[{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"Phylogenetic Analysis Using Parsimony","topics":[{"type":"topic","name":"Swofford 1993","url":"Swofford_1993.htm"},{"type":"topic","name":"Swofford 1998","url":"Swofford_1998.htm"}]},{"type":"key","name":"Phylogenetic Approach","topics":[{"type":"topic","name":"Page and Holmes 1998","url":"Page_and_Holmes_1998.htm"}]},{"type":"key","name":"Phylogenetic Inference","topics":[{"type":"topic","name":"Phylogenetic Inference","url":"Phylogenetic_Inference.htm"},{"type":"topic","name":"Swofford et al. 1996","url":"Swofford_et_al_1996.htm"}]},{"type":"key","name":"Phylogenetic Studies","topics":[{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"}],"keys":[{"type":"key","name":"DNA Sequences","topics":[{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"}]}]},{"type":"key","name":"Phylogenetic Tree","topics":[{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"}],"keys":[{"type":"key","name":"Making","topics":[{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"}]}]},{"type":"key","name":"Phylogenetics","topics":[{"type":"topic","name":"Nei and Kumar 2000","url":"Nei_and_Kumar_2000.htm"}]},{"type":"key","name":"Phylogeny","topics":[{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Bootstrap Test of Phylogeny","url":"Bootstrap_Test_of_Phylogeny.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Interior Branch Test of Phylogeny","url":"Interior_Branch_Test_of_Phylogeny.htm"},{"type":"topic","name":"Phylogeny Menu","url":"Phylogeny_Menu.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Translate/Untranslate (in Sequence Data Explorer)","url":"Translate_Untranslate_(in_Sequence_Data_Explorer).htm"}],"keys":[{"type":"key","name":"Bootstrap Test","topics":[{"type":"topic","name":"Bootstrap Test of Phylogeny","url":"Bootstrap_Test_of_Phylogeny.htm"}]},{"type":"key","name":"Interior Branch Test","topics":[{"type":"topic","name":"Interior Branch Test of Phylogeny","url":"Interior_Branch_Test_of_Phylogeny.htm"}]},{"type":"key","name":"select","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"Test","topics":[{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]}]},{"type":"key","name":"Phylogeny | Any","topics":[{"type":"topic","name":"Tree Explorer","url":"TreeExplorer_HC_files/Tree_Explorer.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
