{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"distances by","topics":[{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"}]},{"type":"key","name":"Distances Display Box","topics":[{"type":"topic","name":"Options dialog box","url":"Options_dialog_box.htm"}]},{"type":"key","name":"Distances Menu","topics":[{"type":"topic","name":"Distances Menu","url":"Distances_Menu.htm"}]},{"type":"key","name":"Divergence Times","topics":[{"type":"topic","name":"Divergence Times in the Outgroup Cluster are not Visible","url":"Divergence_Times_in_the_Outgroup_Cluster_are_not_Visible.htm"},{"type":"topic","name":"Tamura et al. 2012","url":"Tamura_et_al_2012.htm"}],"keys":[{"type":"key","name":"Estimating","topics":[{"type":"topic","name":"Tamura et al. 2012","url":"Tamura_et_al_2012.htm"}]}]},{"type":"key","name":"dmg","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"}]},{"type":"key","name":"dmg file","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"}]},{"type":"key","name":"dN","topics":[{"type":"topic","name":"Nei-Gojobori Method","url":"Nei-Gojobori_Method.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"}]},{"type":"key","name":"DNA","topics":[{"type":"topic","name":"About BLAST","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/BLAST/About_BLAST.htm"},{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"Alignment Gaps","url":"Alignment_Gaps.htm"},{"type":"topic","name":"Alignment Gaps and Sites with Missing Information","url":"Alignment_Gaps_and_Sites_with_Missing_Information.htm"},{"type":"topic","name":"Alignment Menu","url":"Alignment_Menu.htm"},{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"},{"type":"topic","name":"ClustalW","url":"ClustalW.htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Color Cells","url":"Color_Cells.htm"},{"type":"topic","name":"Computing the Gamma Parameter (a)","url":"Computing_the_Gamma_Parameter_a.htm"},{"type":"topic","name":"Converting IG Format","url":"Converting_IG_Format.htm"},{"type":"topic","name":"Converting MSF Format","url":"Converting_MSF_Format.htm"},{"type":"topic","name":"Converting XML Format","url":"Converting_XML_Format.htm"},{"type":"topic","name":"Data Explorer","url":"Data_Explorer.htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Display Menu (in Sequence Data Explorer)","url":"Display_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Distance Model Options","url":"Distance_Model_Options.htm"},{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"},{"type":"topic","name":"Estimate Selection for each Codon (HyPhy)","url":"Estimate_Selection_for_each_Codon_HyPhy.htm"},{"type":"topic","name":"Exon","url":"Exon.htm"},{"type":"topic","name":"Find Best DNA/Protein Models (ML)","url":"Find_Best_DNA_Protein_Models_ML.htm"},{"type":"topic","name":"Gaps In Distance Estimation","url":"Gaps_In_Distance_Estimation.htm"},{"type":"topic","name":"Goldman 1993","url":"Goldman_1993.htm"},{"type":"topic","name":"Hedges et al. 1992","url":"Hedges_et_al_1992.htm"},{"type":"topic","name":"Indels","url":"Indels.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"},{"type":"topic","name":"Intron","url":"Intron.htm"},{"type":"topic","name":"Invalid special symbol in molecular sequences","url":"Invalid_special_symbol_in_molecular_sequences.htm"},{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Kishino and Hasegawa 1989","url":"Kishino_and_Hasegawa_1989.htm"},{"type":"topic","name":"MEGA Format","url":"Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Format.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Minimum Evolution","url":"Minimum_Evolution.htm"},{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"},{"type":"topic","name":"Nei 1986","url":"Nei_1986.htm"},{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"},{"type":"topic","name":"Nucleotide and Amino Acid Compositions","url":"Nucleotide_and_Amino_Acid_Compositions.htm"},{"type":"topic","name":"Positive selection","url":"Positive_selection.htm"},{"type":"topic","name":"Preface","url":"Preface.htm"},{"type":"topic","name":"Purifying selection","url":"Purifying_selection.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"},{"type":"topic","name":"Sequencer Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Sequencer_Menu.htm"},{"type":"topic","name":"Sequencer Menu (in Alignment Explorer)","url":"Sequencer_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Sourdis and Krimbas 1987","url":"Sourdis_and_Krimbas_1987.htm"},{"type":"topic","name":"Staden","url":"Staden.htm"},{"type":"topic","name":"Tajima 1983","url":"Tajima_1983.htm"},{"type":"topic","name":"Tajima 1989","url":"Tajima_1989.htm"},{"type":"topic","name":"Tajima and Nei 1982","url":"Tajima_and_Nei_1982.htm"},{"type":"topic","name":"Tamura and Nei 1993","url":"Tamura_and_Nei_1993.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Translation","url":"Translation.htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"DNA Sequences","topics":[{"type":"topic","name":"Aligning Coding Sequences via Protein Sequences","url":"Part_II_Assembling_Data_For_Analysis/Aligning_Coding_Sequences_via_Protein_Sequences.htm"},{"type":"topic","name":"Alignment Explorer","url":"Alignment_Explorer.htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Data Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Data_Menu.htm"},{"type":"topic","name":"Data Menu (in Alignment Explorer)","url":"Data_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Introduction to Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Introduction_to_Alignment_Explorer.htm"},{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"},{"type":"topic","name":"Staden","url":"Staden.htm"}],"keys":[{"type":"key","name":"Phylogenetic Studies","topics":[{"type":"topic","name":"Nei 1991","url":"Nei_1991.htm"}]}]},{"type":"key","name":"DNA Weight Matrix","topics":[{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"DNA/RNA","topics":[{"type":"topic","name":"IUPAC Single Letter Codes","url":"IUPAC_Single_Letter_Codes.htm"}]},{"type":"key","name":"dnd","topics":[{"type":"topic","name":"Newick Format","url":"Newick_Format.htm"}]},{"type":"key","name":"DOCTYPE","topics":[{"type":"topic","name":"Converting XML Format","url":"Converting_XML_Format.htm"}]},{"type":"key","name":"Documents","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"},{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"doi","topics":[{"type":"topic","name":"Kumar et al. EvoD 2012","url":"Kumar_et_al_EvoD_2012.htm"},{"type":"topic","name":"Pauline and Henikoff 2003","url":"Pauline_and_Henikoff_2003.htm"}]},{"type":"key","name":"Doing","topics":[{"type":"topic","name":"Gene Duplication Inference","url":"Gene_Duplication_Inference.htm"}],"keys":[{"type":"key","name":"Gene Duplication Analysis","topics":[{"type":"topic","name":"Gene Duplication Inference","url":"Gene_Duplication_Inference.htm"}]}]},{"type":"key","name":"Domain Definitions","topics":[{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}],"keys":[{"type":"key","name":"Compute Pairwise","topics":[{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]}]},{"type":"key","name":"Domain Editor","topics":[{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"}]},{"type":"key","name":"Domain Organization window","topics":[{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
