{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"use Add","topics":[{"type":"topic","name":"Uninstalling MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Uninstalling_MEGA.htm"}]},{"type":"key","name":"Use All Selected Sites","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Statistics Menu (in Sequence Data Explorer)","url":"Statistics_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Use All Selected Sites","url":"Use_All_Selected_Sites.htm"}]},{"type":"key","name":"Use Colors","topics":[{"type":"topic","name":"Display Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Display_Menu.htm"},{"type":"topic","name":"Display Menu (in Alignment Explorer)","url":"Display_Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"use computing","topics":[{"type":"topic","name":"Preface","url":"Preface.htm"}]},{"type":"key","name":"Use Identical Symbol","topics":[{"type":"topic","name":"Display Menu (in Sequence Data Explorer)","url":"Display_Menu_in_Sequence_Data_Explorer.htm"},{"type":"topic","name":"Use Identical Symbol","url":"Use_Identical_Symbol.htm"}]},{"type":"key","name":"Use Negative Matrix","topics":[{"type":"topic","name":"Clustal Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/Clustal_Options_(Protein).htm"},{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"Use only Highlighted Sites","topics":[{"type":"topic","name":"Use only Highlighted Sites","url":"Use_only_Highlighted_Sites.htm"}]},{"type":"key","name":"use Partial","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"}]},{"type":"key","name":"use Partial Deletion","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"},{"type":"topic","name":"Analysis Preferences (Distance Computation)","url":"Analysis_Preferences_Distance_Computation.htm"},{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Analysis Preferences (Minimum Evolution)","url":"Analysis_Preferences_Minimum_Evolution.htm"},{"type":"topic","name":"Analysis Preferences (NJ/UPGMA)","url":"Analysis_Preferences_NJ_UPGMA.htm"},{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"},{"type":"topic","name":"Analysis Preferences (Z-test of Selection)","url":"Analysis_Preferences_Z-test_of_Selection.htm"}]},{"type":"key","name":"Use Topology Editor","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"}]},{"type":"key","name":"used","topics":[{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"},{"type":"topic","name":"Jukes-Cantor Distance Failed","url":"Jukes-Cantor_Distance_Failed.htm"},{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"},{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}],"keys":[{"type":"key","name":"Add","topics":[{"type":"topic","name":"Setup/Select Genes & Domains Dialog","url":"Setup_Select_Genes_Domains_Dialog.htm"}]},{"type":"key","name":"Align","topics":[{"type":"topic","name":"ClustalW Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/ClustalW/ClustalW_Options_(DNA).htm"}]},{"type":"key","name":"calculate nucleotide distances and","topics":[{"type":"topic","name":"Jukes-Cantor Distance Failed","url":"Jukes-Cantor_Distance_Failed.htm"}]}]},{"type":"key","name":"used by MEGA and","topics":[{"type":"topic","name":"MEGA-CC","url":"MEGA-CC.htm"}]},{"type":"key","name":"used in this tutorial","topics":[{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"}]},{"type":"key","name":"used to","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"User Account Control","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"}]},{"type":"key","name":"User Accounts","topics":[{"type":"topic","name":"Installing MEGA","url":"Part_I_Getting_Started/Installing_MEGA/Installing_MEGA.htm"}]},{"type":"key","name":"User Stopped Computation","topics":[{"type":"topic","name":"User Stopped Computation","url":"User_Stopped_Computation.htm"}]},{"type":"key","name":"User Tree","topics":[{"type":"topic","name":"Display Newick Tree From File","url":"Display_Newick_Tree_From_File.htm"},{"type":"topic","name":"Display Newick Trees from File","url":"Display_Newick_Trees_from_File.htm"},{"type":"topic","name":"Gene Duplication Inference","url":"Gene_Duplication_Inference.htm"},{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"},{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"User Tree Computation","topics":[{"type":"topic","name":"Analyze User Tree by Least Squares","url":"Analyze_User_Tree_by_Least_Squares.htm"},{"type":"topic","name":"Analyze User Tree by Maximum Likelihood","url":"Analyze_User_Tree_by_Maximum_Likelihood.htm"},{"type":"topic","name":"Analyze User Tree by Parsimony","url":"Analyze_User_Tree_by_Parsimony.htm"}]},{"type":"key","name":"User Tree File","topics":[{"type":"topic","name":"Analysis Preferences (Estimate Selection for each Codon)","url":"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm"}]},{"type":"key","name":"User Tree Name","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}],"keys":[{"type":"key","name":"Map","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]}]},{"type":"key","name":"User-Entered Text","topics":[{"type":"topic","name":"Guide to Notations Used","url":"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm"}]},{"type":"key","name":"Users/UserName","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"},{"type":"topic","name":"Computing Sequence Statistics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Editing Data Files","url":"Editing_Data_Files.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"},{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"},{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]},{"type":"key","name":"Users/UserName/Documents","topics":[{"type":"topic","name":"Introduction","url":"Introduction.htm"}]},{"type":"key","name":"uses estimates","topics":[{"type":"topic","name":"Fisher's Exact Test Has Failed","url":"Fisher_s_Exact_Test_Has_Failed.htm"}],"keys":[{"type":"key","name":"number of","topics":[{"type":"topic","name":"Fisher's Exact Test Has Failed","url":"Fisher_s_Exact_Test_Has_Failed.htm"}]}]},{"type":"key","name":"uses JTT","topics":[{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"}]},{"type":"key","name":"uses Maximum Likelihood","topics":[{"type":"topic","name":"Estimate Rate Variation among Sites (ML)","url":"Estimate_Rate_Variation_among_Sites_ML.htm"}]},{"type":"key","name":"uses PAM 001","topics":[{"type":"topic","name":"Dayhoff and JTT distances (Gamma rates)","url":"Dayhoff_and_JTT_distances_Gamma_rates.htm"}]},{"type":"key","name":"Usin","topics":[{"type":"topic","name":"Aligning Sequences","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm"},{"type":"topic","name":"Working With Genes and Domains","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Working_With_Genes_and_Domains.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
