{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Macromolecules","topics":[{"type":"topic","name":"Sankoff and Cedergren 1983","url":"Sankoff_and_Cedergren_1983.htm"}]},{"type":"key","name":"Maddison","topics":[{"type":"topic","name":"Maddison and Maddison 1992","url":"Maddison_and_Maddison_1992.htm"}]},{"type":"key","name":"Maddison 1992","topics":[{"type":"topic","name":"Maddison and Maddison 1992","url":"Maddison_and_Maddison_1992.htm"}]},{"type":"key","name":"Maddison DR","topics":[{"type":"topic","name":"Maddison and Maddison 1992","url":"Maddison_and_Maddison_1992.htm"}]},{"type":"key","name":"Maddison WP","topics":[{"type":"topic","name":"Maddison and Maddison 1992","url":"Maddison_and_Maddison_1992.htm"}]},{"type":"key","name":"MAFFT","topics":[{"type":"topic","name":"About Muscle","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/About_Muscle.htm"}]},{"type":"key","name":"Main","topics":[{"type":"topic","name":"Amino Acid","url":"Amino_Acid.htm"},{"type":"topic","name":"Compute Between Groups Means","url":"Compute_Between_Groups_Means.htm"},{"type":"topic","name":"Compute Net Between Groups Means","url":"Compute_Net_Between_Groups_Means.htm"},{"type":"topic","name":"Compute Pairwise","url":"Compute_Pairwise.htm"},{"type":"topic","name":"Compute Within Groups Mean","url":"Compute_Within_Groups_Mean.htm"},{"type":"topic","name":"Data Explorer","url":"Data_Explorer.htm"},{"type":"topic","name":"Exit","url":"Exit.htm"},{"type":"topic","name":"Menus in MEGA are so short; where are all the options?","url":"Menus_in_MEGA_are_so_short_where_are_all_the_options.htm"},{"type":"topic","name":"Select Genetic Code (in Sequence Data Explorer)","url":"Select_Genetic_Code_(in_Sequence_Data_Explorer).htm"},{"type":"topic","name":"Setup/Select Taxa & Groups (in Sequence Data Explorer)","url":"Setup_Select_Taxa_Groups_in_Sequence_Data_Explorer.htm"}]},{"type":"key","name":"Main dialog","topics":[{"type":"topic","name":"Compute Overall Mean","url":"Compute_Overall_Mean.htm"},{"type":"topic","name":"Select Genetic Code Table","url":"Select_Genetic_Code_Table.htm"},{"type":"topic","name":"Setup/Select Taxa & Groups","url":"Setup_Select_Taxa_Groups.htm"}]},{"type":"key","name":"Main File Menu","topics":[{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"}]},{"type":"key","name":"Main icon","topics":[{"type":"topic","name":"Open Data","url":"Open_Data.htm"}]},{"type":"key","name":"Main it in MEGA","topics":[{"type":"topic","name":"Input Data Format Dialog","url":"Input_Data_Format_Dialog.htm"}]},{"type":"key","name":"Main MEGA menu","topics":[{"type":"topic","name":"Introduction","url":"Introduction.htm"}]},{"type":"key","name":"Main MEGA Window","topics":[{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"Main Toolbar","topics":[{"type":"topic","name":"Main MEGA Window","url":"Main_MEGA_Window.htm"}]},{"type":"key","name":"Major Histocompatibility Complex","topics":[{"type":"topic","name":"Site Label","url":"Site_Label.htm"},{"type":"topic","name":"What is a Site Label?","url":"What_is_a_Site_Label_.htm"}]},{"type":"key","name":"Majority","topics":[{"type":"topic","name":"Consensus Tree","url":"Consensus_Tree.htm"}]},{"type":"key","name":"Making","topics":[{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"}],"keys":[{"type":"key","name":"Phylogenetic Tree","topics":[{"type":"topic","name":"Building Trees With Distance Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm"}]}]},{"type":"key","name":"Mammal","topics":[{"type":"topic","name":"Writing Command Statements for Defining Groups of Taxa","url":"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm"}]},{"type":"key","name":"Mammalian Protein Metabolism","topics":[{"type":"topic","name":"Jukes and Cantor 1969","url":"Jukes_and_Cantor_1969.htm"}]},{"type":"key","name":"Managing","topics":[{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"}],"keys":[{"type":"key","name":"Taxa With Groups","topics":[{"type":"topic","name":"Managing Taxa With Groups","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm"}]}]},{"type":"key","name":"Managing Taxa","topics":[{"type":"topic","name":"MEGA Basics","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm"}]},{"type":"key","name":"Many Clocks","topics":[{"type":"topic","name":"Analysis Options","url":"Analysis_Options.htm"}]},{"type":"key","name":"Map","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}],"keys":[{"type":"key","name":"User Tree Name","topics":[{"type":"topic","name":"Tree Topology Editor","url":"Tree_Topology_Editor.htm"}]}]},{"type":"key","name":"Map Species Names","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}]},{"type":"key","name":"Map Species To Taxa","topics":[{"type":"topic","name":"Inferring Gene Duplications","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm"}]},{"type":"key","name":"Margoliash","topics":[{"type":"topic","name":"Fitch and Margoliash 1967","url":"Fitch_and_Margoliash_1967.htm"}]},{"type":"key","name":"Margoliash 1967","topics":[{"type":"topic","name":"Fitch and Margoliash 1967","url":"Fitch_and_Margoliash_1967.htm"}]},{"type":"key","name":"Mark/Unmark Site","topics":[{"type":"topic","name":"Alignment Menu (in Alignment Explorer)","url":"Alignment_Menu_in_Alignment_Explorer.htm"},{"type":"topic","name":"Menu in Alignment Explorer","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Menu_in_Alignment_Explorer.htm"}]},{"type":"key","name":"Maryland","topics":[{"type":"topic","name":"Dayhoff 1978","url":"Dayhoff_1978.htm"},{"type":"topic","name":"Eck and Dayhoff 1966","url":"Eck_and_Dayhoff_1966.htm"}]},{"type":"key","name":"MAS file","topics":[{"type":"topic","name":"Alignment session","url":"Alignment_session.htm"}]},{"type":"key","name":"Masatoshi Nei","topics":[{"type":"topic","name":"Acknowledgements","url":"Acknowledgements.htm"}]},{"type":"key","name":"Mask Downstream","topics":[{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"}]},{"type":"key","name":"Mask Upstream","topics":[{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Part_II_Assembling_Data_For_Analysis/Trace_Data_File_Viewer_Editor.htm"},{"type":"topic","name":"Trace Data File Viewer/Editor","url":"Trace_Data_File_Viewer_Editor.htm"}]},{"type":"key","name":"Massachusetts","topics":[{"type":"topic","name":"Maddison and Maddison 1992","url":"Maddison_and_Maddison_1992.htm"},{"type":"topic","name":"Swofford et al. 1996","url":"Swofford_et_al_1996.htm"}]},{"type":"key","name":"MatchChar","topics":[{"type":"topic","name":"Keywords for Format Statement (Sequence Data)","url":"Keywords_for_Format_Statement_(Sequence_Data).htm"},{"type":"topic","name":"Writing Command Statement for Defining Genes and Domains","url":"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm"}]},{"type":"key","name":"Mathematical Biosciences 23","topics":[{"type":"topic","name":"Estabrook et al. 1975","url":"Estabrook_et_al_1975.htm"}]},{"type":"key","name":"Mathematical Biosciences 59","topics":[{"type":"topic","name":"Hendy and Penny 1982","url":"Hendy_and_Penny_1982.htm"}]},{"type":"key","name":"Matrix","topics":[{"type":"topic","name":"Converting NEXUS Format","url":"Converting_NEXUS_Format.htm"},{"type":"topic","name":"First Time User","url":"First_Time_User.htm"},{"type":"topic","name":"Missing data or invalid distances in the matrix","url":"Missing_data_or_invalid_distances_in_the_matrix.htm"},{"type":"topic","name":"Options dialog box","url":"Options_dialog_box.htm"}]},{"type":"key","name":"Matrix Explorer","topics":[{"type":"topic","name":"File Menu (in Distance Matrix Explorer)","url":"File_Menu_in_Distance_Matrix_Explorer.htm"}]},{"type":"key","name":"Matrix Format","topics":[{"type":"topic","name":"Input Data Format Dialog","url":"Input_Data_Format_Dialog.htm"}]},{"type":"key","name":"Max","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Branch-and-Bound algorithm","url":"Branch-and-Bound_algorithm.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"}],"keys":[{"type":"key","name":"choosing","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]},{"type":"key","name":"employs","topics":[{"type":"topic","name":"Branch-and-Bound algorithm","url":"Branch-and-Bound_algorithm.htm"}]}]},{"type":"key","name":"Max Diagonal Length","topics":[{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Max Divergence Time","topics":[{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]},{"type":"key","name":"Max Iterations","topics":[{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Max Memory","topics":[{"type":"topic","name":"Muscle Options (DNA)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/Muscle_Options_(DNA).htm"},{"type":"topic","name":"MUSCLE Options (Protein)","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/MUSCLE/MUSCLE_Options_(Protein).htm"}]},{"type":"key","name":"Max-mini branch","topics":[{"type":"topic","name":"Max-mini branch-and-bound search","url":"Max-mini_branch-and-bound_search.htm"}]},{"type":"key","name":"maximizez","topics":[{"type":"topic","name":"Estimate Rate Variation among Sites (ML)","url":"Estimate_Rate_Variation_among_Sites_ML.htm"}]},{"type":"key","name":"Maximum","topics":[{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"}],"keys":[{"type":"key","name":"constructing","topics":[{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"}]}]},{"type":"key","name":"Maximum Composite Likelihood","topics":[{"type":"topic","name":"Compute MCL Substitution Matrix","url":"Compute_MCL_Substitution_Matrix.htm"},{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"},{"type":"topic","name":"Maximum Composite Likelihood","url":"Maximum_Composite_Likelihood.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Gamma Rates and Heterogeneous Patterns)","url":"Maximum_Composite_Likelihood_Gamma_Rates_and_Heterogeneous_Patterns.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Gamma Rates)","url":"Maximum_Composite_Likelihood_Gamma_Rates.htm"},{"type":"topic","name":"Maximum Composite Likelihood (Heterogeneous Patterns)","url":"Maximum_Composite_Likelihood_Heterogeneous_Patterns.htm"}]},{"type":"key","name":"Maximum Composite Likelihood Method","topics":[{"type":"topic","name":"Maximum Composite Likelihood","url":"Maximum_Composite_Likelihood.htm"},{"type":"topic","name":"Maximum Composite Likelihood Method","url":"Maximum_Composite_Likelihood_Method.htm"}]},{"type":"key","name":"Maximum Composite Likelihood Model","topics":[{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
