{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

(function() {
var index =  {"type":"data","keys":[{"type":"key","name":"Models","topics":[{"type":"topic","name":"Analysis Preferences (Maximum Likelihood)","url":"Analysis_Preferences_Maximum_Likelihood.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Constructing Likelihood Trees","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm"},{"type":"topic","name":"Estimating Evolutionary Distances","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm"},{"type":"topic","name":"Introduction","url":"Introduction.htm"},{"type":"topic","name":"Models for estimating distances","url":"Models_for_estimating_distances.htm"},{"type":"topic","name":"Testing for Selection","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm"},{"type":"topic","name":"Testing Tree Reliability","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm"}]},{"type":"key","name":"Models | Compute MCL Substitution Matrix","topics":[{"type":"topic","name":"Compute MCL Substitution Matrix","url":"Compute_MCL_Substitution_Matrix.htm"}]},{"type":"key","name":"Models | Compute MCL Transversion","topics":[{"type":"topic","name":"Compute MCL Transversion/Transition bias","url":"Compute_MCL_Transversion_Transition_bias.htm"}]},{"type":"key","name":"Models | Estimate Substitution Matrix","topics":[{"type":"topic","name":"Estimate Substitution Matrix (ML)","url":"Estimate_Substitution_Matrix_ML.htm"}]},{"type":"key","name":"Models | Estimate Transition","topics":[{"type":"topic","name":"Estimate Transition/Transversion Bias (ML)","url":"Estimate_Transition_Transversion_Bias_ML.htm"}]},{"type":"key","name":"Models | Find Best","topics":[{"type":"topic","name":"Find Best DNA/Protein Models (ML)","url":"Find_Best_DNA_Protein_Models_ML.htm"}]},{"type":"key","name":"Modified","topics":[{"type":"topic","name":"Analysis Preferences (Fisher's Exact Test)","url":"Analysis_Preferences_Fisher_s_Exact_Test.htm"},{"type":"topic","name":"Modified Nei-Gojobori Method","url":"Modified_Nei-Gojobori_Method.htm"}]},{"type":"key","name":"Modify All Bases","topics":[{"type":"topic","name":"Edit Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Edit_Menu.htm"},{"type":"topic","name":"Edit Menu (in Alignment Explorer)","url":"Edit_Menu_in_Alignment_Explorer.htm"}],"keys":[{"type":"key","name":"Uppercase","topics":[{"type":"topic","name":"Edit Menu","url":"Part_II_Assembling_Data_For_Analysis/Building_Sequence_Alignments/Menu_Items_In_Alignment_Explorer/Edit_Menu.htm"},{"type":"topic","name":"Edit Menu (in Alignment Explorer)","url":"Edit_Menu_in_Alignment_Explorer.htm"}]}]},{"type":"key","name":"Mol Biol Evol","topics":[{"type":"topic","name":"Le and Gascuel 2008","url":"Le_and_Gascuel_2008.htm"}]},{"type":"key","name":"Molecular Biology","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"},{"type":"topic","name":"DeBry 1992","url":"DeBry_1992.htm"},{"type":"topic","name":"Filipski et al. 2014","url":"Filipski_et_al_2014.htm"},{"type":"topic","name":"Gu and Zhang 1997","url":"Gu_and_Zhang_1997.htm"},{"type":"topic","name":"Lake 1987","url":"Lake_1987.htm"},{"type":"topic","name":"Li et al. 1985","url":"Li_et_al_1985.htm"},{"type":"topic","name":"Nei and Gojobori 1986","url":"Nei_and_Gojobori_1986.htm"},{"type":"topic","name":"Nei and Jin 1989","url":"Nei_and_Jin_1989.htm"},{"type":"topic","name":"Nei et al. 1985","url":"Nei_et_al_1985.htm"},{"type":"topic","name":"Pamilo and Bianchi 1993","url":"Pamilo_and_Bianchi_1993.htm"},{"type":"topic","name":"Pamilo and Nei 1988","url":"Pamilo_and_Nei_1988.htm"},{"type":"topic","name":"Rzhetsky and Nei 1992","url":"Rzhetsky_and_Nei_1992.htm"},{"type":"topic","name":"Rzhetsky and Nei 1993","url":"Rzhetsky_and_Nei_1993.htm"},{"type":"topic","name":"Saitou and Nei 1987","url":"Saitou_and_Nei_1987.htm"},{"type":"topic","name":"Sourdis and Krimbas 1987","url":"Sourdis_and_Krimbas_1987.htm"},{"type":"topic","name":"Sourdis and Nei 1988","url":"Sourdis_and_Nei_1988.htm"},{"type":"topic","name":"Studier and Keppler 1988","url":"Studier_and_Keppler_1988.htm"},{"type":"topic","name":"Tajima and Nei 1984","url":"Tajima_and_Nei_1984.htm"},{"type":"topic","name":"Takahashi and Nei 2000","url":"Takahashi_and_Nei_2000.htm"},{"type":"topic","name":"Takezaki et al. 1995","url":"Takezaki_et_al_1995.htm"},{"type":"topic","name":"Tamura 1992","url":"Tamura_1992.htm"},{"type":"topic","name":"Tamura 1994","url":"Tamura_1994.htm"},{"type":"topic","name":"Tamura and Kumar 2002","url":"Tamura_and_Kumar_2002.htm"},{"type":"topic","name":"Tamura and Nei 1993","url":"Tamura_and_Nei_1993.htm"},{"type":"topic","name":"Tamura et al. 2007","url":"Tamura_et_al_2007.htm"},{"type":"topic","name":"Tanaka and Nei 1989","url":"Tanaka_and_Nei_1989.htm"},{"type":"topic","name":"Tateno et al. 1994","url":"Tateno_et_al_1994.htm"},{"type":"topic","name":"Zhang et al. 1997","url":"Zhang_et_al_1997.htm"}]},{"type":"key","name":"Molecular Clock Hypothesis","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"}]},{"type":"key","name":"Molecular Clock Test","topics":[{"type":"topic","name":"Molecular Clock Test (ML)","url":"Molecular_Clock_Test_ML.htm"},{"type":"topic","name":"Time Tree Tool","url":"Time_Tree_Tool.htm"},{"type":"topic","name":"Time Trees","url":"Time_Trees.htm"}]},{"type":"key","name":"Molecular Clocks","topics":[{"type":"topic","name":"Tajima's Test (Relative Rate)","url":"Tajima_s_Test_Relative_Rate.htm"}]},{"type":"key","name":"Molecular Evolution","topics":[{"type":"topic","name":"Li 1997","url":"Li_1997.htm"},{"type":"topic","name":"Nei and Kumar 2000","url":"Nei_and_Kumar_2000.htm"},{"type":"topic","name":"Page and Holmes 1998","url":"Page_and_Holmes_1998.htm"}]},{"type":"key","name":"Molecular Evolutionary Genetics Analysis","topics":[{"type":"topic","name":"Citing MEGA in Publications","url":"Citing_MEGA_In_Publications.htm"},{"type":"topic","name":"Introduction to myPEG","url":"Introduction_to_myPEG.htm"},{"type":"topic","name":"Kumar et al. 1993","url":"Kumar_et_al_1993.htm"},{"type":"topic","name":"Tamura et al. 2007","url":"Tamura_et_al_2007.htm"}]},{"type":"key","name":"molecular sequences","topics":[{"type":"topic","name":"Invalid special symbol in molecular sequences","url":"Invalid_special_symbol_in_molecular_sequences.htm"}]},{"type":"key","name":"Molecular Systematics","topics":[{"type":"topic","name":"Hillis et al. 1996","url":"Hillis_et_al_1996.htm"},{"type":"topic","name":"Swofford et al. 1996","url":"Swofford_et_al_1996.htm"}]},{"type":"key","name":"Monophyletic","topics":[{"type":"topic","name":"Monophyletic","url":"Monophyletic.htm"}]},{"type":"key","name":"Monospace","topics":[{"type":"topic","name":"Guide to Notations Used","url":"Part_I_Getting_Started/Features_and_Support/Guide_to_Notations_Used.htm"}]},{"type":"key","name":"Monte","topics":[{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"}],"keys":[{"type":"key","name":"conduct","topics":[{"type":"topic","name":"Analysis Preferences (Pattern Homogeneity Analysis)","url":"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm"}]}]},{"type":"key","name":"more taxa","topics":[{"type":"topic","name":"Setup/Select Taxa & Groups Dialog","url":"Setup_Select_Taxa_Groups_Dialog.htm"}]},{"type":"key","name":"Moritz","topics":[{"type":"topic","name":"Hillis et al. 1996","url":"Hillis_et_al_1996.htm"},{"type":"topic","name":"Swofford et al. 1996","url":"Swofford_et_al_1996.htm"}]},{"type":"key","name":"Mosurski KR","topics":[{"type":"topic","name":"Sharp et al. 1986","url":"Sharp_et_al_1986.htm"}]},{"type":"key","name":"Motif","topics":[{"type":"topic","name":"Find Motif","url":"Find_Motif.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"}],"keys":[{"type":"key","name":"specify","topics":[{"type":"topic","name":"Find Motif","url":"Find_Motif.htm"},{"type":"topic","name":"Sequence Data Explorer","url":"Sequence_Data_Explorer.htm"}]}]},{"type":"key","name":"Moves text with","topics":[{"type":"topic","name":"Text Editor","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Text_Editor.htm"}]},{"type":"key","name":"MP","topics":[{"type":"topic","name":"Alignment Gaps and Sites with Missing Information","url":"Alignment_Gaps_and_Sites_with_Missing_Information.htm"},{"type":"topic","name":"Analysis Preferences (Maximum Parsimony)","url":"Analysis_Preferences_Maximum_Parsimony.htm"},{"type":"topic","name":"Branch-and-Bound algorithm","url":"Branch-and-Bound_algorithm.htm"},{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Condensed Trees","url":"Condensed_Trees.htm"},{"type":"topic","name":"Consensus Tree","url":"Consensus_Tree.htm"},{"type":"topic","name":"Inferring Ancestral Sequences (MP)","url":"Inferring_Ancestral_Sequences_MP.htm"},{"type":"topic","name":"Max-mini branch-and-bound search","url":"Max-mini_branch-and-bound_search.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"},{"type":"topic","name":"Min-mini algorithm","url":"Min-mini_algorithm.htm"},{"type":"topic","name":"Mutation Detail View","url":"Mutation_Detail_View.htm"},{"type":"topic","name":"Random addition trees","url":"Random_addition_trees.htm"}],"keys":[{"type":"key","name":"constructing","topics":[{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"}]},{"type":"key","name":"find","topics":[{"type":"topic","name":"Close-Neighbor-Interchange (CNI)","url":"Close-Neighbor-Interchange_CNI.htm"},{"type":"topic","name":"Min-mini algorithm","url":"Min-mini_algorithm.htm"}]},{"type":"key","name":"searching","topics":[{"type":"topic","name":"Max-mini branch-and-bound search","url":"Max-mini_branch-and-bound_search.htm"}]}]},{"type":"key","name":"MP Search Level","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}],"keys":[{"type":"key","name":"Change","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"MP Search Method","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}],"keys":[{"type":"key","name":"Mini Heuristic","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"MP trees","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"},{"type":"topic","name":"Maximum Parsimony (Construct Phylogeny)","url":"Maximum_Parsimony_Construct_Phylogeny.htm"},{"type":"topic","name":"Maximum Parsimony (MP) Method","url":"Maximum_Parsimony_MP_Method.htm"}],"keys":[{"type":"key","name":"Constructing","topics":[{"type":"topic","name":"Building Trees From Sequence Data","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm"}]}]},{"type":"key","name":"MRCA","topics":[{"type":"topic","name":"Calibration Dialog","url":"Calibration_Dialog.htm"},{"type":"topic","name":"Calibration File Format","url":"Calibration_File_Format.htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}],"keys":[{"type":"key","name":"value","topics":[{"type":"topic","name":"Calibration File Format","url":"Calibration_File_Format.htm"}]}]},{"type":"key","name":"MRCA Node Label","topics":[{"type":"topic","name":"Calibration Dialog","url":"Calibration_Dialog.htm"},{"type":"topic","name":"Constructing a Timetree (ML)","url":"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm"}]},{"type":"key","name":"mRNA","topics":[{"type":"topic","name":"mRNA","url":"mRNA.htm"}]},{"type":"key","name":"mRNA ID","topics":[{"type":"topic","name":"Gene Search Tab","url":"Gene_Search_Tab.htm"}]},{"type":"key","name":"MS Excel file","topics":[{"type":"topic","name":"Mutation Explorer","url":"Mutation_Explorer.htm"}]},{"type":"key","name":"msdx","topics":[{"type":"topic","name":"Open A File","url":"Open_A_File.htm"}]},{"type":"key","name":"msf","topics":[{"type":"topic","name":"Convert to MEGA Format","url":"Part_II_Assembling_Data_For_Analysis/Text_Editor_And_Utilities/Convert_to_MEGA_Format.htm"},{"type":"topic","name":"Convert to Mega Format (in Text Editor)","url":"Convert_to_Mega_Format_in_Text_Editor.htm"},{"type":"topic","name":"Convert To MEGA Format (Main File Menu)","url":"Convert_To_MEGA_Format_Main_File_Menu.htm"},{"type":"topic","name":"Convert to MEGA Format1","url":"Convert_to_MEGA_Format1.htm"},{"type":"topic","name":"Converting Phylip (Interleaved) Format","url":"Converting_Phylip_(Interleaved)_Format.htm"},{"type":"topic","name":"Importing Data","url":"Importing_Data.htm"}]},{"type":"key","name":"MSF Format","topics":[{"type":"topic","name":"Converting MSF Format","url":"Converting_MSF_Format.htm"}],"keys":[{"type":"key","name":"Converting","topics":[{"type":"topic","name":"Converting MSF Format","url":"Converting_MSF_Format.htm"}]}]}]};
window.rh.model.publish(rh.consts('KEY_TEMP_DATA'), index, { sync:true });
})();
