{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"507\" rd=\"Takahashi_and_Nei_2000.htm|Takahashi and Nei 2000\" ct=\"Takahashi K &amp; Nei M ( 2000 ) Efficiencies of fast algorithms of phylogenetic inference under the cri ...\" bc=\"References\"/><di ky=\"508\" rd=\"Takezaki_et_al_1995.htm|Takezaki et al. 1995\" ct=\"Takezaki N, Rzhetsky A &amp; Nei M ( 2004 ) Phylogenetic test of the molecular clock and linearized tree ...\" bc=\"References\"/><di ky=\"509\" rd=\"Tamura-Nei_distance.htm|Tamura-Nei distance\" ct=\"Tamura-Nei  distance The Tamura-Nei model ( 1993 )  corrects for multiple hits, taking into account  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"510\" rd=\"Tamura-Nei_Distance_Could_Not_Be_Computed.htm|Tamura-Nei Distance Could Not Be Computed\" ct=\"The Tamura-Nei distance formula contains  many log terms. If some of their arguments approach zero t ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"511\" rd=\"Tamura-Nei_distance_Gamma_rates_and_Heterogeneous_patterns.htm|Tamura-Nei distance (Gamma rates and Heterogeneous patterns)\" ct=\"Tamura-Nei  distance (Gamma rates and Heterogeneous patterns) The  Tamura-Nei ( 1993 )  distance wit ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns &gt; Gamma Rates\"/><di ky=\"512\" rd=\"Tamura-Nei_distance_Heterogeneous_Patterns.htm|Tamura-Nei distance (Heterogeneous Patterns)\" ct=\"Tamura-Nei  distance (Heterogeneous Patterns) The Tamura-Nei model ( 1993 ) corrects for multiple  h ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns\"/><di ky=\"513\" rd=\"Tamura-Nei_gamma_distance.htm|Tamura-Nei gamma distance\" ct=\"Tamura-Nei  gamma distance  The  Tamura-Nei ( 1993 ) distance with the gamma model corrects  for mul ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"514\" rd=\"Tamura_1992.htm|Tamura 1992\" ct=\"Tamura K ( 1992 ) Estimation of the number of nucleotide substitutions when there are strong  transi ...\" bc=\"References\"/><di ky=\"515\" rd=\"Tamura_1992_Distance_Could_Not_Be_Computed.htm|Tamura (1992) Distance Could Not Be Computed\" ct=\"Tamura (1992) Distance Could Not Be  Computed For one or more pairs of sequences, the Tajima-Nei  co ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"516\" rd=\"Tamura_1994.htm|Tamura 1994\" ct=\"Tamura K ( 1994 ) Model selection in the estimation of the number of nucleotide substitutions.  Mole ...\" bc=\"References\"/><di ky=\"517\" rd=\"Tamura_3-parameter_distance.htm|Tamura 3-parameter distance\" ct=\"Tamura 3-parameter  distance Tamura’s 3-parameter model corrects for multiple  hits, taking into acc ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"518\" rd=\"Tamura_3-parameter_Gamma.htm|Tamura 3-parameter (Gamma)\" ct=\"Tamura 3-parameter  (Gamma) Tamura’s 3-parameter model corrects for multiple  hits, taking into acco ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"519\" rd=\"Tamura_3_parameter_Gamma_rates_and_Heterogeneous_patterns.htm|Tamura 3 parameter (Gamma rates and Heterogeneous patterns)\" ct=\"Tamura 3  parameter (Gamma rates and Heterogeneous patterns) Tamura’s 3-parameter model corrects  fo ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns &gt; Gamma Rates\"/><di ky=\"520\" rd=\"Tamura_3_parameter_Heterogeneous_patterns.htm|Tamura 3 parameter (Heterogeneous patterns)\" ct=\"Tamura 3  parameter (Heterogeneous patterns) Tamura’s 3-parameter model corrects  for multiple hits, ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns\"/><di ky=\"521\" rd=\"Tamura_and_Kumar_2002.htm|Tamura and Kumar 2002\" ct=\"Tamura K &amp; S Kumar ( 2002 ) Evolutionary distance estimation under heterogeneous substitution patter ...\" bc=\"References\"/><di ky=\"522\" rd=\"Tamura_and_Nei_1993.htm|Tamura and Nei 1993\" ct=\"Tamura K &amp; Nei M ( 1993 ) Estimation of the number of nucleotide substitutions in the control region ...\" bc=\"References\"/><di ky=\"523\" rd=\"Tamura_et_al_2004.htm|Tamura et al. 2004\" ct=\"Tamura K, Nei M &amp; Kumar S ( 2004 ) Prospects for inferring very large phylogenies by using the neigh ...\" bc=\"References\"/><di ky=\"524\" rd=\"Tamura_et_al_2007.htm|Tamura et al. 2007\" ct=\"Tamura K, Dudley J, Nei M &amp; Kumar S ( 2007 ) MEGA4: Molecular Evolutionary Genetics Analysis (MEGA)  ...\" bc=\"References\"/><di ky=\"525\" rd=\"Tamura_et_al_2012.htm|Tamura et al. 2012\" ct=\"        Tamura K, Battistuzzi FU, Billing-Ross P, Murillo O, Filipski A, Kumar S. (2012) Estimating  ...\" bc=\"References\"/><di ky=\"526\" rd=\"Tanaka_and_Nei_1989.htm|Tanaka and Nei 1989\" ct=\"Tanaka T &amp; Nei M ( 1989 ) Positive Darwinian selection observed at the variable-region genes of immu ...\" bc=\"References\"/><di ky=\"527\" rd=\"Tao_et_al._2019.htm|Tao et al. 2019\" ct=\"Tao Q, Tamura K, Battistuzzi F, Kumar S. 2019. A machine learning method  for detecting autocorrelat ...\" bc=\"References\"/><di ky=\"528\" rd=\"Tateno_et_al_1982.htm|Tateno et al. 1982\" ct=\"Tateno Y, Nei M &amp; Tajima F ( 1982 ) Accuracy of estimated phylogenetic trees from molecular data. I. ...\" bc=\"References\"/><di ky=\"529\" rd=\"Tateno_et_al_1994.htm|Tateno et al. 1994\" ct=\"Tateno Y, Takezaki N &amp; Nei M ( 1994 ) Relative efficiencies of the maximum likelihood, neighbor-join ...\" bc=\"References\"/><di ky=\"530\" rd=\"Taxa.htm|Taxa\" ct=\"A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on  ...\" bc=\"Glossary\"/><di ky=\"531\" rd=\"Time_Tree_Tool.htm|Time Tree Tool\" ct=\"Time Tree  Tool The Timetree  tool in the  Tree Explorer  is  used for calculating relative and abso ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Time Trees\"/><di ky=\"532\" rd=\"Time_Trees.htm|Time Trees\" ct=\"Time Trees can be computed in  MEGA where divergence times are estimated for all branching points in ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Time Trees\"/><di ky=\"533\" rd=\"Toolbars_in_Alignment_Explorer.htm|Toolbars in Alignment Explorer\" ct=\"Toolbars  in Alignment Explorer   	 	 	 		 Basic 		 Functions 			 	 		 		 This 		 prepares  Alignmen ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Alignment Explorer\"/><di ky=\"534\" rd=\"Topological_distance.htm|Topological distance\" ct=\"The topological distance quantifies the extent of  topological  differences between two  given trees ...\" bc=\"Glossary\"/><di ky=\"535\" rd=\"Topology.htm|Topology\" ct=\"The branching pattern of a tree is its topology.   ...\" bc=\"Glossary\"/><di ky=\"536\" rd=\"Trace_Data_File_Viewer_Editor.htm|Trace Data File Viewer/Editor\" ct=\"Using this function, you can view and edit trace data produced by an automated DNA sequencer in  ABI ...\" bc=\"\"/><di ky=\"537\" rd=\"Transition.htm|Transition\" ct=\"A transition occurs when a purine is substituted by a  purine , or a  pyrimidine   by a pyrimidine. ...\" bc=\"Glossary\"/><di ky=\"538\" rd=\"Transition_Matrix.htm|Transition Matrix\" ct=\"A  transition  matrix specifies the probability of every possible substitution among the nucleotides ...\" bc=\"Glossary\"/><di ky=\"539\" rd=\"Transition_Transversion_Ratio_R.htm|Transition/Transversion Ratio (R)\" ct=\"This is the ratio of the number of  transitions  to the number of  transversions  for a pair of sequ ...\" bc=\"Glossary\"/><di ky=\"540\" rd=\"Translate_Untranslate_(in_Sequence_Data_Explorer).htm|Translate/Untranslate (in Sequence Data Explorer)\" ct=\"Data | Translate/Untranslate This command is available only if the data contain  protein-coding nucl ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Data Menu\"/><di ky=\"541\" rd=\"Translation.htm|Translation\" ct=\"Translation is the process whereby each  codon   in the  mRNA  is translated into a particular  amin ...\" bc=\"Glossary\"/><di ky=\"542\" rd=\"Transversion.htm|Transversion\" ct=\"A change from a  purine  to a  pyrimidine , or vice versa, is a  transversion.     ...\" bc=\"Glossary\"/><di ky=\"543\" rd=\"Tree_Bisection_Reconnection_TBR.htm|Tree Bisection Reconnection (TBR)\" ct=\"Tree Bisection and Reconnection is a search heuristic which reduces the number of topologies searche ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Parsimony (MP) Method &gt; Heuristic Search\"/><di ky=\"544\" rd=\"Tree_length.htm|Tree length\" ct=\"Tree length  is the  criterion used by the  Maximum Parsimony  method to search  for the best tree.  ...\" bc=\"Glossary\"/><di ky=\"545\" rd=\"Tree_tab_in_Options_dialog_box.htm|Tree tab (in Options dialog box)\" ct=\"Tree tab (in  Options dialog box) This allows  you to manipulate aspects of the tree, depending on t ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"546\" rd=\"Tree_Topology_Editor.htm|Tree Topology Editor\" ct=\"Tree Topology  Editor   The tree  topology   editor shows a single tree&apos;s topology in a way in which ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"547\" rd=\"TreeExplorer_HC_files/File_Menu_in_Tree_Explorer.htm|File Menu (in Tree Explorer)\" ct=\"File Menu  (in Tree Explorer) This menu has  the following options: Save Current  Session :  This br ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"548\" rd=\"TreeExplorer_HC_files/Tree_Explorer.htm|Tree Explorer\" ct=\"Phylogeny |  Any tree-building option The  Tree  Explorer  displays the evolutionary tree based on t ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/></ck>";
