{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"197\" rd=\"Get_Data_from_Saved_Session.htm|Get Data from Saved Session\" ct=\"Data | Open A File/Session…   Choose this menu item to activate a saved session file (.mdsx).  This  ...\" bc=\"\"/><di ky=\"198\" rd=\"Get_more_information_about_the_codon_based_Z-test_for_selection.htm|Get more information about the codon based Z-test for selection\" ct=\"Get more information  about the codon based Z-test for selection The  codon  based Z-test for select ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"199\" rd=\"Go_to_Line_in_Text_Editor.htm|Go to Line (in Text Editor)\" ct=\"Edit | Go to Line # This opens a small dialog box that allows you  to enter a number indicating the  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"200\" rd=\"Goldman_1993.htm|Goldman 1993\" ct=\"Goldman N ( 1993 ) Statistical tests of models of DNA substitution.  Journal of Molecular Evolution ...\" bc=\"References\"/><di ky=\"201\" rd=\"Groups_of_taxa.htm|Groups of taxa\" ct=\"A group of  taxa  is a set of one or more  taxa . Members of a group can be specified  in the input  ...\" bc=\"Glossary\"/><di ky=\"202\" rd=\"Gu_and_Zhang_1997.htm|Gu and Zhang 1997\" ct=\"Gu X &amp; Zhang J ( 1997 ) A simple method for estimating the parameter of substitution rate variation  ...\" bc=\"References\"/><di ky=\"203\" rd=\"Hedges_et_al_1992.htm|Hedges et al. 1992\" ct=\"Hedges SB, Kumar S, Tamura K, &amp; Stoneking M ( 1992 ). Human origins and analysis of mitochondrial DN ...\" bc=\"References\"/><di ky=\"204\" rd=\"Help_Menu.htm|Help Menu\" ct=\"Help Menu This menu provides access to the help index as well as the  About  dialog box, which provi ...\" bc=\"\"/><di ky=\"205\" rd=\"Hendy_and_Penny_1982.htm|Hendy and Penny 1982\" ct=\"Hendy MD &amp; Penny (D) ( 1982 )  Branch  and bound algorithms to determine minimal evolutionary trees. ...\" bc=\"References\"/><di ky=\"206\" rd=\"Hendy_and_Penny_1989.htm|Hendy and Penny 1989\" ct=\"Hendy M D &amp; Penny D ( 1989 ) A framework for the quantitative study of evolutionary trees.  Systemat ...\" bc=\"References\"/><di ky=\"207\" rd=\"Hide_Motif.htm|Hide Motif\" ct=\"If you have searched for a motif, this will  hide all the search result(s).  When the search result( ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"208\" rd=\"Hide_Name_Result.htm|Hide Name Result\" ct=\"If you have searched for a Sequence name, this  will hide that search result.  When the search resul ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"209\" rd=\"Highlight_0-fold_Degenerate_Sites.htm|Highlight 0-fold Degenerate Sites\" ct=\"Highlight 0-fold Degenerate  Sites Highlight | 0-fold Degenerate Sites Use this command to highlight ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"210\" rd=\"Highlight_2-fold_Degenerate_Sites.htm|Highlight 2-fold Degenerate Sites\" ct=\"Highlight 2-fold Degenerate  Sites Highlight | 2-fold Degenerate Sites Use this command to highlight ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"211\" rd=\"Highlight_4-fold_Degenerate_Sites.htm|Highlight 4-fold Degenerate Sites\" ct=\"Highlight 4-fold Degenerate  Sites Highlight | 4-fold Degenerate Sites Use this command to highlight ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"212\" rd=\"Highlight_Conserved_Sites.htm|Highlight Conserved Sites\" ct=\"Highlight | Conserved Sites Use this command to highlight  constant  sites       ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"213\" rd=\"Highlight_Coverage.htm|Highlight Coverage\" ct=\"Highlights sites where there are a certain  percentage (or higher) of unambiguous nucleotides or ami ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu &gt; Highlight Special\"/><di ky=\"214\" rd=\"Highlight_CpG_TpG_CpA.htm|Highlight CpG/TpG/CpA\" ct=\"Highlights sites which have a C followed by  a G, T by a G, or C by a A.  This also uses coverage, t ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu &gt; Highlight Special\"/><di ky=\"215\" rd=\"Highlight_Menu_in_Sequence_Data_Explorer.htm|Highlight Menu (in Sequence Data Explorer)\" ct=\"Highlight Menu (in Sequence  Data Explorer) This menu can be used to highlight certain  types of sit ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"216\" rd=\"Highlight_Parsimony_Informative_Sites.htm|Highlight Parsimony Informative Sites\" ct=\"Highlight Parsimony Informative  Sites Highlight | Parsim-Info Sites Use this command to highlight  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"217\" rd=\"Highlight_Singleton_Sites.htm|Highlight Singleton Sites\" ct=\"Highlight |  Singleton Sites Use this command to highlight  singleton  sites.     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"218\" rd=\"Highlight_Variable_Sites.htm|Highlight Variable Sites\" ct=\"Highlight | Variable Sites Use this command to highlight  variable sites   sites.     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Highlight Menu\"/><di ky=\"219\" rd=\"Hillis_and_Bull_1993.htm|Hillis and Bull 1993\" ct=\"Hillis DM &amp; Bull JJ ( 1993 ) An empirical test of bootstrapping as a method for assessing confidence ...\" bc=\"References\"/><di ky=\"220\" rd=\"Hillis_et_al_1996.htm|Hillis et al. 1996\" ct=\"Hillis DM, Moritz C &amp; Mable BK ( 1996 )  Molecular Systematics . 2 edition. Sunderland, MA: Sinauer  ...\" bc=\"References\"/><di ky=\"221\" rd=\"Hotkeys_and_Shortcut_keys.htm|Hotkeys and Shortcut keys\" ct=\"Many menu items have a  hotkey  and/or a  shortcut  key.  These are special key combinations that ar ...\" bc=\"\"/><di ky=\"222\" rd=\"How_can_I_ignore_the_current_update_available_message_in_MEGA_s_main_window.htm|How can I ignore the current update available message in MEGA&apos;s main window\" ct=\"How can I  ignore the current update available message in MEGA&apos;s main window By clicking  the blue [ ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"223\" rd=\"How_do_I_prevent_the_MEGA_Update_Available_message_showing_up.htm|How do I prevent the &quot;MEGA Update Available&quot; message showing up\" ct=\"How do I prevent  the &quot;MEGA Update Available&quot; message showing up Go to your  MEGA installation direc ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"224\" rd=\"How_to_Label_Sites.htm|How to Label Sites\" ct=\"Sites in a sequence alignment can  be categorized and   labeled    with user-defined symbols. Each   ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Labelling Individual Sites\"/><di ky=\"225\" rd=\"Image_Menu_in_Tree_Explorer.htm|Image Menu (in Tree Explorer)\" ct=\"Image Menu  (in Tree Explorer) The image menu  contains three options: Copy to Clipboard : This  cop ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"226\" rd=\"Importing_Data.htm|Importing Data\" ct=\"MEGA    supports  conversions from several different file formats into   MEGA    formats .  Each for ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats\"/><di ky=\"227\" rd=\"Inapplicable_Computation_Requested.htm|Inapplicable Computation Requested\" ct=\"You have requested a computation that  is not allowed or is unavailable for the currently active dat ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"228\" rd=\"Include_Sites_Option.htm|Include Sites Option\" ct=\"With this command you can set the options  for handling gaps and missing data in the analysis, such  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Handling Missing Data and Alignment Gaps\"/><di ky=\"229\" rd=\"Incorrect_Command_Used.htm|Incorrect Command Used\" ct=\"The selected command or option is not valid here.  Please look at the brief description provided in  ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"230\" rd=\"Indels.htm|Indels\" ct=\"Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be alig ...\" bc=\"Glossary\"/><di ky=\"231\" rd=\"Independent_Sites.htm|Independent Sites\" ct=\"In a sequence alignment, all sites that have not been assigned to any   gene  or  domain  are classi ...\" bc=\"Glossary\"/><di ky=\"232\" rd=\"Index_1.htm|Index\" ct=\"Help | Index This command provides access to the help file index and keyword searching facilities.   ...\" bc=\"\"/><di ky=\"233\" rd=\"Inferred_Tree.htm|Inferred Tree\" ct=\"A tree reconstructed from the observed sequence or other appropriate data using any tree-making meth ...\" bc=\"Glossary\"/><di ky=\"234\" rd=\"Inferring_Ancestral_Sequences_ML.htm|Inferring Ancestral Sequences (ML)\" ct=\"Ancestral Sequences | Infer Ancestral Sequences (ML) This option uses the Maximum Likelihood method  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Ancestral States\"/><di ky=\"235\" rd=\"Inferring_Ancestral_Sequences_MP.htm|Inferring Ancestral Sequences (MP)\" ct=\"Ancestral Sequences | Infer Ancestral Sequences (Parsimony) When the sequence diversity is low Maxim ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Ancestral States\"/></ck>";
