{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"79\" rd=\"Converting_Clustal_Format.htm|Converting Clustal Format\" ct=\"Converting CLUSTAL Format The sequence alignment  outputs from CLUSTAL software often are given the  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"80\" rd=\"Converting_FASTA_Format.htm|Converting FASTA Format\" ct=\"The FASTA file format is  very simple and is quite similar to the   MEGA    file  format. This is an ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"81\" rd=\"Converting_GCG_Format.htm|Converting GCG Format\" ct=\"These files consist of  one or more groups of non-blank lines separated by one or more blank lines;  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"82\" rd=\"Converting_IG_Format.htm|Converting IG Format\" ct=\"These files consist of  one or more groups of non-blank lines separated by one or more blank lines.  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"83\" rd=\"Converting_MSF_Format.htm|Converting MSF Format\" ct=\"These files consist of  one or more groups of non-blank lines separated by one or more blank lines.  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"84\" rd=\"Converting_NBRF_Format.htm|Converting NBRF Format\" ct=\"NBRF files consist of one  or more groups of non-blank lines separated by one or more blank lines;   ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"85\" rd=\"Converting_NEXUS_Format.htm|Converting NEXUS Format\" ct=\"The NEXUS file format has  a header with lines identifying the name of each of the sequences in the  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"86\" rd=\"Converting_Phylip_(Interleaved)_Format.htm|Converting Phylip (Interleaved) Format\" ct=\"Converting  the PHYLIP interleaved file format The  PHYLIP format is interleaved, similar to the MSF ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"87\" rd=\"Converting_Phylip_(Non-interleaved)_Format.htm|Converting Phylip (Non-interleaved) Format\" ct=\"Converting  PHYLIP non-interleaved format While  otherwise similar to the PHYLIP interleaved format, ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"88\" rd=\"Converting_PIR_Format.htm|Converting PIR Format\" ct=\"These files consist of  groups of non-blank lines that look similar to this:   ENTRY G006uaah TITLE  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"89\" rd=\"Converting_XML_Format.htm|Converting XML Format\" ct=\"These files consist of  a group of XML tags and attribute values. A DOCTYPE header may or may  not b ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats &gt; Format Specific Notes\"/><di ky=\"90\" rd=\"Copy_in_Text_Editor.htm|Copy (in Text Editor)\" ct=\"Edit | Copy This places a copy of the selected text on  the  Windows  clipboard, leaving the origina ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; Edit Menu\"/><di ky=\"91\" rd=\"Copyright.htm|Copyright\" ct=\"Copyright  © 1993   -   2021. This  software is protected under the copyright law. No part of this m ...\" bc=\"\"/><di ky=\"92\" rd=\"Cut_in_Text_Editor.htm|Cut (in Text Editor)\" ct=\"Edit | Cut This command places a copy of the selected  text on the  Windows  clipboard, removing the ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; Edit Menu\"/><di ky=\"93\" rd=\"Cutoff_Values_Tab.htm|Cutoff Values Tab\" ct=\"Cutoff Values  Tab In this tab,  you can specify a cut-off level for the  condensed  or  consensus   ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"94\" rd=\"Data_Coverage.htm|Data Coverage\" ct=\"Data coverage at a given internal  node   in a tree is defined as the proportion of sites where ther ...\" bc=\"Glossary\"/><di ky=\"95\" rd=\"Data_Explorer.htm|Data Explorer\" ct=\"Data | Data Explorer Data Explorers used to view the currently  active data set, calculate its basic ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Data Menu\"/><di ky=\"96\" rd=\"Data_File_Parsing_Error.htm|Data File Parsing Error\" ct=\"An error occurred while parsing the input data file.  Pay close attention to the message provided, t ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"97\" rd=\"Data_Menu.htm|Data Menu\" ct=\"This allows you to explore the active data set, and establish various data attributes, and data subs ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Data Menu\"/><di ky=\"98\" rd=\"Data_Menu_(Sequence_Data_Explorer).htm|Data Menu (Sequence Data Explorer)\" ct=\"This menu provides commands for working with selected  data in the  Sequence Data Explorer The comma ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Data Menu\"/><di ky=\"99\" rd=\"Data_Menu_in_Alignment_Explorer.htm|Data Menu (in Alignment Explorer)\" ct=\"This menu provides commands for creating a new alignment, opening/closing sequence data files, savin ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Alignment Explorer &gt; Menu Items\"/><di ky=\"100\" rd=\"Dayhoff_1978.htm|Dayhoff 1978\" ct=\"Dayhoff MO ( 1978 ) Survey of new data and computer methods of analysis. In Dayhoff MO, ed.,  Atlas  ...\" bc=\"References\"/><di ky=\"101\" rd=\"Dayhoff_1979.htm|Dayhoff 1979\" ct=\"Schwarz R &amp; Dayhoff M ( 1979 ) Matrices for detecting distant relationships. In Dayhoff M, editor,  ...\" bc=\"References\"/><di ky=\"102\" rd=\"Dayhoff_and_JTT_distances_Gamma_rates.htm|Dayhoff and JTT distances (Gamma rates)\" ct=\"Dayhoff  and JTT distances (Gamma rates) The PAM and  JTT distances correct for multiple substitutio ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Amino Acid Substitution Models &gt; Gamma Distances\"/><di ky=\"103\" rd=\"Dayhoff_and_JTT_Models.htm|Dayhoff and JTT Models\" ct=\"Dayhoff  and JTT Models The PAM and  JTT distances correct for multiple substitutions based on the m ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Amino Acid Substitution Models\"/><di ky=\"104\" rd=\"Dayhoff_JTT_Distance_Could_Not_Be_Computed.htm|Dayhoff/JTT Distance Could Not Be Computed\" ct=\"Dayhoff/JTT Distance Could Not Be  Computed The Dayhoff/JTT matrix-based correction  could not be ap ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"105\" rd=\"DeBry_1992.htm|DeBry 1992\" ct=\"DeBry RW ( 1992 ) The consistency of several phylogeny-inference methods under varying evolutionary  ...\" bc=\"References\"/><di ky=\"106\" rd=\"Defining_Species_and_Populations.htm|Defining Species and Populations\" ct=\"Beginning with MEGA7, it  is possible to write command statements in .meg files for grouping  taxa   ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Defining Groups and Meta Data\"/><di ky=\"107\" rd=\"Degeneracy.htm|Degeneracy\" ct=\"0-fold degenerate sites  are those at which all changes  are  nonsynonymous . 2-fold degenerate site ...\" bc=\"Glossary\"/><di ky=\"108\" rd=\"Delete_in_Text_Editor.htm|Delete (in Text Editor)\" ct=\"Edit | Delete This deletes the selected (highlighted) text.  It  is NOT copied to the clipboard.     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"109\" rd=\"Diagnose_Mutations.htm|Diagnose Mutations\" ct=\"Forecast the deleteriousness of nsSNVs using  multiple methods and explore them in the context of th ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Diagnose Menu\"/><di ky=\"110\" rd=\"Disclaimer.htm|Disclaimer\" ct=\"Although  the utmost care has been taken to ensure the correctness of the software,  it is provided  ...\" bc=\"\"/><di ky=\"111\" rd=\"Disparity_Index.htm|Disparity Index\" ct=\"Disparity Index measures the observed difference in substitution patterns for a pair of sequences.   ...\" bc=\"Glossary\"/><di ky=\"112\" rd=\"Disparity_Index_Test.htm|Disparity Index Test\" ct=\"Tests the null hypothesis that sequences have evolved with the same  pattern of substitution, as jud ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"113\" rd=\"Display_Menu_in_Alignment_Explorer.htm|Display Menu (in Alignment Explorer)\" ct=\"This menu provides access to commands that control the display of toolbars in the alignment grid.   ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Alignment Explorer &gt; Menu Items\"/><di ky=\"114\" rd=\"Display_Menu_in_Distance_Data_Explorer.htm|Display Menu (in Distance Data Explorer)\" ct=\"Display Menu (in Distance  Data Explorer) The  Display  menu consists of four main  commands: Show O ...\" bc=\"Part III: Input Data Types and File Formats &gt; Distance Data Explorer\"/><di ky=\"115\" rd=\"Display_Menu_in_Distance_Matrix_Explorer.htm|Display Menu (in Distance Matrix Explorer)\" ct=\"Display  Menu (in Distance Matrix Explorer) The display  menu consists of four main commands: Show P ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Distance Matrix Explorer\"/><di ky=\"116\" rd=\"Display_Menu_in_Sequence_Data_Explorer.htm|Display Menu (in Sequence Data Explorer)\" ct=\"Display Menu (in Sequence  Data Explorer) This menu provides commands  for adjusting the display of  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/></ck>";
