{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"549\" rd=\"Undo_in_Text_Editor.htm|Undo (in Text Editor)\" ct=\"Edit | Undo Choose this command to undo your most recent  action.  Repeated use of this command will ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; Edit Menu\"/><di ky=\"550\" rd=\"Unexpected_Error.htm|Unexpected Error\" ct=\"While carrying out the requested task,  an unexpected error has occurred in  MEGA .  Please contact  ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"551\" rd=\"Unrooted_tree.htm|Unrooted tree\" ct=\"An unrooted tree is one in which no assumption is made regarding the ancestor of all the  taxa RH_Ta ...\" bc=\"Glossary\"/><di ky=\"552\" rd=\"UPGMA.htm|UPGMA\" ct=\"This method assumes that the rate of nucleotide or amino acid substitution is the same for all evolu ...\" bc=\"\"/><di ky=\"553\" rd=\"UPGMA_Construct_Phylogeny.htm|UPGMA (Construct Phylogeny)\" ct=\"Phylogeny | Construct/Test UPGMA Tree… This command is used to construct a UPGMA tree.  This tree-ma ...\" bc=\"\"/><di ky=\"554\" rd=\"Upload_a_text_file_with_the_coordinate_information_for_all_nSNVs_of_interest.htm|Upload a text file with the coordinate information for all nSNVs of interest\" ct=\"Upload a text file with the coordinate  information for all nSNVs of interest Create a  text file wi ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; Input Data\"/><di ky=\"555\" rd=\"Use_All_Selected_Sites.htm|Use All Selected Sites\" ct=\"Statistics | Use All Selected Sites Analysis is conducted on all sites in the sequences,  irrespecti ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"556\" rd=\"Use_Identical_Symbol.htm|Use Identical Symbol\" ct=\"Display | Use Identical Symbol Data that contain multiple aligned sequences  may be easier to view i ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"557\" rd=\"Use_only_Highlighted_Sites.htm|Use only Highlighted Sites\" ct=\"Statistics | Use only Highlighted Sites Sites can be selected according to various  criteria (see  H ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"558\" rd=\"User_Stopped_Computation.htm|User Stopped Computation\" ct=\"You have aborted the current process by pressing the  Stop   process  button on the progress indicat ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"559\" rd=\"Using_the_Visual_Tool.htm|Using the Visual Tool\" ct=\"Data | Select  Taxa   &amp; Groups   This invokes the   Select Taxa &amp; Groups    dialog box for including ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Defining Groups and Meta Data\"/><di ky=\"560\" rd=\"Variable_site.htm|Variable site\" ct=\"A variable site contains at least two types of nucleotides or amino  acids.  Some variable sites can ...\" bc=\"Glossary\"/><di ky=\"561\" rd=\"View_Edit_Sequencer_Files.htm|View/Edit Sequencer Files\" ct=\"Align | Edit/View Sequencer Files (Trace)… Use this option to view/edit the sequence  data in   ABI ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Align Menu\"/><di ky=\"562\" rd=\"View_Menu_in_Tree_Explorer.htm|View Menu (in Tree Explorer)\" ct=\"View Menu  (in Tree Explorer) This menu brings  up several viewing options: Topology only :   This d ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"563\" rd=\"Web_Browser_Data_Miner.htm|Web Browser &amp; Data Miner\" ct=\"MEGA  contains a fully functional  Web Browser  to assist users in sequence data retrieval and web e ...\" bc=\"\"/><di ky=\"564\" rd=\"Web_Menu_in_Alignment_Explorer.htm|Web Menu (in Alignment Explorer)\" ct=\"Web Menu  (in Alignment Explorer) This menu provides access to commands for querying GenBank and doi ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Alignment Explorer &gt; Menu Items\"/><di ky=\"565\" rd=\"What_is_a_Site_Label_.htm|What is a Site Label?\" ct=\"The individual sites in nucleotide  or amino acid data can be labeled to construct non-contiguous se ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Labelling Individual Sites\"/><di ky=\"566\" rd=\"Why_can_t_I_display_branch_lengths_for_a_bootstrap_consensus_tree.htm|Why can&apos;t I display branch lengths for a bootstrap consensus tree?\" ct=\"Why can&apos;t  I display branch lengths for a bootstrap consensus tree? MEGA does not  computer or provi ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"567\" rd=\"Word_Wrap_in_Text_Editor.htm|Word Wrap (in Text Editor)\" ct=\"Display | Word Wrap This item can be checked (on) or un-checked  (off) to show whether lines in the  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"568\" rd=\"Writing_Command_Statement_for_Defining_Genes_and_Domains.htm|Writing Command Statement for Defining Genes and Domains\" ct=\"	 	 	 	 	 		 Command 		 Setting 		 Remark 		 Example 	 	 		 DataType 		 DNA, 		 RNA, nucleotide, pro ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Defining Genes and Domains\"/><di ky=\"569\" rd=\"Writing_Command_Statements_for_Defining_Groups_of_Taxa.htm|Writing Command Statements for Defining Groups of Taxa\" ct=\"Writing Command Statements for Defining Groups of Taxa and for Annotating  Taxa with Meta Data The  ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Defining Groups and Meta Data\"/><di ky=\"570\" rd=\"Writing_only_4-fold_degenerate_sites_to_an_output_file.htm|Writing only 4-fold degenerate sites to an output file\" ct=\"Writing only  4-fold degenerate sites to an output file All sequence data subset  facilities are acc ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"571\" rd=\"Yang_1999.htm|Yang 1999\" ct=\"Yang Z ( 1999 ) PAML: Phylogenetic analysis by maximum likelihood, Version 2. 0 . University College ...\" bc=\"References\"/><di ky=\"572\" rd=\"Zhang_and_Gu_1998.htm|Zhang and Gu 1998\" ct=\"Zhang J &amp; Gu X ( 1998 ) Correlation between the substitution rate and rate variation among sites in  ...\" bc=\"References\"/><di ky=\"573\" rd=\"Zhang_et_al_1997.htm|Zhang et al. 1997\" ct=\"Zhang J, Kumar S &amp; Nei M ( 1997 ) Small-sample tests of episodic adaptive evolution: a case study of ...\" bc=\"References\"/><di ky=\"574\" rd=\"Zhang_et_al_1998.htm|Zhang et al. 1998\" ct=\"Zhang J, Rosenberg HF &amp; Nei M ( 1998 ).  Positive Darwinian selection after gene duplication in prim ...\" bc=\"References\"/><di ky=\"575\" rd=\"Zuckerkandl_and_Pauling_1965.htm|Zuckerkandl and Pauling 1965\" ct=\"Zuckerkandl E &amp; Pauling L ( 1965 ) Evolutionary divergence and convergence in proteins, pp. 97-166 i ...\" bc=\"References\"/></ck>";
