{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"320\" rd=\"No_of_differences_Amino_acids.htm|No. of differences (Amino acids)\" ct=\"No. of differences  (Amino acids)   This  distance is the number of sites at which two sequences bei ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Amino Acid Substitution Models\"/><di ky=\"321\" rd=\"No_of_differences_Nucleotide.htm|No. of differences (Nucleotide)\" ct=\"No.  of differences (Nucleotide)   This distance is the number of sites at  which the two compared s ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"322\" rd=\"Node.htm|Node\" ct=\"A node in a phylogenetic tree represents a  taxon ,  the  external  or  terminal  nodes  represent t ...\" bc=\"Glossary\"/><di ky=\"323\" rd=\"Nonsynonymous_change.htm|Nonsynonymous change\" ct=\"A nucleotide change is nonsynonymous if it changes the amino acid encoded by the original  codon .   ...\" bc=\"Glossary\"/><di ky=\"324\" rd=\"Not_Enough_Groups_Selected.htm|Not Enough Groups Selected\" ct=\"The currently active dataset or subset  does not contain enough groups to conduct the desired analys ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"325\" rd=\"Not_Enough_Taxa_Selected.htm|Not Enough Taxa Selected\" ct=\"The currently active dataset or subset does not contain enough sequences or  taxa RH_Taxa  to conduc ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"326\" rd=\"Not_Yet_Implemented.htm|Not Yet Implemented\" ct=\"The task you requested was not activated.  This  function either was not being available in your rel ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"327\" rd=\"Nucleotide_and_Amino_Acid_Compositions.htm|Nucleotide and Amino Acid Compositions\" ct=\"The relative frequencies of the four nucleotides  (nucleotide composition) or of the 20 amino acid r ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Basic Statistical Quantities for Sequence Data\"/><di ky=\"328\" rd=\"Nucleotide_Composition.htm|Nucleotide Composition\" ct=\"Statistics | Nucleotide Composition This command is visible only if the data consist  of nucleotide  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"329\" rd=\"Nucleotide_Pair_Frequencies.htm|Nucleotide Pair Frequencies\" ct=\"Nucleotide  Pair Frequencies Statistics  | Nucleotide Pair Frequencies This command  is visible only ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"330\" rd=\"OLS_branch_length_estimates.htm|OLS branch length estimates\" ct=\"            The ordinary least squares estimate of a branch length ( b ) is given by where  d ij  is ...\" bc=\"Glossary\"/><di ky=\"331\" rd=\"Open_A_File.htm|Open A File\" ct=\"Data | Open A File/Session…   When you choose this option you will be prompted to select a file to l ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Data Menu\"/><di ky=\"332\" rd=\"Open_Data.htm|Open Data\" ct=\"Data | Activate Data for Analysis | Get Data from MEGA text File Choose this command to load a data  ...\" bc=\"\"/><di ky=\"333\" rd=\"Open_in_Text_Editor.htm|Open (in Text Editor)\" ct=\"File | Open Use this command to open an existing file in  the  Text Editor .     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"334\" rd=\"Open_Saved_Alignment_Session.htm|Open Saved Alignment Session\" ct=\"Align | Open Saved  Alignment Session … Use this command to display a previously saved  Alignment Ex ...\" bc=\"\"/><di ky=\"335\" rd=\"Options_dialog_box.htm|Options dialog box\" ct=\"At the top of the options dialog box is an  option for the output format (Publication and  MEGA ) wi ...\" bc=\"Part III: Input Data Types and File Formats &gt; Distance Data Explorer\"/><di ky=\"336\" rd=\"Options_dialog_box_in_Tree_Explorer.htm|Options dialog box (in Tree Explorer)\" ct=\"Options dialog  box (in Tree Explorer) Through this  dialog box, you can specify various drawing att ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"337\" rd=\"Orthologous_Genes.htm|Orthologous Genes\" ct=\"Two genes are said to be orthologous if they are the result of a speciation event.   ...\" bc=\"Glossary\"/><di ky=\"338\" rd=\"Out-group.htm|Out-group\" ct=\"An out-group is a sequence (or set of sequences) that is known to be a sister  taxa  to all other se ...\" bc=\"Glossary\"/><di ky=\"339\" rd=\"Overview.htm|Overview\" ct=\"In order to retrieve predictions  for a given nsSNV, MEGA-MD requires three pieces of information: 1 ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; Input Data\"/><di ky=\"340\" rd=\"p-distance_Amino_acids.htm|p-distance (Amino acids)\" ct=\"p-distance  (Amino acids) This distance is the proportion ( p )  of amino acid sites at which the tw ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Amino Acid Substitution Models\"/><di ky=\"341\" rd=\"p-distance_Nucleotide.htm|p-distance (Nucleotide)\" ct=\"p-distance  (Nucleotide) This distance is the proportion ( p )  of nucleotide sites at which two seq ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"342\" rd=\"p_distance_is_found_to_be_1.htm|p distance is found to be &gt; 1\" ct=\"This peculiar situation can occur in  the computation of the proportion of  synonymous   (or  nonsyn ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"343\" rd=\"Page_and_Holmes_1998.htm|Page and Holmes 1998\" ct=\"Page RDM &amp; Holmes EC ( 1998 )  Molecular Evolution: A Phylogenetic Approach .  Blackwell Science, Ox ...\" bc=\"References\"/><di ky=\"344\" rd=\"Pairwise-deletion_option.htm|Pairwise-deletion option\" ct=\"In the  pairwise-deletion option , sites containing missing data or  alignment gaps  are removed fro ...\" bc=\"Glossary\"/><di ky=\"345\" rd=\"Pamilo-Bianchi-Li_Method.htm|Pamilo-Bianchi-Li Method\" ct=\"Pamilo-Bianchi-Li  Method   This  method ( Pamilo and Bianchi  1993 ;  Li  1993 )  is a modification ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Synonymous and Nonsynonymous Substitution Models\"/><di ky=\"346\" rd=\"Pamilo_and_Bianchi_1993.htm|Pamilo and Bianchi 1993\" ct=\"Pamilo P&amp; Bianchi NO ( 1993 ) Evolution of the Zfx and Zfy, genes: Rates and interdependence between ...\" bc=\"References\"/><di ky=\"347\" rd=\"Pamilo_and_Nei_1988.htm|Pamilo and Nei 1988\" ct=\"Pamilo P &amp; Nei M ( 1988 ) Relationships between gene trees and species trees.  Molecular Biology and ...\" bc=\"\"/><di ky=\"348\" rd=\"Parsimony-informative_site.htm|Parsimony-informative site\" ct=\"A site is parsimony-informative if it contains at least two types of nucleotides (or amino acids), a ...\" bc=\"Glossary\"/><di ky=\"349\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm|Aligning Sequences\" ct=\"In this   tutorial,  we will   show how to    create a   multiple sequence   alignment  from protein ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"350\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm|Building Trees From Sequence Data\" ct=\"In  this tutorial, we will   illustrate  the procedures   for    building trees and in-memory sequen ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"351\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_With_Distance_Data.htm|Building Trees With Distance Data\" ct=\"This    tutorial   i llustrates   proced ures for building phylogenetic trees using    distance data ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"352\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Computing_Sequence_Statistics.htm|Computing Sequence Statistics\" ct=\"The  “Drosophila_Adh.meg” data file, which is   used  in this tutorial ,    can be found in the   ME ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"353\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_a_Timetree_(ML).htm|Constructing a Timetree (ML)\" ct=\"This example shows how   to generate   a timetree in   MEGA . For this analysis,   MEGA    uses a Ti ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"354\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Constructing_Likelihood_Trees.htm|Constructing Likelihood Trees\" ct=\"MEGA    provides options for performing various  calculations relating to likelihood. In this   tuto ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"355\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm|Estimating Evolutionary Distances\" ct=\"In  this   tutorial , we will   estimate    evolutionary   distances  for sequences from 11   Drosop ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"356\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Inferring_Gene_Duplications.htm|Inferring Gene Duplications\" ct=\"This  example shows how   to identify   gene  duplications (and optionally speciation events) in   M ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"357\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Managing_Taxa_With_Groups.htm|Managing Taxa With Groups\" ct=\"The  “Crab_rRNA.meg” file, which is   used  in this tutorial, can be found in the   MEGA/Examples    ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"358\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm|MEGA Basics\" ct=\"In  this tutorial, we will focus on opening and manipulating data files and  saving results.   All o ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"359\" rd=\"Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_for_Selection.htm|Testing for Selection\" ct=\"In  this example, we   describe  how to perform a  codon -based test of  positive selection   for  f ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/></ck>";
