{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"39\" rd=\"Calibrate_Timetree_with_Molecular_Clock.htm|Calibrate Timetree with Molecular Clock\" ct=\"MEGA provides two options for calibrating the molecular clock for RelTime  trees (where no calibrati ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Time Trees\"/><di ky=\"40\" rd=\"Calibration_Dialog.htm|Calibration Dialog\" ct=\"Calibration  Dialog The  Calibration  Editor  allows you to define multiple divergence time calibrat ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Time Trees\"/><di ky=\"41\" rd=\"Calibration_File_Format.htm|Calibration File Format\" ct=\"The  calibration file is used to provide divergence time calibration constraints  to MEGA so that ME ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Time Trees\"/><di ky=\"42\" rd=\"Change_Font.htm|Change Font...\" ct=\"Display | Change Font… This command brings up the  Change Font   dialog box, which allows you to cha ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"43\" rd=\"Citing_MEGA_In_Publications.htm|Citing MEGA in Publications\" ct=\"If you wish to cite  MEGA  in  your publications, we suggest the following: (1) When referring to  M ...\" bc=\"\"/><di ky=\"44\" rd=\"Close-Neighbor-Interchange.htm|Close-Neighbor-Interchange\" ct=\"  ...\" bc=\"\"/><di ky=\"45\" rd=\"Close-Neighbor-Interchange_CNI.htm|Close-Neighbor-Interchange (CNI)\" ct=\"In any method, examining all possible  topologies  is very  time consuming.  This algorithm reduces  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Minimum Evolution Method &gt; Heuristic Search\"/><di ky=\"46\" rd=\"Close_Data.htm|Close Data\" ct=\"This deactivates the currently open data file. Before issuing this command, save any modifications t ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Data Menu\"/><di ky=\"47\" rd=\"Close_File_in_Text_Editor.htm|Close File (in Text Editor)\" ct=\"File | Close File This closes the current file.     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"48\" rd=\"ClustalW.htm|ClustalW\" ct=\"ClustalW is a general purpose multiple sequence alignment program for DNA or proteins. You can learn ...\" bc=\"Glossary\"/><di ky=\"49\" rd=\"Code_Table_Editor.htm|Code Table Editor\" ct=\"Code Table Editor    The   Code Table Editor  allows you to create new  genetic  codes  and to edit  ...\" bc=\"Part III: Input Data Types and File Formats &gt; Genetic Code Tables\"/><di ky=\"50\" rd=\"Codon.htm|Codon\" ct=\"A codon is triplet of nucleotides that codes for a specific amino acid.   ...\" bc=\"Glossary\"/><di ky=\"51\" rd=\"Codon_Based_Fisher_s_Exact_Test.htm|Codon Based Fisher&apos;s Exact Test\" ct=\"Distance |  Codon   Based Fisher’s Exact Test This provides a test of selection based  on the compar ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Selection Menu\"/><di ky=\"52\" rd=\"Codon_Based_Z-Test_large_sample.htm|Codon Based Z-Test (large sample)\" ct=\"Distance |  Codon   Based Z-test (large sample) One way to test whether  positive selection  is oper ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Selection Menu\"/><di ky=\"53\" rd=\"Codon_Usage.htm|Codon Usage\" ct=\"Codon  Usage Statistics  |  Codon   Usage This command  is visible only if the data contains protein ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"54\" rd=\"Color_Cells.htm|Color Cells\" ct=\"Display | Color cells This command colors individual cells in the  two-dimensional display grid acco ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"55\" rd=\"Comeron_1995.htm|Comeron 1995\" ct=\"Comeron JM ( 1995 ) A method for estimating the numbers of synonymous and nonsynonymous substitution ...\" bc=\"References\"/><di ky=\"56\" rd=\"Complete-Deletion_Option.htm|Complete-Deletion Option\" ct=\"In the complete-deletion option, sites containing missing data or  alignment gaps  are removed befor ...\" bc=\"Glossary\"/><di ky=\"57\" rd=\"Composition_Distance.htm|Composition Distance\" ct=\"Composition distance is a measure of the difference in nucleotide (or  amino acid) composition for a ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"58\" rd=\"Compress_Uncompress.htm|Compress/Uncompress\" ct=\"This command changes the cursor to the &apos;Compress/Uncompress&apos; icon.  If  you click on an interior  br ...\" bc=\"Glossary\"/><di ky=\"59\" rd=\"Compute_Between_Groups_Means.htm|Compute Between Groups Means\" ct=\"Distances | Compute Between Groups Means… This computes  the average distances between   groups of t ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Distances Menu\"/><di ky=\"60\" rd=\"Compute_MCL_Substitution_Matrix.htm|Compute MCL Substitution Matrix\" ct=\"Models   |  Compute MCL Substitution Matrix   This option estimates and displays the substitution ra ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"61\" rd=\"Compute_MCL_Transversion_Transition_bias.htm|Compute MCL Transversion/Transition bias\" ct=\"Models   |  Compute MCL  Transversion / Transition  bias   This option estimates the  Transition  /  ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"62\" rd=\"Compute_Menu_in_Tree_Explorer.htm|Compute Menu (in Tree Explorer)\" ct=\"Compute Menu  (in Tree Explorer) This menu makes  available various tree computations, including  Co ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"63\" rd=\"Compute_Net_Between_Groups_Means.htm|Compute Net Between Groups Means\" ct=\"Distances | Compute Net Between Groups Means… This command computes the  net   average distances bet ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Distances Menu\"/><di ky=\"64\" rd=\"Compute_Overall_Mean.htm|Compute Overall Mean\" ct=\"Distances | Compute Overall Mean… This calculates the mean pairwise distance  and standard error for ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Distances Menu\"/><di ky=\"65\" rd=\"Compute_Pairwise.htm|Compute Pairwise\" ct=\"Distances | Compute Pairwise… Choose this to compute the distances  and standard errors bet ween pai ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Distances Menu\"/><di ky=\"66\" rd=\"Compute_Sequence_Diversity.htm|Compute Sequence Diversity\" ct=\"Distances | Compute Sequence Diversity The Sequence Diversity  submenu provides  four commands for c ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Diversity Menu\"/><di ky=\"67\" rd=\"Compute_Within_Groups_Mean.htm|Compute Within Groups Mean\" ct=\"Distances | Compute Within Groups Means… This computes the mean pairwise distances  with in  groups  ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Distances Menu\"/><di ky=\"68\" rd=\"Computing_statistics_on_only_highlighted_sites_in_Data_Explorer.htm|Computing statistics on only highlighted sites in Data Explorer\" ct=\"Computing  statistics on only highlighted sites in Data Explorer Go to the  Statistics   menu in the ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"69\" rd=\"Computing_the_Gamma_Parameter_a.htm|Computing the Gamma Parameter (a)\" ct=\"Computing  the Gamma Parameter (a) In the computation of gamma distances,  it is necessary to know t ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"70\" rd=\"Concatenation_Utility.htm|Concatenation Utility\" ct=\"MEGA provides a utility for concatenating  multiple files containing sequence data into a single seq ...\" bc=\"Part II: Assembling Data for Analysis &gt; Building Sequence Alignments &gt; Concatenate Sequence Data\"/><di ky=\"71\" rd=\"Condensed_Tree.htm|Condensed Tree\" ct=\"When interior branches in a phylogenetic tree do not have statistically significant lengths, choosin ...\" bc=\"Glossary\"/><di ky=\"72\" rd=\"Condensed_Trees.htm|Condensed Trees\" ct=\"When several interior branches of a phylogenetic tree have low statistical  support ( P C  or  P B ) ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Statistical Tests of a Tree Obtained\"/><di ky=\"73\" rd=\"Consensus_Tree.htm|Consensus Tree\" ct=\"The MP method produces many equally parsimonious trees.  Choosing  this command produces a composite ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Parsimony (MP) Method\"/><di ky=\"74\" rd=\"Constant_Site.htm|Constant Site\" ct=\" A site containing the same nucleotide or amino acid in all sequences is referred to as a constant s ...\" bc=\"Glossary\"/><di ky=\"75\" rd=\"Convert_to_MEGA_Format1.htm|Convert to MEGA Format1\" ct=\"Convert to MEGA Format File    |   Convert    File Format   to  MEGA This item allows you to  choose ...\" bc=\"Part III: Input Data Types and File Formats &gt; Importing Data From Other Formats\"/><di ky=\"76\" rd=\"Convert_to_MEGA_Format_Dialog_Box.htm|Convert to MEGA Format Dialog Box\" ct=\"Use the  MEGA  format dialog box to select a file and file format to convert to  MEGA  format.  Deta ...\" bc=\"\"/><di ky=\"77\" rd=\"Convert_to_Mega_Format_in_Text_Editor.htm|Convert to Mega Format (in Text Editor)\" ct=\"Utilities | Convert to Mega Format This item converts the sequence data in the current edit window,  ...\" bc=\"\"/><di ky=\"78\" rd=\"Convert_To_MEGA_Format_Main_File_Menu.htm|Convert To MEGA Format (Main File Menu)\" ct=\"File | Convert File Format to MEGA This item allows you to choose the file and/or the format that yo ...\" bc=\"\"/></ck>";
