{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"117\" rd=\"Display_Newick_Tree_From_File.htm|Display Newick Tree From File\" ct=\"User  Tree   |   Display Newick Tree Use this to retrieve and display one or more trees written  in ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Tree Input Data\"/><di ky=\"118\" rd=\"Display_Newick_Trees_from_File.htm|Display Newick Trees from File\" ct=\"User Tree  | Display Newick Tree Use this to retrieve and display one or more trees written in  Newi ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; User Tree Menu\"/><di ky=\"119\" rd=\"Display_Results_in_XL_CSV_Text.htm|Display Results in XL/CSV/Text\" ct=\"Items in statistic viewer which have output  will be written in one of these three formats.  If text ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"120\" rd=\"Display_Saved_Tree_Session.htm|Display Saved Tree Session\" ct=\"Phylogeny/User  Tree | Open Tree Session Use this command to display a previously  saved  Tree Explo ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Phylogeny Menu\"/><di ky=\"121\" rd=\"Distance_Data_Explorer.htm|Distance Data Explorer\" ct=\"The  Distance Data Explorer  shows the  pair-wise distance data.  This explorer is flexible and it p ...\" bc=\"Part III: Input Data Types and File Formats &gt; Distance Data Explorer\"/><di ky=\"122\" rd=\"Distance_Data_Subset_Selection.htm|Distance Data Subset Selection\" ct=\"You may select   Select    Taxa    (sequences) or   Groups    of  taxa  through the   Setup/Select   ...\" bc=\"Part III: Input Data Types and File Formats &gt; Visual Tools for Data Management &gt; Data Subset Selection\"/><di ky=\"123\" rd=\"Distance_Matrix_Explorer.htm|Distance Matrix Explorer\" ct=\"Distance  Matrix Explorer The  Distance  Matrix Explorer  is used to display results from the pairwi ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Distance Matrix Explorer\"/><di ky=\"124\" rd=\"Distance_Model_Options.htm|Distance Model Options\" ct=\"With this option, you can choose the  general attributes of the substitution models for DNA and prot ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Specifying Distance Estimation Options\"/><di ky=\"125\" rd=\"Distances_Menu.htm|Distances Menu\" ct=\"Distances Menu Use this menu to compute: pairwise and average distances between sequences; within, b ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Distances Menu\"/><di ky=\"126\" rd=\"Divergence_Times_in_the_Outgroup_Cluster_are_not_Visible.htm|Divergence Times in the Outgroup Cluster are not Visible\" ct=\"Divergence  Times in the Outgroup Cluster are not Visible When calibration  constraints are used in  ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"127\" rd=\"Do_BLAST_Search.htm|Do BLAST Search\" ct=\"Align | Do BLAST Search Use this to launch the  BLAST HC_About_BLAST  search in the  MEGA   Web Brow ...\" bc=\"\"/><di ky=\"128\" rd=\"Domains.htm|Domains\" ct=\"A domain is a continuous block of sites in a sequence alignment.  A  domain can be free-standing or  ...\" bc=\"Glossary\"/><di ky=\"129\" rd=\"Domains_Cannot_Overlap.htm|Domains Cannot Overlap\" ct=\"Any given site can belong to only one  domain ,    at most.  If you would like to assign a site or r ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"130\" rd=\"Dopazo_1994.htm|Dopazo 1994\" ct=\"Dopazo J ( 1994 ) Estimating errors and confidence intervals for  branch  lengths in phylogenetic tr ...\" bc=\"References\"/><di ky=\"131\" rd=\"Eck_and_Dayhoff_1966.htm|Eck and Dayhoff 1966\" ct=\"Eck RV &amp; Dayhoff MO ( 1966 )  Atlas of Protein Sequence and Structure . National Biomedical Research ...\" bc=\"References\"/><di ky=\"132\" rd=\"Edit_Menu_in_Alignment_Explorer.htm|Edit Menu (in Alignment Explorer)\" ct=\"This menu provides access to commands for editing the sequence data in the alignment grid .   The co ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Alignment Explorer &gt; Menu Items\"/><di ky=\"133\" rd=\"Editing_Data_Files.htm|Editing Data Files\" ct=\"There  may be times when you want to make changes to a data file. With the    MEGA   Alignment  Expl ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"134\" rd=\"Efron_1982.htm|Efron 1982\" ct=\"Efron B ( 1982 )  The Jackknife, the Bootstrap and Other Resampling Plans . CBMS-NSF Regional Confer ...\" bc=\"References\"/><di ky=\"135\" rd=\"Equal_Input_Correction_Failed.htm|Equal Input Correction Failed\" ct=\"This error message means that, the  Equal Input Model-based correction could not be applied for the  ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"136\" rd=\"Equal_Input_Model_Amino_acids.htm|Equal Input Model (Amino acids)\" ct=\"Equal Input  Model (Amino acids) In  real data, frequencies   usually  vary among different kind of  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Amino Acid Substitution Models\"/><di ky=\"137\" rd=\"Equal_Input_Model_Gamma.htm|Equal Input Model (Gamma)\" ct=\"In  real data,   amino acid frequencies    usually vary among the different kinds  of amino acids an ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"138\" rd=\"Equal_Input_Model_Gamma_rates_and_Heterogeneous_Patterns.htm|Equal Input Model (Gamma rates and Heterogeneous Patterns)\" ct=\"In real data,   amino acid frequencies   usually vary among different kind of amino acids.    Theref ...\" bc=\"\"/><di ky=\"139\" rd=\"Equal_Input_Model_Heterogeneous_Patterns.htm|Equal Input Model (Heterogeneous Patterns)\" ct=\"In  real data,   amino acid frequencies    usually vary among different kinds of  amino acids. In th ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns &gt; Gamma Rates\"/><di ky=\"140\" rd=\"Estabrook_et_al_1975.htm|Estabrook et al. 1975\" ct=\"Estabrook GF, Johnson CS &amp; McMorris FR ( 1975 ) An idealized concept of the true cladistic character ...\" bc=\"References\"/><di ky=\"141\" rd=\"Estimate_Gamma_Parameter_for_Site_Rates_(ML).htm|Estimate Gamma Parameter for Site Rates (ML)\" ct=\"Estimate the value of the shape parameter for the discrete Gamma distribution. ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Rates Menu\"/><di ky=\"142\" rd=\"Estimate_Position-by-Position_Rates_ML.htm|Estimate Position-by-Position Rates (ML)\" ct=\"Rates | Estimate Position-by-Position Rates (ML)   This option uses the Maximum Likelihood method to ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Rates Menu\"/><di ky=\"143\" rd=\"Estimate_Rate_Variation_among_Sites_ML.htm|Estimate Rate Variation among Sites (ML)\" ct=\"Rates | Estimate  Gamma Parameter  for Site Rates (ML)   This option uses Maximum Likelihood to esti ...\" bc=\"\"/><di ky=\"144\" rd=\"Estimate_Selection_for_each_Codon_HyPhy.htm|Estimate Selection for each Codon (HyPhy)\" ct=\"Selection | Estimate Selection for each  Codon  (HyPhy)   This option estimates the strength of sele ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Selection Menu\"/><di ky=\"145\" rd=\"Estimate_Substitution_Matrix_ML.htm|Estimate Substitution Matrix (ML)\" ct=\"Models | Estimate Substitution Matrix   This option estimates and displays the nucleotide substituti ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"146\" rd=\"Estimate_Transition_Transversion_Bias_ML.htm|Estimate Transition/Transversion Bias (ML)\" ct=\"Models | Estimate  Transition / Transversion  Bias (ML)   This option estimates the  Transition / Tr ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"147\" rd=\"Evolutionary_Probabilities.htm|Evolutionary Probabilities\" ct=\"The Evolutionary Probabilities (EP) analysis in MEGA is used for predicting  permissible and forbidd ...\" bc=\"Part IV: Evolutionary Analysis\"/><di ky=\"148\" rd=\"Exit.htm|Exit\" ct=\"Data | Exit This command closes the currently active  data file and all other windows. If you want t ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Data Menu\"/><di ky=\"149\" rd=\"Exit_Editor_in_Text_Editor.htm|Exit Editor (in Text Editor)\" ct=\"File | Exit Editor This closes the currently open file.  If  the file was modified, but the modifica ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"150\" rd=\"Exon.htm|Exon\" ct=\"A protein-coding  gene  typically consists of multiple  coding regions, known as exons, interspersed ...\" bc=\"Glossary\"/><di ky=\"151\" rd=\"Export_Data.htm|Export Data\" ct=\"Data | Export Data This command activates the appropriate input data explorer, presents a dialog box ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Data Menu\"/><di ky=\"152\" rd=\"Export_Data_(Sequence_Data_Explorer).htm|Export Data (Sequence Data Explorer)\" ct=\"Data | Export Data The  Exporting Sequence Data  dialog box first  displays an edit box for entering ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Data Menu\"/><di ky=\"153\" rd=\"Extant_Taxa.htm|Extant Taxa\" ct=\"The  taxa RH_Taxa  whose sequences, other genetic information or morphological characters, etc. are  ...\" bc=\"Glossary\"/><di ky=\"154\" rd=\"Felsenstein_1978.htm|Felsenstein 1978\" ct=\"Felsenstein J ( 1978 ) Cases in which parsimony or compatibility methods will be positively misleadi ...\" bc=\"References\"/><di ky=\"155\" rd=\"Felsenstein_1985.htm|Felsenstein 1985\" ct=\"Felsenstein J ( 1985 ) Confidence limits on phylogenies: An approach using the bootstrap.  Evolution ...\" bc=\"References\"/></ck>";
