{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"236\" rd=\"Information_Box.htm|Information Box\" ct=\"Information  Box The information  box in the  Tree Explorer  lists the various statistical attribute ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"237\" rd=\"Input_Data_Format_Dialog.htm|Input Data Format Dialog\" ct=\"The Input Data Format dialog is displayed if  MEGA  does not find enough information about the type  ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; MEGA Dialogs\"/><di ky=\"238\" rd=\"Interior_Branch_Test_of_Phylogeny.htm|Interior Branch Test of Phylogeny\" ct=\"Phylogeny | Construct/Test Neighbor-Joining Tree Or Phylogeny  | Construct/Test Minimum-Evolution Tr ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Statistical Tests of a Tree Obtained &gt; Interior Branch Tests\"/><di ky=\"239\" rd=\"Introduction.htm|Introduction\" ct=\"This walk-through provides several  brief tutorials that explain how to perform common tasks in   ME ...\" bc=\"Part I: Getting Started &gt; A Walk Through MEGA\"/><di ky=\"240\" rd=\"Introduction_to_myPEG.htm|Introduction to myPEG\" ct=\"Computational  diagnosis of amino acid variants in the human exome is the first step  in assessing t ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs\"/><di ky=\"241\" rd=\"Intron.htm|Intron\" ct=\"Introns are the non-coding segments of DNA in a  gene   that are interspersed among the  exons .     ...\" bc=\"Glossary\"/><di ky=\"242\" rd=\"Invalid_special_symbol_in_molecular_sequences.htm|Invalid special symbol in molecular sequences\" ct=\"Unique ASCII characters, except letters and &apos;*&apos;, can be used as special symbols for  alignment gaps ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"243\" rd=\"IUPAC_Single_Letter_Codes.htm|IUPAC Single Letter Codes\" ct=\"Nucleotide or amino acid  sequences should be written in IUPAC single-letter codes. The single-lette ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data\"/><di ky=\"244\" rd=\"Jones_et_al_1992.htm|Jones et al. 1992\" ct=\"Jones DT, Taylor WR &amp; Thornton JM ( 1992 ) The rapid generation of mutation data matrices from prote ...\" bc=\"References\"/><di ky=\"245\" rd=\"Jukes-Cantor_distance.htm|Jukes-Cantor distance\" ct=\"Jukes-Cantor  distance In the  Jukes  and Cantor (1969)  model, the rate of nucleotide substitution  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"246\" rd=\"Jukes-Cantor_Distance_Failed.htm|Jukes-Cantor Distance Failed\" ct=\"The Jukes-Cantor correction is used  to calculate nucleotide distances and  synonymous  and  nonsyno ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"247\" rd=\"Jukes-Cantor_Gamma_distance.htm|Jukes-Cantor Gamma distance\" ct=\"Jukes-Cantor  Gamma distance   In  the  Jukes and  Cantor (1969)  model, the rate of nucleotide subs ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"248\" rd=\"Jukes_and_Cantor_1969.htm|Jukes and Cantor 1969\" ct=\"Jukes TH &amp; Cantor CR ( 1969 ) Evolution of protein molecules. In Munro HN, editor,  Mammalian Protei ...\" bc=\"References\"/><di ky=\"249\" rd=\"Keywords_for_Command_Statements_(Genes_Domains).htm|Keywords for Command Statements (Genes/Domains)\" ct=\"	 	 	 	 	 		 Command 		 Setting 		 Remark 		 Example 	 	 		 Domain 		 A 		 name 		 This 		 instructi ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data &gt; Defining Genes and Domains\"/><di ky=\"250\" rd=\"Keywords_for_Format_Statement.htm|Keywords for Format Statement\" ct=\"	 	 	 	 	 		 Command 		 Setting 		 Remark 		 Example 	 	 		 DataType 		 Distance 		 Specifies 		 tha ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Distance Input Data\"/><di ky=\"251\" rd=\"Keywords_for_Format_Statement_(Sequence_Data).htm|Keywords for Format Statement (Sequence Data)\" ct=\"	 	 	 	 	 		 Command 		 Setting 		 Remark 		 Example 	 	 		 DataType 		 DNA, 		 RNA, nucleotide, pro ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data\"/><di ky=\"252\" rd=\"Kimura_1980.htm|Kimura 1980\" ct=\"Kimura M ( 1980 ) A simple method for estimating evolutionary rate of base substitutions through com ...\" bc=\"References\"/><di ky=\"253\" rd=\"Kimura_2-parameter_distance.htm|Kimura 2-parameter distance\" ct=\"Kimura 2-parameter  distance Kimura’s two parameter model ( 1980 )  corrects for multiple hits, taki ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"254\" rd=\"Kimura_Distance_Failed.htm|Kimura Distance Failed\" ct=\"The Kimura (1980) distance correction is  used in a number of operations, including calculating nucl ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"255\" rd=\"Kimura_gamma_distance.htm|Kimura gamma distance\" ct=\"Kimura gamma  distance Kimura’s  two-parameter gamma model corrects for multiple hits, taking into a ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"256\" rd=\"Kishino_and_Hasegawa_1989.htm|Kishino and Hasegawa 1989\" ct=\"Kishino H &amp; Hasegawa M ( 1989 ) Evaluation of the maximum likelihood estimate of the evolutionary tr ...\" bc=\"References\"/><di ky=\"257\" rd=\"Kumar_and_Gadagkar_2001.htm|Kumar and Gadagkar 2001\" ct=\"Kumar S &amp; Gadagkar SR ( 2001 )  Disparity Index : A simple statistic to measure and test the homogen ...\" bc=\"References\"/><di ky=\"258\" rd=\"Kumar_et_al_1993.htm|Kumar et al. 1993\" ct=\"Kumar S, Tamura K &amp; Nei M ( 1993 )   MEGA: Molecular Evolutionary Genetics Analysis . Pennsylvania S ...\" bc=\"References\"/><di ky=\"259\" rd=\"Kumar_et_al_2012.htm|Kumar et al. 2012\" ct=\"            Kumar S, Sanderford M, Gray, VE, Ye J, Liu Li.  (2012)  Evolutionary diagnosis method fo ...\" bc=\"References\"/><di ky=\"260\" rd=\"Kumar_et_al_EvoD_2012.htm|Kumar et al. EvoD 2012\" ct=\"Kumar S, Sanderford M, Gray VE, Ye J, Liu Li.      Evolutionary diagnosis method for variants in per ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; References\"/><di ky=\"261\" rd=\"Kumar_Method.htm|Kumar Method\" ct=\"  This  method is a modification of the  Pamilo-Bianchi-Li   and  Comeron   (1995) methods and is ab ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Synonymous and Nonsynonymous Substitution Models\"/><di ky=\"262\" rd=\"Labeled_Sites.htm|Labeled Sites\" ct=\"Sites in a sequence alignment can be categorized and  labeled  with user-defined symbols.  Each cate ...\" bc=\"Glossary\"/><di ky=\"263\" rd=\"Labels_tab_in_Options_dialog_box.htm|Labels tab (in Options dialog box)\" ct=\"Labels tab  (in Options dialog box) This tab has options for  the following: Display  Taxon Names:   ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"264\" rd=\"Lake_1987.htm|Lake 1987\" ct=\"Lake JA ( 1987 ) A rate-independent technique for analysis of nucleic acid sequences: Evolutionary p ...\" bc=\"References\"/><di ky=\"265\" rd=\"Large_Sample_Tests_of_Selection.htm|Large Sample Tests of Selection\" ct=\"One way to test whether  positive selection  is operating on a   gene  is to compare the  relative a ...\" bc=\"Part IV: Evolutionary Analysis &gt; Tests of Selection &gt; Synonymous/Nonsynonymous Tests\"/><di ky=\"266\" rd=\"Le_and_Gascuel_2008.htm|Le and Gascuel 2008\" ct=\"            Le S.Q, Gascuel O.  (2008) . An Improved General Amino Acid Replacement Matrix.  Mol Bio ...\" bc=\"References\"/><di ky=\"267\" rd=\"Li-Wu-Luo_Method.htm|Li-Wu-Luo Method\" ct=\"Li-Wu-Luo  Method In  this method ( Li et al 1985 ),  each site in a  codon   is allocated to  0-fol ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Synonymous and Nonsynonymous Substitution Models\"/><di ky=\"268\" rd=\"Li_1993.htm|Li 1993\" ct=\"Li W-H ( 1993 ) Unbiased estimation of the rates of synonymous and nonsynonymous substitution.  Jour ...\" bc=\"References\"/><di ky=\"269\" rd=\"Li_1997.htm|Li 1997\" ct=\"Li W-H ( 1997 )  Molecular Evolution . Sunderland, MA: Sinauer Associates. ...\" bc=\"References\"/><di ky=\"270\" rd=\"Li_et_al_1985.htm|Li et al. 1985\" ct=\"Li W-H, Wu C-I &amp; Luo C-C ( 1985 ) A new method for estimating synonymous and nonsynonymous rates of  ...\" bc=\"References\"/><di ky=\"271\" rd=\"Liu_et_al._2016.htm|Liu et al. 2016\" ct=\"Liu L, Tamura K, Sanderford MD, Gray V, Kumar S. 2016. A molecular evolutionary  reference or the hu ...\" bc=\"References\"/><di ky=\"272\" rd=\"LogDet_Distance_Could_Not_Be_Computed.htm|LogDet Distance Could Not Be Computed\" ct=\"The formula used for calculating distances  contains many log terms. If some of their arguments appr ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"273\" rd=\"Maddison_and_Maddison_1992.htm|Maddison and Maddison 1992\" ct=\"Maddison WP &amp; Maddison DR ( 1992 ) MacClade: Analysis of phylogeny and character evolution. Version  ...\" bc=\"References\"/><di ky=\"274\" rd=\"Main_MEGA_Window.htm|Main MEGA Window\" ct=\"The main window in  MEGA  contains  a menu bar, a main toolbar (just beneath the menu bar), a second ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus\"/><di ky=\"275\" rd=\"Max-mini_branch-and-bound_search.htm|Max-mini branch-and-bound search\" ct=\"This is an algorithm for searching for the  MP   tree using the  branch-and  bound   search method.  ...\" bc=\"Glossary\"/><di ky=\"276\" rd=\"Maximum_Composite_Likelihood.htm|Maximum Composite Likelihood\" ct=\"In general, a composite likelihood is defined as a sum of log-likelihoods  for related estimates. In ...\" bc=\"Glossary\"/><di ky=\"277\" rd=\"Maximum_Composite_Likelihood_Gamma_Rates.htm|Maximum Composite Likelihood (Gamma Rates)\" ct=\"Maximum  Composite Likelihood (Gamma Rates) The  Tamura-Nei (1993)  distance with the gamma model es ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/></ck>";
