{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"0\" rd=\"ABI_File_Format.htm|ABI File Format\" ct=\"The ABI File Format is a binary file that is produced by ABI sequencer software. This data file, ref ...\" bc=\"Glossary\"/><di ky=\"1\" rd=\"About.htm|About\" ct=\"Help | About… This command will display the  About  dialog box showing the copyright, authors, and v ...\" bc=\"\"/><di ky=\"2\" rd=\"Acknowledgements.htm|Acknowledgements\" ct=\"Many  friends and colleagues have provided encouragement and assistance in the  development of   MEG ...\" bc=\"\"/><di ky=\"3\" rd=\"Adding_Modifying_Genetic_Code_Tables.htm|Adding/Modifying Genetic Code Tables\" ct=\"You may add new genetic code  tables and/or edit existing code tables in the   Genetic  Code Selecto ...\" bc=\"Part III: Input Data Types and File Formats &gt; Genetic Code Tables\"/><di ky=\"4\" rd=\"Adzhubei_et_al_2010.htm|Adzhubei et al. 2010\" ct=\"Adzhubei IA, Schmidt S, Peshkin L, Ramensky V, Gerasimova A, Bork P, Kondrashov A, Sunyaev S.  A met ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; References\"/><di ky=\"5\" rd=\"Alignment_Explorer.htm|Alignment Explorer\" ct=\"Alignment  Explorer The  Alignment Explorer   provides options to (1) view and manually edit alignme ...\" bc=\"\"/><di ky=\"6\" rd=\"Alignment_Gaps.htm|Alignment Gaps\" ct=\"Phylogenetic analysis on two or more DNA or amino acid sequences requires  that the sequences be ali ...\" bc=\"Glossary\"/><di ky=\"7\" rd=\"Alignment_Gaps_and_Sites_with_Missing_Information.htm|Alignment Gaps and Sites with Missing Information\" ct=\"In  MEGA , gap sites are ignored in the MP analysis, but there are three different ways to treat the ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Parsimony (MP) Method\"/><di ky=\"8\" rd=\"Alignment_Menu.htm|Alignment Menu\" ct=\"Align Menu This menu provides access to options for viewing and building DNA and protein sequence al ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Align Menu\"/><di ky=\"9\" rd=\"Alignment_Menu_in_Alignment_Explorer.htm|Alignment Menu (in Alignment Explorer)\" ct=\"Alignment Menu (in Alignment  Explorer) This menu provides access  to commands for editing the seque ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Alignment Explorer &gt; Menu Items\"/><di ky=\"10\" rd=\"Alignment_session.htm|Alignment session\" ct=\"When working in MEGA’s Alignment Explorer you can choose to save the current state of all data and s ...\" bc=\"Glossary\"/><di ky=\"11\" rd=\"Amino_Acid.htm|Amino Acid\" ct=\"Amino acid sequences are compared residue-by-residue.  These distances can be computed for protein s ...\" bc=\"\"/><di ky=\"12\" rd=\"Amino_Acid_Composition.htm|Amino Acid Composition\" ct=\"Statistics | Amino acid Composition This command is visible only if the data consists  of amino acid ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"13\" rd=\"Analysis_Options.htm|Analysis Options\" ct=\"In this dialog box you can select and view desired options in the  Options Summary .  Options are or ...\" bc=\"\"/><di ky=\"14\" rd=\"Analysis_Preferences_Distance_Computation.htm|Analysis Preferences (Distance Computation)\" ct=\"In this dialog box you can select and  view the desired options in the  Options Summary .  Options   ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Specifying Distance Estimation Options\"/><di ky=\"15\" rd=\"Analysis_Preferences_Estimate_Selection_for_each_Codon.htm|Analysis Preferences (Estimate Selection for each Codon)\" ct=\"Analysis Preferences (Estimate Selection for each  Codon ) In this dialog box, you can view and sele ...\" bc=\"\"/><di ky=\"16\" rd=\"Analysis_Preferences_Fisher_s_Exact_Test.htm|Analysis Preferences (Fisher&apos;s Exact Test)\" ct=\"When the numbers of codons or the total  numbers of synonymous and/or nonsynonymous substitutions ar ...\" bc=\"Part IV: Evolutionary Analysis &gt; Tests of Selection &gt; Synonymous/Nonsynonymous Tests\"/><di ky=\"17\" rd=\"Analysis_Preferences_Maximum_Likelihood.htm|Analysis Preferences (Maximum Likelihood)\" ct=\"Analysis Preferences (ML) In this dialog box, you can view and  select desired settings for options  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Likelihood (ML) Methods\"/><di ky=\"18\" rd=\"Analysis_Preferences_Maximum_Parsimony.htm|Analysis Preferences (Maximum Parsimony)\" ct=\"This dialog box contains a set of analysis  options for use in Maximum Parsimony analysis.  Informat ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Parsimony (MP) Method\"/><di ky=\"19\" rd=\"Analysis_Preferences_Minimum_Evolution.htm|Analysis Preferences (Minimum Evolution)\" ct=\"In this dialog box you can select and  view desired options in the  Options Summary .  Options  are  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Minimum Evolution Method\"/><di ky=\"20\" rd=\"Analysis_Preferences_NJ_UPGMA.htm|Analysis Preferences (NJ/UPGMA)\" ct=\"In this dialog box, you can view and  select desired options in the  Options Summary .  Options  are ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; NJ/UPGMA Methods\"/><di ky=\"21\" rd=\"Analysis_Preferences_Pattern_Homogeneity_Analysis.htm|Analysis Preferences (Pattern Homogeneity Analysis)\" ct=\"In this dialog box, you can select and view  options in the  Options Summary  for the different patt ...\" bc=\"Part IV: Evolutionary Analysis &gt; Tests of Selection &gt; Synonymous/Nonsynonymous Tests\"/><di ky=\"22\" rd=\"Analysis_Preferences_Z-test_of_Selection.htm|Analysis Preferences (Z-test of Selection)\" ct=\"In this dialog box, you can view and  select options in the  Options Summary .  Options are organize ...\" bc=\"Part IV: Evolutionary Analysis &gt; Tests of Selection &gt; Synonymous/Nonsynonymous Tests\"/><di ky=\"23\" rd=\"Analyze_and_Prototype_Modes.htm|Analyze and Prototype Modes\" ct=\"The MEGA graphical user interface (GUI) can be run in two different  modes depending on what is need ...\" bc=\"Part I: Getting Started &gt; Features and Support\"/><di ky=\"24\" rd=\"Analyze_User_Tree_by_Least_Squares.htm|Analyze User Tree by Least Squares\" ct=\"User Tree Computation   |  Analyze User Tree by Least Squares   This option estimates the  branch  l ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; User Tree Menu\"/><di ky=\"25\" rd=\"Analyze_User_Tree_by_Maximum_Likelihood.htm|Analyze User Tree by Maximum Likelihood\" ct=\"User Tree Computation | Analyzer User Tree by Maximum Likelihood   This option estimates the  branch ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; User Tree Menu\"/><di ky=\"26\" rd=\"Analyze_User_Tree_by_Parsimony.htm|Analyze User Tree by Parsimony\" ct=\"User Tree Computation | Analyze User Tree by Parsimony   This will test the tree which you provide,  ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; User Tree Menu\"/><di ky=\"27\" rd=\"Autocorrelated_Rates_Test.htm|Autocorrelated Rates Test\" ct=\"When using Bayesian methods for estimating times of divergence in a  phylogenetic tree, the selectio ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Time Trees\"/><di ky=\"28\" rd=\"Average_Menu_in_Distance_Data_Explorer.htm|Average Menu (in Distance Data Explorer)\" ct=\"Average Menu (in Distance  Data Explorer) This menu is used for the computation of average  values u ...\" bc=\"Part III: Input Data Types and File Formats &gt; Distance Data Explorer\"/><di ky=\"29\" rd=\"Average_Menu_in_Distance_Matrix_Explorer.htm|Average Menu (in Distance Matrix Explorer)\" ct=\"Average  Menu (in Distance Matrix Explorer) With this menu,  you can compute the following average v ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Distance Matrix Explorer\"/><di ky=\"30\" rd=\"Bifurcating_Tree.htm|Bifurcating Tree\" ct=\"A bifurcating tree is one in which each ancestral lineage gives rise to exactly two descendent linea ...\" bc=\"Glossary\"/><di ky=\"31\" rd=\"Blank_Names_Are_Not_Permitted.htm|Blank Names Are Not Permitted\" ct=\"As this error message suggests, you  cannot leave the name of a sequence,  taxa ,  domain ,  or   ge ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"32\" rd=\"Bootstrap_method_to_compute_standard_error_of_distance_estimates.htm|Bootstrap method to compute standard error of distance estimates\" ct=\"Bootstrap  method to compute standard error of distance estimates When you choose the bootstrap meth ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Specifying Distance Estimation Options\"/><di ky=\"33\" rd=\"Bootstrap_Test_of_Phylogeny.htm|Bootstrap Test of Phylogeny\" ct=\"Phylogeny | Construct/Test Neighbor-Joining Tree Or Phylogeny | Construct/Test Minimum-Evolution Tre ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Statistical Tests of a Tree Obtained &gt; Bootstrap Tests\"/><di ky=\"34\" rd=\"Branch-and-Bound_algorithm.htm|Branch-and-Bound algorithm\" ct=\"The branch-and-bound algorithm is used to find all the  MP  trees.  It  guarantees to find all the M ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Parsimony (MP) Method\"/><di ky=\"35\" rd=\"Branch.htm|Branch\" ct=\"A branch is a line connecting either two internal nodes to each other or an external  node  to an in ...\" bc=\"Glossary\"/><di ky=\"36\" rd=\"Branch_Swap_Filter.htm|Branch Swap Filter\" ct=\"For analyses that use a Maximum Likelihood approach, the ‘ branch   swap  filter’ analysis setting i ...\" bc=\"\"/><di ky=\"37\" rd=\"Branch_tab_in_Options_dialog_box.htm|Branch tab (in Options dialog box)\" ct=\"Branch tab  (in Options dialog box) This tab has  options for the following aspects of the tree: Lin ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"38\" rd=\"Built-in_Genetic_Codes.htm|Built-in Genetic Codes\" ct=\"MEGA   contains  four commonly used genetic code tables: (1) Standard, (2) Vertebrate mitochondrial, ...\" bc=\"Part III: Input Data Types and File Formats &gt; Genetic Code Tables\"/></ck>";
