{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"278\" rd=\"Maximum_Composite_Likelihood_Gamma_Rates_and_Heterogeneous_Patterns.htm|Maximum Composite Likelihood (Gamma Rates and Heterogeneous Patterns)\" ct=\"Maximum  Composite Likelihood (Gamma Rates and Heterogeneous Patterns) The  Tamura-Nei (1993)  dista ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns &gt; Gamma Rates\"/><di ky=\"279\" rd=\"Maximum_Composite_Likelihood_Heterogeneous_Patterns.htm|Maximum Composite Likelihood (Heterogeneous Patterns)\" ct=\"Maximum  Composite Likelihood (Heterogeneous Patterns) The  Tamura-Nei distance (1993)  estimated by ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns\"/><di ky=\"280\" rd=\"Maximum_Composite_Likelihood_Method.htm|Maximum Composite Likelihood Method\" ct=\"Maximum  Composite Likelihood Method A composite likelihood is defined as a sum  of related log-like ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"281\" rd=\"Maximum_Likelihood_ML.htm|Maximum Likelihood (ML)\" ct=\"Phylogeny   |  Construct/Test Maximum Likelihood Tree (ML)   A phylogenetic tree is constructed for  ...\" bc=\"\"/><di ky=\"282\" rd=\"Maximum_Parsimony_Construct_Phylogeny.htm|Maximum Parsimony (Construct Phylogeny)\" ct=\"Phylogeny | Construct/Test Maximum Parsimony… This command is used to construct phylogenetic trees u ...\" bc=\"\"/><di ky=\"283\" rd=\"Maximum_Parsimony_MP_Method.htm|Maximum Parsimony (MP) Method\" ct=\"Maximum parsimony RH_Maximum_Parsimony_Principle  (MP) methods originally were developed for morphol ...\" bc=\"\"/><di ky=\"284\" rd=\"Maximum_Parsimony_Principle.htm|Maximum Parsimony Principle\" ct=\"For any given  topology , the sum of the minimum possible substitutions over all sites is known as t ...\" bc=\"Glossary\"/><di ky=\"285\" rd=\"MEGA-CC.htm|MEGA-CC\" ct=\"MEGA-CC is the compute  core that is used by MEGA and is installed as a command-line application.  T ...\" bc=\"\"/><di ky=\"286\" rd=\"MEGA-CC_Overview.htm|MEGA-CC Overview\" ct=\"Description MEGA-CC is the command-line interface for using MEGA and it is included  with the graphi ...\" bc=\"Part I: Getting Started &gt; Using the Command-line Interface (MEGA-CC)\"/><di ky=\"287\" rd=\"MEGA-MDW_Server.htm|MEGA-MDW Server\" ct=\"All EvoD,  PolyPhen-2, and SIFT predictions are pre-computed and stored on the MEGA-MDW  web server. ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs\"/><di ky=\"288\" rd=\"MEGA_Analysis_Options_File.htm|MEGA Analysis Options File\" ct=\"The MEGA Analysis Options (.mao) file is used to specify analysis options  that are needed when usin ...\" bc=\"Glossary\"/><di ky=\"289\" rd=\"MEGA_Software_Development_Team.htm|MEGA Software Development Team\" ct=\"Project   Director  Sudhir  Kumar Application   Programmers  Koichiro  Tamura and Glen Stecher Websi ...\" bc=\"\"/><di ky=\"290\" rd=\"Menus_in_MEGA_are_so_short_where_are_all_the_options.htm|Menus in MEGA are so short; where are all the options?\" ct=\"Menus in MEGA  are so short; where are all the options? Our aim in developing  the objectively drive ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"291\" rd=\"Mid-point_rooting.htm|Mid-point rooting\" ct=\"In the mid-point rooting method, the root of an  unrooted tree  is placed at the mid-point of the lo ...\" bc=\"Glossary\"/><di ky=\"292\" rd=\"Min-mini_algorithm.htm|Min-mini algorithm\" ct=\"This is a heuristic search algorithm for finding the  MP   tree, and is somewhat similar to the  bra ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Parsimony (MP) Method &gt; Heuristic Search\"/><di ky=\"293\" rd=\"Minimum_Evolution.htm|Minimum Evolution\" ct=\"In the ME method, distance measures that correct for multiple hits at  the same sites are used, and  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Minimum Evolution Method\"/><di ky=\"294\" rd=\"Minimum_Evolution_Construct_Phylogeny.htm|Minimum Evolution (Construct Phylogeny)\" ct=\"Phylogeny | Construct/Test Minimum Evolution Tree… This command is used to construct a phylogenetic  ...\" bc=\"\"/><di ky=\"295\" rd=\"Missing_data_or_invalid_distances_in_the_matrix.htm|Missing data or invalid distances in the matrix\" ct=\"Missing data or invalid distances  in the matrix The selected set of  taxa   contains one or more pa ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"296\" rd=\"Miura_et_al_2018.htm|Miura et al 2018\" ct=\"Miura S, Tamura K, Tao Q, Huuki LA, Pond SLK, Priest J, Deng J, Kumar  S. 2018. A new method for inf ...\" bc=\"References\"/><di ky=\"297\" rd=\"Models_for_estimating_distances.htm|Models for estimating distances\" ct=\"Models  for estimating distances The evolutionary distance between  a pair of sequences usually is m ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models\"/><di ky=\"298\" rd=\"Modified_Nei-Gojobori_Method.htm|Modified Nei-Gojobori Method\" ct=\"Modified  Nei-Gojobori Method The  modified Nei-Gojobori distance differs from the original Nei-Gojo ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Synonymous and Nonsynonymous Substitution Models\"/><di ky=\"299\" rd=\"Molecular_Clock_Test_ML.htm|Molecular Clock Test (ML)\" ct=\"Clocks | Test Molecular Clock(ML)   This option performs a Maximum Likelihood test of the molecular  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Molecular Clock Tests\"/><di ky=\"300\" rd=\"Monophyletic.htm|Monophyletic\" ct=\"A cluster of  taxa  that shared a common ancestor comparatively recently in the evolutionary history ...\" bc=\"Glossary\"/><di ky=\"301\" rd=\"mRNA.htm|mRNA\" ct=\"Protein-coding genes are first  transcribed  into messenger RNAs (mRNA), which are, in turn,  transl ...\" bc=\"Glossary\"/><di ky=\"302\" rd=\"Mutation_Detail_View.htm|Mutation Detail View\" ct=\"The Mutation  Detail View window displays all available information for the currently  active record ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; MEGA-MD Windows\"/><di ky=\"303\" rd=\"Mutation_Explorer.htm|Mutation Explorer\" ct=\"The  Mutation  Explorer  window displays predictions and data associated with the  nsSNVs being expl ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; MEGA-MD Windows\"/><di ky=\"304\" rd=\"NCBI.htm|NCBI\" ct=\"An acronym that stands for “National Center for Biotechnology Information”. NCBI is a federally fund ...\" bc=\"Glossary\"/><di ky=\"305\" rd=\"Nearest-Neighbor-Interchange_NNI.htm|Nearest-Neighbor-Interchange (NNI)\" ct=\"The Nearest-Neighbor-Interchange is a heuristic to improve the likelihood of a tree by performing th ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Likelihood (ML) Methods &gt; Heuristic Search\"/><di ky=\"306\" rd=\"Nei-Gojobori_Method.htm|Nei-Gojobori Method\" ct=\"Nei-Gojobori  Method This  method computes the numbers of  synonymous   and  nonsynonymous   substit ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Synonymous and Nonsynonymous Substitution Models\"/><di ky=\"307\" rd=\"Nei_1986.htm|Nei 1986\" ct=\"Nei M ( 1986 )  Stochastic errors in DNA evolution and molecular phylogeny. In Gershowitz H, Rucknag ...\" bc=\"References\"/><di ky=\"308\" rd=\"Nei_1991.htm|Nei 1991\" ct=\"Nei M ( 1991 ) Relative efficiencies of different tree making methods for molecular data. In Miyamot ...\" bc=\"References\"/><di ky=\"309\" rd=\"Nei_and_Gojobori_1986.htm|Nei and Gojobori 1986\" ct=\"Nei M &amp; Gojobori T ( 1986 ) Simple methods for estimating the numbers of synonymous and nonsynonymou ...\" bc=\"References\"/><di ky=\"310\" rd=\"Nei_and_Jin_1989.htm|Nei and Jin 1989\" ct=\"Nei M &amp; Jin L ( 1989 ) Variances of the average numbers of nucleotide substitutions within and betwe ...\" bc=\"References\"/><di ky=\"311\" rd=\"Nei_and_Kumar_2000.htm|Nei and Kumar 2000\" ct=\"Nei M &amp; Kumar S ( 2000 )   Molecular Evolution and Phylogenetics . Oxford University Press, New York ...\" bc=\"References\"/><di ky=\"312\" rd=\"Nei_et_al_1976.htm|Nei et al. 1976\" ct=\"Nei M, Chakraborty R &amp; Fuerst PA ( 1976 ) Infinite allele model with varying mutation rate.  Proceed ...\" bc=\"References\"/><di ky=\"313\" rd=\"Nei_et_al_1985.htm|Nei et al. 1985\" ct=\"Nei M, Stephens JC &amp; Saitou N ( 1985 ) Methods for computing the standard errors of branching points ...\" bc=\"References\"/><di ky=\"314\" rd=\"Nei_et_al_1998.htm|Nei et al. 1998\" ct=\"Nei M, Kumar S &amp; Takahashi ( 1998 ) The optimization principle in phylogenetic analysis tends to giv ...\" bc=\"References\"/><di ky=\"315\" rd=\"Neighbor-Joining_NJ_Method.htm|Neighbor-Joining (NJ) Method\" ct=\"This method ( Saitou and Nei 1987 Saitou_and_Nei_1987 ) is a simplified version of the minimum evolu ...\" bc=\"\"/><di ky=\"316\" rd=\"Neighbor_Joining_Construct_Phylogeny.htm|Neighbor Joining (Construct Phylogeny)\" ct=\"Phylogeny | Construct/Test Neighbor-Joining Tree… This command is used to construct (or  Test) a nei ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Statistical Tests of a Tree Obtained &gt; Interior Branch Tests\"/><di ky=\"317\" rd=\"New_in_Text_Editor.htm|New (in Text Editor)\" ct=\"File | New Use this command to create a new file in the   Text Editor .     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"318\" rd=\"Newick_Format.htm|Newick Format\" ct=\"NEWICK is a simple format used to write out trees in a text file.  While  this is a hard-to-read for ...\" bc=\"Glossary\"/><di ky=\"319\" rd=\"No_Common_Sites.htm|No Common Sites\" ct=\"For the sequences and data subset options selected,  MEGA  found zero common sites.  If you selected ...\" bc=\"Appendix C: Error Messages\"/></ck>";
