{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"156\" rd=\"Felsenstein_1986.htm|Felsenstein 1986\" ct=\"Felsenstein J ( 1986 ) Distance Methods: Reply to Farris.  Cladistics   2 :130-143. ...\" bc=\"References\"/><di ky=\"157\" rd=\"Felsenstein_1988.htm|Felsenstein 1988\" ct=\"Felsenstein J ( 1988 ) Phylogenies from molecular sequences: Inference and reliability.  Annual Revi ...\" bc=\"References\"/><di ky=\"158\" rd=\"Felsenstein_1993.htm|Felsenstein 1993\" ct=\"Felsenstein J ( 1993 ) Phylogeny Inference Package (PHYLIP). Version 3.5. University of Washington,  ...\" bc=\"References\"/><di ky=\"159\" rd=\"Felsenstein_and_Kishino_1993.htm|Felsenstein and Kishino 1993\" ct=\"Felsenstein J &amp; Kishino H ( 1993 ) Is there something wrong with the bootstrap on phylogenies? A rep ...\" bc=\"References\"/><di ky=\"160\" rd=\"File_Menu.htm|File Menu\" ct=\"File Menu This allows you to perform various important tasks, including activating a data file, edit ...\" bc=\"\"/><di ky=\"161\" rd=\"File_Menu_in_Distance_Data_Explorer.htm|File Menu (in Distance Data Explorer)\" ct=\"File Menu (in Distance Data  Explorer) The  File  menu consists of three commands: Select  &amp; Edit  T ...\" bc=\"Part III: Input Data Types and File Formats &gt; Distance Data Explorer\"/><di ky=\"162\" rd=\"File_Menu_in_Distance_Matrix_Explorer.htm|File Menu (in Distance Matrix Explorer)\" ct=\"File Menu (in  Distance Matrix Explorer) The file menu  consists of three commands: Show Input  Data ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Distance Matrix Explorer\"/><di ky=\"163\" rd=\"Filipski_et_al_2014.htm|Filipski et al. 2014\" ct=\"Filipski A, Murillo O, Freydenzon A, Tamura K and Kumar S ( 2014 ) Prospects for building large time ...\" bc=\"References\"/><di ky=\"164\" rd=\"Find_Again_in_Text_Editor.htm|Find Again (in Text Editor)\" ct=\"Search | Find Again Choose this to repeat the last  Find   command.     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; Search Menu\"/><di ky=\"165\" rd=\"Find_Best_DNA_Protein_Models_ML.htm|Find Best DNA/Protein Models (ML)\" ct=\"Models | Find Best DNA/Protein Models (ML)   This option tests a data file (nucleotide or amino acid ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Models Menu\"/><di ky=\"166\" rd=\"Find_in_Text_Editor.htm|Find (in Text Editor)\" ct=\"Search | Find Choose this command to display the  Find  Text  dialog box.     ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; Search Menu\"/><di ky=\"167\" rd=\"Find_Motif.htm|Find Motif\" ct=\"You may specify a Motif to search for in the  sequence data.  This Motif supports IUPAC codes such a ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"168\" rd=\"Find_Next_Motif.htm|Find Next Motif\" ct=\"If you have already searched for a motif, this  will find the next instance of the search term in re ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"169\" rd=\"Find_Next_Name.htm|Find Next Name\" ct=\"If you have already searched for a sequence  name, this will find the next instance of the search te ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"170\" rd=\"Find_Prev_Motif.htm|Find Prev Motif\" ct=\"If you have already searched for a motif, this  will find the previous instance of the search term i ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"171\" rd=\"Find_Prev_Name.htm|Find Prev Name\" ct=\"If you have already searched for a sequence  name, this will find the previous instance of the searc ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"172\" rd=\"Find_Sequence_Name.htm|Find Sequence Name\" ct=\"This allows you to specify a sequence name  to find.  Search results are bolded and the row is highl ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Search Menu\"/><di ky=\"173\" rd=\"Finding_the_number_of_sites_in_pairwise_comparisons.htm|Finding the number of sites in pairwise comparisons\" ct=\"Finding  the number of sites in pairwise comparisons If you want to find  the number of sites betwee ...\" bc=\"Appendix A: Frequently Asked Questions\"/><di ky=\"174\" rd=\"First_Time_User.htm|First Time User\" ct=\"Thank  you for choosing to use   MEGA   in your research.   This manual provides comprehensive docum ...\" bc=\"Part I: Getting Started\"/><di ky=\"175\" rd=\"Fisher_s_Exact_Test_Has_Failed.htm|Fisher&apos;s Exact Test Has Failed\" ct=\"Fisher&apos;s exact test uses estimates  of the number of  synonymous   sites (S), the number of  nonsyno ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"176\" rd=\"Fitch_1971.htm|Fitch 1971\" ct=\"Fitch WM ( 1971 ) Towards defining the course of evolution: Minimum change for a specific tree  topo ...\" bc=\"References\"/><di ky=\"177\" rd=\"Fitch_and_Margoliash_1967.htm|Fitch and Margoliash 1967\" ct=\"Fitch WM &amp; Margoliash E ( 1967 ) Construction of phylogenetic trees.  Science   155 :279 284. ...\" bc=\"References\"/><di ky=\"178\" rd=\"Flip.htm|Flip\" ct=\"This command changes the cursor to the &apos;Flip&apos; icon. Then, if you click  on an interior  branch ,   M ...\" bc=\"Glossary\"/><di ky=\"179\" rd=\"Font_in_Text_Editor.htm|Font (in Text Editor)\" ct=\"Display | Set Font Choose this command to activate a dialog box  with which you can change the displ ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; Edit Menu\"/><di ky=\"180\" rd=\"Format_command.htm|Format command\" ct=\"A format command in a data file begins with! Format and contains at least the data type included in  ...\" bc=\"Glossary\"/><di ky=\"181\" rd=\"Format_Selected_Sequence.htm|Format Selected Sequence\" ct=\"Utilities | Format Selected Sequence This submenu presents four other menu items that offer some com ...\" bc=\"\"/><di ky=\"182\" rd=\"Gamma_distance_Amino_acids.htm|Gamma distance (Amino acids)\" ct=\"Gamma distance  (Amino acids)   The  Gamma distance improves upon the Poisson correction distance by ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Amino Acid Substitution Models &gt; Gamma Distances\"/><di ky=\"183\" rd=\"Gamma_Distance_Failed_Because_p_0_99.htm|Gamma Distance Failed Because p &gt; 0.99\" ct=\"Gamma Distance Failed Because p &gt;  0.99 For amino acid distance estimation, if the  proportion of am ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"184\" rd=\"Gamma_parameter.htm|Gamma parameter\" ct=\"According to the gamma distribution, the substitution rate often varies from site to site within a s ...\" bc=\"Glossary\"/><di ky=\"185\" rd=\"Gaps_In_Distance_Estimation.htm|Gaps In Distance Estimation\" ct=\"Gaps often are inserted  during the alignment of homologous regions of sequences and represent  dele ...\" bc=\"Part IV: Evolutionary Analysis &gt; Handling Missing Data and Alignment Gaps\"/><di ky=\"186\" rd=\"Gene.htm|Gene\" ct=\"A gene is a collection of  domains .  The  domains  included  in a gene need not be consecutive or o ...\" bc=\"Glossary\"/><di ky=\"187\" rd=\"Gene_Duplication_Inference.htm|Gene Duplication Inference\" ct=\"  Gene  Duplication Wizard MEGA7 introduces a new wizard-style system for identifying  gene  duplica ...\" bc=\"Part IV: Evolutionary Analysis &gt; Identifying Gene Duplications\"/><di ky=\"188\" rd=\"Gene_Duplication_Wizard.htm|Gene Duplication Wizard\" ct=\"The Gene Duplication Wizard guides users through the steps required for the Gene Duplication inferen ...\" bc=\"\"/><di ky=\"189\" rd=\"Gene_Names_Must_Be_Unique.htm|Gene Names Must Be Unique\" ct=\"MEGA   requires that all  gene RH_Gene  names in a genome be unique, although, for convenience, many ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"190\" rd=\"Gene_Search_Tab.htm|Gene Search Tab\" ct=\"The  Gene  Search  tab facilitates  searching for genes by keyword (based on  gene  product) or alte ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; MEGA-MD Windows\"/><di ky=\"191\" rd=\"General_Comments_on_Statistical_Tests.htm|General Comments on Statistical Tests\" ct=\"There are two different types of methods for testing the reliability  of an obtained tree. One is to ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Statistical Tests of a Tree Obtained\"/><di ky=\"192\" rd=\"General_Considerations.htm|General Considerations\" ct=\"The sequence data must consist of two or more sequences of  equal length. All sequences must be alig ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Sequence Input Data\"/><di ky=\"193\" rd=\"General_Considerations_(Distance_Data_Format).htm|General Considerations (Distance Data Format)\" ct=\"For a set of   m   sequences (or   taxa ), there are   m ( m -1)/2 pairwise  distances. These distan ...\" bc=\"Part III: Input Data Types and File Formats &gt; MEGA Input Data Formats &gt; Distance Input Data\"/><di ky=\"194\" rd=\"General_Description.htm|General Description\" ct=\"Data | Export Data The  Exporting Sequence Data  dialog box first  displays an edit box for entering ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Data Menu\"/><di ky=\"195\" rd=\"Genetic_Code_Statistical_Attributes.htm|Genetic Code Statistical Attributes\" ct=\"There is a significant amount of redundancy in the genetic code because  most amino acids are encode ...\" bc=\"Part III: Input Data Types and File Formats &gt; Genetic Code Tables\"/><di ky=\"196\" rd=\"Genetic_Codes.htm|Genetic Codes\" ct=\"A genetic code table specifies the amino acid residues encoded by the  various  codons .  Vertebrate ...\" bc=\"Glossary\"/></ck>";
