{
	Copyright 1992-2024 Sudhir Kumar and Koichiro Tamura

	This file is part of the MEGA (Molecular Evolutionary Genetics Analyis) software.

	MEGA (Molecular Evolutionary Genetics Analysis) is free software:
	you can redistribute it and/or modify it under the terms of the
	GNU General Public License as published by the Free Software
	Foundation, either version 3 of the License, or (at your option)
	any later version.

	This program is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with this program.  If not, see <https://www.gnu.org/licenses/>.

   Contributor(s):   The MEGA source code and software is made available in the hopes that it will be useful. 
   In keeping with the spirit and intent that the MEGA project is developed under, the authors of MEGA request that before
   distributing any significant changes to the MEGA source code (or derivatives thereof), you share
   those changes with the MEGA authors so that they may have the opportunity to test that
   the changes do not introduce errors into the code or otherwise negatively impact the correctness
   or performance of the MEGA software.
   
	Please email inqiries to s.kumar@temple.edu
}

gXMLBuffer="<?xml version=\"1.0\" encoding=\"UTF-8\"?><ck><di ky=\"467\" rd=\"Show_Line_Numbers_in_Text_Editor.htm|Show Line Numbers (in Text Editor)\" ct=\"Show Line Numbers (in Text  Editor) Display | Show Line Numbers This item can be checked (on) or un- ...\" bc=\"Part III: Input Data Types and File Formats &gt; Text Editor &gt; Using Text File Editor &gt; File Menu\"/><di ky=\"468\" rd=\"Show_Only_Selected_Sequences.htm|Show Only Selected Sequences\" ct=\"Display | Show only Selected Sequences The check boxes in the left column of the display  grid can b ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"469\" rd=\"Show_Sequence_Names.htm|Show Sequence Names\" ct=\"Display | Show Sequence Names This option displays the full sequence names  in  Sequence  Data Explo ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"470\" rd=\"Show_Web_Browser.htm|Show Web Browser\" ct=\"Align | Show Web Browser Use this option to launch the  MEGA    Web Browser .     ...\" bc=\"Appendix B: Main Menu Items and Dialogs Reference &gt; Main MEGA Menus &gt; Align Menu\"/><di ky=\"471\" rd=\"Singleton_Sites.htm|Singleton Sites\" ct=\"A singleton site contains at least two types of nucleotides (or amino acids) with, at most, one occu ...\" bc=\"Glossary\"/><di ky=\"472\" rd=\"Site_Label.htm|Site Label\" ct=\"The individual sites in nucleotide or amino acid data can be labeled  to construct non-contiguous se ...\" bc=\"Glossary\"/><di ky=\"473\" rd=\"Sneath_and_Sokal_1973.htm|Sneath and Sokal 1973\" ct=\"Sneath PHA &amp; Sokal RR ( 1973 )  Numerical Taxonomy. Freeman , San Francisco. ...\" bc=\"References\"/><di ky=\"474\" rd=\"Sort_Sequences.htm|Sort Sequences\" ct=\"Display | Sort Sequences The sequences in the data set can be sorted  based on several options:  seq ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"475\" rd=\"Sort_Sequences_As_per_Taxa_Group_Organizer.htm|Sort Sequences As per Taxa/Group Organizer\" ct=\"Sort Sequences As per Taxa/Group  Organizer Display | Sort Sequences | As per Taxa/Group  Organizer ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"476\" rd=\"Sort_Sequences_by_Group_and_Sequence_Names.htm|Sort Sequences by Group and Sequence Names\" ct=\"Sort Sequences by Group  and Sequence Names Display | Sort Sequences | By Group and Sequence  Names ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"477\" rd=\"Sort_Sequences_by_Group_Name.htm|Sort Sequences by Group Name\" ct=\"Sort Sequences by Group  Name Display | Sort Sequences | By Group Name Sequences that have been grou ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"478\" rd=\"Sort_Sequences_By_Sequence_Name.htm|Sort Sequences By Sequence Name\" ct=\"Sort Sequences By Sequence  Name Display | Sort Sequences | By Sequence Name The sequences are sorte ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Display Menu\"/><di ky=\"479\" rd=\"Sourdis_and_Krimbas_1987.htm|Sourdis and Krimbas 1987\" ct=\"Sourdis J &amp; Krimbas C ( 1987 ) Accuracy of phylogenetic trees estimated from DNA sequence data.  Mol ...\" bc=\"References\"/><di ky=\"480\" rd=\"Sourdis_and_Nei_1988.htm|Sourdis and Nei 1988\" ct=\"Sourdis J &amp; Nei M ( 1988 ) Relative efficiencies of the maximum parsimony and distance-matrix method ...\" bc=\"References\"/><di ky=\"481\" rd=\"Specify_the_coordinate_information_using_the_Sequence_Data_Explorer.htm|Specify the coordinate information using the Sequence Data Explorer\" ct=\"Specify the coordinate information  using the Sequence Data Explorer If a 46-species sequence alignm ...\" bc=\"Part IV: Evolutionary Analysis &gt; Exploring the Impact of nsSNVs &gt; Input Data\"/><di ky=\"482\" rd=\"Staden.htm|Staden\" ct=\"The Staden file format is used to store data from DNA sequencing instruments. Each file contains the ...\" bc=\"Glossary\"/><di ky=\"483\" rd=\"Statements_in_input_files.htm|Statements in input files\" ct=\"All statements in  MEGA  files start with an exclamation mark (!) and end with a semicolon (;).  The ...\" bc=\"Glossary\"/><di ky=\"484\" rd=\"Statistics_Menu_in_Sequence_Data_Explorer.htm|Statistics Menu (in Sequence Data Explorer)\" ct=\"Statistics Menu (in Sequence  Data Explorer) Various summary statistics of the sequences  can be com ...\" bc=\"Part III: Input Data Types and File Formats &gt; Viewing And Exploring Input Data &gt; Sequence Data Explorer &gt; Statistics Menu\"/><di ky=\"485\" rd=\"Studier_and_Keppler_1988.htm|Studier and Keppler 1988\" ct=\"Studier JA &amp; Keppler KL ( 1988 ) A note on the neighbor-joining algorithm of Saitou and Nei.  Molecu ...\" bc=\"References\"/><di ky=\"486\" rd=\"Subtree-Pruning-Regrafting_SPR.htm|Subtree-Pruning-Regrafting (SPR)\" ct=\"For any tree searching method, exhaustive search, where all possible topologies are considered is un ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Maximum Likelihood (ML) Methods &gt; Heuristic Search\"/><di ky=\"487\" rd=\"Subtree_Drawing_Options_in_Tree_Explorer.htm|Subtree Drawing Options (in Tree Explorer)\" ct=\"Subtree Drawing  Options (in Tree Explorer) This dialog box provides  choices options for changing v ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"488\" rd=\"Subtree_Menu_in_Tree_Explorer.htm|Subtree Menu (in Tree Explorer)\" ct=\"Subtree Menu  (in Tree Explorer) This menu contains  the tree manipulation options  Swap ,  Flip Fli ...\" bc=\"Part V: Visualizing and Exploring Data &gt; Tree Explorer\"/><di ky=\"489\" rd=\"Swap.htm|Swap\" ct=\"This command changes the cursor to the &apos; Flip &apos;  icon.  Then, you click on an interior  branch ,   M ...\" bc=\"Glossary\"/><di ky=\"490\" rd=\"Swofford_1993.htm|Swofford 1993\" ct=\"Swofford DL ( 1993 ) Phylogenetic Analysis Using Parsimony (PAUP), Version 3.1.1. University of Illi ...\" bc=\"References\"/><di ky=\"491\" rd=\"Swofford_1998.htm|Swofford 1998\" ct=\"Swofford DL ( 1998 ) PAUP*: Phylogenetic Analysis Using Parsimony (and Other Methods) Sunderland, MA ...\" bc=\"References\"/><di ky=\"492\" rd=\"Swofford_et_al_1996.htm|Swofford et al. 1996\" ct=\"Swofford DL, Olsen GJ, Waddell PJ &amp; Hillis DM ( 1996 ). Phylogenetic Inference. In Hiillis DM, Morit ...\" bc=\"References\"/><di ky=\"493\" rd=\"Syn-Nonsynonymous.htm|Syn-Nonsynonymous\" ct=\"Sequences are compared  codon -by- codon .  These distances can only be computed for protein-coding  ...\" bc=\"\"/><di ky=\"494\" rd=\"Synonymous_change.htm|Synonymous change\" ct=\"A nucleotide change is synonymous if it does not cause the  codon RH_Codon  to code for a different  ...\" bc=\"Glossary\"/><di ky=\"495\" rd=\"Tajima-Nei_distance.htm|Tajima-Nei distance\" ct=\"Tajima-Nei  distance   In  real data, nucleotide frequencies often deviate substantially from 0.25.  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models\"/><di ky=\"496\" rd=\"Tajima-Nei_Distance_Could_Not_Be_Computed.htm|Tajima-Nei Distance Could Not Be Computed\" ct=\"For one or more pairs of sequences,  the  Tajima-Nei   correction could not be applied, which usuall ...\" bc=\"Appendix C: Error Messages\"/><di ky=\"497\" rd=\"Tajima_1983.htm|Tajima 1983\" ct=\"Tajima F ( 1983 ) Evolutionary relationship of DNA sequences in finite populations.  Genetics   105 ...\" bc=\"References\"/><di ky=\"498\" rd=\"Tajima_1989.htm|Tajima 1989\" ct=\"Tajima F ( 1989 ) Statistical methods to test for nucleotide mutation hypothesis by DNA polymorphism ...\" bc=\"References\"/><di ky=\"499\" rd=\"Tajima_1993.htm|Tajima 1993\" ct=\"Tajima F ( 1993 ) Simple methods for testing molecular clock hypothesis.  Genetics   135 :599-607. ...\" bc=\"References\"/><di ky=\"500\" rd=\"Tajima_and_Nei_1982.htm|Tajima and Nei 1982\" ct=\"Tajima F &amp; Nei M ( 1982 ) Biases of the estimates of DNA divergence obtained by the restriction enzy ...\" bc=\"References\"/><di ky=\"501\" rd=\"Tajima_and_Nei_1984.htm|Tajima and Nei 1984\" ct=\"Tajima F &amp; Nei M ( 1984 ) Estimation of evolutionary distance between nucleotide sequences.  Molecul ...\" bc=\"References\"/><di ky=\"502\" rd=\"Tajima_Nei_distance_Gamma_rates.htm|Tajima Nei distance (Gamma rates)\" ct=\"Tajima Nei  distance (Gamma rates) In  real data, nucleotide frequencies often deviate substantially ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Gamma Distances\"/><di ky=\"503\" rd=\"Tajima_Nei_Distance_Gamma_Rates_and_Heterogeneous_patterns.htm|Tajima Nei Distance (Gamma Rates and Heterogeneous patterns)\" ct=\"Tajima Nei  Distance (Gamma Rates and Heterogeneous patterns) In  real data, nucleotide frequencies  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns &gt; Gamma Rates\"/><di ky=\"504\" rd=\"Tajima_Nei_Distance_Heterogeneous_patterns.htm|Tajima Nei Distance (Heterogeneous patterns)\" ct=\"Tajima Nei  Distance (Heterogeneous patterns) In  real data, nucleotide frequencies often deviate su ...\" bc=\"Part IV: Evolutionary Analysis &gt; Computing Evolutionary Distances &gt; Distance Models &gt; Nucleotide Substitution Models &gt; Heterogeneous Patterns\"/><di ky=\"505\" rd=\"Tajima_s_Test_of_Neutrality.htm|Tajima&apos;s Test of Neutrality\" ct=\"Selection | Tajima’s Test of Neutrality This conducts Tajima’s test of neutrality  ( Tajima 1989 ),  ...\" bc=\"Part IV: Evolutionary Analysis &gt; Tests of Selection &gt; Other Tests\"/><di ky=\"506\" rd=\"Tajima_s_Test_Relative_Rate.htm|Tajima&apos;s Test (Relative Rate)\" ct=\"  Molecular Clocks | Tajima’s Relative Rate  Test Use this to conduct Tajima’s relative  rate test ( ...\" bc=\"Part IV: Evolutionary Analysis &gt; Constructing Phylogenetic Trees &gt; Molecular Clock Tests\"/></ck>";
