<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Clustal Options (Protein)</title>
<link rel="StyleSheet" href="../../../default.css" type="text/css" />
 <link rel="StyleSheet" href="../../../default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="../../../template/scripts/rh.min.js"></script>
<script type="text/javascript" src="../../../template/scripts/common.min.js"></script>
<script type="text/javascript" src="../../../template/scripts/topic.min.js"></script>
<script type="text/javascript" src="../../../template/scripts/constants.js"></script>
<script type="text/javascript" src="../../../template/scripts/utils.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhutils.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhlang.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhver.js"></script>
<script type="text/javascript" src="../../../template/scripts/settings.js"></script>
<script type="text/javascript" src="../../../template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="../../../template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="../../../template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="../../../template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="../../../template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="../../../template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = "../../..";
gCommonRootRelPath = "../../..";
gTopicId = "2.2.2.0_3";
</script>
 <meta name="topic-breadcrumbs" content="Part II: Assembling Data for Analysis > Building Sequence Alignments > ClustalW" />
</head>

<body>
<script type="text/javascript" src="../../../ehlpdhtm.js"></script>


<h1>Clustal Options (Protein)</h1>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><font style="font-size: 11pt;" 
	 color="#333333" size="2">This dialog box displays a single tab containing 
 a set of organized parameters that are used by</font> <span style="font-size: 11pt;"><a 
	 href="About_ClustalW.htm">ClustalW</a></span> <font style="font-size: 11pt;" 
														 color="#333333" 
														 size="2">to align 
 Protein sequences.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><font style="font-size: 11pt; 
	 font-family: Arial;" color="#333333" size="2" face="Arial">In this 
 dialog box, you will see the following options:</font></p>
<h5 style="font-size: 12pt; text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><i><font 
	 style="font-size: 11pt; font-family: Arial; font-weight: normal; font-style: italic;" 
	 color="#333333" size="2" face="Arial">Parameters for Pairwise Sequence 
 Alignment</font></i></h5>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Gap 
 Opening Penalty</font></b><font style="font-size: 11pt; font-family: Arial;" 
								 color="#333333" size="2" face="Arial">: 
 The penalty for opening a gap in the alignment. Increasing this value 
 makes the gaps less frequent.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Gap 
 Extension Penalty</font></b><font style="font-size: 11pt; font-family: Arial;" 
									 color="#333333" size="2" face="Arial">: 
 The penalty for extending a gap by one residue. Increasing this value 
 will make the gaps shorter. Terminal gaps are not penalized.</font></p>
<h5 style="font-size: 12pt; text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><i><font 
	 style="font-size: 11pt; font-family: Arial; font-weight: normal; font-style: italic;" 
	 color="#333333" size="2" face="Arial">Parameters for Multiple Sequence 
 Alignment</font></i></h5>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Gap 
 Opening Penalty</font></b><font style="font-size: 11pt; font-family: Arial;" 
								 color="#333333" size="2" face="Arial">: 
 The penalty for opening a gap in the alignment. Increasing this value 
 makes the gaps less frequent.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Gap 
 Extension Penalty</font></b><font style="font-size: 11pt; font-family: Arial;" 
									 color="#333333" size="2" face="Arial">: 
 The penalty for extending a gap by one residue. Increasing this value 
 will make the gaps shorter. Terminal gaps are not penalized.</font></p>
<h6 style="font-size: 12pt; text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><i><u><font 
	 style="font-size: 11pt; font-family: Arial; font-weight: normal; font-style: italic; 
	 text-decoration: underline; " color="#333333" size="2" face="Arial">Common 
 Parameters</font></u></i></h6>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Protein</font></b> 
 <b><font style="font-size: 11pt; font-family: Arial; font-weight: bold;" 
			 color="#333333" size="2" face="Arial">Weight Matrix</font></b><font 
	 style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
	 face="Arial">: The scores assigned to matches and mismatches (including 
 IUB ambiguity codes).</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Residue-specific 
 Penalties</font></b><font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">: Amino 
 acid specific gap penalties that reduce or increase the gap opening penalties 
 at each position or sequence in the alignment. For example, positions 
 that are rich in glycine are more likely to have an adjacent gap than 
 positions that are rich in valine. See the documentation for details.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Hydrophilic 
 Penalties</font></b><font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">: Used 
 to increase the chances of a gap within a run (5 or more residues) of 
 hydrophilic amino acids; these are likely to be loop or random coil regions 
 in which gaps are more common.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Gap 
 Separation Distance</font></b><font style="font-size: 11pt; font-family: Arial;" 
									 color="#333333" size="2" face="Arial">: 
 Tries to decrease the chances of gaps being too close to each other. Gaps 
 that are less than this distance apart are penalized more than other gaps. 
 This does not prevent close gaps; it makes them less frequent, promoting 
 a block-like appearance of the alignment.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Use 
 Negative Matrix</font></b><font style="font-size: 11pt; font-family: Arial;" 
								 color="#333333" size="2" face="Arial">: 
 When enabled negative weight matrix values will be used if they are found; 
 otherwise the matrix will be automatically adjusted to all positive values.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Delay 
 Divergent Cutoff (%)</font></b><font style="font-size: 11pt; font-family: Arial;" 
										 color="#333333" size="2" face="Arial">: 
 Delays the alignment of the most distantly-related sequences until after 
 the alignment of the most closely-related sequences. The setting shows 
 the percent identity level required to delay the addition of a sequence; 
 sequences that are less identical than this level will be aligned later.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Keep 
 Predefined Gaps</font></b><font style="font-size: 11pt; font-family: Arial;" 
								 color="#333333" size="2" face="Arial">: 
 When checked, any alignment positions in which ANY of the sequences have 
 a gap will be ignored.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Specify 
 Guide Tree</font></b><font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">:</font> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">Browser for and select a guide tree (in <a href="javascript:TextPopup(this)" id="HotSpot18620" class="glossterm">Newick format</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot18620', 'POPUP18620');
//]]></script><span class="glosstext" id="POPUP18620" style="display: none;"> NEWICK is a simple format used to write out trees in a text file.  While this is a hard-to-read format for humans, it is very useful for exchanging trees between different types of software. An example of the contents of a NEWICK format tree file is given below (note that semi-colon is needed to end the tree).  Further information on this format can be found at Joe Felsenstein’s website.   ((raccoon, bear),((sea_lion,seal),((monkey,cat), weasel)),dog);  The above tree with branch lengths will look as follows:  ((raccoon:19.19959,bear:6.80041):0.84600,((sea_lion:11.99700, seal:12.00300):7.52973,((monkey:100.85930,cat:47.14069):20.59201, weasel:18.87953):2.09460):3.87382,dog:25.46154);  If you wish to specify bootstrap values then they could appear before the branch lengths (e.g., in .DND files produced by CLUSTAL) or after the branch lengths (e.g., in .PHB files produced by CLUSTAL).  In these cases, the format might look like:  ((raccoon:19.19959,bear:6.80041)50:0.84600,((sea_lion:11.99700, seal:12.00300)100:7.52973,((monkey:100.85930,cat:47.14069)80:20.59201, weasel:18.87953)75:2.09460)50:3.87382,dog:25.46154);  or  ((raccoon:19.19959,bear:6.80041):0.84600[50],((sea_lion:11.99700, seal:12.00300):7.52973[100],((monkey:100.85930,cat:47.14069):20.59201[80], weasel:18.87953):2.09460[75]):3.87382[50],dog:25.46154);</span>) 
 to be used for the alignment. If this option is not used, then a Neighbor-Join 
 tree will be created and automatically used as the guide tree.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;">&#160;</p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;">&#160;</p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><font style="font-size: 11pt; 
	 font-family: Arial;" color="#333333" size="2" face="Arial">NOTE: All 
 definitions are derived from <a href="javascript:TextPopup(this)" id="HotSpot31105" class="glossterm">CLUSTALW</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot31105', 'POPUP31105');
//]]></script><span class="glosstext" id="POPUP31105" style="display: none;"> ClustalW is a general purpose multiple sequence alignment program for DNA or proteins. You can learn more about ClustalW by visiting its website (http://www.ebi.ac.uk/clustalw/).</span> manual.</font></p>
</body>
</html>
