<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="Adobe RoboHelp 2017" />
<meta name="OriginalFile" content="CentralDialogBox_HC.doc" />
 <title>Analysis Preferences (Estimate Selection for each Codon)</title>
<link rel="stylesheet" href="CentralDialogBox_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "";
</script>
</head>
<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Analysis Preferences (Estimate Selection for each Codon)</h1>
<p><b><span style=" font-size:11.0pt;">Analysis Preferences (Estimate Selection for each <a href="javascript:TextPopup(this)" id="HotSpot41225" class="glossterm">Codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot41225', 'POPUP41225');
//]]></script><span class="glosstext" id="POPUP41225" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span>)</span></b></p>
<p><span style=" font-size:11.0pt;">In this dialog box, you can view and select desired settings for options which are listed in the <b>Options Summary</b> tab. Available options are organized into logical sections. A yellow row indicates an analysis option for which there are multiple settings available. To view the available settings for an option, click on its row. The three primary categories of options available for analysis are:</span></p>
<p><b><i><span style=" font-size:11.0pt;">Tree to Use</span></i></b><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:.5in;"><u><span style=" font-size:11.0pt;">Tree to Use</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">MEGA requires a tree to start the analysis. This tree can either be provided by a user or automatically generated by HyPhy.  If you wish to specify a tree, change the value of this option to “Specify a tree from a file” or “Use <a href="javascript:TextPopup(this)" id="HotSpot17011" class="glossterm">Topology</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot17011', 'POPUP17011');
//]]></script><span class="glosstext" id="POPUP17011" style="display: none;"> The branching pattern of a tree is its topology.</span> Editor”.  If you are specifying a tree from a file, another option “User Tree File” will become enabled.</span></p>
<p style=" margin-left:.5in;"><u><span style=" font-size:11.0pt;">User Tree File</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">This option contains the path to a tree file you would like to use for this analysis.&#160; This option is only on when “Tree to Use” is set to “Specify a tree form a file”.&#160; To choose a file click on the row once and you will see an ellipsis button […], click it.</span></p>
<p><b><i><span style=" font-size:11.0pt;">Substitution Model</span></i></b><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">In this set of options, you can select various attributes of the substitution models for DNA and AA sequences.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Substitutions Type</span></u><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:35.45pt; text-indent:.55pt;"><span style=" font-size:11.0pt;">For this analysis, only the Syn-Nonsynonymous option is available.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Model/Method</span></u><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">Here you can select a stochastic <u style=" text-underline:double;"><span style=" color:green;">model</span></u><span style=" display:none;">HC_Models_for_estimating_distances</span>&#160; for estimating evolutionary distance by clicking on the row then selecting from one of the models that are applicable for the currently selected <i>Substitutions Type</i>.</span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">In this particular analysis there are only 4 models available (Felstein 1981, Hasegawa-Kishino-Yano, Tamura-Nei, and General Time Reversable).&#160; These are the models supported by HyPhy which MEGA uses as its calculation engine for this particular analysis.</span></p>
<p><b><i><span style=" font-size:11.0pt;">Data Subset to Use</span></i></b><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">These are options for handling gaps and missing data, including or excluding <a href="javascript:TextPopup(this)" id="HotSpot5598" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot5598', 'POPUP5598');
//]]></script><span class="glosstext" id="POPUP5598" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> positions, and restricting the analysis to <a href="javascript:TextPopup(this)" id="HotSpot37080" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot37080', 'POPUP37080');
//]]></script><span class="glosstext" id="POPUP37080" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, if applicable.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Gaps/Missing Data Treatment</span></u><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">For this analysis setting, you may choose to remove all sites containing <a href="javascript:TextPopup(this)" id="HotSpot35476" class="glossterm">alignment gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot35476', 'POPUP35476');
//]]></script><span class="glosstext" id="POPUP35476" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span> and missing information before the calculation begins (<u><span style=" color:green;">Complete-deletion</span></u><span style=" display:none;">RH_Complete_Deletion_Option</span> option). Alternatively, you may choose to retain all such sites or you may use <u><span style=" color:green;">Partial-Deletion</span></u><span style=" display:none;">RH_Partial_Deletion_Option</span> (<i>Site Coverage Cutoff</i> ) as a percentage.</span></p>
</body>
</html>
