<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="CentralDialogBox_HC.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Analysis Preferences (Maximum Likelihood)</title>
<link rel="stylesheet" href="CentralDialogBox_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "4.3.4.0_1";
</script>
 <meta name="topic-breadcrumbs" content="Part IV: Evolutionary Analysis > Constructing Phylogenetic Trees > Maximum Likelihood (ML) Methods" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Analysis Preferences (Maximum Likelihood)</h1>
<p><b><span style="font-size: 11.0pt;">Analysis Preferences (ML)</span></b></p>
<p><span style="font-size: 11.0pt;">In this dialog box, you can view and 
 select desired settings for options which are listed in the <b>Options 
 Summary</b> tab. Available options are organized into logical sections. 
 A yellow row indicates an analysis option for which there are multiple 
 settings available. To view the available settings for an option, click 
 on its row. The five primary categories of options available for ML analysis 
 are:</span></p>
<p><b><i><span style="font-size: 11.0pt;">Phylogeny Test</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">To assess 
 the reliability of a phylogenetic tree, <i>MEGA</i> provides the </span><a 
	 href="Bootstrap_Test_of_Phylogeny.htm" style="color: #007fbf;"><u 
	 style="text-underline: double; color: #007fbf;"><span style="color: #007fbf;">Bootstrap 
 test</span></u><span style="display: none; color: #007fbf;">HC_Bootstrap_Test_Phylogeny</span></a><span 
	 style="font-size: 11.0pt;">. When selected, this test requires you 
 to enter the desired <i>number of replicates</i>.</span></p>
<p><b><i><span style="font-size: 11.0pt;">Substitution Model</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">In this 
 set of options, you can select various attributes of the substitution 
 models for DNA and protein sequences.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Substitutions 
 Type</span></u></p>
<p style="margin-left: 35.45pt; text-indent: .55pt;"><span style="font-size: 11.0pt;">Here 
 you may select a substitution type of either Nucleotide or Amino Acid, 
 depending on the active sequence data. The selection in this row determines 
 the models of evolution that will be available in the <i>Model/Method</i> 
 row.</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;"><u>Genetic 
 Code Table </u>(displayed for protein coding nucleotide data only)</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">By accessing 
 this option, you can select from a list of genetic code tables for automatically 
 translating codons into amino acids. Alternatively, you can open a </span><u 
	 style="text-underline: double; color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Code_Table_Editor.htm" style="color: #007fbf;">Code Table Editor</a></span></u><span 
	 style="display: none;">HC_Code_table_editor</span><span style="font-size: 11.0pt;"> 
 from within this option.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Model/Method</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Here you 
 can select a stochastic </span><a href="Models_for_estimating_distances.htm" 
									 style="color: #007fbf;"><u style="text-underline: double; 
	 color: #007fbf;"><span style="color: #007fbf;">model</span></u><span 
	 style="display: none; color: #007fbf;">HC_Models_for_estimating_distances</span></a><span 
	 style="font-size: 11.0pt;">&#160;&#160;for estimating evolutionary 
 distance by clicking on the row then selecting from one of the models 
 that are applicable for the currently selected <i>Substitutions Type</i>.</span></p>
<p><b><i><span style="font-size: 11.0pt;">Rates and Patterns</span></i></b></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Rates 
 among Sites</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 will be enabled if the selected model has formulas that allow rate variation 
 among sites. If you choose gamma-distributed rates, then the <i>No of 
 Discrete Gamma Categories</i> option becomes visible (see&#160;&#160;<u><span 
	 style="color: #008000;"><a href="javascript:TextPopup(this)" id="HotSpot34759" class="glossterm">Gamma parameter</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot34759', 'POPUP34759');
//]]></script><span class="glosstext" id="POPUP34759" style="display: none;"> According to the gamma distribution, the substitution rate often varies from site to site within a sequence.  The shape of this distribution is determined by a value known as the gamma parameter, which is also known as the shape parameter.</span></span></u><span 
	 style="display: none;">RH_Gamma_parameter</span>).</span></p>
<p><b><i><span style="font-size: 11.0pt;">Data Subset to Use</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">These are 
 options for handling gaps and missing data, including or excluding <a href="javascript:TextPopup(this)" id="HotSpot6477" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot6477', 'POPUP6477');
//]]></script><span class="glosstext" id="POPUP6477" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> positions, and restricting the 
 analysis to <a href="javascript:TextPopup(this)" id="HotSpot32813" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot32813', 'POPUP32813');
//]]></script><span class="glosstext" id="POPUP32813" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, 
 if applicable.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Gaps/Missing 
 Data Treatment</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For this 
 analysis setting, you may choose to remove all sites containing <a href="javascript:TextPopup(this)" id="HotSpot40435" class="glossterm">alignment gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot40435', 'POPUP40435');
//]]></script><span class="glosstext" id="POPUP40435" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span> and missing information 
 before the calculation begins (</span><u style="color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Complete-Deletion_Option.htm" style="color: #007fbf;">Complete-deletion</a></span></u><span 
	 style="display: none;">RH_Complete_Deletion_Option</span><span style="font-size: 11.0pt;"> 
 option). Alternatively, you may choose to retain all such sites initially, 
 excluding them as necessary in the pairwise distance estimation (</span><u 
	 style="color: #007fbf;"><span style="color: #007fbf;"><a href="Pairwise-deletion_option.htm" 
	 style="color: #007fbf;">Pairwise-deletion</a></span></u><span style="display: none;">RH_Pairwise_deletion_option</span><span 
	 style="font-size: 11.0pt;"> option), or you may use </span><a href="Partial-Deletion_Option.htm" 
	 style="color: #007fbf;"><u style="color: #007fbf;"><span style="color: #007fbf;">Partial-Deletion</span></u><span 
	 style="display: none; color: #007fbf;">RH_Partial_Deletion_Option</span></a><span 
	 style="font-size: 11.0pt;"> (<i>Site Coverage Cutoff</i> ) as a percentage.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Codon 
 Positions</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Check or 
 uncheck the boxes for any combination of 1<sup>st</sup>, 2<sup>nd</sup>, 
 3<sup>rd</sup>, and non-coding positions for analysis. This option is 
 available only if you are analyzing nucleotide sequences which contain 
 protein-coding regions and you have selected a nucleotide-by-nucleotide 
 analysis.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Labeled 
 Sites</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 is available only if some or all of the sites have associated </span><u 
	 style="text-underline: double; color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Labeled_Sites.htm" style="color: #007fbf;">labels</a></span></u><span 
	 style="display: none;">RH_Labeled_Sites</span><span style="font-size: 11.0pt;">. 
 By clicking on the row, you will be provided with the option of including 
 sites with selected labels. If you choose to include only <a href="javascript:TextPopup(this)" id="HotSpot26158" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot26158', 'POPUP26158');
//]]></script><span class="glosstext" id="POPUP26158" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, then these sites will 
 be the first extracted from the data. Then all other options mentioned 
 above will be enforced. Note that labels associated with all three positions 
 in the <a href="javascript:TextPopup(this)" id="HotSpot34065" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot34065', 'POPUP34065');
//]]></script><span class="glosstext" id="POPUP34065" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> must be 
 included for a full <a href="javascript:TextPopup(this)" id="HotSpot4371" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot4371', 'POPUP4371');
//]]></script><span class="glosstext" id="POPUP4371" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 to be incorporated in the analysis.</span></p>
<p><b><i><span style="font-size: 11.0pt;">Tree Inference Options</span></i></b></p>
<p style="margin-left: .5in;"><u><span style="font-size: 11.0pt;">ML Heuristic 
 Method</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 allows you to specify either the </span><u style="text-underline: double; 
											 color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Nearest-Neighbor-Interchange_NNI.htm" style="color: #007fbf;">Nearest-Neighbor-Interchange 
 (NNI)</a></span></u><span style="display: none;">Nearest_Neighbor_Interchange_NNI_</span><span 
	 style="font-size: 11.0pt;"> or </span><u style="text-underline: double; 
												 color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Subtree-Pruning-Regrafting_SPR.htm" 
								 style="color: #007fbf;">Subtree-Pruning-Regrafting 
 (SPR)</a></span></u><span style="display: none;">Subtree_Pruning_Regrafting</span><span 
	 style="font-size: 11.0pt;"> </span><span style="display: none;">RH_Close_Neighbor_Interchange_CNI</span><span 
	 style="font-size: 11.0pt;"> tree searching strategy.</span></p>
<p>&#160;</p>
<p>&#160;</p>
<p>&#160;</p>
</body>
</html>
