<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="CentralDialogBox_HC.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Analysis Preferences (Minimum Evolution)</title>
<link rel="stylesheet" href="CentralDialogBox_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "4.3.2.0_2";
</script>
 <meta name="topic-breadcrumbs" content="Part IV: Evolutionary Analysis > Constructing Phylogenetic Trees > Minimum Evolution Method" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Analysis Preferences (Minimum Evolution)</h1>
<p><span style="font-size: 11.0pt;">In this dialog box you can select and 
 view desired options in the <b>Options Summary</b>.&#160;&#160;Options 
 are organized in logical sections.&#160;&#160;A yellow row indicates that 
 you have a choice for that particular attribute.&#160;&#160;&#160;The 
 primary sets of options available in this dialog box are:</span></p>
<p><b><i><span style="font-size: 11.0pt;">Analysis</span></i></b></p>
<p><b><i><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;</span></i></b>&#160;<u><span 
	 style="font-size: 11.0pt;">Test of Phylogeny</span></u></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">To 
 assess the reliability of a phylogenetic tree, <i>MEGA</i> provides two 
 different types of tests: the <i>Bootstrap test</i> and the <i>Interior 
 <a href="javascript:TextPopup(this)" id="HotSpot25654" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot25654', 'POPUP25654');
//]]></script><span class="glosstext" id="POPUP25654" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span> test</i>.&#160;&#160;Both 
 of these tests use the bootstrap re-sampling strategy, so you need to 
 enter the <i>number of replicates</i>. For a given data set applicable 
 tests and the phylogeny inference method are enabled.&#160;</span></p>
<p class="Leftalignarial"><b><i><span style="font-size: 11.0pt;">Substitution 
 Model</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">In this 
 set of options, you can choose various attributes of the substitution 
 models for DNA and protein sequences.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Substitutions 
 Type</span></u></p>
<p style="margin-left: 35.45pt; text-indent: .55pt;"><span style="font-size: 11.0pt;">Here 
 you may select a substitutions type of Nucleotide, Syn-Nonsynonymous, 
 or Amino Acid. The selection in this row affects the available models 
 in the <i>model</i> row.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Model</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Here you 
 select a stochastic </span><u style="text-underline: double; color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Models_for_estimating_distances.htm" 
								 style="color: #007fbf;">model</a></span></u><span 
	 style="display: none;">HC_Models_for_estimating_distances</span><span 
	 style="font-size: 11.0pt;"> for estimating evolutionary distance by 
 clicking on the row then selecting a model for the current <i>Substitutions 
 Type</i>.&#160;</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Substitutions 
 to Include</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Depending 
 on the distance model or method selected, the evolutionary distance can 
 be teased into two or more components.&#160;&#160;By clicking on the row, 
 you will be provided with a list of components relevant to the chosen 
 model.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Transition/Transversion 
 Ratio</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 will be visible if the chosen model requires you to provide a value for 
 the </span><a href="Transition_Transversion_Ratio_R.htm" style="color: #007fbf;"><u 
	 style="color: #007fbf;"><span style="color: #007fbf;">Transition/Transversion 
 ratio (R)</span></u><span style="display: none; color: #007fbf;">RH_Transition_Transversion_Ratio_R</span></a><span 
	 style="font-size: 11.0pt;">.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Pattern 
 among Lineages</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 becomes available if the selected model has formulas that allow the relaxation 
 of the assumption of homogeneity of substitution patterns among lineages.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Rates 
 among Sites</span></u></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">This 
 option becomes available if the selected distance model has formulas that 
 allow rate variation among sites.&#160;&#160;If you choose gamma-distributed 
 rates, then the <u><span style="color: #008000;"><a href="javascript:TextPopup(this)" id="HotSpot17531" class="glossterm">Gamma 
 parameter</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot17531', 'POPUP17531');
//]]></script><span class="glosstext" id="POPUP17531" style="display: none;"> According to the gamma distribution, the substitution rate often varies from site to site within a sequence.  The shape of this distribution is determined by a value known as the gamma parameter, which is also known as the shape parameter.</span></span></u><span style="display: none;">RH_Gamma_parameter</span> 
 option becomes visible.</span></p>
<p class="Leftalignarial"><b><i><span style="font-size: 11.0pt;">Include 
 Sites</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">These are 
 options for handling gaps and missing data, including or excluding <a href="javascript:TextPopup(this)" id="HotSpot32219" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot32219', 'POPUP32219');
//]]></script><span class="glosstext" id="POPUP32219" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> positions, and restricting the 
 analysis to <a href="javascript:TextPopup(this)" id="HotSpot11661" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot11661', 'POPUP11661');
//]]></script><span class="glosstext" id="POPUP11661" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, 
 if applicable.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Gaps 
 and Missing Data</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">You may 
 choose to remove all sites containing <a href="javascript:TextPopup(this)" id="HotSpot32166" class="glossterm">alignment 
 gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot32166', 'POPUP32166');
//]]></script><span class="glosstext" id="POPUP32166" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span> and missing information before the calculation begins 
 (</span><u style="color: #007fbf;"><span style="color: #007fbf;"><a href="Complete-Deletion_Option.htm" 
	 style="color: #007fbf;">Complete-deletion</a></span></u><span style="display: none;">RH_Complete_Deletion_Option</span><span 
	 style="font-size: 11.0pt;"> option).&#160;&#160;Alternatively, you 
 may choose to retain all such sites initially, excluding them as necessary 
 in the pairwise distance estimation (</span><u style="color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Pairwise-deletion_option.htm" style="color: #007fbf;">Pairwise-deletion</a></span></u><span 
	 style="display: none;">RH_Pairwise_deletion_option</span><span style="font-size: 11.0pt;"> 
 option), or you may use </span><u style="color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Partial-Deletion_Option.htm" style="color: #007fbf;">Partial 
 Deletion</a></span></u><span style="display: none;">RH_Partial_Deletion_Option</span><span 
	 style="font-size: 11.0pt;"> (Site coverage) as a percentage.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;"><a href="javascript:TextPopup(this)" id="HotSpot33757" class="glossterm">Codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot33757', 'POPUP33757');
//]]></script><span class="glosstext" id="POPUP33757" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> Positions</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Check or 
 uncheck the boxes for any combination of 1<sup>st</sup>, 2<sup>nd</sup>, 
 3<sup>rd</sup>, and non-coding positions for analysis.&#160;&#160;This 
 option is available only if the nucleotide sequences contain protein-coding 
 regions and you have selected a nucleotide-by-nucleotide analysis.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;"><a href="javascript:TextPopup(this)" id="HotSpot23837" class="glossterm">Labeled Sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot23837', 'POPUP23837');
//]]></script><span class="glosstext" id="POPUP23837" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span></span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 is available only if some or all of the sites have associated labels.&#160;&#160;By 
 clicking on the row, you will be provided with the option of including 
 sites with selected labels.&#160;&#160;If you choose to include only <a href="javascript:TextPopup(this)" id="HotSpot3205" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot3205', 'POPUP3205');
//]]></script><span class="glosstext" id="POPUP3205" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, then these sites will 
 be the first extracted from the data.&#160;&#160;Then all other options 
 mentioned above will be enforced.&#160;&#160;Note that labels associated 
 with all three positions in the <a href="javascript:TextPopup(this)" id="HotSpot19187" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot19187', 'POPUP19187');
//]]></script><span class="glosstext" id="POPUP19187" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 must be included for a full <a href="javascript:TextPopup(this)" id="HotSpot8423" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot8423', 'POPUP8423');
//]]></script><span class="glosstext" id="POPUP8423" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 to be incorporated in the analysis.</span></p>
<p class="Leftalignarial">&#160;</p>
<p class="Leftalignarial"><b><i><span style="font-size: 11.0pt;">Tree Inference 
 Options</span></i></b></p>
<p class="Leftalignarial" style="margin-left: .5in;"><i><u><span style="font-size: 11.0pt;">ME 
 Heuristic Method</span></u></i></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">&#160;<i>MEGA</i> 
 employs the <u style="text-underline: double;"><span style="color: #008000;"><a href="javascript:TextPopup(this)" id="HotSpot40466" class="glossterm">Close-Neighbor-Interchange (CNI)</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot40466', 'POPUP40466');
//]]></script><span class="glosstext" id="POPUP40466" style="display: none;"> In any method, examining all possible topologies is very time consuming.  This algorithm reduces the time spent searching by first producing a temporary tree, (e.g., an NJ tree when an ME tree is being sought), and then examining all of the topologies that are different from this temporary tree by a topological distance of dT = 2 and 4.  If this is repeated many times, and all the topologies previously examined are avoided, one can usually obtain the tree being sought.    For the MP method, the CNI search can start with a tree generated by the random addition of sequences.  This process can be repeated multiple times to find the MP tree.  See Nei &amp; Kumar (2000) for details.</span></span></u><span 
	 style="display: none;">RH_Close_Neighbor_Interchange_CNI</span> algorithm 
 for finding the ME tree.&#160;&#160;It is a <a href="javascript:TextPopup(this)" id="HotSpot44771" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot44771', 'POPUP44771');
//]]></script><span class="glosstext" id="POPUP44771" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span> 
 swapping method, which begins with an initial NJ tree.</span></p>
<p class="Leftalignarial" style="margin-left: .5in;"><i><u><span style="font-size: 11.0pt;">Initial 
 Tree For ME</span></u></i></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">This 
 is obtained by using Neighbor Joining.</span></p>
<p class="Leftalignarial" style="margin-left: .5in;"><u><span style="font-size: 11.0pt;">ME 
 Search Level</span></u></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">Select 
 a search level 1 or 2.</span></p>
<p>&#160;</p>
<p>&#160;</p>
<b><span style="font-size: 11.0pt; font-family: Arial, sans-serif; color: #0000ff;"><br 
	 style="clear: all;" clear="all" />
</span></b> 
</body>
</html>
