<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="CentralDialogBox_HC.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Analysis Preferences (Z-test of Selection)</title>
<link rel="stylesheet" href="CentralDialogBox_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "4.6.1.0_2";
</script>
 <meta name="topic-breadcrumbs" content="Part IV: Evolutionary Analysis > Tests of Selection > Synonymous/Nonsynonymous Tests" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Analysis Preferences (Z-test of Selection)</h1>
<p><span style="font-size: 11.0pt;">In this dialog box, you can view and 
 select options in the <b>Options Summary</b>.&#160;&#160;Options are organized 
 in logical sections.&#160;&#160;A yellow row indicates that you have a 
 choice for that particular attribute.&#160;&#160;The three primary sets 
 of options available in this dialog box are:</span></p>
<p class="TipNoteHeading"><i><span style="font-size: 11.0pt;">Analysis</span></i></p>
<p class="Leftalignarial" style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Analysis 
 Scope</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Use this 
 option to specify whether to conduct an analysis for sequence pairs, an 
 overall average, or within sequence groups (if sequence groups are specified).&#160; 
 </span></p>
<p style="margin-left: .5in;"><u><span style="font-size: 11.0pt;">Test 
 Hypothesis</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">One way 
 to test whether <a href="javascript:TextPopup(this)" id="HotSpot55888" class="glossterm">positive selection</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot55888', 'POPUP55888');
//]]></script><span class="glosstext" id="POPUP55888" style="display: none;"> At the DNA sequence level, positive selection refers to selection in favor of nonsynonymous substitutions.  In this case, the evolutionary distance based on nonsynonymous substitutions is expected to be greater than synonymous substitutions.</span> 
 is operating on a <a href="javascript:TextPopup(this)" id="HotSpot19159" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot19159', 'POPUP19159');
//]]></script><span class="glosstext" id="POPUP19159" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> 
 is to compare the relative abundance of synonymous and nonsynonymous substitutions 
 within the <a href="javascript:TextPopup(this)" id="HotSpot24607" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot24607', 'POPUP24607');
//]]></script><span class="glosstext" id="POPUP24607" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> sequences.&#160;&#160;For 
 a pair of sequences, this is done by first estimating the number of synonymous 
 substitutions per synonymous site (<i>d</i><sub>S</sub>) and the number 
 of nonsynonymous substitutions per nonsynonymous site (<i>d</i><sub>N</sub>), 
 and their variances: Var(<i>d</i><sub>S</sub>) and Var(<i>d</i><sub>N</sub>), 
 respectively.&#160;&#160;With this information, we can test the null hypothesis 
 that H<sub>0</sub>: <i>d</i><sub>N</sub> = <i>d</i><sub>S</sub> using 
 a Z-test:</span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Z 
 = (<i>d</i><sub>N</sub> - <i>d</i><sub>S</sub>) / SQRT(Var(<i>d</i><sub>S</sub>) 
 + Var(<i>d</i><sub>N</sub>))</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">The level 
 of significance at which the null hypothesis is rejected depends on the 
 alternative hypothesis (HA):</span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;H<sub>0</sub>:&#160;&#160;&#160;&#160;&#160;&#160;&#160;<i>d</i><sub>N</sub> 
 = <i>d</i><sub>S</sub></span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;HA: 
 &#160;&#160;&#160;&#160;&#160;(a)&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;<i>d</i><sub>N</sub></span> 
 <span style="font-size: 11.0pt; font-family: Symbol;">ยน</span> <span style="font-size: 11.0pt;"><i>d</i><sub>S</sub>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(test 
 of neutrality).</span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(b)&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;<i>d</i><sub>N</sub> 
 &gt; <i>d</i><sub>S</sub> &#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(<a href="javascript:TextPopup(this)" id="HotSpot22280" class="glossterm">positive selection</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot22280', 'POPUP22280');
//]]></script><span class="glosstext" id="POPUP22280" style="display: none;"> At the DNA sequence level, positive selection refers to selection in favor of nonsynonymous substitutions.  In this case, the evolutionary distance based on nonsynonymous substitutions is expected to be greater than synonymous substitutions.</span>).</span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(c 
 )&#160;&#160;&#160;&#160;&#160;&#160;&#160;<i>d</i><sub>N</sub> &lt; <i>d</i><sub>S</sub> 
 &#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(<a href="javascript:TextPopup(this)" id="HotSpot39795" class="glossterm">purifying selection</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot39795', 'POPUP39795');
//]]></script><span class="glosstext" id="POPUP39795" style="display: none;"> Purifying selection refers to selection against nonsynonymous substitutions at the DNA level.  In this case, the evolutionary distance based on synonymous substitutions is expected to be greater than the distance based on nonsynonymous substitutions.</span>).</span></p>
<p>&#160;</p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For alternative 
 hypotheses (b) and (c), we use a one-tailed test and for (a) we use a 
 two-tailed test. These three tests can be conducted directly for pairs 
 of sequences, overall sequences, or within groups of sequences.&#160;&#160;For 
 testing for selection in a pairwise manner, you can compute the variance 
 of (<i>d</i><sub>N</sub> - <i>d</i><sub>S</sub>) by using either the analytical 
 formulas or the bootstrap resampling method.</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For data 
 sets containing more than two sequences, you can compute the average number 
 of synonymous substitutions and the average number of nonsynonymous substitutions 
 to conduct a Z-test in a manner similar to the one mentioned above.&#160;&#160;The 
 variance of the difference between these two quantities can be estimated 
 by the bootstrap method (<u><span style="color: #008000;">Nei and Kumar 
 [2000]</span></u><span style="display: none;">Nei_and_Kumar_2000</span>, 
 page 56).&#160;&#160;</span></p>
<p class="Leftalignarial" style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Variance 
 Estimation Method</span></u></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">Depending 
 on the scope of the analysis (pairwise versus other), you may compute 
 standard errors using analytical formulas or the bootstrap method.&#160;&#160;Whenever 
 standard errors are estimated by the bootstrap method, you will be prompted 
 for the number of bootstrap replicates and a random number seed.&#160;</span></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">When 
 the selected test involves the computation of average distance, only the 
 bootstrap method is available for computing standard errors.</span></p>
<p class="Leftalignarial"><b><i><span style="font-size: 11.0pt;">Substitution 
 Model</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">In this 
 set of options, you can choose various attributes of the substitution 
 models for DNA and protein sequences.</span></p>
<p style="margin-left: .5in;"><u><span style="font-size: 11.0pt;">Substitutions 
 Type</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This is 
 limited to Syn-Nonsynonymous.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Model</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">By clicking 
 on the row of the currently selected model, you may select a stochastic 
 </span><u style="text-underline: double; color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Models_for_estimating_distances.htm" style="color: #007fbf;">model</a></span></u><span 
	 style="display: none;">HC_Models_for_estimating_distances</span><span 
	 style="font-size: 11.0pt;"> for estimating evolutionary distance (click 
 on the yellow row first).&#160;&#160;This will reveal a </span><u style="text-underline: double;"><span 
	 style="color: #008000;">menu</span></u><span style="display: none;">HC_Models_for_estimating_distances</span><span 
	 style="font-size: 11.0pt;"> containing many different distance methods 
 and models.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Transition/Transversion 
 Ratio</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 will be visible if the chosen model requires you to provide a value for 
 the </span><u style="color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Transition_Transversion_Ratio_R.htm" style="color: #007fbf;">Transition/Transversion 
 ratio (R)</a></span></u><span style="display: none;">RH_Transition_Transversion_Ratio_R</span><span 
	 style="font-size: 11.0pt;">.</span></p>
<p class="Leftalignarial"><b><i><span style="font-size: 11.0pt;">Data Subset 
 to Use</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">These are 
 options for handling gaps and missing data and restricting the analysis 
 to <a href="javascript:TextPopup(this)" id="HotSpot18132" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot18132', 'POPUP18132');
//]]></script><span class="glosstext" id="POPUP18132" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, if 
 applicable.</span></p>
<p style="margin-left: .5in;"><u><span style="font-size: 11.0pt;">Gaps 
 and Missing Data</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">You may 
 choose to remove all sites containing <a href="javascript:TextPopup(this)" id="HotSpot9219" class="glossterm">alignment 
 gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot9219', 'POPUP9219');
//]]></script><span class="glosstext" id="POPUP9219" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span> and missing information before the calculation begins 
 (</span><a href="Complete-Deletion_Option.htm" style="color: #007fbf;"><u 
	 style="color: #007fbf;"><span style="color: #007fbf;">Complete-deletion</span></u><span 
	 style="display: none; color: #007fbf;">RH_Complete_Deletion_Option</span></a><span 
	 style="font-size: 11.0pt;"> option).&#160;&#160;Alternatively, you 
 may choose to retain all such sites initially, excluding them as necessary 
 in the pairwise distance estimation (</span><a href="Pairwise-deletion_option.htm"><u 
	 style="color: #007fbf;"><span style="color: #007fbf;">Pairwise-deletion</span></u><span 
	 style="display: none;">RH_Pairwise_deletion_option</span></a><span 
	 style="font-size: 11.0pt;"> option), or you may use </span><u style="color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Partial-Deletion_Option.htm" style="color: #007fbf;">Partial 
 Deletion</a></span></u><span style="display: none;">RH_Partial_Deletion_Option</span><span 
	 style="font-size: 11.0pt;"> (Site coverage) as a percentage.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Labeled 
 Sites</span></u></p>
<p class="Leftalignarial" style="margin-left: .5in;"><span style="font-size: 11.0pt;">This 
 option is available only if there are labels associated with some or all 
 of the sites in the data.&#160;&#160;By clicking on the yellow row, you 
 will have the option of including sites with selected labels.&#160;&#160;If 
 you chose to include only <a href="javascript:TextPopup(this)" id="HotSpot19860" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot19860', 'POPUP19860');
//]]></script><span class="glosstext" id="POPUP19860" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, 
 they will be first extracted from the data and all of the other options 
 mentioned above will be enforced.&#160;&#160;Note that labels associated 
 with all three positions in the <a href="javascript:TextPopup(this)" id="HotSpot45407" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot45407', 'POPUP45407');
//]]></script><span class="glosstext" id="POPUP45407" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 must be included for a full <a href="javascript:TextPopup(this)" id="HotSpot24914" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot24914', 'POPUP24914');
//]]></script><span class="glosstext" id="POPUP24914" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 in the analysis.</span></p>
<p>&#160;</p>
<b><span style="font-size: 11.0pt; font-family: Arial, sans-serif; color: #0000ff;"><br 
	 style="clear: all;" clear="all" />
</span></b> 
</body>
</html>
