<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="Adobe RoboHelp 2017" />
<meta name="OriginalFile" content="CentralDialogBox_HC.doc" />
 <title>Analysis Options</title>
<link rel="stylesheet" href="CentralDialogBox_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "";
</script>
</head>
<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Analysis Options</h1>
<p><span style=" font-size:11.0pt;">In this dialog box you can select and view desired options in the <b>Options Summary</b>.&#160; Options are organized in logical sections.&#160; A yellow row indicates that you have a choice for that particular attribute. The primary sets of options available in this dialog box are:</span></p>
<p class="Leftalignarial"><b><i><span style=" font-size:11.0pt;">Clock Settings</span></i></b></p>
<p style=" margin-left:.5in;" class="Leftalignarial"><span style=" font-size:11.0pt;">The RelTime analysis provides several options that are specific to divergence time estimation.</span></p>
<p style=" text-indent:.5in;" class="Leftalignarial"><u><span style=" font-size:11.0pt;">Clock Type</span></u></p>
<p style=" margin-left:.5in;" class="Leftalignarial"><span style=" font-size:11.0pt;">A RelTime analysis assumes that there is no global clock. You cannot change this parameter.</span></p>
<p style=" text-indent:.5in;" class="Leftalignarial"><u><span style=" font-size:11.0pt;">Clock Stringency</span></u></p>
<p style=" margin-left:.5in;" class="Leftalignarial"><span style=" font-size:11.0pt;">This selects how much separation clocks need to have before they can be merged. All Clocks keeps all of the clocks without merging any.&#160; Many Clocks merges all clocks which are within 1 standard deviation of each other.&#160; Few merges within 2 standard deviations, and Fewer merges within 3 standard deviations.</span></p>
<p style=" text-indent:.5in;" class="Leftalignarial"><u><span style=" font-size:11.0pt;">Variance Estimation Method</span></u></p>
<p style=" margin-left:.5in;" class="Leftalignarial"><span style=" font-size:11.0pt;">This offers the same options as the Phylogeny Test option in regular ML Tree construction (Bootstrap and Analytical).&#160; For RelTime trees, MEGA will use the selected option to calculate the variance for estimated divergence times.<b><i></i></b></span></p>
<p class="Leftalignarial"><b><i><span style=" font-size:11.0pt;">Phylogeny Test and Options</span></i></b></p>
<p style=" margin-left:.5in;" class="Leftalignarial"><span style=" font-size:11.0pt;">To assess the reliability of a phylogenetic tree, <i>MEGA</i> provides the <i>Bootstrap test</i>.&#160; This uses the bootstrap re-sampling strategy, so you need to enter the <i>number of replicates</i>.</span></p>
<p class="Leftalignarial"><b><i><span style=" font-size:11.0pt;">Substitution Model</span></i></b></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">In this set of options, you can choose various attributes of the substitution models for DNA and protein sequences.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Substitutions Type</span></u></p>
<p style=" margin-left:35.45pt; text-indent:.55pt;"><span style=" font-size:11.0pt;">Here you may select a substitutions type of Nucleotide, Syn-Nonsynonymous, or Amino Acid.&#160; The selection in this row affects the available models in the <i>model</i> row.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Model</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">Here you select a stochastic <u style=" text-underline:double;"><span style=" color:green;">model</span></u><span style=" display:none;">HC_Models_for_estimating_distances</span> for estimating evolutionary distance by clicking on the row then selecting a model for the current <i>Substitutions Type</i>.&#160;</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Substitutions to Include</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">Depending on the distance model or method selected, the evolutionary distance can be teased into two or more components.&#160; By clicking on the row, you will be provided with a list of components relevant to the chosen model.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;"><a href="javascript:TextPopup(this)" id="HotSpot20370" class="glossterm">Transition</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot20370', 'POPUP20370');
//]]></script><span class="glosstext" id="POPUP20370" style="display: none;"> A transition occurs when a purine is substituted by a purine, or a pyrimidine by a pyrimidine.</span>/<a href="javascript:TextPopup(this)" id="HotSpot13559" class="glossterm">Transversion</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot13559', 'POPUP13559');
//]]></script><span class="glosstext" id="POPUP13559" style="display: none;"> A change from a purine to a pyrimidine, or vice versa, is a transversion.</span> Ratio</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">This option will be visible if the chosen model requires you to provide a value for the <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot23873" class="glossterm">Transition</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot23873', 'POPUP23873');
//]]></script><span class="glosstext" id="POPUP23873" style="display: none;"> A transition occurs when a purine is substituted by a purine, or a pyrimidine by a pyrimidine.</span>/<a href="javascript:TextPopup(this)" id="HotSpot46320" class="glossterm">Transversion</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot46320', 'POPUP46320');
//]]></script><span class="glosstext" id="POPUP46320" style="display: none;"> A change from a purine to a pyrimidine, or vice versa, is a transversion.</span> ratio (R)</span></u><span style=" display:none;">RH_Transition_Transversion_Ratio_R</span>.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Pattern among Lineages</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">This option becomes available if the selected model has formulas that allow the relaxation of the assumption of homogeneity of substitution patterns among lineages.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Rates among Sites</span></u></p>
<p style=" margin-left:.5in;" class="Leftalignarial"><span style=" font-size:11.0pt;">This option becomes available if the selected distance model has formulas that allow rate variation among sites.&#160; If you choose gamma-distributed rates, then an <span style=" display:none;">RH_Gamma_parame</span>option to specify the number of discrete categories for modeling the gamma distribution becomes visible.</span></p>
<p class="Leftalignarial"><b><i><span style=" font-size:11.0pt;">Data Subset to Use</span></i></b></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">These are options for handling gaps and missing data, including or excluding <a href="javascript:TextPopup(this)" id="HotSpot44747" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot44747', 'POPUP44747');
//]]></script><span class="glosstext" id="POPUP44747" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> positions, and restricting the analysis to <a href="javascript:TextPopup(this)" id="HotSpot10751" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot10751', 'POPUP10751');
//]]></script><span class="glosstext" id="POPUP10751" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, if applicable.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Gaps and Missing Data</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">You may choose to remove all sites containing <a href="javascript:TextPopup(this)" id="HotSpot56989" class="glossterm">alignment gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot56989', 'POPUP56989');
//]]></script><span class="glosstext" id="POPUP56989" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span> and missing information before the calculation begins (<u><span style=" color:green;">Complete-deletion</span></u><span style=" display:none;">RH_Complete_Deletion_Option</span> option).&#160; Alternatively, you may choose to retain all such sites initially, excluding them as necessary in the pairwise distance estimation (<u><span style=" color:green;">Pairwise-deletion</span></u><span style=" display:none;">RH_Pairwise_deletion_option</span> option), or you may use <u><span style=" color:green;">Partial Deletion</span></u><span style=" display:none;">RH_Partial_Deletion_Option</span> (Site coverage) as a percentage.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;"><a href="javascript:TextPopup(this)" id="HotSpot24170" class="glossterm">Codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot24170', 'POPUP24170');
//]]></script><span class="glosstext" id="POPUP24170" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> Positions</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">Check or uncheck the boxes for any combination of 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>, and non-coding positions for analysis.&#160; This option is available only if the nucleotide sequences contain protein-coding regions and you have selected a nucleotide-by-nucleotide analysis.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;"><a href="javascript:TextPopup(this)" id="HotSpot23904" class="glossterm">Labeled Sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot23904', 'POPUP23904');
//]]></script><span class="glosstext" id="POPUP23904" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span></span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">This option is available only if some or all of the sites have associated labels.  By clicking on the row, you will be provided with the option of including sites with selected labels.  If you choose to include only <a href="javascript:TextPopup(this)" id="HotSpot31052" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot31052', 'POPUP31052');
//]]></script><span class="glosstext" id="POPUP31052" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, then these sites will be the first extracted from the data.  Then all other options mentioned above will be enforced.  Note that labels associated with all three positions in the <a href="javascript:TextPopup(this)" id="HotSpot31571" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot31571', 'POPUP31571');
//]]></script><span class="glosstext" id="POPUP31571" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> must be included for a full <a href="javascript:TextPopup(this)" id="HotSpot37885" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot37885', 'POPUP37885');
//]]></script><span class="glosstext" id="POPUP37885" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> to be incorporated in the analysis.</span></p>
<p><span style=" font-size:11.0pt;">&#160;</span></p>
<p><b><i><span style=" font-size:11.0pt;">Tree Inference Options</span></i></b></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">ML Heuristic Method</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">There are two options available</span> <u style=" text-underline:double;"><span style=" font-size:11.0pt; color:green;">Nearest-Neighbor-Interchange (NNI)</span></u><span style=" font-size:11.0pt; display:none;">Nearest_Neighbor_Interchange_NNI_</span> <span style=" font-size:11.0pt;">or <u style=" text-underline:double;"><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot26864" class="glossterm">Subtree-Pruning-Regrafting (SPR)</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot26864', 'POPUP26864');
//]]></script><span class="glosstext" id="POPUP26864" style="display: none;"> For any tree searching method, exhaustive search, where all possible topologies are considered is unfeasible for even a small number of taxa. Subtree Pruning And Regrafting is a tree topology search heuristic which reduces the number of topologies searched by performing the following operations on the tree.        First, a subtree of the current best tree is selected and detached (pruned). Second, the detached subtree is regrafted onto another branch of the remaining tree, in such a way that a new topology is created and then likelihood of the new topology is calculated. This procedure is repeated for all regrafting positions that produce new topologies using the pruned subtree. The procedure is also repeated for each subtree (within the designated search level) and if the topology with best likelihood among those scored gives sufficient improvement over the current best tree, that topology becomes the current best tree. This is repeated until no significant further likelihood improvements are obtained.        A single pass of the SPR algorithm examines O(N2) new trees, where N is the number of leaves in the original tree. This is because, for each subtree there are O(N) possible regraftings, and there are O(N) possible subtrees to consider. In contrast, NNI examines O(N) topologies at each pass of the algorithm.</span></span></u><span style=" display:none;">Subtree_Pruning_Regrafting</span>. For SPR, two search levels are available; search level 3, which searches less exhaustively and search level 5 which performs an extensive search of the tree space.</span></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Initial Tree for ML</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">The initial tree for ML can be supplied in several ways. Mega can simply create it on the fly using Neighbor Joining (NJ), BioNJ, Maximum Parsimony, or try both NJ and BioNJ and select the superior tree or you may specify your own tree by providing a newick file.</span></p>
<p style=" margin-left:.5in;"><u><span style=" font-size:11.0pt;"><a href="javascript:TextPopup(this)" id="HotSpot32230" class="glossterm">Branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot32230', 'POPUP32230');
//]]></script><span class="glosstext" id="POPUP32230" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span> <a href="javascript:TextPopup(this)" id="HotSpot25355" class="glossterm">Swap</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot25355', 'POPUP25355');
//]]></script><span class="glosstext" id="POPUP25355" style="display: none;"> This command changes the cursor to the 'Flip' icon.  Then, you click on an interior branch, MEGA swaps the two subtrees defined by this branch.  If each of the subtrees is an individual taxon, then Swap is the same as Flip.  The cursor will revert to the arrow if you click on the arrow icon on the left-hand side of the Tree Explorer.</span> Filter</span></u></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">For analyses that use a Maximum Likelihood approach, the ‘<a href="javascript:TextPopup(this)" id="HotSpot37578" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot37578', 'POPUP37578');
//]]></script><span class="glosstext" id="POPUP37578" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span> <a href="javascript:TextPopup(this)" id="HotSpot36668" class="glossterm">swap</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot36668', 'POPUP36668');
//]]></script><span class="glosstext" id="POPUP36668" style="display: none;"> This command changes the cursor to the 'Flip' icon.  Then, you click on an interior branch, MEGA swaps the two subtrees defined by this branch.  If each of the subtrees is an individual taxon, then Swap is the same as Flip.  The cursor will revert to the arrow if you click on the arrow icon on the left-hand side of the Tree Explorer.</span> filter’ analysis setting is used to set the stringency of optimization with respect to <a href="javascript:TextPopup(this)" id="HotSpot58088" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot58088', 'POPUP58088');
//]]></script><span class="glosstext" id="POPUP58088" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span> lengths and improvements in log likelihood values. A weaker filter will result in more exhaustive optimization so that the search may take longer but potentially, a larger search space will be explored. A stronger filter will result in less exhaustive optimization so that the search may require less time but potentially, a smaller search space will be explored.</span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">&#160;</span></p>
<p><b><i><span style=" font-size:11.0pt;">System Resource Usage</span></i></b></p>
<p style=" text-indent:.5in;"><u><span style=" font-size:11.0pt;">Number of Threads</span></u><span style=" font-size:11.0pt;"></span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">Beginning in MEGA5.1, multithreaded ML phylogeny inference can be performed on computers that have multiple CPUs or cores. If MEGA detects multiple processing resources, an option for setting the number of search threads will be provided in the analysis preferences dialog. However, this option is only available when running a single ML inference analysis. If you wish to run multiple ML based analyses simultaneously, set the number of threads to ‘1’ when initializing the first analysis.</span></p>
<p style=" margin-left:.5in;"><span style=" font-size:11.0pt;">&#160;</span></p>
<p><span style=" font-size:11.0pt;">&#160;</span></p>
<p><span style=" font-size:11.0pt;">&#160;</span></p>
<b><span style=" font-size:11.0pt; font-family:'Arial' ,sans-serif; color:blue;"><br clear="all" /></span></b>
</body>
</html>
