<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="CentralDialogBox_HC.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Analysis Preferences (Maximum Parsimony)</title>
<link rel="stylesheet" href="CentralDialogBox_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "4.3.3.0_4";
</script>
 <meta name="topic-breadcrumbs" content="Part IV: Evolutionary Analysis > Constructing Phylogenetic Trees > Maximum Parsimony (MP) Method" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Analysis Preferences (Maximum Parsimony)</h1>
<p><span style="font-size: 11.0pt;">This dialog box contains a set of analysis 
 options for use in Maximum Parsimony analysis.&#160;&#160;Information 
 from this dialog is used in the requested analysis, so it is important 
 that you examine the options selected before pressing <i>OK</i> to proceed 
 with an analysis.</span></p>
<p><b><i><span style="font-size: 11.0pt;">Phylogeny Test and Options</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">To assess 
 the reliability of the MP trees, <i>MEGA</i> provides the bootstrap test.&#160;&#160;You 
 need to enter the <i>number of replicates</i> for this test if it is selected.</span></p>
<p class="TipNoteHeading"><i><span style="font-size: 11.0pt;">Tree Inference 
 Options</span></i></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">Use this 
 to select between the </span><u style="text-underline: double; color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Subtree-Pruning-Regrafting_SPR.htm" 
								 style="color: #007fbf;">Subtree-Pruning-Regrafting 
 (SPR)</a></span></u><span style="display: none;">Subtree_Pruning_Regrafting</span><span 
	 style="font-size: 11.0pt;">, </span><a href="Tree_Bisection_Reconnection_TBR.htm" 
											 style="color: #007fbf;"><u 
	 style="text-underline: double; color: #007fbf;"><span style="color: #007fbf;">Tree-Bisection-Reconnection 
 (TBR)</span></u><span style="display: none; color: #007fbf;">Tree_Bisection_Reconnection_TBR_</span></a><span 
	 style="font-size: 11.0pt;">, </span><a href="Min-mini_algorithm.htm" 
											 style="color: #007fbf;"><u 
	 style="text-underline: double; color: #007fbf;"><span style="color: #007fbf;">Min-Mini</span></u><span 
	 style="display: none; color: #007fbf;">RH_Min_mini_algorithm</span></a><span 
	 style="font-size: 11.0pt;">, and </span><u style="text-underline: double; 
												 color: #007fbf;"><span 
	 style="color: #007fbf;"><a href="Max-mini_branch-and-bound_search.htm" 
								 style="color: #007fbf;">Max-Mini Branch-and-Bound</a></span></u><span 
	 style="display: none;">RH_Max_mini_branch_and_bound_search</span><span 
	 style="font-size: 11.0pt;">, heuristic search methods.&#160;</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For the 
 SPR and TBR methods, you may automatic</span><span style="display: none;">RH_tree_length</span><span 
	 style="font-size: 11.0pt;">ally obtain a set of initial trees by using 
 the </span><u style="color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Random_addition_trees.htm" style="color: #007fbf;">random addition 
 option</a></span></u><span style="display: none;">RH_Random_addition_trees</span><span 
	 style="font-size: 11.0pt;"> to produce the initial trees.</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For all 
 methods except for Max-Mini <a href="javascript:TextPopup(this)" id="HotSpot28359" class="glossterm">Branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot28359', 'POPUP28359');
//]]></script><span class="glosstext" id="POPUP28359" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span>-and-Bound, 
 the MP search level can be set.</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For the 
 <a href="javascript:TextPopup(this)" id="HotSpot26856" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot26856', 'POPUP26856');
//]]></script><span class="glosstext" id="POPUP26856" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span>-and-bound search, 
 an optimized Max-Mini Branch-and-Bound<span style="display: none;">RH_Max_mini_branch_and_bound_search</span> 
 algorithm is used.&#160;&#160;While this algorithm is guaranteed to find 
 all the MP trees, a branch-and-bound search often is too time-consuming 
 for more than 15 sequences, although this number varies from data set 
 to data set.&#160;</span></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For all 
 methods, you can set the maximum number of equally parsimonious trees 
 to retain.</span></p>
<p><b><i><span style="font-size: 11.0pt;">Substitution Model</span></i></b></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Substitution 
 Type</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">For protein-coding 
 data, the analysis can be performed at the nucleotide or the amino acid 
 level.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Genetic 
 Code Table</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">If protein-coding 
 data is to be analyzed at the amino-acid level, a genetic code for translating 
 the data must be selected.</span></p>
<p><b><i><span style="font-size: 11.0pt;">Data Subset to Use&#160;</span></i></b></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This provides 
 options for handling gaps and missing data in the analysis, specifying 
 inclusion and exclusion of <a href="javascript:TextPopup(this)" id="HotSpot41930" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot41930', 'POPUP41930');
//]]></script><span class="glosstext" id="POPUP41930" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 positions, and restricting the analysis to only some types of <a href="javascript:TextPopup(this)" id="HotSpot31066" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot31066', 'POPUP31066');
//]]></script><span class="glosstext" id="POPUP31066" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span> (if applicable).</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Gaps 
 and Missing Data</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">You may 
 choose to remove all sites containing <a href="javascript:TextPopup(this)" id="HotSpot46419" class="glossterm">alignment 
 gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot46419', 'POPUP46419');
//]]></script><span class="glosstext" id="POPUP46419" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span> and missing-information before the parsimony analysis 
 begins using the </span><u><span style="color: #008000;">Complete-deletion</span></u><span 
	 style="display: none;">RH_Complete_Deletion_Option</span><span style="font-size: 11.0pt;"> 
 option.&#160;&#160;Alternatively, you may choose to retain all such sites.&#160;&#160;In 
 this case, all missing-information and alignment gap sites are treated 
 as missing data in the calculation of </span><u><span style="color: #008000;"><a href="javascript:TextPopup(this)" id="HotSpot32267" class="glossterm">tree length</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot32267', 'POPUP32267');
//]]></script><span class="glosstext" id="POPUP32267" style="display: none;"> Tree length is the criterion used by the Maximum Parsimony method to search for the best tree.  It is defined as the sum of the minimum numbers of substitutions over all sites for the given topology.   To compute the tree length for the unweighted parsimony method, we use the procedure described in Fitch (1971), which is based on the two rules described below.  For a given site these rules are applied to each node and the sum of substitutions over all nodes and over all sites are taken.  Note that the estimation of the minimum number of substitutions is not affected by the position of the root.  Rule 1.  When the two descendent nodes of an ancestral node have some states (nucleotides or amino acids) in common, the ancestral node is assigned to the set of common states.  In this case, the most parsimonious explanation does not require any substitutions.  Rule 2.  When the two descendant nodes have no states in common, then all states in the descendent nodes are combined to form the set of possible states at the ancestral node.  In this case, one substitution is required.</span></span></u><span style="display: none;">RH_Tree_length</span><span 
	 style="font-size: 11.0pt;">.&#160;&#160;Your last option is </span><u 
	 style="color: #007fbf;"><span style="color: #007fbf;"><a href="Partial-Deletion_Option.htm" 
	 style="color: #007fbf;">Partial Deletion</a></span></u><span style="display: none;">RH_Partial_Deletion_Option</span><span 
	 style="font-size: 11.0pt;"> a.k.a. coverage where you may select a 
 percentage where only sites above that percentage of unambiguity will 
 be counted.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Select 
 Codon Positions</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">You may 
 select any combination of 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>, 
 and non-coding positions for analysis.&#160;&#160;This option is available 
 <i>only if</i> the nucleotide sequences contain protein-coding regions.&#160;&#160;If 
 they do, you can choose between the analysis of nucleotide sequences or 
 translated protein sequences.&#160;&#160;If you choose the latter, <i>MEGA</i> 
 will translate all protein-coding regions into amino acid sequences and 
 conduct the protein sequence parsimony analysis.</span></p>
<p style="text-indent: .5in;"><u><span style="font-size: 11.0pt;">Labeled 
 Sites</span></u></p>
<p style="margin-left: .5in;"><span style="font-size: 11.0pt;">This option 
 is available only if there are labels associated with some or all of the 
 sites in the data.&#160;&#160;By clicking on the ellipses, you will have 
 the option of including sites with selected labels.&#160;&#160;If you 
 choose to include only <a href="javascript:TextPopup(this)" id="HotSpot29841" class="glossterm">labeled sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot29841', 'POPUP29841');
//]]></script><span class="glosstext" id="POPUP29841" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span>, 
 then these sites will be the first extracted from the data and all other 
 options mentioned above will be enforced.&#160;&#160;Note that labels 
 associated with all three positions in the <a href="javascript:TextPopup(this)" id="HotSpot38119" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot38119', 'POPUP38119');
//]]></script><span class="glosstext" id="POPUP38119" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 must be included for a full <a href="javascript:TextPopup(this)" id="HotSpot31447" class="glossterm">codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot31447', 'POPUP31447');
//]]></script><span class="glosstext" id="POPUP31447" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 to be incorporated in the analysis.</span></p>
<p>&#160;</p>
<b><span style="font-size: 11.0pt; font-family: Arial, sans-serif; color: #0000ff;"><br 
	 style="clear: all;" clear="all" />
</span></b> 
</body>
</html>
