<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="ContextHelp_HC.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Codon Based Z-Test (large sample)</title>
<link rel="stylesheet" href="ContextHelp_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<style type="text/css">
<!--
A:visited { color:#800080; }
A:link { color:#0000ff; }
-->
</style>
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "7.1.9.0_3";
</script>
 <meta name="topic-breadcrumbs" content="Appendix B: Main Menu Items and Dialogs Reference > Main MEGA Menus > Selection Menu" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1><span style="font-size: 11.0pt;">Codon Based Z-Test (large sample)</span></h1>
<p class="keyword">Distance | <a href="javascript:TextPopup(this)" id="HotSpot15433" class="glossterm">Codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot15433', 'POPUP15433');
//]]></script><span class="glosstext" id="POPUP15433" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span> 
 Based Z-test (large sample)</p>
<p><span style="font-size: 11.0pt;">One way to test whether <a href="javascript:TextPopup(this)" id="HotSpot26587" class="glossterm">positive selection</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot26587', 'POPUP26587');
//]]></script><span class="glosstext" id="POPUP26587" style="display: none;"> At the DNA sequence level, positive selection refers to selection in favor of nonsynonymous substitutions.  In this case, the evolutionary distance based on nonsynonymous substitutions is expected to be greater than synonymous substitutions.</span> is operating on a 
 <a href="javascript:TextPopup(this)" id="HotSpot29600" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot29600', 'POPUP29600');
//]]></script><span class="glosstext" id="POPUP29600" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> is to compare the 
 relative abundance of synonymous and nonsynonymous substitutions that 
 have occurred in the <a href="javascript:TextPopup(this)" id="HotSpot31283" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot31283', 'POPUP31283');
//]]></script><span class="glosstext" id="POPUP31283" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> 
 sequences.&#160;&#160;For a pair of sequences, this is done by first estimating 
 the number of synonymous substitutions per synonymous site (<i>d</i><sub>S</sub>) 
 and the number of nonsynonymous substitutions per nonsynonymous site (<i>d</i><sub>N</sub>), 
 and their variances: Var(<i>d</i><sub>S</sub>) and Var(<i>d</i><sub>N</sub>), 
 respectively.&#160;&#160;With this information, we can test the null hypothesis 
 that H<sub>0</sub>: <i>d</i><sub>N</sub> = <i>d</i><sub>S</sub> using 
 a Z-test:</span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Z 
 = (<i>d</i><sub>N</sub> - <i>d</i><sub>S</sub>) / SQRT(Var(<i>d</i><sub>S</sub>) 
 + Var(<i>d</i><sub>N</sub>))</span></p>
<p><span style="font-size: 11.0pt;">The level of significance at which 
 the null hypothesis is rejected depends on the alternative hypothesis 
 (HA)</span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;H<sub>0</sub>:&#160;&#160;&#160;&#160;&#160;&#160;&#160;<i>d</i><sub>N</sub> 
 = <i>d</i><sub>S</sub></span></p>
<p><span style="font-size: 11.0pt;">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;HA: 
 &#160;&#160;&#160;&#160;&#160;(a)&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;<i>d</i><sub>N</sub></span> 
 <span style="font-size: 11.0pt; font-family: Symbol;">ยน</span><span style="font-size: 11.0pt;"><i>d</i><sub>S</sub>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(test 
 of neutrality).</span></p>
<p style="margin-left: 1.0in;"><span style="font-size: 11.0pt;">(b)<i>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;d</i><sub>N</sub> 
 &gt; <i>d</i><sub>S</sub> &#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(<a href="javascript:TextPopup(this)" id="HotSpot22507" class="glossterm">positive selection</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot22507', 'POPUP22507');
//]]></script><span class="glosstext" id="POPUP22507" style="display: none;"> At the DNA sequence level, positive selection refers to selection in favor of nonsynonymous substitutions.  In this case, the evolutionary distance based on nonsynonymous substitutions is expected to be greater than synonymous substitutions.</span>).</span></p>
<p style="margin-left: 1.0in;"><span style="font-size: 11.0pt;">(c)<i>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;d</i><sub>N</sub> 
 &lt; <i>d</i><sub>S</sub> &#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;(<a href="javascript:TextPopup(this)" id="HotSpot25917" class="glossterm">purifying selection</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot25917', 'POPUP25917');
//]]></script><span class="glosstext" id="POPUP25917" style="display: none;"> Purifying selection refers to selection against nonsynonymous substitutions at the DNA level.  In this case, the evolutionary distance based on synonymous substitutions is expected to be greater than the distance based on nonsynonymous substitutions.</span>).</span></p>
<p class="equation">&#160;</p>
<p><span style="font-size: 11.0pt;">For alternative hypotheses (b) and 
 (c), we use a one-tailed test and for (a) we use a two-tailed test.&#160;&#160;These 
 three tests can be conducted directly for pairs of sequences, overall 
 sequences, or within groups of sequences.&#160;&#160;For testing for selection 
 in a pairwise manner, you can compute the variance of (<i>d</i><sub>N</sub> 
 - <i>d</i><sub>S</sub>) by using either the analytical formulas or the 
 bootstrap resampling method.&#160;</span></p>
<p><span style="font-size: 11.0pt;">For data sets containing more than 
 two sequences, you can compute the average number of synonymous substitutions 
 and the average number of nonsynonymous substitutions to conduct a Z-test 
 in a manner similar to the one mentioned above.&#160;&#160;The variance 
 of the difference between these two quantities is estimated by the bootstrap 
 method (See </span><u style="color: #007fbf;"><span style="color: #007fbf;"><a 
	 href="Nei_and_Kumar_2000.htm" style="color: #007fbf;">Nei and Kumar 
 (2000)</a></span></u><span style="display: none;">Nei_and_Kumar_2000</span><span 
	 style="font-size: 11.0pt;"> page 55).</span></p>
<p>&#160;</p>
<b><span style="font-size: 11.0pt; font-family: Arial, sans-serif;"><br 
	 style="clear: all;" clear="all" />
</span></b> 
</body>
</html>
