<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="myPEG_RH.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Introduction to myPEG</title>
<link rel="stylesheet" href="myPEG_RH.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<style type="text/css">
<!--
A:visited { color:#954F72; }
A:link { color:#0000ff; }
-->
</style>
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "4.7.0_1";
</script>
 <meta name="topic-breadcrumbs" content="Part IV: Evolutionary Analysis > Exploring the Impact of nsSNVs" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1><span style="font-size: 11.0pt;">Introduction to myPEG</span></h1>
<p style="margin-left: .5in;"><span style="font-size: 10.0pt;">Computational 
 diagnosis of amino acid variants in the human exome is the first step 
 in assessing the disruptive impacts of non-synonymous single nucleotide 
 variants (nsSNVs) on human health and disease. MEGA-MD (<b>M</b>olecular 
 <b>E</b>volutionary <b>G</b>enetics <b>A</b>nalysis â€“ <b>M</b>utation 
 <b>D</b>iagnosis) is a client-server application used to forecast the 
 deleteriousness of nsSNVs using multiple methods and explore them in the 
 context of the variability permitted in the long-term evolution of the 
 affected positions.</span></p>
<p style="margin-left: .5in; font-size: 10.0pt;">MEGA-MD accesses a relational 
 database (MD-DB) resident on our servers that contains pre-computed diagnoses, 
 and associated information, for all possible mutations at all amino acid 
 positions in the human exome. We have included three primary methods (<u 
	 style="text-decoration: none; ">PolyPhen-2</u><span style="display: none;">PolyPhen_2</span>, 
 <u style="text-decoration: none; ">SIFT</u><span style="display: none; 
													 text-decoration: underline; ">SIFT_1</span>, 
 and <u style="text-decoration: none; ">EvoD</u><span style="display: none;">EvoD_Evolutionary_Diagnosis_</span>) 
 of predicting the functional impact of amino acid variants. The first 
 two are the most popular methods and the third significantly improves 
 the performance for nSNVs found at ultra-conserved and at fast-evolving 
 positions (Kumar et al., 2012). The PolyPhen-2 and SIFT diagnoses were 
 obtained from <u style="text-decoration: none; ">dbNSFP</u><span style="display: none;">dbNSFP_1</span>. 
 We have also included results from a multi-method consensus diagnosis, 
 because they have been shown to be more reliable. In this case, we use 
 the evolutionarily-balanced (see <u style="text-decoration: none; ">Liu 
 and Kumar 2013</u><span style="display: none;">Evolutionary_balancing_is_described_in_</span>) 
 versions of PolyPhen-2 and SIFT diagnosis.</p>
<p style="margin-left: .5in; font-size: 10.0pt;">In addition to retrieving 
 pre-computed predictions for variants in the human exome, MEGA-MD provides 
 a facility to infer ancestral states for the position where a given amino 
 acid mutation is found. Maximum parsimony and maximum likelihood approaches 
 are supported by this utility which uses the 46 species reference phylogeny 
 along with the 46 species peptide alignment for the relevant <a href="javascript:TextPopup(this)" id="HotSpot14924" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot14924', 'POPUP14924');
//]]></script><span class="glosstext" id="POPUP14924" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> (obtained from the <u style="text-decoration: none; ">UCSC 
 resource</u><span style="display: none;">UCSC_resource</span>).</p>
<p style="margin-left: .5in; font-size: 10.0pt;">MEGA-MD is developed using 
 the <u style="text-decoration: none; "><span style="text-decoration: none; ">MEGA</span></u><span 
	 style="display: none;">MEGA6</span> (Molecular Evolutionary Genetics 
 Analysis) software package.</p>
<p>&#160;</p>
<p>&#160;</p>
<p>&#160;</p>
<b><span style="font-size: 8.0pt; font-family: Arial, sans-serif;"><br 
	 style="clear: all;" clear="all" />
</span></b> 
</body>
</html>
