<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="Adobe RoboHelp 2017" />
<meta name="OriginalFile" content="PhylogeneticTrees_RH.doc" />
 <title>Maximum Parsimony (MP) Method</title>
<link rel="stylesheet" href="PhylogeneticTrees_RH.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "";
</script>
</head>
<body link="blue" vlink="#954F72">
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Maximum Parsimony (MP) Method</h1>
<p><u><span style=" color:green;">Maximum parsimony</span></u><span style=" display:none;">RH_Maximum_Parsimony_Principle</span> (MP) methods originally were developed for morphological characters, and there are many different versions (see <u><span style=" color:green;">Nei and Kumar [2000]</span></u><span style=" display:none;">Nei_and_Kumar_2000</span> for a review).&#160; In <i>MEGA</i>, we consider both of these methods for nucleotide and amino acid sequence data (<u><span style=" color:green;">Eck and Dayhoff 1966</span></u><span style=" display:none;">Eck_and_Dayhoff_1966</span>; <u><span style=" color:green;">Fitch 1971</span></u><span style=" display:none;">Fitch_1971</span>).</p>
<p class="BodyText2">For constructing an MP tree, only sites at which there are at least two different kinds of nucleotides or amino acids, each represented at least twice, are used (parsimony-informative sites). Other variable sites are not used for constructing an MP tree, although they are informative for distance and maximum-likelihood methods.</p>
<p><i>MEGA</i> estimates MP tree <a href="javascript:TextPopup(this)" id="HotSpot17192" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot17192', 'POPUP17192');
//]]></script><span class="glosstext" id="POPUP17192" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span> lengths by using the average pathway method for unrooted trees (see <u><span style=" color:green;">Nei and Kumar [2000]</span></u><span style=" display:none;">Nei_and_Kumar_2000</span>, page 132).</p>
<p>To search for MP Trees, <i>MEGA</i> provides four different types of searches: the <u><span style=" color:green;">max-mini <a href="javascript:TextPopup(this)" id="HotSpot53038" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot53038', 'POPUP53038');
//]]></script><span class="glosstext" id="POPUP53038" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span>-and-bound search</span></u><span style=" display:none;">RH_Max_mini_branch_and_bound_search</span>, <u><span style=" color:green;">min-mini heuristic search</span></u><span style=" display:none;">RH_Min_mini_algorithm</span>, <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot23675" class="glossterm">Subtree-Pruning-Regrafting (SPR)</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot23675', 'POPUP23675');
//]]></script><span class="glosstext" id="POPUP23675" style="display: none;"> For any tree searching method, exhaustive search, where all possible topologies are considered is unfeasible for even a small number of taxa. Subtree Pruning And Regrafting is a tree topology search heuristic which reduces the number of topologies searched by performing the following operations on the tree.        First, a subtree of the current best tree is selected and detached (pruned). Second, the detached subtree is regrafted onto another branch of the remaining tree, in such a way that a new topology is created and then likelihood of the new topology is calculated. This procedure is repeated for all regrafting positions that produce new topologies using the pruned subtree. The procedure is also repeated for each subtree (within the designated search level) and if the topology with best likelihood among those scored gives sufficient improvement over the current best tree, that topology becomes the current best tree. This is repeated until no significant further likelihood improvements are obtained.        A single pass of the SPR algorithm examines O(N2) new trees, where N is the number of leaves in the original tree. This is because, for each subtree there are O(N) possible regraftings, and there are O(N) possible subtrees to consider. In contrast, NNI examines O(N) topologies at each pass of the algorithm.</span></span></u><span style=" display:none;">Subtree_Pruning_Regrafting</span>, and <u><span style=" color:green;">Tree-Bisection-Reconnection (TBR)</span></u><span style=" display:none;">Tree_Bisection_Reconnection_TBR_</span> <span style=" display:none;">RH_Close_Neighbor_Interchange_CNI</span> heuristic search.  Only the <a href="javascript:TextPopup(this)" id="HotSpot52794" class="glossterm">branch</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot52794', 'POPUP52794');
//]]></script><span class="glosstext" id="POPUP52794" style="display: none;"> A branch is a line connecting either two internal nodes to each other or an external node to an internal node in a phylogenetic tree.  The length of a branch denotes the genetic distance (e.g., number of substitutions per unit time) between the two taxa it connects.</span>-and-bound search is guaranteed to find all the MP trees, but it takes prohibitive amount of time if the number of sequences is large (>15).  For details, please see chapter 7 in <u><span style=" color:green;">Nei and Kumar (2000)</span></u><span style=" display:none;">Nei_and_Kumar_2000</span></p>
<p>&#160;</p>
<b><span style=" font-size:10.0pt; font-family:'Arial' ,sans-serif; color:blue;"><br clear="all" /></span></b>
</body>
</html>
