<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="OriginalFile" content="myPEG_RH.doc" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Mutation Detail View</title>
<link rel="stylesheet" href="myPEG_RH.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<style type="text/css">
<!--
A:visited { color:#954F72; }
A:link { color:#0000ff; }
-->
</style>
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "4.7.1.0_2";
</script>
 <meta name="topic-breadcrumbs" content="Part IV: Evolutionary Analysis > Exploring the Impact of nsSNVs > MEGA-MD Windows" />
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>

<h1><span style="font-size: 11.0pt;">Mutation Detail View</span></h1>
<p style="margin-left: .5in;"><span style="font-size: 10.0pt;">The Mutation 
 Detail View window displays all available information for the currently 
 active record (selected in the </span><u style="text-underline: double; 
											 font-size: 11pt; color: #007fbf;"><span 
	 style="font-size: 11pt; color: #007fbf;"><a href="Mutation_Explorer.htm" 
												 style="font-size: 11pt; 
												 color: #007fbf;">Mutation 
 Explorer</a></span></u><span style="display: none;">Mutation_Explorer</span><span 
	 style="font-size: 10.0pt;"> window). Additionally, this window provides 
 access to the 46-species reference alignment for the given gene as well 
 as the ability to infer ancestral alleles using the Maximum Likelihood 
 (ML) or Maximum Parsimony (MP) methods.</span></p>
<p style="margin-left: .5in;"><span style="font-size: 10.0pt;">When the 
 <i>Explore Alignment</i> button is clicked, MEGA-MD will retrieve the 
 46-species reference alignment from the MEGA-MDW server and display it 
 in the Sequence Data Explorer, from where it can be exported or further 
 exploration can be done.</span></p>
<p style="margin-left: .5in; font-size: 10.0pt;">When the <i>Explore Ancestors</i> 
 button is clicked, the choice of ML and MP methods are presented. If the 
 ML approach is selected, the <u style="text-underline: double; text-decoration: none; "><span 
	 style="text-decoration: none; ">Analysis Preferences Dialog</span></u><span 
	 style="display: none;">Analysis_Preferences_Dialog</span> is displayed 
 from which the analysis can be launched with custom settings (e.g. substitution 
 model, distribution of rates, etcâ€¦). If the MP approach is selected, the 
 analysis is launched immediately as not custom settings are available 
 for this method. When the analysis is completed, the reference topology 
 will be displayed in the <u style="text-underline: double;"><span style="color: #008000;">Tree 
 Explorer</span></u><span style="display: none;">Tree_Explorer</span> along 
 with inferred ancestral alleles for the amino acid site designated earlier.</p>
<p>&#160;</p>
<p><img src="myPEG_RH_files/image008.jpg" alt="" border="0" style="width:100%; max-width:595px" /></p>
<b><span style="font-size: 8.0pt; font-family: Arial, sans-serif;"><br 
	 style="clear: all;" clear="all" />
</span></b> 
</body>
</html>
