<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Preface</title>
<link rel="stylesheet" href="default.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "0_2";
</script>
</head>

<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1>Preface</h1>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">Genome 
 sequencing is generating vast amounts of DNA sequence data from a wide 
 range of organisms. As a result, <a href="javascript:TextPopup(this)" id="HotSpot36266" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot36266', 'POPUP36266');
//]]></script><span class="glosstext" id="POPUP36266" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> 
 sequence databases are growing rapidly. In order to conduct efficient 
 analyses of these data, there is a need for easy-to-use computer programs, 
 containing fast computational algorithms and useful statistical methods.</font></p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">The 
 objective of the</font> <i><font style="font-size: 12pt; font-family: Arial; 
									 font-style: italic;" size="4" face="Arial">MEGA</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">software 
 has been to provide tools for exploring, discovering, and analyzing DNA 
 and protein sequences from an evolutionary perspective. The first version 
 was developed for the limited computational resources that were available 
 on the average personal computer in early 1990s.</font> <i><font style="font-size: 12pt; 
	 font-family: Arial; font-style: italic;" size="4" face="Arial">MEGA1</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">made 
 many methods of evolutionary analysis easily accessible to the scientific 
 community for research and education.</font> <i><font style="font-size: 12pt; 
														 font-family: Arial; 
														 font-style: italic;" 
														 size="4" face="Arial">MEGA2</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">was 
 designed to harness the exponentially greater computing power</font> <font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">and 
 a graphical interface of the late 1990’s</font><font style="font-size: 12pt; 
														 font-family: Arial;" 
														 size="4" face="Arial">, 
 fulfilling the fast-growing need for more extensive biological sequence 
 analysis and exploration software. It expanded the scope of its predecessor 
 from single <a href="javascript:TextPopup(this)" id="HotSpot29472" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot29472', 'POPUP29472');
//]]></script><span class="glosstext" id="POPUP29472" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> to genome 
 wide analyses. Two versions were developed (2.0 and 2.1), each supporting 
 the analyses of molecular sequence (DNA and protein sequences) and pairwise 
 distance data. Both</font> <font style="font-size: 12pt; font-family: Arial;" 
									 size="4" face="Arial">could specify 
 <a href="javascript:TextPopup(this)" id="HotSpot28388" class="glossterm">domains</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot28388', 'POPUP28388');
//]]></script><span class="glosstext" id="POPUP28388" style="display: none;"> A domain is a continuous block of sites in a sequence alignment.  A domain can be free-standing or assigned to genes and protein-coding (e.g., exons) or non-coding (e.g., introns). Domains can be defined in the input data, and can be defined and edited in the Setup Genes Domains dialog.</span> and genes for 
 multi-<a href="javascript:TextPopup(this)" id="HotSpot15973" class="glossterm">gene</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot15973', 'POPUP15973');
//]]></script><span class="glosstext" id="POPUP15973" style="display: none;"> A gene is a collection of domains. The domains included in a gene need not be consecutive or of the same type.  Genes and domains can be defined in the input data, and can be defined and edited in the Setup Genes and Domains dialog.  Genes can be selected or unselected from an analysis.  When a gene is unselected, all its domains are automatically unselected. However, a gene can be selected, with some of its domains unselected.</span> comparative 
 sequence analysis and could create groups of sequences that would facilitate 
 the estimation of within- and among- group diversities and infer the higher-level 
 evolutionary relationships of genes and species.</font> <i><font style="font-size: 12pt; 
	 font-family: Arial; font-style: italic;" size="4" face="Arial">MEGA2</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">implemented 
 many methods for the estimation of evolutionary distances, the calculation 
 of molecular sequence and genetic diversities within and among groups, 
 and the inference of phylogenetic trees under minimum evolution and maximum 
 parsimony criteria. It included the bootstrap and the confidence probability 
 tests of reliability of the inferred phylogenies, and the <a href="javascript:TextPopup(this)" id="HotSpot23840" class="glossterm">disparity index</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot23840', 'POPUP23840');
//]]></script><span class="glosstext" id="POPUP23840" style="display: none;"> Disparity Index measures the observed difference in substitution patterns for a pair of sequences.  It works by comparing the nucleotide (or amino acid) frequencies in given pair of sequences and using the number of observed differences between sequences.  MEGA computes and presents the Disparity Index per site, which is given by the total disparity index between two sequences divided by the number of positions compared, excluding gaps and missing data.  It is more powerful than a chi-square test of the equality of base frequencies between sequences.</span> test for examining the 
 heterogeneity of substitution pattern between lineages.</font></p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">MEGA</font></i><i><font style="font-size: 12pt; font-family: Arial; 
											 font-style: italic;" size="4" 
											 face="Arial">4</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">continues 
 where</font> <i><font style="font-size: 12pt; font-family: Arial; font-style: italic;" 
						 size="4" face="Arial">MEGA2</font></i> <font style="font-size: 12pt; 
	 font-family: Arial;" size="4" face="Arial">left off, emphasizing the 
 integration of sequence acquisition with evolutionary analysis. It contains 
 an array of input data and multiple results explorers for visual representation; 
 the handling and editing of sequence data, sequence alignments, inferred 
 phylogenetic trees; and estimated evolutionary distances. The results 
 explorers allow users to browse, edit, summarize, export, and generate 
 publication-quality captions for their results.</font> <i><font style="font-size: 12pt; 
	 font-family: Arial; font-style: italic;" size="4" face="Arial">MEGA</font></i><i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">4</font></i> <font style="font-size: 12pt; font-family: Arial;" 
									 size="4" face="Arial">also includes 
 distance matrix and phylogeny explorers as well as advanced graphical 
 modules for the visual representation of input data and output results. 
 These features, which we discuss below, set</font> <i><font style="font-size: 12pt; 
															 font-family: Arial; 
															 font-style: italic;" 
															 size="4" face="Arial">MEGA</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">apart 
 from other comparative sequence analysis programs</font></p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">As 
 with previous versions,</font> <i><font style="font-size: 12pt; font-family: Arial; 
										 font-style: italic;" size="4" 
										 face="Arial">MEGA</font></i><i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">5</font></i> <font style="font-size: 12pt; font-family: Arial;" 
									 size="4" face="Arial">wa</font><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">s 
 specifically designed to reduce the time needed for mundane tasks in data 
 analysis and to provide statistical methods of molecular evolutionary 
 genetic analysis in an easy-to-use computing workbench. While</font> <i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">MEGA</font></i><i><font style="font-size: 12pt; font-family: Arial; 
											 font-style: italic;" size="4" 
											 face="Arial">5</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">wa</font><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">s 
 distinct</font> <font style="font-size: 12pt; font-family: Arial;" size="4" 
						 face="Arial">from previous versions, we</font> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">made 
 a special effort to retain the user-friendly interface that researchers 
 have come to identify with</font> <i><font style="font-size: 12pt; font-family: Arial; 
											 font-style: italic;" size="4" 
											 face="Arial">MEGA</font></i><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">.</font> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">We 
 have simplified the file activation process, where you may select an analysis 
 before needing to open a file</font><font style="font-size: 12pt; font-family: Arial;" 
											 size="4" face="Arial">.</font></p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">MEGA</font></i><i><font style="font-size: 12pt; font-family: Arial; 
											 font-style: italic;" size="4" 
											 face="Arial">6</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">represents 
 a leap forward in terms of performance. The multithreaded ML system has 
 been optimized for maximum efficiency. A new memory manager and updated 
 compiler have made it possible for</font> <i><font style="font-size: 12pt; 
													 font-family: Arial; 
													 font-style: italic;" 
													 size="4" face="Arial">MEGA</font></i> 
 <font style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">to 
 allocate twice as much memory on 64-bit systems as</font> <i><font style="font-size: 12pt; 
	 font-family: Arial; font-style: italic;" size="4" face="Arial">MEGA</font></i><i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">5</font></i> <font style="font-size: 12pt; font-family: Arial;" 
									 size="4" face="Arial">could. The naïve 
 timing methods that were added in</font> <i><font style="font-size: 12pt; 
													 font-family: Arial; 
													 font-style: italic;" 
													 size="4" face="Arial">MEGA</font></i><i><font 
	 style="font-size: 12pt; font-family: Arial; font-style: italic;" size="4" 
	 face="Arial">5</font></i> <font style="font-size: 12pt; font-family: Arial;" 
									 size="4" face="Arial">have been replaced 
 by a RelTime based system which is as accurate as (or better than) contemporary 
 methodologies but with speeds &gt;1000 times faster.</font></p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><i><font 
	 style="font-size: 12pt; font-style: italic;" size="4">MEGA</font></i><i><font 
	 style="font-size: 12pt; font-style: italic;" size="4">7</font></i> 
 <font style="font-size: 12pt;" size="4">is a major refactoring of the</font> 
 <i><font style="font-size: 12pt; font-style: italic;" size="4">MEGA</font></i> 
 <font style="font-size: 12pt;" size="4">source code and achieves another 
 leap forward in terms of performance.</font> <i><font style="font-size: 12pt; 
														 font-style: italic;" 
														 size="4">MEGA</font></i> 
 <font style="font-size: 12pt;" size="4">is now optimized for 64-bit processor 
 architectures and can now utilize many GB of memory. In addition, the</font> 
 <i><font style="font-size: 12pt; font-style: italic;" size="4">Tree Explorer</font></i> 
 <font style="font-size: 12pt;" size="4">window has been re-factored to 
 handle trees with &gt; 100k <a href="javascript:TextPopup(this)" id="HotSpot34166" class="glossterm">taxa</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot34166', 'POPUP34166');
//]]></script><span class="glosstext" id="POPUP34166" style="display: none;"> A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on the study, “taxa” may refer to species, populations, individuals, or sequences within an individual.</span> 
 (previously it could only handle ~4k <a href="javascript:TextPopup(this)" id="HotSpot28222" class="glossterm">taxa</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot28222', 'POPUP28222');
//]]></script><span class="glosstext" id="POPUP28222" style="display: none;"> A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on the study, “taxa” may refer to species, populations, individuals, or sequences within an individual.</span>), 
 depending on the available graphics processing resources.</font> <font 
	 style="font-size: 12pt;" size="4">Beyond increased performance, improvements 
 have been made to the user interface. The</font> <i><font style="font-size: 12pt; 
															 font-style: italic;" 
															 size="4">Timetree</font></i> 
 <font style="font-size: 12pt;" size="4">system has been completely restyled 
 to use a wizard system for guiding the user through the steps to create 
 a timetree using the</font> <i><font style="font-size: 12pt; font-style: italic;" 
										 size="4">Reltime</font></i> <font 
	 style="font-size: 12pt;" size="4">method.</font></p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt; 
	 font-size: 12pt;">MEGA X &#160;advances MEGA for cross-platform distribution. 
 In addition to MS Windows, MEGA X now runs natively on Linux systems. 
 Additionally, MEGA X has advanced performance capabilities for bootstrap 
 tree construction and model selection. For bootstrap tree construction 
 when using distance or parsimony methods, multiple bootstrap replicates 
 run simultaneously (ML bootstrap tree construction already uses fine-grain 
 parallelization). For the model selection analysis, multiple models are 
 analyzed simultaneously. Finally, the user interface has been modernized, 
 while retaining the same structure that MEGA users are accustomed to. 
 &#160;</p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt; 
	 font-size: 12pt;">MEGA X version 10.1 introduces advanced methods 
 for constructing timetrees. The <a href="javascript:TextPopup(this)" id="HotSpot47460" class="glossterm">RTDT 
 (RelTime with Dated Tips)</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot47460', 'POPUP47460');
//]]></script><span class="glosstext" id="POPUP47460" style="display: none;"> RelTime method applied using tip dates for calibrating the timetree. To use this method, tip dates must be available for all molecular ingroup taxa. RTDT is suitable for the analysis of DNA or protein sequence from fast evolving pathogens or those generated from ancient data.</span> method allows construction of 
 timetrees using the sample dates for molecular data as calibration constraints. 
 For the RelTime method, statistical calibration densities (normal, exponential, 
 and uniform distributions) can be used in addition to minimum and maximum 
 time constraints for calibrating timetrees. For RelTime trees, users can 
 now calibrate them using a molecular clock (a global evolutionary rate 
 when no calibrations are known or a single know divergence time).</p>
<p style="margin-bottom: 2pt; margin-top: 6pt; text-align: left; margin-right: 6.5pt;"><font 
	 style="font-size: 12pt; font-family: Arial;" size="4" face="Arial">MEGA 
 11 is optimized for working with larger data sets. The ML framework has 
 been re-factored to utilize memory more efficiently by compressing data 
 based on common site configurations. All distance calculations are now 
 performed outside of the main GUI thread so that when using very large 
 data sets, distance calculation do not cause the GUI to be unresponsive. 
 Many other optimizations for working with larger data sets have also been 
 made but are too numerous to mention. In addition to these optimizations, 
 the RTDT analysis has been updated to support actual sample dates. The 
 original RTDT implementation only used sample years as dated tip calibrations 
 but the updated version uses year, month, and day for dated tip calibrations.</font></p>
</body>
</html>
