<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="Adobe RoboHelp 2017" />
<meta name="OriginalFile" content="DataExplorerHelp_HC.doc" />
 <title>Sequence Data Subset Selection</title>
<link rel="stylesheet" href="DataExplorerHelp_HC.css" type="text/css" />
 <link rel="StyleSheet" href="default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="template/scripts/rh.min.js"></script>
<script type="text/javascript" src="template/scripts/common.min.js"></script>
<script type="text/javascript" src="template/scripts/topic.min.js"></script>
<script type="text/javascript" src="template/scripts/constants.js"></script>
<script type="text/javascript" src="template/scripts/utils.js"></script>
<script type="text/javascript" src="template/scripts/mhutils.js"></script>
<script type="text/javascript" src="template/scripts/mhlang.js"></script>
<script type="text/javascript" src="template/scripts/mhver.js"></script>
<script type="text/javascript" src="template/scripts/settings.js"></script>
<script type="text/javascript" src="template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = ".";
gCommonRootRelPath = ".";
gTopicId = "3.7.1.0_1";
</script>
 <meta name="topic-breadcrumbs" content="Part III: Input Data Types and File Formats > Visual Tools for Data Management > Data Subset Selection" />
</head>
<body>
<script type="text/javascript" src="./ehlpdhtm.js"></script>


<h1 style=" margin-left:0in;">Sequence Data Subset Selection</h1>
<p>Any subset of sequence data can be selected for analysis using the options in the <i><u style=" text-underline:double;"><span style=" color:green;">Data</span></u></i> <u style=" text-underline:double;"><span style=" color:green;">menu</span></u><span style=" display:none;">HC_Data_Menu_in_sequence_Data_Explorer</span>.&#160; You may:</p>
<p><i>Select <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot9083" class="glossterm">Taxa</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot9083', 'POPUP9083');
//]]></script><span class="glosstext" id="POPUP9083" style="display: none;"> A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on the study, “taxa” may refer to species, populations, individuals, or sequences within an individual.</span></span></u></i><span style=" display:none;">RH_Taxa</span> (sequences) <i>or <u><span style=" color:green;">Groups</span></u></i><span style=" display:none;">RH_Groups_of_taxa</span> of <a href="javascript:TextPopup(this)" id="HotSpot40114" class="glossterm">taxa</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot40114', 'POPUP40114');
//]]></script><span class="glosstext" id="POPUP40114" style="display: none;"> A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on the study, “taxa” may refer to species, populations, individuals, or sequences within an individual.</span> through the <i><u style=" text-underline:double;"><span style=" color:green;">Setup/Select <a href="javascript:TextPopup(this)" id="HotSpot42434" class="glossterm">Taxa</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot42434', 'POPUP42434');
//]]></script><span class="glosstext" id="POPUP42434" style="display: none;"> A taxon is the individual unit whose evolutionary relationship is being investigated.  Depending on the study, “taxa” may refer to species, populations, individuals, or sequences within an individual.</span> & Groups</span></u></i><span style=" display:none;">HC_Setup_Taxa_Groups_Dlg</span> dialog box,</p>
<p><i>Choose <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot47212" class="glossterm">Domains</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot47212', 'POPUP47212');
//]]></script><span class="glosstext" id="POPUP47212" style="display: none;"> A domain is a continuous block of sites in a sequence alignment.  A domain can be free-standing or assigned to genes and protein-coding (e.g., exons) or non-coding (e.g., introns). Domains can be defined in the input data, and can be defined and edited in the Setup Genes Domains dialog.</span></span></u></i><span style=" display:none;">RH_Domains</span> <i>and <u><span style=" color:green;">Genes</span></u></i><span style=" display:none;">RH_Gene</span> through the <i><u style=" text-underline:double;"><span style=" color:green;">Setup/Select Genes & <a href="javascript:TextPopup(this)" id="HotSpot26337" class="glossterm">Domains</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot26337', 'POPUP26337');
//]]></script><span class="glosstext" id="POPUP26337" style="display: none;"> A domain is a continuous block of sites in a sequence alignment.  A domain can be free-standing or assigned to genes and protein-coding (e.g., exons) or non-coding (e.g., introns). Domains can be defined in the input data, and can be defined and edited in the Setup Genes Domains dialog.</span></span></u></i><span style=" display:none;">HC_Genes_Domains_Dialog</span> dialog box,</p>
<p>Items 1 and 2 lead to the construction of a primary data subset, which is maintained until it is modified in the two dialog boxes mentioned in the above items or in the <i><u style=" text-underline:double;"><span style=" color:green;">Sequence Data Explorer</span></u></i><span style=" display:none;">HC_Sequence_Data_Explorer</span>.</p>
<p>Select any combination of <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot33369" class="glossterm">Codon</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot33369', 'POPUP33369');
//]]></script><span class="glosstext" id="POPUP33369" style="display: none;"> A codon is triplet of nucleotides that codes for a specific amino acid.</span></span></u><span style=" display:none;">RH_Codon</span> Positions to use through the <i>Analysis Preferences/Options dialog box</i> from the <i><u style=" text-underline:double;"><span style=" color:green;">Data | Select Preferences</span></u></i><span style=" display:none;">HC_Select_Preferences</span> menu item in the main interface.&#160;</p>
<p>Choose to include only the <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot53482" class="glossterm">Labeled Sites</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot53482', 'POPUP53482');
//]]></script><span class="glosstext" id="POPUP53482" style="display: none;"> Sites in a sequence alignment can be categorized and labeled with user-defined symbols. Each category is represented by a letter or a number. Each site can be assigned to only one category, although any combination of categories can be selected for analysis.  Labeled sites work independently of and in addition to genes and domains, thus allowing complex subsets of sites to be defined easily.</span></span></u><span style=" display:none;">RH_Labeled_Sites</span> through the <i><u style=" text-underline:double;"><span style=" color:green;">Data | Select Preferences</span></u></i><span style=" display:none;">HC_Select_Preferences</span> menu item.</p>
<p>Decide to enforce <u><span style=" color:green;">Complete-Deletion</span></u><span style=" display:none;">RH_Complete_Deletion_Option</span> or <u><span style=" color:green;">Pairwise-Deletion</span></u><span style=" display:none;">RH_Pairwise_deletion_option</span> of the missing data and <u><span style=" color:green;"><a href="javascript:TextPopup(this)" id="HotSpot17738" class="glossterm">alignment gaps</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot17738', 'POPUP17738');
//]]></script><span class="glosstext" id="POPUP17738" style="display: none;"> Phylogenetic analysis on two or more DNA or amino acid sequences requires that the sequences be aligned so that the substitutions can be accurately enumerated.  During alignment, gaps must be introduced in sequences that have undergone deletions or insertions.  These gaps are known as alignment gaps or indels.</span></span></u><span style=" display:none;">RH_Alignment_Gaps</span>.</p>
<p>Items 3, 4, and 5 provide the second level of data subset options.&#160; You are given relevant choices immediately prior to the start of the analysis.&#160; Therefore, these choices are secondary in nature and are specific to the currently requested analysis.&#160; The <i>Analysis Preferences</i> dialog box remembers them for your convenience and provides them as a default the next time you conduct an analysis that utilizes those options.</p>
<p>&#160;</p>
<p>&#160;</p>
</body>
</html>
