<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Adobe RoboHelp 2017" />
<title>Data Menu</title>
<link rel="StyleSheet" href="../../../default.css" type="text/css" />
 <link rel="StyleSheet" href="../../../default.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
p.WebHelpNavBar { text-align:right; }
-->
</style>

<script type="text/javascript" src="../../../template/scripts/rh.min.js"></script>
<script type="text/javascript" src="../../../template/scripts/common.min.js"></script>
<script type="text/javascript" src="../../../template/scripts/topic.min.js"></script>
<script type="text/javascript" src="../../../template/scripts/constants.js"></script>
<script type="text/javascript" src="../../../template/scripts/utils.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhutils.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhlang.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhver.js"></script>
<script type="text/javascript" src="../../../template/scripts/settings.js"></script>
<script type="text/javascript" src="../../../template/scripts/XmlJsReader.js"></script>
<script type="text/javascript" src="../../../template/scripts/loadparentdata.js"></script>
<script type="text/javascript" src="../../../template/scripts/loadscreen.js"></script>
<script type="text/javascript" src="../../../template/scripts/loadprojdata.js"></script>
<script type="text/javascript" src="../../../template/scripts/mhtopic.js"></script>
 <link rel="stylesheet" type="text/css" href="../../../template/styles/widgets.min.css">
 <link rel="stylesheet" type="text/css" href="../../../template/styles/topic.min.css">
<script type="text/javascript" >
gRootRelPath = "../../..";
gCommonRootRelPath = "../../..";
gTopicId = "2.2.4.0_5";
</script>
 <meta name="topic-breadcrumbs" content="Part II: Assembling Data for Analysis > Building Sequence Alignments > Menu Items in the Alignment Explorer" />
</head>

<body>
<script type="text/javascript" src="../../../ehlpdhtm.js"></script>


<h1>Data Menu</h1>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><font style="font-size: 11pt; 
	 font-family: Arial;" color="#333333" size="2" face="Arial">This menu 
 provides commands for creating a new alignment, opening/closing sequence 
 data files, saving alignment sessions to a file, exporting sequence data 
 to a file, changing alignment seq</font><font style="font-size: 11pt; 
												 font-family: Arial;" color="#333333" 
												 size="2" face="Arial">uence 
 properties, reverse comple</font><font style="font-size: 11pt; font-family: Arial;" 
										 color="#333333" size="2" face="Arial">menting 
 sequences in the alignment, and exiting</font> <i><font style="font-size: 11pt; 
														 font-family: Arial; 
														 font-style: italic;" 
														 color="#333333" 
														 size="2" face="Arial">Alignment 
 Explorer.</font></i> <font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">The 
 commands in this menu are:</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Create 
 New Alignment</font></b><font style="font-size: 11pt; font-family: Arial;" 
								 color="#333333" size="2" face="Arial">: 
 This tells</font> <i><font style="font-size: 11pt; font-family: Arial; 
							 font-style: italic;" color="#333333" size="2" 
							 face="Arial">Alignment Explorer</font></i> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">to prepare for a new alignment. Any sequence data 
 currently loaded into</font> <i><font style="font-size: 11pt; font-family: Arial; 
										 font-style: italic;" color="#333333" 
										 size="2" face="Arial">Alignment 
 Builder</font></i> <font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">is discarded.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-weight: bold;" color="#333333" size="2">Open:</font></b> <font 
	 style="font-size: 11pt;" color="#333333" size="2">This submenu provides 
 two options: opening an existing sequence</font> <font style="font-size: 11pt;" 
														 size="2"><a href="../../../Alignment_session.htm">alignment 
 session</a></font> <font style="font-size: 11pt;" color="#333333" size="2">(previously 
 saved from</font> <i><font style="font-size: 11pt; font-style: italic;" 
							 color="#333333" size="2">Alignment Explorer</font></i><font 
	 style="font-size: 11pt;" color="#333333" size="2">), and reading a 
 text file containing sequences in one of many formats (including, MEGA, 
 PAUP, FASTA, NBRF, etc.). Based on the option you choose, you will be 
 prompted for the file name that you wish to read.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Reopen:</font></b> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">Displays a list of recently opened files that can 
 be activated in Alignment Explorer.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Close:</font></b> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">This closes the currently active data in the</font> 
 <i><font style="font-size: 11pt; font-family: Arial; font-style: italic;" 
			 color="#333333" size="2" face="Arial">Alignment Explorer</font></i><font 
	 style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
	 face="Arial">.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Phylogenetic 
 Analysis:</font></b> <font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">Clicking 
 this item will prepare the data in the active sequence alignment for further 
 analysis in MEGA so that the alignment does not have to be saved to a 
 file on disk and then reopened for analysis in MEGA.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Save 
 Session:</font></b> <font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">This 
 allows you to save the current sequence alignment to an</font> <font style="font-size: 11pt; 
	 font-family: Arial;" size="2" face="Arial"><a href="javascript:TextPopup(this)" id="HotSpot22886" class="glossterm">alignment 
 session</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot22886', 'POPUP22886');
//]]></script><span class="glosstext" id="POPUP22886" style="display: none;"> When working in MEGAâ€™s Alignment Explorer you can choose to save the current state of all data and settings in the alignment explorer to a file so you can archive your work, or save it to resume editing in the future. An alignment session is a binary file format that is saved with the .MAS file extension.</span></font><font style="font-size: 11pt; font-family: Arial;" 
									 color="#333333" size="2" face="Arial">. 
 You will be requested to give a file name to write the data to.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-weight: bold;" 
	 color="#333333">Export Alignment:</font></b> <font color="#333333">This 
 allows you to export the current sequence alignment to a file. There are 
 three</font> <font color="#333333">formats to choose from:</font> <font 
	 color="#007FBF"><a href="../../../Part_III_Input_Data_Types_and_File_Format/MEGA_Input_Data_File_Formats/MEGA_Format.htm" id="a1" style="color: #007fbf;" class="BSSCPopup">MEGA</a></font><font 
	 color="#333333">,</font> <a href="../../../Converting_FASTA_Format.htm" id="a2" style="color: #007fbf;" class="BSSCPopup">FASTA</a> 
 <font color="#333333">or</font> <a href="../../../Converting_NEXUS_Format.htm" id="a3" style="color: #007fbf;" class="BSSCPopup">PAUP/NEXUS</a> 
 <font color="#333333">formats. You will be requested to give a file name 
 to write the data to.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">DNA 
 Sequences</font></b><font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">: Use 
 this item to specify that the input data is DNA. If DNA is selected, then 
 all sites are treated as nucleotides. The</font> <i><font style="font-size: 11pt; 
															 font-family: Arial; 
															 font-style: italic;" 
															 color="#333333" 
															 size="2" face="Arial">Translated 
 Protein Sequences</font></i> <font style="font-size: 11pt; font-family: Arial;" 
									 color="#333333" size="2" face="Arial">tab 
 contains the protein sequences. If the data is non-coding, then ignore 
 the second tab, as it has no affect on the on the DNA sequence tab. However, 
 any changes you make in the</font> <i><font style="font-size: 11pt; font-family: Arial; 
											 font-style: italic;" color="#333333" 
											 size="2" face="Arial">Protein 
 Sequence</font></i> <font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">tab 
 are applied to the</font> <i><font style="font-size: 11pt; font-family: Arial; 
									 font-style: italic;" color="#333333" 
									 size="2" face="Arial">DNA Sequences</font></i> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">tab window. Note that you can UNDO these changes 
 by using the undo button.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Protein 
 Sequences</font></b><font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">: Use 
 this item to specify that the input data is amino acid sequences. If selected, 
 then all sites are treated as amino acid residues.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Translate/Untranslate</font></b><font 
	 style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
	 face="Arial">: This item only will be available if protein-coding 
 DNA sequences are available in the alignment grid. It will translate protein-coding 
 DNA sequences into their respective amino acid sequences using the selected 
 genetic code table.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Select 
 Genetic Code Table</font></b><font style="font-size: 11pt; font-family: Arial;" 
									 color="#333333" size="2" face="Arial">: 
 This displays the</font> <i><font style="font-size: 11pt; font-family: Arial; 
									 font-style: italic;" color="#333333" 
									 size="2" face="Arial">Select Genetic 
 Code</font></i> <font style="font-size: 11pt; font-family: Arial;" color="#333333" 
						 size="2" face="Arial">dialog window, which can 
 select the genetic code table that is used when translating protein-coding 
 DNA sequence data.</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Reverse 
 Complement</font></b><font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">: This 
 becomes available when an entire sequence of row(s) is selected. It will 
 update the selected rows to contain the reverse compliment of the originally 
 selected sequence(s).</font></p>
<p style="text-align: left; margin: 4pt 6.5pt 0pt 36pt;"><b><font style="font-size: 11pt; 
	 font-family: Arial; font-weight: bold;" color="#333333" size="2" face="Arial">Exit 
 Aln</font></b><b><font style="font-size: 11pt; font-family: Arial; font-weight: bold;" 
						 color="#333333" size="2" face="Arial">Explorer</font></b><font 
	 style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
	 face="Arial">: This closes the</font> <i><font style="font-size: 11pt; 
													 font-family: Arial; 
													 font-style: italic;" 
													 color="#333333" size="2" 
													 face="Arial">Alignment 
 Explorer</font></i> <font style="font-size: 11pt; font-family: Arial;" 
							 color="#333333" size="2" face="Arial">window 
 and returns to the main</font> <i><font style="font-size: 11pt; font-family: Arial; 
										 font-style: italic;" color="#333333" 
										 size="2" face="Arial">MEGA</font></i> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">application window. When selected, a message box 
 appears asking if you would like to save the current <a href="javascript:TextPopup(this)" id="HotSpot52095" class="glossterm">alignment 
 session</a><script type="text/javascript">//<![CDATA[
TextPopupInit('HotSpot52095', 'POPUP52095');
//]]></script><span class="glosstext" id="POPUP52095" style="display: none;"> When working in MEGAâ€™s Alignment Explorer you can choose to save the current state of all data and settings in the alignment explorer to a file so you can archive your work, or save it to resume editing in the future. An alignment session is a binary file format that is saved with the .MAS file extension.</span> to a file. Then a second message box appears asking 
 if you would like to save the current alignment to a</font> <i><font style="font-size: 11pt; 
	 font-family: Arial; font-style: italic;" color="#333333" size="2" 
	 face="Arial">MEGA</font></i> <font style="font-size: 11pt; font-family: Arial;" 
										 color="#333333" size="2" face="Arial">file. 
 If the current alignment is saved to a</font> <i><font style="font-size: 11pt; 
														 font-family: Arial; 
														 font-style: italic;" 
														 color="#333333" 
														 size="2" face="Arial">MEGA</font></i> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">file, a third message box will appear asking if you 
 would like to open the saved</font> <i><font style="font-size: 11pt; font-family: Arial; 
												 font-style: italic;" color="#333333" 
												 size="2" face="Arial">MEGA</font></i> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">file in the main</font> <i><font style="font-size: 11pt; 
														 font-family: Arial; 
														 font-style: italic;" 
														 color="#333333" 
														 size="2" face="Arial">MEGA</font></i> 
 <font style="font-size: 11pt; font-family: Arial;" color="#333333" size="2" 
		 face="Arial">application.</font></p>

</body>
</html>
